-- MariaDB dump 10.19  Distrib 10.5.9-MariaDB, for debian-linux-gnu (x86_64)
--
-- Host: localhost    Database: CAMEL
-- ------------------------------------------------------
-- Server version	10.5.9-MariaDB-1:10.5.9+maria~focal

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `experiments`
--

DROP TABLE IF EXISTS `experiments`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `experiments` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `name` (`name`)
) ENGINE=InnoDB AUTO_INCREMENT=763 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `experiments`
--

LOCK TABLES `experiments` WRITE;
/*!40000 ALTER TABLE `experiments` DISABLE KEYS */;
INSERT INTO `experiments` VALUES (737,'A fresh experiment'),(279,'Aarnio TH_1991'),(585,'Acar M_2008'),(635,'AceGlu'),(421,'Acevedo A_2014'),(306,'Adams HM_2017'),(166,'Agashe D_2016'),(712,'AgResEcoEvo'),(586,'Agudelo-Romero P_2008'),(60,'Aguilar-Rodríguez J_2016'),(671,'Akanuma S_1998'),(632,'Alcántara-Díaz D_2004'),(289,'Ali SS_2014'),(461,'Alto BW_2013'),(61,'Alvarez-Ponce D_2016'),(298,'AMP Staph'),(724,'AMPEvo'),(462,'Angst DC_2013'),(494,'Arnoldini M_2012'),(7,'AspTriazEVO'),(547,'Atsumi S_2010'),(706,'Atwood KC_1951'),(107,'Auriol C_2011'),(263,'Avrahami-Moyal L_2012'),(62,'Azevedo M_2016'),(162,'Bachmann H_2013'),(725,'BacteriocinEvo'),(380,'Bailey SF_2015'),(672,'Baranowski E_1998'),(234,'Barberis P_2018'),(625,'Barrett RDH_2005'),(220,'Barrett RDH_2006'),(565,'Barrick JE_2009'),(548,'Barrick JE_2010'),(255,'BarTrack_C'),(240,'BarTrack_N'),(294,'Baumgartner M_2017'),(381,'Baym M_2015'),(63,'Baym M_2016'),(29,'Beabout K_2015'),(327,'Beabout K_2017'),(522,'Beardmore RE_2011'),(566,'Beaumont HJE_2009'),(496,'Bedhomme S_2012'),(328,'Bedhomme S_2017'),(678,'Bell G_1997'),(270,'Bell G_2009'),(267,'Bell G_2011'),(463,'Bell G_2013'),(587,'Benmayor R_2008'),(694,'Bennett AF_1993'),(683,'Bennett AF_1996'),(598,'Bennett AF_2007'),(233,'Bernardi AC_2015'),(178,'Betteridge AL_2017'),(193,'Betts A_2016'),(497,'Blaby IK_2012'),(549,'Bloom JD_2010'),(498,'Blount ZD_2012'),(241,'Bódi Z_2017'),(329,'Bonhoeffer S_2017'),(180,'Bono LM_2012'),(179,'Bono LM_2015'),(423,'Bos J_2014'),(32,'Bosshard L_2017'),(33,'Bottery MJ_2017'),(19,'Boyd M_2018'),(633,'Brockhurst M a_2004'),(8,'Bruggeman J_2003'),(183,'Bull JJ_1997'),(659,'Burch CL_2000'),(634,'Burch CL_2004'),(424,'Burke MK_2014'),(15,'BurkMA'),(177,'Cairns J_2016'),(221,'Cairns J_2016_1'),(296,'Cao S_2017'),(425,'Carja O_2014'),(426,'Carja O_2014_1'),(169,'Carroll SM_2013'),(382,'Carroll SM_2015'),(307,'Carsenti-Etesse H_1999'),(722,'CFPseudoCollSens'),(700,'Chao L_1990'),(34,'Chib S_2017'),(22,'ChlamMA'),(523,'Chou H-H_2011'),(499,'Chou H-HH_2012'),(500,'Chubiz LM_2012'),(375,'Churton NW V_2016'),(291,'Clark TA_2004'),(696,'Clarke DK_1993'),(616,'Codoñer FM_2006'),(588,'Coffey LL_2008'),(525,'Coffey LL_2011'),(567,'Collins S_2009'),(116,'Conrad TM_2009'),(641,'Cooper TF_2003'),(550,'Cooper TF_2010'),(660,'Cooper VS_2000'),(643,'Cooper VS_2002'),(427,'Cooper VS_2014'),(152,'CoopEvo'),(170,'Copin R_2016'),(64,'Cottinet D_2016'),(155,'Couñago R_2006'),(551,'Cox J_2010'),(36,'Creamer KE_2016'),(568,'Crecy E de_2009'),(649,'Cullum AJ_2001'),(290,'Davidson CJ_2008'),(210,'Davies E V._2016'),(4,'de Crécy E_2007'),(130,'de Crécy-Lagard V a_2001'),(319,'de la Iglesia F_2007'),(684,'de Visser JAGM_1996'),(679,'de Visser JAGM_1997'),(664,'de Visser JAGM_1999'),(644,'de Visser JAGM_2002'),(302,'DefensEvo'),(243,'Delneri D_2017'),(526,'Deng Y_2011'),(599,'Desai MM_2007'),(27,'DesulfoOsmo'),(273,'Dettman JR_2007'),(176,'Dettman JR_2008'),(194,'Dettman JR_2016'),(274,'Dickinson WJ_2008'),(65,'Dillon MM_2016'),(151,'Dionisio F_2005'),(98,'DomesticEvo'),(253,'Donald_2016'),(186,'Dößelmann B_2017'),(161,'Douillard FP_2016'),(292,'Dover JA_2016'),(206,'Drago L_2005'),(103,'Dragosits M_2013'),(699,'Duarte E a_1992'),(617,'Duffy S_2006'),(223,'Duncan AB_2016'),(282,'Dunn B_2013'),(384,'Durão P_2015'),(79,'EcM9glu'),(422,'EcMouse'),(100,'EcoliABCombi'),(149,'EcoliMacro'),(102,'EcOsm'),(685,'Elena SF_1996'),(680,'Elena SF_1997'),(665,'Elena SF_1999'),(645,'Elena SF_2002'),(552,'Ellis CN_2010'),(385,'Ellis CN_2015'),(305,'Enav H_2018'),(332,'Erickson KE_2017'),(300,'Evgrafov MR De_2015'),(278,'Ferea TL_1999'),(618,'Fiegna F_2006'),(38,'Finn TJ_2017'),(217,'Flohr RCE_2013'),(218,'Flohr RCE_2013_1'),(464,'Flynn KM_2013'),(195,'Flynn KM_2016'),(127,'Fong SS_2004'),(124,'Fong SS_2006'),(429,'Fothergill JL_2014'),(39,'Fraebel DT_2017'),(280,'Francis JC_1972'),(430,'Frenkel EM_2014'),(431,'Fridman O_2014'),(465,'Friman V-P_2013'),(432,'Friman V-P_2014'),(386,'Friman V-P_2015'),(333,'Fronhofer EA_2017'),(387,'Fumagalli M_2015'),(117,'Gac M Le_2009'),(108,'Gaffé J_2011'),(603,'Gallet R_2007'),(334,'Gallet R_2017'),(388,'Gallie J_2015'),(711,'GaResEcoEvo'),(323,'Garijo R_2016'),(710,'gepotidacinEVO'),(674,'Gerrish PJ_1998'),(466,'Gifford DR_2013'),(196,'Gifford DR_2016'),(650,'Giraud A_2001'),(249,'Gjuvsland AB_2016'),(276,'Goddard MR_2005'),(527,'Goldman RP_2011'),(467,'Gong LI_2013'),(118,'Goodarzi H_2009'),(554,'Goodarzi H_2010'),(502,'Gordo I_2012'),(224,'Gorter FA_2016'),(212,'Gounand I_2016'),(433,'Graham JK_2014'),(187,'Granato ET_2017'),(389,'Graves JL_2015'),(626,'Greene IP_2005'),(271,'Gresham D_2008'),(171,'Guerrini V_2016'),(528,'Gullberg E_2011'),(391,'Guo J_2015'),(434,'Guyot S_2014'),(159,'Haas CE_1990'),(619,'Habets MGJL_2006'),(529,'Hall AR_2011'),(336,'Hall JPJ_2017'),(338,'Hall RN_2017'),(66,'Hammerling MJ_2016'),(569,'Handel A_2009'),(604,'Hansen SK_2007'),(392,'Hao Y-Q_2015'),(89,'Harden MM_2015'),(555,'Harriott MM_2010'),(570,'Harris DR_2009'),(228,'Harrison E_2015'),(393,'Harrison E_2015_1'),(339,'Harrison E_2017'),(251,'Healey D_2016'),(18,'Healey KR_2016'),(125,'Hegreness MJ_2006'),(121,'Hegreness MJ_2008'),(435,'Heilbron K_2014'),(141,'Helling R_1987'),(436,'Helliwell KE_2014'),(468,'Herron MD_2013'),(20,'Herron MD_2018'),(225,'Hesse E_2016'),(762,'Hickman2017'),(165,'Hillesland KL_2010'),(317,'Hillung J_2014'),(437,'Hiltunen T_2014'),(394,'Hiltunen T_2015'),(150,'Hiltunen T_2017'),(340,'Holmskov M_2017'),(256,'Hong J_2014'),(142,'Hong J_2016'),(246,'Hope EA_2017'),(666,'Hopper KR_1999'),(556,'Horinouchi T_2010'),(99,'Horinouchi T_2014'),(395,'Horinouchi T_2015'),(42,'Horinouchi T_2017'),(207,'Houte S Van_2016'),(167,'Hu B_2016'),(123,'Hughes BS_2007'),(605,'Hughes BS_2007_1'),(503,'Hughes JM_2012'),(651,'Imhof M_2001'),(197,'Inglis RF_2016'),(198,'Inglis RF_2016_1'),(606,'Jasmin JN_2007'),(530,'Jayaraman R_2011'),(469,'Jeong J_2013'),(3,'Jezequel N_2013'),(341,'Jia N_2017'),(470,'Jiang W_2013'),(297,'Jiao YJ_2016'),(199,'Jochumsen N_2016'),(571,'Johnson PJT_2009'),(188,'Jorth P_2017'),(636,'Joseph SB_2004'),(213,'Jousset A_2016'),(471,'Jørgensen KM_2013'),(164,'Ju SY_2016'),(310,'Jung H-C_2017'),(67,'Kacar B_2017'),(342,'Kacar B_2017_1'),(344,'Kalily E_2017'),(272,'Kao KC_2008'),(90,'Karve SM_2015'),(68,'Karve SM_2016'),(675,'Kassen R_1998'),(620,'Kassen R_2006'),(646,'Kerr B_2002'),(396,'Ketola T_2015'),(531,'Khan AI_2011'),(91,'Khare A_2015'),(133,'Kibota T_1996'),(257,'Kildegaard KR_2014'),(439,'Kim S_2014'),(219,'Kim W_2008'),(440,'Kim W_2014'),(572,'Kinnersley M a_2009'),(441,'Kinnersley M a_2014'),(557,'Kishimoto T_2010'),(558,'Kohanski M a_2010'),(258,'Kohn LM_2014'),(23,'Korona R_1994'),(472,'Koschwanez JH_2013'),(248,'Kosheleva K_2017'),(397,'Koskella B_2015'),(185,'Kottara A_2016'),(573,'Kraigsley AM_2009'),(376,'Kram KE_2016'),(265,'Kryazhimskiy S_2012'),(259,'Kryazhimskiy S_2014'),(299,'Kubicek-Sutherland JZ_2016'),(204,'Kümmerli R_2015'),(138,'Kurlandzka A_1991'),(627,'Kussell E_2005'),(532,'Kvitek DJ_2011'),(473,'Kvitek DJ_2013'),(109,'Kwon Y_2011'),(254,'Laan L_2015'),(345,'Laanto E_2017'),(92,'LaCroix RA_2015'),(346,'LaCroix RA_2017'),(495,'LactMilk'),(442,'Laehnemann D_2014'),(505,'Lalic J_2012'),(506,'Lalić J_2012_1'),(93,'Lambert G_2015'),(145,'LamBOmpF'),(443,'Lang GI_2014'),(474,'Laszlo AH_2013'),(475,'Lázár V_2013'),(444,'Lázár V_2014'),(154,'Lazaro E_2003'),(331,'LeClair JS_2017'),(110,'Lee D-H_2011'),(574,'Lee M-C_2009'),(507,'Lee M-C_2012'),(508,'Leiby N_2012'),(445,'Leiby N_2014'),(347,'Leister D_2017'),(704,'Lenski RE_1986'),(702,'Lenski RE_1988'),(697,'Lenski RE_1993'),(691,'Lenski RE_1994'),(692,'Leroi AM_1994'),(69,'Lescat M_2016'),(275,'Leu JY_2006'),(43,'Levin-Reisman I_2017'),(509,'Levy SF_2012'),(321,'Lin H_2016'),(320,'Lin H_2017'),(399,'Lin W_2015'),(216,'Lind PA_2015'),(189,'Lind PA_2016'),(476,'Lindsey H a_2013'),(128,'Loewe L_2003'),(70,'Long H_2016'),(46,'Lopatkin AJ_2017'),(349,'López García de Lomana A_2017'),(190,'Lowery NV_2017'),(11,'LP'),(35,'LTEE'),(51,'LTEE_1'),(84,'LTEE_2'),(94,'LTEE_3'),(115,'LTEE_4'),(120,'LTEE_5'),(135,'LTEE_6'),(139,'LTEE_7'),(330,'LTEE_8'),(335,'LTEE_9'),(418,'LTEE_X'),(417,'LTEE_XI'),(690,'LTEE_XII'),(85,'LTEE_XIII'),(559,'MacLean RC_2010'),(401,'Maddamsetti R_2015'),(30,'Maddamsetti R_2018'),(126,'Maharjan RP_2006'),(607,'Maharjan RP_2007'),(104,'Maharjan RP_2012'),(478,'Maharjan RP_2013'),(479,'Maharjan RP_2013_1'),(377,'Manhart M_2016'),(47,'Marchal M_2017'),(402,'Marietou A_2015'),(608,'Maughan H_2007'),(309,'McCarthy S_2016'),(2,'McLeman A_2016'),(350,'Mcnamara KB_2017'),(232,'Meaden S_2017'),(403,'Mechler L_2015'),(191,'Mehta HH_2017'),(192,'Melnyk AH_2017'),(560,'Meyer JR_2010'),(163,'Meyer JR_2016'),(590,'Michel J-B_2008'),(209,'Michiels JE_2016'),(262,'Miller AW_2013'),(404,'Minter EJA_2015'),(534,'Minty JJ_2011'),(667,'Miralles R_1999'),(1,'Moffatt JH_2010'),(48,'Mohamed ET_2017'),(686,'Mongold J a_1996'),(181,'Montville R_2005'),(71,'Moreau PL_2016'),(208,'Morley D_2016'),(293,'Morley VJ_2015'),(324,'Morley VJ_2016'),(351,'Morley VJ_2017'),(510,'Morris JJ_2012'),(446,'Morris JJ_2014'),(95,'Moura De Sousa J_2015'),(352,'Moura De Sousa J_2017'),(353,'Moxon R_2017'),(49,'Mundhada H_2017'),(173,'MXSwarmEvo'),(175,'MXSwarmEvo_1'),(512,'Nasvall J_2012'),(168,'Nayak DDD_2016'),(447,'New AM_2014'),(354,'Niehus R_2017'),(287,'Nilsson AI_2005'),(355,'Nishimura I_2017'),(726,'NoiseEvo'),(129,'Notley-McRobb L_2002'),(647,'Notley-McRobb L_2002_1'),(687,'Novella IS_1996'),(264,'NyastatEvo'),(652,'Ooper VASC_2001'),(591,'Ostrowski E a_2008'),(629,'Ostrowski EA_2005'),(448,'Oz T_2014'),(721,'PACeftaAvi'),(407,'Palmer AC_2015'),(668,'Papadopoulos D_1999'),(705,'Paquin C_1983'),(561,'Paterson S_2010'),(72,'Peabody G_2016'),(50,'Peabody V GL_2017'),(26,'Pellerin A_2015'),(481,'Pena-Miller R_2013'),(610,'Perfeito L_2007'),(449,'Perfeito L_2014'),(205,'Perron GG_2007'),(592,'Perron GG_2008'),(408,'Perry EB_2015'),(87,'PersDeEVO'),(86,'PersEVO'),(24,'Pfeifer E_2017'),(45,'pgiEvo'),(112,'pgiEvo_1'),(356,'Phan K_2017'),(576,'Philippe N_2009'),(73,'Phillips KN_2016'),(281,'Pieczynska MD_2016'),(513,'Piotrowski JS_2012'),(450,'Plucain J_2014'),(74,'Plucain J_2016'),(75,'Plucain J_2016_1'),(143,'Porse A_2016'),(122,'Portnoy V a_2008'),(31,'PropanEvo'),(113,'PropanEvo_1'),(723,'PseudoResist'),(226,'PseuPhi'),(229,'PseuPhi_1'),(482,'Puentes-Téllez PE_2013'),(451,'Puentes-Téllez PE_2014'),(200,'Qi Q_2016'),(452,'Quandt EM_2014'),(231,'Quesada JM_2016'),(562,'Racey D_2010'),(676,'Rainey PB_1998'),(535,'Rainey PB_2011'),(76,'Rajaraman E_2016'),(453,'Rajon E_2014'),(514,'Ram Y_2012'),(483,'Ratcliff WC_2013'),(484,'Ratcliff WC_2013_1'),(454,'Raynes Y_2014'),(681,'Reboud X_1997'),(653,'Remold SK_2001'),(638,'Remold SK_2004'),(409,'Rendueles O_2015'),(410,'Rendueles O_2015_1'),(174,'Rendueles O_2016'),(515,'Reyes LH_2012'),(516,'Reyes LH_2012_1'),(485,'Reyes LH_2013'),(239,'Reyes LH_2018'),(52,'Reynolds TS_2017'),(235,'RhizoEv'),(236,'RhizoEv_1'),(237,'RhizoEv_2'),(412,'Rice DP_2015'),(654,'Riehle MM_2001'),(77,'Rigato E_2016'),(486,'Rodríguez-Verdugo A_2013'),(378,'Rogozin IB_2016'),(593,'Rokyta DR_2008'),(577,'Rokyta DR_2009'),(536,'Rokyta DR_2011'),(9,'Rosenberg J_2017'),(693,'Rosenzweig FR_1994'),(661,'Rozen DE_2000'),(648,'Rozen DE_2002'),(630,'Rozen DE_2005'),(611,'Rozen DE_2007'),(594,'Rozen DE_2008'),(578,'Rozen DE_2009'),(96,'Sabater-Munoz B_2015'),(357,'Sabater-Muñoz B_2017'),(242,'SachGluEVO'),(261,'SachGluEVO_1'),(268,'SachGluEVO_2'),(358,'Sackman AM_2017'),(160,'Saini P_2017'),(537,'Salverda MLM_2011'),(359,'Salverda MLM_2017'),(286,'Samani P_2010'),(283,'Samani P_2016'),(284,'Samani P_2016_1'),(285,'Samani P_2016_2'),(78,'San Millan A_2016'),(360,'Sanchez MR_2017'),(101,'Sandberg TE_2014'),(53,'Sandberg TE_2017'),(157,'Santos-Lopez A_2016'),(413,'Satterwhite RS_2015'),(579,'Saxer G_2009'),(563,'Saxer G_2010'),(414,'Scanlan PD_2015'),(487,'Schenk MF_2013'),(455,'Schenk MF_2014'),(538,'Schiffels S_2011'),(28,'Schlu ter L_2016'),(662,'Schneider D_2000'),(580,'Schoustra SE_2009'),(172,'Sebastian J_2016'),(147,'Shapiro JW_2016'),(146,'Shapiro JW_2018'),(54,'Sievert C_2017'),(182,'Silander OK_2007'),(581,'Simons AM_2009'),(539,'Simons AM_2011'),(311,'Singh R_2015'),(55,'Singh T_2017'),(621,'Sleight SC_2006'),(612,'Sleight SC_2007'),(595,'Sleight SC_2008'),(682,'Sniegowski PD_1997'),(266,'SnowFlakEVO'),(456,'Solopova A_2014'),(230,'Song C_2016'),(361,'Song J_2017'),(727,'SorboseFerm'),(184,'Sota M_2010'),(379,'Souza GD_2016'),(132,'Souza V_1997'),(288,'Søndberg E_2016'),(80,'Spagnolo F_2016'),(596,'Spencer CC_2008'),(363,'Stalder T_2017'),(582,'Stanek MT_2009'),(269,'Stanley D_2010'),(518,'Starrfelt J_2012'),(56,'Steinrueck M_2017'),(457,'Stelkens RB_2014'),(364,'Stern A_2017'),(119,'Stoebel DM_2009'),(81,'Stone LK_2016'),(458,'Sutera V_2014'),(97,'Suzuki S_2015'),(82,'Suzuki S_2016'),(57,'Swings T_2017'),(642,'Szafraniec K_2003'),(260,'Szamecz B_2014'),(83,'Szappanos B_2016'),(214,'Tan J_2016'),(211,'Tan J_2017'),(365,'Tan J_2017_1'),(366,'Tashiro Y_2017'),(416,'Taylor TB_2015'),(37,'TEE'),(655,'Tenaillon O_2001'),(519,'Tenaillon O_2012'),(314,'TEVdupli'),(315,'TEVorder'),(313,'TEVThal'),(367,'Tognon M_2017'),(156,'Tong S_2018'),(105,'Toprak E_2012'),(489,'Torres-Barceló C_2013'),(201,'Torres-Barceló C_2016'),(490,'Traverse CC_2013'),(689,'Travisano M_1995'),(677,'Treves DS_1998'),(583,'Trindade S_2009'),(564,'Trindade S_2010'),(368,'Turkarslan S_2017'),(688,'Turner PE_1996'),(663,'Turner PE_2000'),(88,'Van Hofwegen DJ_2016'),(540,'Vanlint D_2011'),(131,'Vasi FK_1999'),(369,'Vázquez-García I_2017'),(597,'Veening J-W_2008'),(669,'Velicer GJ_1999'),(622,'Velicer GJ_2006'),(541,'Venail PA_2011'),(5,'Vila-Farrés X_2015'),(215,'Vogwill T_2016'),(419,'Voordeckers K_2015'),(520,'Walkiewicz K_2012'),(322,'Walworth NG_2016'),(158,'Wang G_2001'),(370,'Wang J_2017'),(114,'Wang L_2010'),(111,'Wang X_2011'),(238,'Wang XX_2017'),(543,'Wang Y_2011'),(584,'Ward H_2009'),(420,'Wasik BR_2015'),(325,'Wasik BR_2016'),(202,'Wassermann T_2016'),(17,'Weil T_2017'),(623,'Weinreich DM_2006'),(544,'Wenger JW_2011'),(308,'Westhoff S_2016'),(670,'Wichman HA_1999'),(460,'Wildenberg G a_2014'),(656,'Wilke CO_2001'),(371,'Willemsen A_2016_2'),(312,'Willemsen A_2017'),(326,'Williams ESCPCP_2016'),(372,'Winans NJ_2017'),(106,'Winkler JD_2012'),(491,'Wiser MJ_2013'),(657,'Wloch DM_2001'),(658,'Wloch DM_2001_1'),(521,'Wong A_2012'),(624,'Woods RJ_2006'),(545,'Woods RJ_2011'),(12,'WormBacilEVO'),(492,'Wright E a_2013'),(227,'Wright RCT_2016'),(373,'Wu B_2017'),(58,'Wünsche A_2017'),(244,'YeastMetal'),(203,'Yen P_2016'),(493,'Yi X_2013'),(136,'Zambrano_1993'),(59,'Zampieri M_2017'),(374,'Zee PC_2017'),(277,'Zeyl CW_2001'),(546,'Zhang Q_2011'),(295,'Zhang T_2018'),(222,'Zhao X-F_2017');
/*!40000 ALTER TABLE `experiments` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `experiments_fields`
--

DROP TABLE IF EXISTS `experiments_fields`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `experiments_fields` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `experiment_id` int(11) NOT NULL,
  `field_id` int(11) NOT NULL,
  `value_VARCHAR` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `value_TEXT` mediumtext COLLATE utf8_unicode_ci DEFAULT NULL,
  `value_INT` bigint(20) DEFAULT NULL,
  `value_DOUBLE` double DEFAULT NULL,
  `value_BOOL` tinyint(1) DEFAULT NULL,
  `value_ATTACH` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_experiment_id` (`experiment_id`),
  KEY `fk_field_id` (`field_id`),
  CONSTRAINT `fk_experiment_id` FOREIGN KEY (`experiment_id`) REFERENCES `experiments` (`id`) ON DELETE CASCADE,
  CONSTRAINT `fk_field_id` FOREIGN KEY (`field_id`) REFERENCES `fields` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB AUTO_INCREMENT=112880 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `experiments_fields`
--

LOCK TABLES `experiments_fields` WRITE;
/*!40000 ALTER TABLE `experiments_fields` DISABLE KEYS */;
INSERT INTO `experiments_fields` VALUES (106634,1,1,'Acinetobacter baumannii',NULL,NULL,NULL,NULL,NULL),(106635,1,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106636,1,3,NULL,NULL,2,NULL,NULL,NULL),(106637,1,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106638,1,5,'plates',NULL,NULL,NULL,NULL,NULL),(106639,1,6,'MHB',NULL,NULL,NULL,NULL,NULL),(106640,1,7,'colistin',NULL,NULL,NULL,NULL,NULL),(106641,1,8,NULL,NULL,NULL,NULL,0,NULL),(106642,1,10,NULL,NULL,NULL,NULL,0,NULL),(106643,1,12,NULL,NULL,NULL,NULL,1,NULL),(106644,1,13,'agar plates',NULL,NULL,NULL,NULL,NULL),(106645,1,14,NULL,NULL,NULL,NULL,0,NULL),(106646,1,19,NULL,NULL,NULL,NULL,0,NULL),(106647,1,22,'targeted',NULL,NULL,NULL,NULL,NULL),(106648,1,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(106649,1,23,NULL,NULL,NULL,NULL,1,NULL),(106650,1,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(106651,1,25,'Genome Analyzer IIx',NULL,NULL,NULL,NULL,NULL),(106652,1,26,NULL,NULL,2,NULL,NULL,NULL),(106653,1,28,NULL,NULL,110,NULL,NULL,NULL),(106654,1,34,NULL,'Colistin resistance can develop in one step by loss of LPS production.',NULL,NULL,NULL,NULL),(106655,2,1,'Acinetobacter baylyi',NULL,NULL,NULL,NULL,NULL),(106656,2,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106657,2,3,NULL,NULL,24,NULL,NULL,NULL),(106658,2,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106659,2,5,'plastic microcosms',NULL,NULL,NULL,NULL,NULL),(106660,2,6,'LB',NULL,NULL,NULL,NULL,NULL),(106661,2,7,'coevolution',NULL,NULL,NULL,NULL,NULL),(106662,2,8,NULL,NULL,NULL,NULL,0,NULL),(106663,2,10,NULL,NULL,NULL,NULL,1,NULL),(106664,2,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(106665,2,12,NULL,NULL,NULL,NULL,0,NULL),(106666,2,14,NULL,NULL,NULL,NULL,0,NULL),(106667,2,16,NULL,NULL,33,NULL,NULL,NULL),(106668,2,17,NULL,NULL,5,NULL,NULL,NULL),(106669,2,19,NULL,NULL,NULL,NULL,0,NULL),(106670,2,23,NULL,NULL,NULL,NULL,0,NULL),(106671,2,34,NULL,'Natural transformation did not improve the evolution in competition with phages',NULL,NULL,NULL,NULL),(106672,3,1,'Acinetobacter baylyi',NULL,NULL,NULL,NULL,NULL),(106673,3,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106674,3,3,NULL,NULL,1,NULL,NULL,NULL),(106675,3,4,'continuous',NULL,NULL,NULL,NULL,NULL),(106676,3,7,'nitrogen limitation',NULL,NULL,NULL,NULL,NULL),(106677,3,16,NULL,NULL,2800,NULL,NULL,NULL),(106678,3,17,NULL,NULL,128,NULL,NULL,NULL),(106679,3,22,'targeted',NULL,NULL,NULL,NULL,NULL),(106680,3,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(106681,3,23,NULL,NULL,NULL,NULL,1,NULL),(106682,3,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(106683,3,26,NULL,NULL,3,NULL,NULL,NULL),(106684,3,27,NULL,NULL,3,NULL,NULL,NULL),(106685,3,30,NULL,NULL,NULL,1,NULL,NULL),(106686,3,32,NULL,NULL,30000000000,NULL,NULL,NULL),(106687,3,33,NULL,NULL,30000000000,NULL,NULL,NULL),(106688,3,34,NULL,'long-term diversity emerges in simple medium in which initial rapid increase is followed by slower increase in fitness, which can be approximated by maximal growth rate',NULL,NULL,NULL,NULL),(106689,4,1,'Acinetobacter baylyi',NULL,NULL,NULL,NULL,NULL),(106690,4,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106691,4,3,NULL,NULL,1,NULL,NULL,NULL),(106692,4,4,'continuous',NULL,NULL,NULL,NULL,NULL),(106693,4,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(106694,4,6,'LB',NULL,NULL,NULL,NULL,NULL),(106695,4,7,'adaptation to cripple gene',NULL,NULL,NULL,NULL,NULL),(106696,4,10,NULL,NULL,NULL,NULL,0,NULL),(106697,4,12,NULL,NULL,NULL,NULL,0,NULL),(106698,4,16,NULL,NULL,195,NULL,NULL,NULL),(106699,4,17,NULL,NULL,17,NULL,NULL,NULL),(106700,4,18,'phenotype on LB agar',NULL,NULL,NULL,NULL,NULL),(106701,4,18,'kanamycin resistance',NULL,NULL,NULL,NULL,NULL),(106702,4,18,'sensitivity to sucrose',NULL,NULL,NULL,NULL,NULL),(106703,4,18,'PCR',NULL,NULL,NULL,NULL,NULL),(106704,4,19,NULL,NULL,NULL,NULL,0,NULL),(106705,4,23,NULL,NULL,NULL,NULL,0,NULL),(106706,4,32,NULL,NULL,5000000000,NULL,NULL,NULL),(106707,4,33,NULL,NULL,5000000000,NULL,NULL,NULL),(106708,4,34,NULL,'Development of a continuous system with a moving tube to evade wall growth.',NULL,NULL,NULL,NULL),(106709,5,1,'Acinetobacter nosocomialis',NULL,NULL,NULL,NULL,NULL),(106710,5,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106711,5,3,NULL,NULL,1,NULL,NULL,NULL),(106712,5,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106713,5,5,'plates',NULL,NULL,NULL,NULL,NULL),(106714,5,7,'colistin',NULL,NULL,NULL,NULL,NULL),(106715,5,8,NULL,NULL,NULL,NULL,0,NULL),(106716,5,10,NULL,NULL,NULL,NULL,1,NULL),(106717,5,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(106718,5,12,NULL,NULL,NULL,NULL,1,NULL),(106719,5,13,'plates',NULL,NULL,NULL,NULL,NULL),(106720,5,14,NULL,NULL,NULL,NULL,0,NULL),(106721,5,19,NULL,NULL,NULL,NULL,0,NULL),(106722,5,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(106723,5,23,NULL,NULL,NULL,NULL,1,NULL),(106724,5,24,'454',NULL,NULL,NULL,NULL,NULL),(106725,5,25,'Titanium',NULL,NULL,NULL,NULL,NULL),(106726,5,26,NULL,NULL,1,NULL,NULL,NULL),(106727,5,34,NULL,'Loss of LPS production also underlies resistance development to colistin in A. nosocomialis',NULL,NULL,NULL,NULL),(106731,7,1,'Aspergillus fumigatus',NULL,NULL,NULL,NULL,NULL),(106732,7,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106733,7,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106734,8,1,'Aspergillus nidulans',NULL,NULL,NULL,NULL,NULL),(106735,8,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106736,8,3,NULL,NULL,80,NULL,NULL,NULL),(106737,8,4,'MA',NULL,NULL,NULL,NULL,NULL),(106738,8,5,'plate',NULL,NULL,NULL,NULL,NULL),(106739,8,6,'minimal medium',NULL,NULL,NULL,NULL,NULL),(106740,8,7,'MA',NULL,NULL,NULL,NULL,NULL),(106741,8,8,NULL,NULL,NULL,NULL,0,NULL),(106742,8,10,NULL,NULL,NULL,NULL,0,NULL),(106743,8,12,NULL,NULL,NULL,NULL,1,NULL),(106744,8,13,'plates',NULL,NULL,NULL,NULL,NULL),(106745,8,14,NULL,NULL,NULL,NULL,0,NULL),(106746,8,17,NULL,NULL,480,NULL,NULL,NULL),(106747,8,19,NULL,NULL,NULL,NULL,1,NULL),(106748,8,20,'colony diameter',NULL,NULL,NULL,NULL,NULL),(106749,8,23,NULL,NULL,NULL,NULL,0,NULL),(106750,8,34,NULL,'During MA, sexual reproduction accumulates less slightly deleterious mutations',NULL,NULL,NULL,NULL),(106751,9,1,'Bacillus subtilis',NULL,NULL,NULL,NULL,NULL),(106752,9,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(106753,9,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106776,11,1,'Bacillus subtilis',NULL,NULL,NULL,NULL,NULL),(106777,11,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(106778,11,3,NULL,NULL,2,NULL,NULL,NULL),(106779,11,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106780,11,5,'flasks',NULL,NULL,NULL,NULL,NULL),(106781,11,6,'LB',NULL,NULL,NULL,NULL,NULL),(106782,11,7,'air pressure',NULL,NULL,NULL,NULL,NULL),(106783,11,8,NULL,NULL,NULL,NULL,0,NULL),(106784,11,10,NULL,NULL,NULL,NULL,0,NULL),(106785,11,12,NULL,NULL,NULL,NULL,0,NULL),(106786,11,14,NULL,NULL,NULL,NULL,0,NULL),(106787,11,16,NULL,NULL,1000,NULL,NULL,NULL),(106788,11,17,NULL,NULL,150,NULL,NULL,NULL),(106789,11,18,'antibiotc resistance',NULL,NULL,NULL,NULL,NULL),(106790,11,19,NULL,NULL,NULL,NULL,1,NULL),(106791,11,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(106792,11,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(106793,11,23,NULL,NULL,NULL,NULL,1,NULL),(106794,11,34,NULL,'B. subtillis evolved in a stepwise manner improved performance at low pressure.',NULL,NULL,NULL,NULL),(106795,12,1,'Bacillus thuringiensis',NULL,NULL,NULL,NULL,NULL),(106796,12,1,'Caenorhabditis elegans',NULL,NULL,NULL,NULL,NULL),(106797,12,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106798,12,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106807,15,1,'Burkholderia cenocepacia',NULL,NULL,NULL,NULL,NULL),(106808,15,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106809,15,3,NULL,NULL,75,NULL,NULL,NULL),(106810,15,4,'MA',NULL,NULL,NULL,NULL,NULL),(106811,15,5,'plates',NULL,NULL,NULL,NULL,NULL),(106812,15,6,'TSA agar',NULL,NULL,NULL,NULL,NULL),(106813,15,7,'MA',NULL,NULL,NULL,NULL,NULL),(106814,15,8,NULL,NULL,NULL,NULL,0,NULL),(106815,15,10,NULL,NULL,NULL,NULL,0,NULL),(106816,15,12,NULL,NULL,NULL,NULL,1,NULL),(106817,15,13,'plates',NULL,NULL,NULL,NULL,NULL),(106818,15,14,NULL,NULL,NULL,NULL,0,NULL),(106819,15,16,NULL,NULL,5554,NULL,NULL,NULL),(106820,15,17,NULL,NULL,217,NULL,NULL,NULL),(106821,15,19,NULL,NULL,NULL,NULL,1,NULL),(106822,15,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(106823,15,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(106824,15,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(106825,15,23,NULL,NULL,NULL,NULL,1,NULL),(106826,15,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(106827,15,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(106828,15,26,NULL,NULL,47,NULL,NULL,NULL),(106829,15,28,NULL,NULL,50,NULL,NULL,NULL),(106830,15,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=dod',NULL,NULL,NULL,NULL,NULL),(106831,15,33,NULL,NULL,1,NULL,NULL,NULL),(106832,15,34,NULL,'MA experiment in B. cenocepacia uncovers the mutational spectrum of spontaneous mutations. The fitness effect of the mutational spectrum obtained under MA in B. cenocepacia confirmed that most mutations were deleterious or neutral, albeit some showed positive effects in alternative environments',NULL,NULL,NULL,NULL),(106857,17,1,'Candida albicans',NULL,NULL,NULL,NULL,NULL),(106858,17,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106859,18,1,'Candida glabrata',NULL,NULL,NULL,NULL,NULL),(106860,18,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106861,18,3,NULL,NULL,16,NULL,NULL,NULL),(106862,18,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106863,18,5,'mouse GI tract',NULL,NULL,NULL,NULL,NULL),(106864,18,7,'caspofungin',NULL,NULL,NULL,NULL,NULL),(106865,18,8,NULL,NULL,NULL,NULL,0,NULL),(106866,18,10,NULL,NULL,NULL,NULL,1,NULL),(106867,18,11,'mouse gut',NULL,NULL,NULL,NULL,NULL),(106868,18,12,NULL,NULL,NULL,NULL,1,NULL),(106869,18,13,'mouse gut',NULL,NULL,NULL,NULL,NULL),(106870,18,14,NULL,NULL,NULL,NULL,1,NULL),(106871,18,15,'mouse gut',NULL,NULL,NULL,NULL,NULL),(106872,18,17,NULL,NULL,21,NULL,NULL,NULL),(106873,18,19,NULL,NULL,NULL,NULL,0,NULL),(106874,18,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(106875,18,23,NULL,NULL,NULL,NULL,0,NULL),(106876,18,34,NULL,'Mutators result in faster evolution of caspofungin resistance in mice model.',NULL,NULL,NULL,NULL),(106877,19,1,'Chlamydomonas reinhardtii',NULL,NULL,NULL,NULL,NULL),(106878,19,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106879,19,3,NULL,NULL,8,NULL,NULL,NULL),(106880,19,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106881,19,5,'24-well plates',NULL,NULL,NULL,NULL,NULL),(106882,19,6,'COMBO medium',NULL,NULL,NULL,NULL,NULL),(106883,19,7,'predation',NULL,NULL,NULL,NULL,NULL),(106884,19,8,NULL,NULL,NULL,NULL,0,NULL),(106885,19,10,NULL,NULL,NULL,NULL,0,NULL),(106886,19,12,NULL,NULL,NULL,NULL,0,NULL),(106887,19,14,NULL,NULL,NULL,NULL,0,NULL),(106888,19,16,NULL,NULL,300,NULL,NULL,NULL),(106889,19,17,NULL,NULL,350,NULL,NULL,NULL),(106890,19,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(106891,19,19,NULL,NULL,NULL,NULL,0,NULL),(106892,19,23,NULL,NULL,NULL,NULL,0,NULL),(106893,19,34,NULL,'Multicellularity evolved upon predation results in a loss of phototaxis, a costly trade-off.',NULL,NULL,NULL,NULL),(106894,20,1,'Chlamydomonas reinhardtii',NULL,NULL,NULL,NULL,NULL),(106895,20,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106896,20,3,NULL,NULL,8,NULL,NULL,NULL),(106897,20,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106898,20,5,'24-well plates',NULL,NULL,NULL,NULL,NULL),(106899,20,6,'COMBO medium',NULL,NULL,NULL,NULL,NULL),(106900,20,7,'predation',NULL,NULL,NULL,NULL,NULL),(106901,20,8,NULL,NULL,NULL,NULL,0,NULL),(106902,20,10,NULL,NULL,NULL,NULL,0,NULL),(106903,20,12,NULL,NULL,NULL,NULL,0,NULL),(106904,20,14,NULL,NULL,NULL,NULL,0,NULL),(106905,20,16,NULL,NULL,300,NULL,NULL,NULL),(106906,20,17,NULL,NULL,350,NULL,NULL,NULL),(106907,20,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(106908,20,19,NULL,NULL,NULL,NULL,0,NULL),(106909,20,23,NULL,NULL,NULL,NULL,0,NULL),(106910,20,34,NULL,'Predation can lead to evolution of multicellularity as a way to protect themselfs by physical engulfment',NULL,NULL,NULL,NULL),(106914,22,1,'Chlamydomonas reinhardtii',NULL,NULL,NULL,NULL,NULL),(106915,22,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106916,22,4,'MA',NULL,NULL,NULL,NULL,NULL),(106917,23,1,'Comamonas sp.',NULL,NULL,NULL,NULL,NULL),(106918,23,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106919,23,3,NULL,NULL,18,NULL,NULL,NULL),(106920,23,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106921,23,7,'2-4-D limitation',NULL,NULL,NULL,NULL,NULL),(106922,23,16,NULL,NULL,1000,NULL,NULL,NULL),(106923,23,17,NULL,NULL,260,NULL,NULL,NULL),(106924,23,18,'streptomycin',NULL,NULL,NULL,NULL,NULL),(106925,23,18,'nalidixic acid resistance',NULL,NULL,NULL,NULL,NULL),(106926,23,21,'streptomycin',NULL,NULL,NULL,NULL,NULL),(106927,23,21,'nalidixic acid resistance',NULL,NULL,NULL,NULL,NULL),(106928,23,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(106929,23,23,NULL,NULL,NULL,NULL,0,NULL),(106930,23,32,NULL,NULL,2600000000,NULL,NULL,NULL),(106931,23,33,NULL,NULL,10000000,NULL,NULL,NULL),(106932,23,34,NULL,'structured environments allow the emergence of more diversity and could speed up evolution',NULL,NULL,NULL,NULL),(106933,24,1,'Corynebacterium glutamicum',NULL,NULL,NULL,NULL,NULL),(106934,24,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(106935,24,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106940,26,1,'Desulfovibrio vulgaris',NULL,NULL,NULL,NULL,NULL),(106941,26,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106942,26,3,NULL,NULL,12,NULL,NULL,NULL),(106943,26,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106944,26,7,'minimal medium',NULL,NULL,NULL,NULL,NULL),(106945,26,16,NULL,NULL,1000,NULL,NULL,NULL),(106946,26,17,NULL,NULL,250,NULL,NULL,NULL),(106947,26,18,'kanamycin resistance',NULL,NULL,NULL,NULL,NULL),(106948,26,18,'16S rRNA',NULL,NULL,NULL,NULL,NULL),(106949,26,21,'barcode',NULL,NULL,NULL,NULL,NULL),(106950,26,23,NULL,NULL,NULL,NULL,0,NULL),(106951,26,34,NULL,'fitness increased rapidly in the beginning but levelled off later',NULL,NULL,NULL,NULL),(106952,27,1,'Desulfovibrio vulgaris',NULL,NULL,NULL,NULL,NULL),(106953,27,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(106954,27,3,NULL,NULL,12,NULL,NULL,NULL),(106955,27,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106956,28,1,'Emiliania huxleyi',NULL,NULL,NULL,NULL,NULL),(106957,28,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(106958,28,3,NULL,NULL,15,NULL,NULL,NULL),(106959,28,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(106960,28,5,'flasks',NULL,NULL,NULL,NULL,NULL),(106961,28,6,'artificial sea water medium',NULL,NULL,NULL,NULL,NULL),(106962,28,7,'CO2 pressure',NULL,NULL,NULL,NULL,NULL),(106963,28,8,NULL,NULL,NULL,NULL,0,NULL),(106964,28,10,NULL,NULL,NULL,NULL,0,NULL),(106965,28,12,NULL,NULL,NULL,NULL,0,NULL),(106966,28,14,NULL,NULL,NULL,NULL,0,NULL),(106967,28,16,NULL,NULL,2100,NULL,NULL,NULL),(106968,28,17,NULL,NULL,1460,NULL,NULL,NULL),(106969,28,19,NULL,NULL,NULL,NULL,1,NULL),(106970,28,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(106971,28,23,NULL,NULL,NULL,NULL,0,NULL),(106972,28,34,NULL,'Calcification in the algae evolved in response on elevated C02 pressures in a complex manner',NULL,NULL,NULL,NULL),(106973,29,1,'Enterococcus faecalis',NULL,NULL,NULL,NULL,NULL),(106974,29,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(106975,29,3,NULL,NULL,2,NULL,NULL,NULL),(106976,29,4,'continuous',NULL,NULL,NULL,NULL,NULL),(106977,29,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(106978,29,7,'tigecycline',NULL,NULL,NULL,NULL,NULL),(106979,29,10,NULL,NULL,NULL,NULL,1,NULL),(106980,29,11,'stepwise increases',NULL,NULL,NULL,NULL,NULL),(106981,29,12,NULL,NULL,NULL,NULL,1,NULL),(106982,29,13,'biofilm growth',NULL,NULL,NULL,NULL,NULL),(106983,29,17,NULL,NULL,24,NULL,NULL,NULL),(106984,29,18,'phenotype on LBHI agar',NULL,NULL,NULL,NULL,NULL),(106985,29,19,NULL,NULL,NULL,NULL,1,NULL),(106986,29,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(106987,29,21,'tigecycline resistance',NULL,NULL,NULL,NULL,NULL),(106988,29,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(106989,29,23,NULL,NULL,NULL,NULL,1,NULL),(106990,29,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(106991,29,25,'Illumina HiSeq',NULL,NULL,NULL,NULL,NULL),(106992,29,26,NULL,NULL,14,NULL,NULL,NULL),(106993,29,27,NULL,NULL,2,NULL,NULL,NULL),(106994,29,28,NULL,NULL,254,NULL,NULL,NULL),(106995,29,29,NULL,NULL,818,NULL,NULL,NULL),(106996,29,30,NULL,NULL,NULL,5,NULL,NULL),(106997,29,32,NULL,NULL,6000000000000,NULL,NULL,NULL),(106998,29,33,NULL,NULL,6000000000000,NULL,NULL,NULL),(106999,29,34,NULL,'Horizontal gene transfer of a resistance carrying transposon can be increased upon selection for resistance',NULL,NULL,NULL,NULL),(107000,30,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107001,30,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107002,30,3,NULL,NULL,24,NULL,NULL,NULL),(107003,30,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107004,30,5,'flasks',NULL,NULL,NULL,NULL,NULL),(107005,30,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107006,30,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(107007,30,8,NULL,NULL,NULL,NULL,0,NULL),(107008,30,10,NULL,NULL,NULL,NULL,0,NULL),(107009,30,12,NULL,NULL,NULL,NULL,0,NULL),(107010,30,14,NULL,NULL,NULL,NULL,0,NULL),(107011,30,16,NULL,NULL,200,NULL,NULL,NULL),(107012,30,17,NULL,NULL,30,NULL,NULL,NULL),(107013,30,18,'T5',NULL,NULL,NULL,NULL,NULL),(107014,30,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107015,30,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107016,30,19,NULL,NULL,NULL,NULL,1,NULL),(107017,30,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107018,30,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107019,30,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107020,30,23,NULL,NULL,NULL,NULL,1,NULL),(107021,30,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107022,30,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107023,30,26,NULL,NULL,40,NULL,NULL,NULL),(107024,30,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA396011',NULL,NULL,NULL,NULL,NULL),(107025,30,32,NULL,NULL,500000000,NULL,NULL,NULL),(107026,30,33,NULL,NULL,5000000,NULL,NULL,NULL),(107027,30,34,NULL,'Plasmid-mediated recombination did not accelerated evolution of the LTEE because of overwhelming effects of incoming DNA that resulted in fixation of donor alleles without any selective benefit.',NULL,NULL,NULL,NULL),(107028,31,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107029,31,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107030,31,3,NULL,NULL,3,NULL,NULL,NULL),(107031,31,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107032,31,5,'flask',NULL,NULL,NULL,NULL,NULL),(107033,31,6,'M9',NULL,NULL,NULL,NULL,NULL),(107034,31,7,'L-1,2-propanediol usage',NULL,NULL,NULL,NULL,NULL),(107035,31,8,NULL,NULL,NULL,NULL,0,NULL),(107036,31,10,NULL,NULL,NULL,NULL,1,NULL),(107037,31,11,'decreasing glycerol concentration',NULL,NULL,NULL,NULL,NULL),(107038,31,12,NULL,NULL,NULL,NULL,0,NULL),(107039,31,14,NULL,NULL,NULL,NULL,0,NULL),(107040,31,16,NULL,NULL,450,NULL,NULL,NULL),(107041,31,17,NULL,NULL,60,NULL,NULL,NULL),(107042,31,18,'PCR',NULL,NULL,NULL,NULL,NULL),(107043,31,18,'16S RNA sequencing',NULL,NULL,NULL,NULL,NULL),(107044,31,19,NULL,NULL,NULL,NULL,1,NULL),(107045,31,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107046,31,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107047,31,23,NULL,NULL,NULL,NULL,1,NULL),(107048,31,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107049,31,25,'GAIIx',NULL,NULL,NULL,NULL,NULL),(107050,31,26,NULL,NULL,1,NULL,NULL,NULL),(107051,31,32,NULL,NULL,500000000,NULL,NULL,NULL),(107052,31,33,NULL,NULL,500000,NULL,NULL,NULL),(107053,31,34,NULL,'By experimental evolution, E. coli gained the ability to grow on a previously unusable carbon source',NULL,NULL,NULL,NULL),(107054,32,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107055,32,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107056,32,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107057,32,8,NULL,NULL,NULL,NULL,1,NULL),(107058,32,9,'mutator',NULL,NULL,NULL,NULL,NULL),(107059,32,12,NULL,NULL,NULL,NULL,1,NULL),(107060,32,13,'plate expansion',NULL,NULL,NULL,NULL,NULL),(107061,33,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107062,33,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107063,34,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107064,34,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107065,34,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107066,35,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107067,35,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107068,36,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107069,36,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107070,36,3,NULL,NULL,24,NULL,NULL,NULL),(107071,36,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107072,36,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(107073,36,6,'LB',NULL,NULL,NULL,NULL,NULL),(107074,36,7,'bennzoate',NULL,NULL,NULL,NULL,NULL),(107075,36,8,NULL,NULL,NULL,NULL,0,NULL),(107076,36,10,NULL,NULL,NULL,NULL,1,NULL),(107077,36,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(107078,36,12,NULL,NULL,NULL,NULL,0,NULL),(107079,36,14,NULL,NULL,NULL,NULL,0,NULL),(107080,36,16,NULL,NULL,2000,NULL,NULL,NULL),(107081,36,17,NULL,NULL,300,NULL,NULL,NULL),(107082,36,19,NULL,NULL,NULL,NULL,1,NULL),(107083,36,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107084,36,20,'yield',NULL,NULL,NULL,NULL,NULL),(107085,36,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107086,36,23,NULL,NULL,NULL,NULL,1,NULL),(107087,36,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107088,36,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107089,36,26,NULL,NULL,16,NULL,NULL,NULL),(107090,36,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP074501.',NULL,NULL,NULL,NULL,NULL),(107091,36,32,NULL,NULL,50000000,NULL,NULL,NULL),(107092,36,33,NULL,NULL,500000,NULL,NULL,NULL),(107093,36,34,NULL,'Selection for benzoate tolerance can result in reversion of antibiotic resistance',NULL,NULL,NULL,NULL),(107094,37,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107095,37,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107096,37,3,NULL,NULL,30,NULL,NULL,NULL),(107097,37,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107098,37,5,'flasks',NULL,NULL,NULL,NULL,NULL),(107099,37,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107100,37,7,'temperature',NULL,NULL,NULL,NULL,NULL),(107101,37,8,NULL,NULL,NULL,NULL,0,NULL),(107102,37,10,NULL,NULL,NULL,NULL,1,NULL),(107103,37,11,'shifting between temperatures',NULL,NULL,NULL,NULL,NULL),(107104,37,12,NULL,NULL,NULL,NULL,0,NULL),(107105,37,14,NULL,NULL,NULL,NULL,0,NULL),(107106,37,16,NULL,NULL,2000,NULL,NULL,NULL),(107107,37,17,NULL,NULL,150,NULL,NULL,NULL),(107108,37,18,'T5',NULL,NULL,NULL,NULL,NULL),(107109,37,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107110,37,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107111,37,19,NULL,NULL,NULL,NULL,1,NULL),(107112,37,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107113,37,21,'Tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107114,37,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107115,37,23,NULL,NULL,NULL,NULL,1,NULL),(107116,37,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107117,37,25,'GAIIx',NULL,NULL,NULL,NULL,NULL),(107118,37,26,NULL,NULL,30,NULL,NULL,NULL),(107119,37,31,'https://www.ncbi.nlm.nih.gov/sra/SRP009429',NULL,NULL,NULL,NULL,NULL),(107120,37,32,NULL,NULL,500000000,NULL,NULL,NULL),(107121,37,33,NULL,NULL,5000000,NULL,NULL,NULL),(107122,37,34,NULL,'Genetic signatures can be found of specialization to thermal niches. \n\nFitness trade-off costs seem to be assymmetric during adaptation to thermal niches.\n\nVariety in rates and extent of adaptation between different temperature groups occur without apparent trade-off costs but with costs for generalist.\n\n	\nAdaptation on 42°C occurs over 200 generations',NULL,NULL,NULL,NULL),(107123,38,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107124,38,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107125,38,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107126,38,7,'anaerobic conditions',NULL,NULL,NULL,NULL,NULL),(107127,39,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107128,39,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107129,39,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107130,39,12,NULL,NULL,NULL,NULL,1,NULL),(107131,39,13,'plates',NULL,NULL,NULL,NULL,NULL),(107137,42,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107138,42,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107139,42,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107140,42,35,NULL,'robotics',NULL,NULL,NULL,NULL),(107141,43,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107142,43,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107143,43,3,NULL,NULL,42,NULL,NULL,NULL),(107144,43,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107145,43,5,'flask',NULL,NULL,NULL,NULL,NULL),(107146,43,6,'LB',NULL,NULL,NULL,NULL,NULL),(107147,43,7,'ampicillin',NULL,NULL,NULL,NULL,NULL),(107148,43,8,NULL,NULL,NULL,NULL,0,NULL),(107149,43,10,NULL,NULL,NULL,NULL,0,NULL),(107150,43,12,NULL,NULL,NULL,NULL,0,NULL),(107151,43,14,NULL,NULL,NULL,NULL,0,NULL),(107152,43,17,NULL,NULL,17,NULL,NULL,NULL),(107153,43,18,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107154,43,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107155,43,23,NULL,NULL,NULL,NULL,1,NULL),(107156,43,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107157,43,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(107158,43,26,NULL,NULL,6,NULL,NULL,NULL),(107159,43,31,'https://www.ncbi.nlm.nih.gov/sra?linkname=bioproject_sra_all&from_uid=361209',NULL,NULL,NULL,NULL,NULL),(107160,43,34,NULL,'Resistant mutants only arise in Amp treatment after tolerance arived and they emerge in tolerant background',NULL,NULL,NULL,NULL),(107186,45,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107187,45,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107188,45,3,NULL,NULL,10,NULL,NULL,NULL),(107189,45,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107190,45,5,'flask',NULL,NULL,NULL,NULL,NULL),(107191,45,6,'M9',NULL,NULL,NULL,NULL,NULL),(107192,45,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(107193,46,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107194,46,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107195,46,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107196,47,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107197,47,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107198,47,3,NULL,NULL,18,NULL,NULL,NULL),(107199,47,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107200,47,5,'tube',NULL,NULL,NULL,NULL,NULL),(107201,47,6,'minimal medium',NULL,NULL,NULL,NULL,NULL),(107202,47,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(107203,47,8,NULL,NULL,NULL,NULL,1,NULL),(107204,47,9,'auxotrophic mutants',NULL,NULL,NULL,NULL,NULL),(107205,47,10,NULL,NULL,NULL,NULL,0,NULL),(107206,47,12,NULL,NULL,NULL,NULL,0,NULL),(107207,47,14,NULL,NULL,NULL,NULL,0,NULL),(107208,47,16,NULL,NULL,46,NULL,NULL,NULL),(107209,47,17,NULL,NULL,56,NULL,NULL,NULL),(107210,47,18,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107211,47,19,NULL,NULL,NULL,NULL,0,NULL),(107212,47,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107213,47,23,NULL,NULL,NULL,NULL,1,NULL),(107214,47,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107215,47,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107216,47,26,NULL,NULL,40,NULL,NULL,NULL),(107217,47,34,NULL,'Evolution of spatial structure by attachment repeatedly evolves during coevolution of imposed mutualists',NULL,NULL,NULL,NULL),(107218,48,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107219,48,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107220,48,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107221,49,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107222,49,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107223,49,3,NULL,NULL,5,NULL,NULL,NULL),(107224,49,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107225,49,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107226,49,6,'M9',NULL,NULL,NULL,NULL,NULL),(107227,49,7,'L-serine',NULL,NULL,NULL,NULL,NULL),(107228,49,8,NULL,NULL,NULL,NULL,0,NULL),(107229,49,10,NULL,NULL,NULL,NULL,1,NULL),(107230,49,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(107231,49,12,NULL,NULL,NULL,NULL,0,NULL),(107232,49,14,NULL,NULL,NULL,NULL,0,NULL),(107233,49,17,NULL,NULL,45,NULL,NULL,NULL),(107234,49,19,NULL,NULL,NULL,NULL,1,NULL),(107235,49,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107236,49,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107237,49,23,NULL,NULL,NULL,NULL,1,NULL),(107238,49,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107239,49,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107240,49,26,NULL,NULL,10,NULL,NULL,NULL),(107241,49,34,NULL,'Tolerance to L-serine was gradually increased and with the more tolerant strain, production was also increased',NULL,NULL,NULL,NULL),(107242,49,35,NULL,'robotics to maintain exponential growth',NULL,NULL,NULL,NULL),(107243,50,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107244,50,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107245,50,3,NULL,NULL,72,NULL,NULL,NULL),(107246,50,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107247,50,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107248,50,6,'glycerol minimal medium',NULL,NULL,NULL,NULL,NULL),(107249,50,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107250,50,8,NULL,NULL,NULL,NULL,1,NULL),(107251,50,9,'temporal mutator',NULL,NULL,NULL,NULL,NULL),(107252,50,10,NULL,NULL,NULL,NULL,1,NULL),(107253,50,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(107254,50,12,NULL,NULL,NULL,NULL,0,NULL),(107255,50,14,NULL,NULL,NULL,NULL,0,NULL),(107256,50,16,NULL,NULL,500,NULL,NULL,NULL),(107257,50,19,NULL,NULL,NULL,NULL,1,NULL),(107258,50,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107259,50,22,'targeted',NULL,NULL,NULL,NULL,NULL),(107260,50,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107261,50,23,NULL,NULL,NULL,NULL,1,NULL),(107262,50,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107263,50,25,'NextSeq500',NULL,NULL,NULL,NULL,NULL),(107264,50,27,NULL,NULL,12,NULL,NULL,NULL),(107265,50,29,NULL,NULL,500,NULL,NULL,NULL),(107266,50,30,NULL,NULL,NULL,7.6,NULL,NULL),(107267,50,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP110624',NULL,NULL,NULL,NULL,NULL),(107268,50,34,NULL,'Increased mutation rate AND recombination proficient E. coli results in a superior evolutionary adaptation to glycerol and some antibiotics, results in maintenance of diversity of beneficial mutations and in in situ recombination events',NULL,NULL,NULL,NULL),(107269,51,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107270,51,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107271,51,3,NULL,NULL,12,NULL,NULL,NULL),(107272,51,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107273,51,5,'flask',NULL,NULL,NULL,NULL,NULL),(107274,51,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107275,51,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(107276,51,8,NULL,NULL,NULL,NULL,0,NULL),(107277,51,10,NULL,NULL,NULL,NULL,0,NULL),(107278,51,12,NULL,NULL,NULL,NULL,0,NULL),(107279,51,14,NULL,NULL,NULL,NULL,0,NULL),(107280,51,16,NULL,NULL,50000,NULL,NULL,NULL),(107281,51,17,NULL,NULL,7500,NULL,NULL,NULL),(107282,51,18,'T5',NULL,NULL,NULL,NULL,NULL),(107283,51,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107284,51,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107285,51,19,NULL,NULL,NULL,NULL,1,NULL),(107286,51,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107287,51,21,'blue white lacZ or tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107288,51,23,NULL,NULL,NULL,NULL,0,NULL),(107289,51,32,NULL,NULL,500000000,NULL,NULL,NULL),(107290,51,33,NULL,NULL,5000000,NULL,NULL,NULL),(107291,51,34,NULL,'Mutations in pykF that fixed in LTEE tend to have complex and dynamic epistatic interactions with different backgrounds during the LTEE',NULL,NULL,NULL,NULL),(107292,52,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107293,52,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107294,53,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107295,53,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107296,53,3,NULL,NULL,21,NULL,NULL,NULL),(107297,53,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107298,53,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107299,53,6,'M9 glucose',NULL,NULL,NULL,NULL,NULL),(107300,53,6,'xylose',NULL,NULL,NULL,NULL,NULL),(107301,53,6,'glycerol',NULL,NULL,NULL,NULL,NULL),(107302,53,6,'acetate',NULL,NULL,NULL,NULL,NULL),(107303,53,7,'shifting carbon sources',NULL,NULL,NULL,NULL,NULL),(107304,53,8,NULL,NULL,NULL,NULL,0,NULL),(107305,53,10,NULL,NULL,NULL,NULL,1,NULL),(107306,53,11,'shifting between carbon source',NULL,NULL,NULL,NULL,NULL),(107307,53,12,NULL,NULL,NULL,NULL,0,NULL),(107308,53,14,NULL,NULL,NULL,NULL,0,NULL),(107309,53,16,NULL,NULL,1180,NULL,NULL,NULL),(107310,53,17,NULL,NULL,55,NULL,NULL,NULL),(107311,53,19,NULL,NULL,NULL,NULL,1,NULL),(107312,53,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107313,53,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107314,53,23,NULL,NULL,NULL,NULL,1,NULL),(107315,53,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107316,53,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107317,53,27,NULL,NULL,31,NULL,NULL,NULL),(107318,53,29,NULL,NULL,100,NULL,NULL,NULL),(107319,53,31,'http://www.pnas.org/content/110/35/14302',NULL,NULL,NULL,NULL,NULL),(107320,53,34,NULL,'Under fluctuating carbon sources, either sequential selection for specialization occurs with subpopulations for each carbon source or generalists emerge',NULL,NULL,NULL,NULL),(107321,54,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107322,54,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107323,54,3,NULL,NULL,3,NULL,NULL,NULL),(107324,54,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107325,54,5,'fermentation vessels',NULL,NULL,NULL,NULL,NULL),(107326,54,6,'AM1 medium',NULL,NULL,NULL,NULL,NULL),(107327,54,7,'xylose fermentation',NULL,NULL,NULL,NULL,NULL),(107328,54,8,NULL,NULL,NULL,NULL,0,NULL),(107329,54,10,NULL,NULL,NULL,NULL,0,NULL),(107330,54,12,NULL,NULL,NULL,NULL,0,NULL),(107331,54,14,NULL,NULL,NULL,NULL,0,NULL),(107332,54,16,NULL,NULL,150,NULL,NULL,NULL),(107333,54,17,NULL,NULL,40,NULL,NULL,NULL),(107334,54,19,NULL,NULL,NULL,NULL,0,NULL),(107335,54,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107336,54,23,NULL,NULL,NULL,NULL,1,NULL),(107337,54,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107338,54,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107339,54,26,NULL,NULL,4,NULL,NULL,NULL),(107340,54,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP083931',NULL,NULL,NULL,NULL,NULL),(107341,54,34,NULL,'Strains selected during xylose fermentation loose catabolite repression and can be used for improved lactate production',NULL,NULL,NULL,NULL),(107342,55,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107343,55,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107344,55,3,NULL,NULL,40,NULL,NULL,NULL),(107345,55,4,'MA',NULL,NULL,NULL,NULL,NULL),(107346,55,5,'plates',NULL,NULL,NULL,NULL,NULL),(107347,55,5,'flasks',NULL,NULL,NULL,NULL,NULL),(107348,55,6,'LB',NULL,NULL,NULL,NULL,NULL),(107349,55,6,'MG',NULL,NULL,NULL,NULL,NULL),(107350,55,7,'MA',NULL,NULL,NULL,NULL,NULL),(107351,55,8,NULL,NULL,NULL,NULL,1,NULL),(107352,55,9,'mutator',NULL,NULL,NULL,NULL,NULL),(107353,55,10,NULL,NULL,NULL,NULL,0,NULL),(107354,55,12,NULL,NULL,NULL,NULL,1,NULL),(107355,55,13,'plates',NULL,NULL,NULL,NULL,NULL),(107356,55,14,NULL,NULL,NULL,NULL,0,NULL),(107357,55,16,NULL,NULL,1250,NULL,NULL,NULL),(107358,55,17,NULL,NULL,50,NULL,NULL,NULL),(107359,55,18,'tetracycline',NULL,NULL,NULL,NULL,NULL),(107360,55,19,NULL,NULL,NULL,NULL,1,NULL),(107361,55,20,'yield',NULL,NULL,NULL,NULL,NULL),(107362,55,23,NULL,NULL,NULL,NULL,0,NULL),(107363,55,33,NULL,NULL,1,NULL,NULL,NULL),(107364,55,34,NULL,'Under MA, mutators decrease their mutation rate in order to survive, if not they go extinct',NULL,NULL,NULL,NULL),(107365,56,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107366,56,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107367,57,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107368,57,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107369,57,3,NULL,NULL,20,NULL,NULL,NULL),(107370,57,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107371,57,7,'increasing ethanol concentrations',NULL,NULL,NULL,NULL,NULL),(107372,57,16,NULL,NULL,500,NULL,NULL,NULL),(107373,57,17,NULL,NULL,712,NULL,NULL,NULL),(107374,57,18,'(differential) fluorescent labels',NULL,NULL,NULL,NULL,NULL),(107375,57,19,NULL,NULL,NULL,NULL,1,NULL),(107376,57,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107377,57,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107378,57,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107379,57,23,NULL,NULL,NULL,NULL,1,NULL),(107380,57,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107381,57,25,'Illumina HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107382,57,26,NULL,NULL,10,NULL,NULL,NULL),(107383,57,27,NULL,NULL,8,NULL,NULL,NULL),(107384,57,28,NULL,NULL,150,NULL,NULL,NULL),(107385,57,29,NULL,NULL,150,NULL,NULL,NULL),(107386,57,30,NULL,NULL,NULL,10,NULL,NULL),(107387,57,31,'https://www.ncbi.nlm.nih.gov/biosample?Db=biosample&DbFrom=bioproject&Cmd=Link&LinkName=bioproject_biosample&LinkReadableName=BioSample&ordinalpos=1&IdsFromResult=344553',NULL,NULL,NULL,NULL,NULL),(107388,57,32,NULL,NULL,27000000000,NULL,NULL,NULL),(107389,57,33,NULL,NULL,270000000,NULL,NULL,NULL),(107390,57,34,NULL,'Mutation rate can change and respond in a flexible and transient way to changes in selective pressure, resulting in repeated evolution of hypermutation under increasing concentrations of ethanol. Decreases in mutation rate once adaptation is fulfilled because of increased cellular mortality.',NULL,NULL,NULL,NULL),(107391,58,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107392,58,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107393,58,3,NULL,NULL,176,NULL,NULL,NULL),(107394,58,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107395,58,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(107396,58,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107397,58,7,'glucose limtation',NULL,NULL,NULL,NULL,NULL),(107398,58,8,NULL,NULL,NULL,NULL,0,NULL),(107399,58,10,NULL,NULL,NULL,NULL,0,NULL),(107400,58,12,NULL,NULL,NULL,NULL,0,NULL),(107401,58,14,NULL,NULL,NULL,NULL,0,NULL),(107402,58,16,NULL,NULL,1000,NULL,NULL,NULL),(107403,58,18,'T5',NULL,NULL,NULL,NULL,NULL),(107404,58,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107405,58,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107406,58,19,NULL,NULL,NULL,NULL,1,NULL),(107407,58,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107408,58,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107409,58,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107410,58,23,NULL,NULL,NULL,NULL,1,NULL),(107411,58,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107412,58,25,'NextSeq500',NULL,NULL,NULL,NULL,NULL),(107413,58,26,NULL,NULL,88,NULL,NULL,NULL),(107414,58,31,'https://www.ncbi.nlm.nih.gov/bioproject/PRJNA356254/',NULL,NULL,NULL,NULL,NULL),(107415,58,32,NULL,NULL,10000000,NULL,NULL,NULL),(107416,58,33,NULL,NULL,5000000,NULL,NULL,NULL),(107417,58,34,NULL,'Declining adaptability is explained by a general decrease in the size of available beneficial mutations.',NULL,NULL,NULL,NULL),(107418,59,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107419,59,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107420,59,3,NULL,NULL,24,NULL,NULL,NULL),(107421,59,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107422,59,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(107423,59,6,'M9 + glucose',NULL,NULL,NULL,NULL,NULL),(107424,59,6,'acetate',NULL,NULL,NULL,NULL,NULL),(107425,59,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107426,59,8,NULL,NULL,NULL,NULL,0,NULL),(107427,59,10,NULL,NULL,NULL,NULL,1,NULL),(107428,59,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(107429,59,12,NULL,NULL,NULL,NULL,0,NULL),(107430,59,14,NULL,NULL,NULL,NULL,0,NULL),(107431,59,16,NULL,NULL,160,NULL,NULL,NULL),(107432,59,19,NULL,NULL,NULL,NULL,0,NULL),(107433,59,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107434,59,23,NULL,NULL,NULL,NULL,1,NULL),(107435,59,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107436,59,25,'GAIIx',NULL,NULL,NULL,NULL,NULL),(107437,59,27,NULL,NULL,27,NULL,NULL,NULL),(107438,59,31,'https://www.ebi.ac.uk/ena/data/view/PRJEB19222',NULL,NULL,NULL,NULL,NULL),(107439,59,34,NULL,'Metabolism can affect resistance development both by restraining in speed or affecting outcome, especially for compensatory mutations.',NULL,NULL,NULL,NULL),(107440,60,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107441,60,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107442,60,3,NULL,NULL,8,NULL,NULL,NULL),(107443,60,4,'MA',NULL,NULL,NULL,NULL,NULL),(107444,60,5,'plates',NULL,NULL,NULL,NULL,NULL),(107445,60,6,'LB',NULL,NULL,NULL,NULL,NULL),(107446,60,7,'MA at 2 temperatures',NULL,NULL,NULL,NULL,NULL),(107447,60,8,NULL,NULL,NULL,NULL,0,NULL),(107448,60,10,NULL,NULL,NULL,NULL,0,NULL),(107449,60,12,NULL,NULL,NULL,NULL,0,NULL),(107450,60,14,NULL,NULL,NULL,NULL,0,NULL),(107451,60,16,NULL,NULL,1870,NULL,NULL,NULL),(107452,60,17,NULL,NULL,85,NULL,NULL,NULL),(107453,60,19,NULL,NULL,NULL,NULL,0,NULL),(107454,60,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107455,60,23,NULL,NULL,NULL,NULL,1,NULL),(107456,60,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107457,60,24,'454',NULL,NULL,NULL,NULL,NULL),(107458,60,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107459,60,25,'454',NULL,NULL,NULL,NULL,NULL),(107460,60,26,NULL,NULL,9,NULL,NULL,NULL),(107461,60,33,NULL,NULL,1,NULL,NULL,NULL),(107462,60,34,NULL,'Overexpression of DnaK can buffer the effect of deleterious mutations and lead to a faster evolution rate',NULL,NULL,NULL,NULL),(107463,61,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107464,61,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107465,61,3,NULL,NULL,2,NULL,NULL,NULL),(107466,61,4,'MA',NULL,NULL,NULL,NULL,NULL),(107467,61,5,'plates',NULL,NULL,NULL,NULL,NULL),(107468,61,6,'LB',NULL,NULL,NULL,NULL,NULL),(107469,61,7,'MA',NULL,NULL,NULL,NULL,NULL),(107470,61,8,NULL,NULL,NULL,NULL,1,NULL),(107471,61,9,'mutator',NULL,NULL,NULL,NULL,NULL),(107472,61,10,NULL,NULL,NULL,NULL,0,NULL),(107473,61,12,NULL,NULL,NULL,NULL,1,NULL),(107474,61,13,'plates',NULL,NULL,NULL,NULL,NULL),(107475,61,14,NULL,NULL,NULL,NULL,0,NULL),(107476,61,16,NULL,NULL,5720,NULL,NULL,NULL),(107477,61,17,NULL,NULL,260,NULL,NULL,NULL),(107478,61,19,NULL,NULL,NULL,NULL,0,NULL),(107479,61,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107480,61,23,NULL,NULL,NULL,NULL,1,NULL),(107481,61,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107482,61,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107483,61,26,NULL,NULL,7,NULL,NULL,NULL),(107484,61,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP062225',NULL,NULL,NULL,NULL,NULL),(107485,61,33,NULL,NULL,1,NULL,NULL,NULL),(107486,61,34,NULL,'Essential genes accumulate less mutations and evolution rate is not correlated with length of the gene/protein or its expression level.',NULL,NULL,NULL,NULL),(107487,62,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107488,62,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107489,62,3,NULL,NULL,20,NULL,NULL,NULL),(107490,62,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107491,62,5,'24-well plate',NULL,NULL,NULL,NULL,NULL),(107492,62,6,'RPMI',NULL,NULL,NULL,NULL,NULL),(107493,62,7,'intracellular macrophage conditions',NULL,NULL,NULL,NULL,NULL),(107494,62,10,NULL,NULL,NULL,NULL,0,NULL),(107495,62,12,NULL,NULL,NULL,NULL,0,NULL),(107496,62,16,NULL,NULL,182,NULL,NULL,NULL),(107497,62,17,NULL,NULL,26,NULL,NULL,NULL),(107498,62,19,NULL,NULL,NULL,NULL,1,NULL),(107499,62,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107500,62,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107501,62,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(107502,62,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107503,62,23,NULL,NULL,NULL,NULL,1,NULL),(107504,62,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107505,62,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107506,62,26,NULL,NULL,26,NULL,NULL,NULL),(107507,62,28,NULL,NULL,35,NULL,NULL,NULL),(107508,62,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP066892',NULL,NULL,NULL,NULL,NULL),(107509,62,32,NULL,NULL,1000000,NULL,NULL,NULL),(107510,62,33,NULL,NULL,10000,NULL,NULL,NULL),(107511,62,34,NULL,'E. coli adapts to intracellular macrophage conditions by loss of function mutations with strong effect, antagonistic pleiotropy and metabolic trade-offs',NULL,NULL,NULL,NULL),(107512,63,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107513,63,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107514,63,3,NULL,NULL,12,NULL,NULL,NULL),(107515,63,4,'continuous',NULL,NULL,NULL,NULL,NULL),(107516,63,5,'plates',NULL,NULL,NULL,NULL,NULL),(107517,63,6,'LB',NULL,NULL,NULL,NULL,NULL),(107518,63,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107519,63,8,NULL,NULL,NULL,NULL,0,NULL),(107520,63,10,NULL,NULL,NULL,NULL,1,NULL),(107521,63,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(107522,63,12,NULL,NULL,NULL,NULL,1,NULL),(107523,63,13,'plates',NULL,NULL,NULL,NULL,NULL),(107524,63,14,NULL,NULL,NULL,NULL,0,NULL),(107525,63,17,NULL,NULL,12,NULL,NULL,NULL),(107526,63,18,'double autoclave cylce',NULL,NULL,NULL,NULL,NULL),(107527,63,18,'antibiotics',NULL,NULL,NULL,NULL,NULL),(107528,63,19,NULL,NULL,NULL,NULL,1,NULL),(107529,63,20,'yield',NULL,NULL,NULL,NULL,NULL),(107530,63,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107531,63,23,NULL,NULL,NULL,NULL,1,NULL),(107532,63,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107533,63,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107534,63,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107535,63,26,NULL,NULL,252,NULL,NULL,NULL),(107536,63,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP077287',NULL,NULL,NULL,NULL,NULL),(107537,63,34,NULL,'Construction and application of MEGA plate setup that allows adaptive radiation of big populations in a dimensional manner accros different gradients/steps of selective agents. Here, they applied antibiotics to show that under migration and evolution in structured environments, massive diversification emerges. While evolution proceeded rapidly, spatial structure and directionality can also result in superior mutants to be come trapped.',NULL,NULL,NULL,NULL),(107538,64,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107539,64,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107540,64,3,NULL,NULL,12,NULL,NULL,NULL),(107541,64,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107542,64,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107543,64,6,'LB',NULL,NULL,NULL,NULL,NULL),(107544,64,7,'starvation',NULL,NULL,NULL,NULL,NULL),(107545,64,8,NULL,NULL,NULL,NULL,0,NULL),(107546,64,10,NULL,NULL,NULL,NULL,0,NULL),(107547,64,12,NULL,NULL,NULL,NULL,1,NULL),(107548,64,13,'static',NULL,NULL,NULL,NULL,NULL),(107549,64,14,NULL,NULL,NULL,NULL,0,NULL),(107550,64,17,NULL,NULL,30,NULL,NULL,NULL),(107551,64,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107552,64,19,NULL,NULL,NULL,NULL,0,NULL),(107553,64,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107554,64,23,NULL,NULL,NULL,NULL,1,NULL),(107555,64,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107556,64,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107557,64,26,NULL,NULL,16,NULL,NULL,NULL),(107558,64,31,'https://www.ncbi.nlm.nih.gov/biosample?Db=biosample&DbFrom=bioproject&Cmd=Link&LinkName=bioproject_biosample&LinkReadableName=BioSample&ordinalpos=1&IdsFromResult=298997',NULL,NULL,NULL,NULL,NULL),(107559,64,34,NULL,'Development of a microfluidic device to track individual cells after evolution',NULL,NULL,NULL,NULL),(107560,65,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107561,65,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107562,65,3,NULL,NULL,24,NULL,NULL,NULL),(107563,65,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107564,65,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107565,65,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107566,65,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(107567,65,8,NULL,NULL,NULL,NULL,0,NULL),(107568,65,10,NULL,NULL,NULL,NULL,0,NULL),(107569,65,12,NULL,NULL,NULL,NULL,0,NULL),(107570,65,14,NULL,NULL,NULL,NULL,0,NULL),(107571,65,16,NULL,NULL,540,NULL,NULL,NULL),(107572,65,17,NULL,NULL,40,NULL,NULL,NULL),(107573,65,18,'T5',NULL,NULL,NULL,NULL,NULL),(107574,65,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107575,65,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107576,65,19,NULL,NULL,NULL,NULL,1,NULL),(107577,65,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107578,65,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107579,65,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107580,65,23,NULL,NULL,NULL,NULL,1,NULL),(107581,65,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107582,65,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107583,65,26,NULL,NULL,24,NULL,NULL,NULL),(107584,65,28,NULL,NULL,80,NULL,NULL,NULL),(107585,65,32,NULL,NULL,250000000,NULL,NULL,NULL),(107586,65,33,NULL,NULL,25000,NULL,NULL,NULL),(107587,65,34,NULL,'Most effects of single step mutations on glucose limitation showed positive pleiotropy and fitness effects in a more adapted background diminished',NULL,NULL,NULL,NULL),(107588,66,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107589,66,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107590,66,3,NULL,NULL,18,NULL,NULL,NULL),(107591,66,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107592,66,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107593,66,6,'LB',NULL,NULL,NULL,NULL,NULL),(107594,66,7,'stability of exp',NULL,NULL,NULL,NULL,NULL),(107595,66,7,'ed genetic code',NULL,NULL,NULL,NULL,NULL),(107596,66,8,NULL,NULL,NULL,NULL,1,NULL),(107597,66,9,'exp',NULL,NULL,NULL,NULL,NULL),(107598,66,9,'ed genetic code',NULL,NULL,NULL,NULL,NULL),(107599,66,9,'RifR mutants',NULL,NULL,NULL,NULL,NULL),(107600,66,10,NULL,NULL,NULL,NULL,0,NULL),(107601,66,12,NULL,NULL,NULL,NULL,0,NULL),(107602,66,14,NULL,NULL,NULL,NULL,0,NULL),(107603,66,16,NULL,NULL,100,NULL,NULL,NULL),(107604,66,17,NULL,NULL,10,NULL,NULL,NULL),(107605,66,18,'antibiotc resistance',NULL,NULL,NULL,NULL,NULL),(107606,66,19,NULL,NULL,NULL,NULL,1,NULL),(107607,66,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107608,66,23,NULL,NULL,NULL,NULL,0,NULL),(107609,66,34,NULL,'The new paths towards rifampicin resistance opened by expanded genetic code does not lead to increased stability of this code by selection on this trait depnendent on it.',NULL,NULL,NULL,NULL),(107610,67,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107611,67,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107612,67,3,NULL,NULL,12,NULL,NULL,NULL),(107613,67,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107614,67,5,'flask',NULL,NULL,NULL,NULL,NULL),(107615,67,6,'DM25',NULL,NULL,NULL,NULL,NULL),(107616,67,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(107617,67,8,NULL,NULL,NULL,NULL,1,NULL),(107618,67,9,'ancient EF-Tu',NULL,NULL,NULL,NULL,NULL),(107619,67,10,NULL,NULL,NULL,NULL,0,NULL),(107620,67,12,NULL,NULL,NULL,NULL,0,NULL),(107621,67,14,NULL,NULL,NULL,NULL,0,NULL),(107622,67,16,NULL,NULL,2000,NULL,NULL,NULL),(107623,67,17,NULL,NULL,300,NULL,NULL,NULL),(107624,67,18,'T5',NULL,NULL,NULL,NULL,NULL),(107625,67,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107626,67,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107627,67,19,NULL,NULL,NULL,NULL,1,NULL),(107628,67,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107629,67,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107630,67,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107631,67,23,NULL,NULL,NULL,NULL,1,NULL),(107632,67,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107633,67,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107634,67,26,NULL,NULL,24,NULL,NULL,NULL),(107635,67,27,NULL,NULL,8,NULL,NULL,NULL),(107636,67,32,NULL,NULL,250000000,NULL,NULL,NULL),(107637,67,33,NULL,NULL,2500000,NULL,NULL,NULL),(107638,67,34,NULL,'E. coli adapts to the presence of an ancient elongation factor by upregulating its expression',NULL,NULL,NULL,NULL),(107639,68,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107640,68,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107641,68,3,NULL,NULL,120,NULL,NULL,NULL),(107642,68,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107643,68,5,'24-well plate',NULL,NULL,NULL,NULL,NULL),(107644,68,6,'NB',NULL,NULL,NULL,NULL,NULL),(107645,68,7,'adaptation to salt',NULL,NULL,NULL,NULL,NULL),(107646,68,7,'pH',NULL,NULL,NULL,NULL,NULL),(107647,68,7,'H2O2',NULL,NULL,NULL,NULL,NULL),(107648,68,8,NULL,NULL,NULL,NULL,0,NULL),(107649,68,10,NULL,NULL,NULL,NULL,1,NULL),(107650,68,11,'r',NULL,NULL,NULL,NULL,NULL),(107651,68,11,'om fluctuation',NULL,NULL,NULL,NULL,NULL),(107652,68,12,NULL,NULL,NULL,NULL,0,NULL),(107653,68,14,NULL,NULL,NULL,NULL,1,NULL),(107654,68,15,'simultaneous pressence of stresses',NULL,NULL,NULL,NULL,NULL),(107655,68,16,NULL,NULL,900,NULL,NULL,NULL),(107656,68,17,NULL,NULL,100,NULL,NULL,NULL),(107657,68,18,'kanamycin resistance',NULL,NULL,NULL,NULL,NULL),(107658,68,19,NULL,NULL,NULL,NULL,1,NULL),(107659,68,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107660,68,20,'yield',NULL,NULL,NULL,NULL,NULL),(107661,68,23,NULL,NULL,NULL,NULL,0,NULL),(107662,68,34,NULL,'Under random fluctuating complex stress, E. coli adapts significantly better to each of the seperate compound stressors and shows least variation accross these conditions. In fact, trade-off costs were totally abolished although the were significantly present for the constant conditions.',NULL,NULL,NULL,NULL),(107663,69,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107664,69,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107665,69,3,NULL,NULL,15,NULL,NULL,NULL),(107666,69,4,'continuous',NULL,NULL,NULL,NULL,NULL),(107667,69,5,'mice',NULL,NULL,NULL,NULL,NULL),(107668,69,6,'mice',NULL,NULL,NULL,NULL,NULL),(107669,69,7,'in situ evolution under streptomycen pressure',NULL,NULL,NULL,NULL,NULL),(107670,69,8,NULL,NULL,NULL,NULL,0,NULL),(107671,69,10,NULL,NULL,NULL,NULL,1,NULL),(107672,69,11,'in situ',NULL,NULL,NULL,NULL,NULL),(107673,69,12,NULL,NULL,NULL,NULL,1,NULL),(107674,69,13,'in situ',NULL,NULL,NULL,NULL,NULL),(107675,69,14,NULL,NULL,NULL,NULL,1,NULL),(107676,69,15,'in situ',NULL,NULL,NULL,NULL,NULL),(107677,69,16,NULL,NULL,6408,NULL,NULL,NULL),(107678,69,17,NULL,NULL,356,NULL,NULL,NULL),(107679,69,19,NULL,NULL,NULL,NULL,0,NULL),(107680,69,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107681,69,23,NULL,NULL,NULL,NULL,1,NULL),(107682,69,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107683,69,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107684,69,26,NULL,NULL,15,NULL,NULL,NULL),(107685,69,34,NULL,'While convergent evolution at the molecular level was observed in the mouse gut, the rate was lower and no global regulators were detected',NULL,NULL,NULL,NULL),(107686,70,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107687,70,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107688,70,3,NULL,NULL,746,NULL,NULL,NULL),(107689,70,4,'MA',NULL,NULL,NULL,NULL,NULL),(107690,70,5,'plates',NULL,NULL,NULL,NULL,NULL),(107691,70,6,'LB',NULL,NULL,NULL,NULL,NULL),(107692,70,7,'norfloxacin concentration',NULL,NULL,NULL,NULL,NULL),(107693,70,7,'MA',NULL,NULL,NULL,NULL,NULL),(107694,70,8,NULL,NULL,NULL,NULL,0,NULL),(107695,70,10,NULL,NULL,NULL,NULL,0,NULL),(107696,70,12,NULL,NULL,NULL,NULL,1,NULL),(107697,70,13,'plates',NULL,NULL,NULL,NULL,NULL),(107698,70,14,NULL,NULL,NULL,NULL,0,NULL),(107699,70,17,NULL,NULL,60,NULL,NULL,NULL),(107700,70,19,NULL,NULL,NULL,NULL,0,NULL),(107701,70,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107702,70,23,NULL,NULL,NULL,NULL,1,NULL),(107703,70,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107704,70,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107705,70,26,NULL,NULL,746,NULL,NULL,NULL),(107706,70,28,NULL,NULL,92,NULL,NULL,NULL),(107707,70,31,'https://www.ncbi.nlm.nih.gov/sra',NULL,NULL,NULL,NULL,NULL),(107708,70,33,NULL,NULL,1,NULL,NULL,NULL),(107709,70,34,NULL,'Norfloxacin enhances genome-wide mutation rates',NULL,NULL,NULL,NULL),(107710,71,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107711,71,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107712,71,3,NULL,NULL,2,NULL,NULL,NULL),(107713,71,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107714,71,6,'MOPS',NULL,NULL,NULL,NULL,NULL),(107715,71,7,'phosphor limitation',NULL,NULL,NULL,NULL,NULL),(107716,71,10,NULL,NULL,NULL,NULL,0,NULL),(107717,71,12,NULL,NULL,NULL,NULL,0,NULL),(107718,71,19,NULL,NULL,NULL,NULL,0,NULL),(107719,71,22,'targeted',NULL,NULL,NULL,NULL,NULL),(107720,71,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107721,71,23,NULL,NULL,NULL,NULL,1,NULL),(107722,71,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107723,71,25,'Genome Analyzer IIx',NULL,NULL,NULL,NULL,NULL),(107724,71,26,NULL,NULL,6,NULL,NULL,NULL),(107725,71,34,NULL,'E. coli adapts to starvation and P limitation by constitutive expression of trk and kdp and inactivation of arcA to maintain TCA activtiy',NULL,NULL,NULL,NULL),(107726,71,35,NULL,'linked with Guillemet et al 2012 Jbac',NULL,NULL,NULL,NULL),(107727,72,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107728,72,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107729,72,3,NULL,NULL,12,NULL,NULL,NULL),(107730,72,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107731,72,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107732,72,6,'LB',NULL,NULL,NULL,NULL,NULL),(107733,72,7,'LB',NULL,NULL,NULL,NULL,NULL),(107734,72,8,NULL,NULL,NULL,NULL,1,NULL),(107735,72,9,'mutator',NULL,NULL,NULL,NULL,NULL),(107736,72,10,NULL,NULL,NULL,NULL,0,NULL),(107737,72,12,NULL,NULL,NULL,NULL,0,NULL),(107738,72,14,NULL,NULL,NULL,NULL,0,NULL),(107739,72,16,NULL,NULL,850,NULL,NULL,NULL),(107740,72,17,NULL,NULL,31,NULL,NULL,NULL),(107741,72,19,NULL,NULL,NULL,NULL,1,NULL),(107742,72,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107743,72,23,NULL,NULL,NULL,NULL,0,NULL),(107744,72,32,NULL,NULL,2000000000,NULL,NULL,NULL),(107745,72,33,NULL,NULL,50,NULL,NULL,NULL),(107746,73,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107747,73,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107748,73,3,NULL,NULL,90,NULL,NULL,NULL),(107749,73,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107750,73,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(107751,73,6,'DM',NULL,NULL,NULL,NULL,NULL),(107752,73,7,'lactose limitation',NULL,NULL,NULL,NULL,NULL),(107753,73,8,NULL,NULL,NULL,NULL,0,NULL),(107754,73,10,NULL,NULL,NULL,NULL,0,NULL),(107755,73,12,NULL,NULL,NULL,NULL,0,NULL),(107756,73,14,NULL,NULL,NULL,NULL,0,NULL),(107757,73,16,NULL,NULL,1000,NULL,NULL,NULL),(107758,73,17,NULL,NULL,150,NULL,NULL,NULL),(107759,73,18,'T5',NULL,NULL,NULL,NULL,NULL),(107760,73,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107761,73,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107762,73,19,NULL,NULL,NULL,NULL,1,NULL),(107763,73,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107764,73,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107765,73,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107766,73,23,NULL,NULL,NULL,NULL,1,NULL),(107767,73,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107768,73,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107769,73,25,'NextSeq500',NULL,NULL,NULL,NULL,NULL),(107770,73,26,NULL,NULL,6,NULL,NULL,NULL),(107771,73,34,NULL,'Evolution on glucose improves evolvability on lactose.',NULL,NULL,NULL,NULL),(107772,74,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107773,74,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107774,74,3,NULL,NULL,16,NULL,NULL,NULL),(107775,74,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107776,74,5,'flasks',NULL,NULL,NULL,NULL,NULL),(107777,74,5,'plates',NULL,NULL,NULL,NULL,NULL),(107778,74,5,'tubes',NULL,NULL,NULL,NULL,NULL),(107779,74,5,'multiwell plates',NULL,NULL,NULL,NULL,NULL),(107780,74,6,'DM',NULL,NULL,NULL,NULL,NULL),(107781,74,7,'acetate',NULL,NULL,NULL,NULL,NULL),(107782,74,7,'glycerol',NULL,NULL,NULL,NULL,NULL),(107783,74,7,'gluconate',NULL,NULL,NULL,NULL,NULL),(107784,74,7,'glucose',NULL,NULL,NULL,NULL,NULL),(107785,74,8,NULL,NULL,NULL,NULL,0,NULL),(107786,74,10,NULL,NULL,NULL,NULL,0,NULL),(107787,74,12,NULL,NULL,NULL,NULL,0,NULL),(107788,74,14,NULL,NULL,NULL,NULL,0,NULL),(107789,74,16,NULL,NULL,1000,NULL,NULL,NULL),(107790,74,18,'T5',NULL,NULL,NULL,NULL,NULL),(107791,74,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107792,74,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107793,74,19,NULL,NULL,NULL,NULL,1,NULL),(107794,74,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107795,74,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107796,74,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107797,74,23,NULL,NULL,NULL,NULL,1,NULL),(107798,74,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107799,74,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107800,74,26,NULL,NULL,14,NULL,NULL,NULL),(107801,74,31,'https://www.ncbi.nlm.nih.gov/sra/?term=PRJEB13018',NULL,NULL,NULL,NULL,NULL),(107802,74,34,NULL,'While similar genes were modified during a second evolution experiment on glucose, the growth rate and fitness dependend on historical contigency of the ancestors.',NULL,NULL,NULL,NULL),(107803,75,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107804,75,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107805,75,3,NULL,NULL,32,NULL,NULL,NULL),(107806,75,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107807,75,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(107808,75,6,'DM',NULL,NULL,NULL,NULL,NULL),(107809,75,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(107810,75,8,NULL,NULL,NULL,NULL,0,NULL),(107811,75,10,NULL,NULL,NULL,NULL,0,NULL),(107812,75,12,NULL,NULL,NULL,NULL,0,NULL),(107813,75,14,NULL,NULL,NULL,NULL,0,NULL),(107814,75,16,NULL,NULL,1000,NULL,NULL,NULL),(107815,75,18,'T5',NULL,NULL,NULL,NULL,NULL),(107816,75,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(107817,75,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107818,75,19,NULL,NULL,NULL,NULL,1,NULL),(107819,75,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107820,75,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(107821,75,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107822,75,23,NULL,NULL,NULL,NULL,1,NULL),(107823,75,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107824,75,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107825,75,26,NULL,NULL,14,NULL,NULL,NULL),(107826,75,31,'https://www.ncbi.nlm.nih.gov/sra/?term=PRJEB13018',NULL,NULL,NULL,NULL,NULL),(107827,75,34,NULL,'While similar genes were modified during a second evolution experiment on glucose, the growth rate and fitness dependend on historical contigency of the ancestors.',NULL,NULL,NULL,NULL),(107828,76,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107829,76,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(107830,76,3,NULL,NULL,1,NULL,NULL,NULL),(107831,76,4,'continuous',NULL,NULL,NULL,NULL,NULL),(107832,76,5,'2.5L chemostat',NULL,NULL,NULL,NULL,NULL),(107833,76,6,'defined medium',NULL,NULL,NULL,NULL,NULL),(107834,76,7,'acetate limitation',NULL,NULL,NULL,NULL,NULL),(107835,76,8,NULL,NULL,NULL,NULL,0,NULL),(107836,76,10,NULL,NULL,NULL,NULL,0,NULL),(107837,76,12,NULL,NULL,NULL,NULL,0,NULL),(107838,76,14,NULL,NULL,NULL,NULL,0,NULL),(107839,76,16,NULL,NULL,450,NULL,NULL,NULL),(107840,76,17,NULL,NULL,10,NULL,NULL,NULL),(107841,76,19,NULL,NULL,NULL,NULL,0,NULL),(107842,76,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107843,76,23,NULL,NULL,NULL,NULL,1,NULL),(107844,76,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107845,76,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107846,76,26,NULL,NULL,2,NULL,NULL,NULL),(107847,76,28,NULL,NULL,100,NULL,NULL,NULL),(107848,76,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA314220',NULL,NULL,NULL,NULL,NULL),(107849,76,34,NULL,'A single mutation in rpoA causes an increased growth rate in acetate.',NULL,NULL,NULL,NULL),(107850,76,35,NULL,'coated with PEG-silane to prevent wall growth',NULL,NULL,NULL,NULL),(107851,77,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107852,77,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107853,77,3,NULL,NULL,39,NULL,NULL,NULL),(107854,77,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107855,77,5,'flask',NULL,NULL,NULL,NULL,NULL),(107856,77,6,'M9 + carbon source',NULL,NULL,NULL,NULL,NULL),(107857,77,7,'glycerol or lactate limitation',NULL,NULL,NULL,NULL,NULL),(107858,77,10,NULL,NULL,NULL,NULL,0,NULL),(107859,77,12,NULL,NULL,NULL,NULL,0,NULL),(107860,77,16,NULL,NULL,266,NULL,NULL,NULL),(107861,77,17,NULL,NULL,28,NULL,NULL,NULL),(107862,77,19,NULL,NULL,NULL,NULL,1,NULL),(107863,77,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107864,77,23,NULL,NULL,NULL,NULL,0,NULL),(107865,77,32,NULL,NULL,100000000000,NULL,NULL,NULL),(107866,77,33,NULL,NULL,10000000,NULL,NULL,NULL),(107867,77,34,NULL,'Higher cryptic variation at the start of an evolution experiment can promote adaptation',NULL,NULL,NULL,NULL),(107868,78,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107869,78,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107870,78,3,NULL,NULL,156,NULL,NULL,NULL),(107871,78,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107872,78,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(107873,78,6,'LB',NULL,NULL,NULL,NULL,NULL),(107874,78,7,'ceftazidime',NULL,NULL,NULL,NULL,NULL),(107875,78,8,NULL,NULL,NULL,NULL,0,NULL),(107876,78,10,NULL,NULL,NULL,NULL,1,NULL),(107877,78,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(107878,78,12,NULL,NULL,NULL,NULL,0,NULL),(107879,78,14,NULL,NULL,NULL,NULL,0,NULL),(107880,78,16,NULL,NULL,112,NULL,NULL,NULL),(107881,78,17,NULL,NULL,16,NULL,NULL,NULL),(107882,78,19,NULL,NULL,NULL,NULL,1,NULL),(107883,78,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(107884,78,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(107885,78,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107886,78,23,NULL,NULL,NULL,NULL,1,NULL),(107887,78,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107888,78,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(107889,78,26,NULL,NULL,9,NULL,NULL,NULL),(107890,78,27,NULL,NULL,56,NULL,NULL,NULL),(107891,78,30,NULL,NULL,NULL,10,NULL,NULL),(107892,78,31,'https://www.ebi.ac.uk/ena/data/view/PRJEB14587',NULL,NULL,NULL,NULL,NULL),(107893,78,34,NULL,'Bearing a multicopy plasmid with a resistance gen can accelerate evolution toward a new antibiotic through copy number and faster appearance of beneficial mutations',NULL,NULL,NULL,NULL),(107894,79,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107895,79,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107896,79,3,NULL,NULL,6,NULL,NULL,NULL),(107897,79,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107898,79,5,'flasks',NULL,NULL,NULL,NULL,NULL),(107899,79,6,'M9 + carbon source',NULL,NULL,NULL,NULL,NULL),(107900,79,7,'C13 glucose',NULL,NULL,NULL,NULL,NULL),(107901,79,8,NULL,NULL,NULL,NULL,0,NULL),(107902,79,10,NULL,NULL,NULL,NULL,0,NULL),(107903,79,12,NULL,NULL,NULL,NULL,0,NULL),(107904,79,14,NULL,NULL,NULL,NULL,0,NULL),(107905,79,16,NULL,NULL,963,NULL,NULL,NULL),(107906,79,17,NULL,NULL,40,NULL,NULL,NULL),(107907,79,19,NULL,NULL,NULL,NULL,1,NULL),(107908,79,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(107909,79,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107910,79,23,NULL,NULL,NULL,NULL,1,NULL),(107911,79,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107912,79,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(107913,79,26,NULL,NULL,6,NULL,NULL,NULL),(107914,79,28,NULL,NULL,90,NULL,NULL,NULL),(107915,79,34,NULL,'Adaptation to C13 glucose does not seem to have any specific effect compared to adaptation to C12 glucose both in terms of phenotypical outcome and mutations',NULL,NULL,NULL,NULL),(107916,80,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107917,80,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107918,80,3,NULL,NULL,8,NULL,NULL,NULL),(107919,80,4,'continuous',NULL,NULL,NULL,NULL,NULL),(107920,80,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(107921,80,6,'Davis Salts minimal media 0.01% glucose',NULL,NULL,NULL,NULL,NULL),(107922,80,7,'streptomycin',NULL,NULL,NULL,NULL,NULL),(107923,80,10,NULL,NULL,NULL,NULL,1,NULL),(107924,80,11,'continuously rising in time',NULL,NULL,NULL,NULL,NULL),(107925,80,12,NULL,NULL,NULL,NULL,0,NULL),(107926,80,16,NULL,NULL,33,NULL,NULL,NULL),(107927,80,17,NULL,NULL,5,NULL,NULL,NULL),(107928,80,19,NULL,NULL,NULL,NULL,0,NULL),(107929,80,21,'T5 resistance (fhuA)',NULL,NULL,NULL,NULL,NULL),(107930,80,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(107931,80,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(107932,80,23,NULL,NULL,NULL,NULL,0,NULL),(107933,80,32,NULL,NULL,3000000000,NULL,NULL,NULL),(107934,80,33,NULL,NULL,3000000000,NULL,NULL,NULL),(107935,80,34,NULL,'Resistant mutants remain at lower density longer than expected only to rise when concentration is far above the MIC',NULL,NULL,NULL,NULL),(107936,80,35,NULL,'contamination in one chemostat (not clear how determined) and each chemostat is actually competition between 2 ancestors but either all of them go exting or both survive',NULL,NULL,NULL,NULL),(107937,81,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107938,81,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107939,81,3,NULL,NULL,12,NULL,NULL,NULL),(107940,81,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107941,81,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(107942,81,6,'LB',NULL,NULL,NULL,NULL,NULL),(107943,81,7,'β-thujaplicin or doxycycline',NULL,NULL,NULL,NULL,NULL),(107944,81,8,NULL,NULL,NULL,NULL,1,NULL),(107945,81,9,'TetR',NULL,NULL,NULL,NULL,NULL),(107946,81,10,NULL,NULL,NULL,NULL,1,NULL),(107947,81,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(107948,81,12,NULL,NULL,NULL,NULL,0,NULL),(107949,81,14,NULL,NULL,NULL,NULL,0,NULL),(107950,81,17,NULL,NULL,10,NULL,NULL,NULL),(107951,81,19,NULL,NULL,NULL,NULL,1,NULL),(107952,81,20,'yield',NULL,NULL,NULL,NULL,NULL),(107953,81,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(107954,81,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107955,81,23,NULL,NULL,NULL,NULL,1,NULL),(107956,81,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(107957,81,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(107958,81,26,NULL,NULL,12,NULL,NULL,NULL),(107959,81,28,NULL,NULL,65,NULL,NULL,NULL),(107960,81,31,'https://www.ncbi.nlm.nih.gov//bioproject/PRJNA317940',NULL,NULL,NULL,NULL,NULL),(107961,81,33,NULL,NULL,10000,NULL,NULL,NULL),(107962,81,34,NULL,'TetR resistance can be selected against by hits from selective advantage screening.',NULL,NULL,NULL,NULL),(107963,82,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107964,82,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(107965,82,3,NULL,NULL,36,NULL,NULL,NULL),(107966,82,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107967,82,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(107968,82,6,'modified M9',NULL,NULL,NULL,NULL,NULL),(107969,82,7,'9 antibiotics',NULL,NULL,NULL,NULL,NULL),(107970,82,12,NULL,NULL,NULL,NULL,0,NULL),(107971,82,17,NULL,NULL,90,NULL,NULL,NULL),(107972,82,19,NULL,NULL,NULL,NULL,0,NULL),(107973,82,22,'N',NULL,NULL,NULL,NULL,NULL),(107974,82,23,NULL,NULL,NULL,NULL,0,NULL),(107975,82,34,NULL,'The further away a resistant strain evolved from its wild type (transcriptome) the higher its cost of resistance',NULL,NULL,NULL,NULL),(107976,82,35,NULL,'follow up of S. Suzuki, T. Horinouchi and C. Furusawa, Prediction of antibiotic resistance by gene expression profiles, Nat. Commun., 2014, 5, 5792.',NULL,NULL,NULL,NULL),(107977,83,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(107978,83,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(107979,83,3,NULL,NULL,24,NULL,NULL,NULL),(107980,83,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(107981,83,5,'plates',NULL,NULL,NULL,NULL,NULL),(107982,83,6,'MS',NULL,NULL,NULL,NULL,NULL),(107983,83,7,'ability to grow on ethylene or propylene glycol',NULL,NULL,NULL,NULL,NULL),(107984,83,8,NULL,NULL,NULL,NULL,1,NULL),(107985,83,9,'mutator,',NULL,NULL,NULL,NULL,NULL),(107986,83,10,NULL,NULL,NULL,NULL,0,NULL),(107987,83,12,NULL,NULL,NULL,NULL,1,NULL),(107988,83,13,'plates',NULL,NULL,NULL,NULL,NULL),(107989,83,14,NULL,NULL,NULL,NULL,0,NULL),(107990,83,17,NULL,NULL,40,NULL,NULL,NULL),(107991,83,19,NULL,NULL,NULL,NULL,0,NULL),(107992,83,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(107993,83,23,NULL,NULL,NULL,NULL,1,NULL),(107994,83,24,'Ion Torrent',NULL,NULL,NULL,NULL,NULL),(107995,83,25,'PGM',NULL,NULL,NULL,NULL,NULL),(107996,83,26,NULL,NULL,2,NULL,NULL,NULL),(107997,83,30,NULL,NULL,NULL,66,NULL,NULL),(107998,83,31,'https://www.ncbi.nlm.nih.gov//sra/?term=SRP062939',NULL,NULL,NULL,NULL,NULL),(107999,83,34,NULL,'In E. coli, the possibility to use ethylene glycol as carbon source is historical contingent on the evolution to use propylene glycol. all-or-none epsistasis between upregulation of 2 genes simultaneously is necessary.',NULL,NULL,NULL,NULL),(108000,84,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108001,84,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108002,84,3,NULL,NULL,12,NULL,NULL,NULL),(108003,84,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108004,84,5,'flask',NULL,NULL,NULL,NULL,NULL),(108005,84,6,'DM25',NULL,NULL,NULL,NULL,NULL),(108006,84,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108007,84,10,NULL,NULL,NULL,NULL,0,NULL),(108008,84,12,NULL,NULL,NULL,NULL,0,NULL),(108009,84,16,NULL,NULL,50000,NULL,NULL,NULL),(108010,84,17,NULL,NULL,7500,NULL,NULL,NULL),(108011,84,18,'T5',NULL,NULL,NULL,NULL,NULL),(108012,84,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108013,84,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108014,84,19,NULL,NULL,NULL,NULL,0,NULL),(108015,84,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108016,84,23,NULL,NULL,NULL,NULL,1,NULL),(108017,84,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108018,84,25,'Genome Analyzer',NULL,NULL,NULL,NULL,NULL),(108019,84,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(108020,84,26,NULL,NULL,224,NULL,NULL,NULL),(108021,84,28,NULL,NULL,80,NULL,NULL,NULL),(108022,84,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA294072',NULL,NULL,NULL,NULL,NULL),(108023,84,32,NULL,NULL,500000000,NULL,NULL,NULL),(108024,84,33,NULL,NULL,5000000,NULL,NULL,NULL),(108025,84,34,NULL,'Most mutations that fixed of the long term fo the LTEE are beneficial but neutrals increase as fitness does. By comparing to a MA experiment, the strong signature of selection becomes apparent',NULL,NULL,NULL,NULL),(108026,85,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108027,85,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108028,85,3,NULL,NULL,15,NULL,NULL,NULL),(108029,85,4,'MA',NULL,NULL,NULL,NULL,NULL),(108030,85,5,'plates',NULL,NULL,NULL,NULL,NULL),(108031,85,6,'DM25',NULL,NULL,NULL,NULL,NULL),(108032,85,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108033,85,10,NULL,NULL,NULL,NULL,0,NULL),(108034,85,12,NULL,NULL,NULL,NULL,0,NULL),(108035,85,16,NULL,NULL,13750,NULL,NULL,NULL),(108036,85,17,NULL,NULL,550,NULL,NULL,NULL),(108037,85,18,'T5',NULL,NULL,NULL,NULL,NULL),(108038,85,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108039,85,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108040,85,19,NULL,NULL,NULL,NULL,1,NULL),(108041,85,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108042,85,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108043,85,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108044,85,23,NULL,NULL,NULL,NULL,1,NULL),(108045,85,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108046,85,25,'Genome Analyzer',NULL,NULL,NULL,NULL,NULL),(108047,85,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(108048,85,26,NULL,NULL,15,NULL,NULL,NULL),(108049,85,28,NULL,NULL,95,NULL,NULL,NULL),(108050,85,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA294072',NULL,NULL,NULL,NULL,NULL),(108051,85,32,NULL,NULL,30000000,NULL,NULL,NULL),(108052,85,33,NULL,NULL,1,NULL,NULL,NULL),(108053,85,34,NULL,'Most mutations that fixed of the long term fo the LTEE are beneficial but neutrals increase as fitness does. By comparing to a MA experiment, the strong signature of selection becomes apparent',NULL,NULL,NULL,NULL),(108054,86,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108055,86,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108056,86,3,NULL,NULL,52,NULL,NULL,NULL),(108057,86,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108058,86,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108059,86,6,'MHB',NULL,NULL,NULL,NULL,NULL),(108060,86,7,'aminoglycoside survival',NULL,NULL,NULL,NULL,NULL),(108061,86,8,NULL,NULL,NULL,NULL,0,NULL),(108062,86,10,NULL,NULL,NULL,NULL,1,NULL),(108063,86,11,'periodically changing between high antibiotic dose',NULL,NULL,NULL,NULL,NULL),(108064,86,11,'no dose',NULL,NULL,NULL,NULL,NULL),(108065,86,12,NULL,NULL,NULL,NULL,0,NULL),(108066,86,14,NULL,NULL,NULL,NULL,0,NULL),(108067,86,17,NULL,NULL,64,NULL,NULL,NULL),(108068,86,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108069,86,19,NULL,NULL,NULL,NULL,1,NULL),(108070,86,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108071,86,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108072,86,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108073,86,23,NULL,NULL,NULL,NULL,1,NULL),(108074,86,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108075,86,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(108076,86,26,NULL,NULL,4,NULL,NULL,NULL),(108077,86,28,NULL,NULL,150,NULL,NULL,NULL),(108078,86,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP051150',NULL,NULL,NULL,NULL,NULL),(108079,86,32,NULL,NULL,100000000000,NULL,NULL,NULL),(108080,86,33,NULL,NULL,100000,NULL,NULL,NULL),(108081,86,34,NULL,'Persistence rapidly evolves in E. coli under frequent aminoglycoside treatment according to treatment frequency',NULL,NULL,NULL,NULL),(108082,87,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108083,87,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108084,87,3,NULL,NULL,12,NULL,NULL,NULL),(108085,87,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108086,87,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108087,87,6,'MHB',NULL,NULL,NULL,NULL,NULL),(108088,87,7,'cost of high persistence',NULL,NULL,NULL,NULL,NULL),(108089,87,8,NULL,NULL,NULL,NULL,1,NULL),(108090,87,9,'high persistence mutants',NULL,NULL,NULL,NULL,NULL),(108091,87,10,NULL,NULL,NULL,NULL,0,NULL),(108092,87,12,NULL,NULL,NULL,NULL,0,NULL),(108093,87,14,NULL,NULL,NULL,NULL,0,NULL),(108094,87,16,NULL,NULL,1280,NULL,NULL,NULL),(108095,87,17,NULL,NULL,64,NULL,NULL,NULL),(108096,87,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108097,87,19,NULL,NULL,NULL,NULL,1,NULL),(108098,87,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108099,87,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108100,87,23,NULL,NULL,NULL,NULL,0,NULL),(108101,87,32,NULL,NULL,100000000000,NULL,NULL,NULL),(108102,87,33,NULL,NULL,100000,NULL,NULL,NULL),(108103,87,34,NULL,'Persistence rapidly evolves in E. coli under frequent aminoglycoside treatment according to treatment frequency',NULL,NULL,NULL,NULL),(108104,88,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108105,88,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108106,88,3,NULL,NULL,77,NULL,NULL,NULL),(108107,88,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108108,88,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108109,88,6,'M9 + carbon source',NULL,NULL,NULL,NULL,NULL),(108110,88,7,'citrate utilization',NULL,NULL,NULL,NULL,NULL),(108111,88,8,NULL,NULL,NULL,NULL,1,NULL),(108112,88,9,'citT',NULL,NULL,NULL,NULL,NULL),(108113,88,9,'dctA',NULL,NULL,NULL,NULL,NULL),(108114,88,9,'rpoS knockouts',NULL,NULL,NULL,NULL,NULL),(108115,88,10,NULL,NULL,NULL,NULL,0,NULL),(108116,88,12,NULL,NULL,NULL,NULL,0,NULL),(108117,88,14,NULL,NULL,NULL,NULL,0,NULL),(108118,88,16,NULL,NULL,100,NULL,NULL,NULL),(108119,88,17,NULL,NULL,134,NULL,NULL,NULL),(108120,88,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(108121,88,19,NULL,NULL,NULL,NULL,0,NULL),(108122,88,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108123,88,23,NULL,NULL,NULL,NULL,1,NULL),(108124,88,24,'Pacbio',NULL,NULL,NULL,NULL,NULL),(108125,88,25,'RS II',NULL,NULL,NULL,NULL,NULL),(108126,88,26,NULL,NULL,4,NULL,NULL,NULL),(108127,88,28,NULL,NULL,100,NULL,NULL,NULL),(108128,88,31,'https://www.ncbi.nlm.nih.gov/bioproject/306781',NULL,NULL,NULL,NULL,NULL),(108129,88,33,NULL,NULL,1000000,NULL,NULL,NULL),(108130,88,34,NULL,'Direct selection for cit+ phenotypes leads to faster evoluton of cit+, mainly because of longer selection in stationary phase, while citT and dctA are essential for the adaptation process.',NULL,NULL,NULL,NULL),(108131,89,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108132,89,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108133,89,3,NULL,NULL,24,NULL,NULL,NULL),(108134,89,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108135,89,5,'flask',NULL,NULL,NULL,NULL,NULL),(108136,89,6,'LBK',NULL,NULL,NULL,NULL,NULL),(108137,89,7,'low pH (4.8 to 4.6) HOMOPIPES',NULL,NULL,NULL,NULL,NULL),(108138,89,10,NULL,NULL,NULL,NULL,1,NULL),(108139,89,11,'Decreased pH at generation 730',NULL,NULL,NULL,NULL,NULL),(108140,89,12,NULL,NULL,NULL,NULL,0,NULL),(108141,89,16,NULL,NULL,2000,NULL,NULL,NULL),(108142,89,19,NULL,NULL,NULL,NULL,1,NULL),(108143,89,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108144,89,21,'Lac',NULL,NULL,NULL,NULL,NULL),(108145,89,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108146,89,23,NULL,NULL,NULL,NULL,1,NULL),(108147,89,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108148,89,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(108149,89,26,NULL,NULL,8,NULL,NULL,NULL),(108150,89,28,NULL,NULL,30,NULL,NULL,NULL),(108151,89,34,NULL,'Acid adaptted populations show progressive loss of activity of lysine decarboxylase (CadA), a major stress enzyme',NULL,NULL,NULL,NULL),(108152,90,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108153,90,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108154,90,3,NULL,NULL,6,NULL,NULL,NULL),(108155,90,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108156,90,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108157,90,6,'NB',NULL,NULL,NULL,NULL,NULL),(108158,90,7,'adaptation to salt',NULL,NULL,NULL,NULL,NULL),(108159,90,7,'pH',NULL,NULL,NULL,NULL,NULL),(108160,90,7,'H2O2',NULL,NULL,NULL,NULL,NULL),(108161,90,8,NULL,NULL,NULL,NULL,0,NULL),(108162,90,10,NULL,NULL,NULL,NULL,1,NULL),(108163,90,11,'r',NULL,NULL,NULL,NULL,NULL),(108164,90,11,'om fluctuation',NULL,NULL,NULL,NULL,NULL),(108165,90,12,NULL,NULL,NULL,NULL,0,NULL),(108166,90,14,NULL,NULL,NULL,NULL,1,NULL),(108167,90,15,'simultaneous pressence of stresses',NULL,NULL,NULL,NULL,NULL),(108168,90,16,NULL,NULL,170,NULL,NULL,NULL),(108169,90,17,NULL,NULL,30,NULL,NULL,NULL),(108170,90,19,NULL,NULL,NULL,NULL,1,NULL),(108171,90,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(108172,90,20,'yield',NULL,NULL,NULL,NULL,NULL),(108173,90,23,NULL,NULL,NULL,NULL,0,NULL),(108174,90,34,NULL,'Small scale evolution experiment under random fluctuating complex stress revealed no increase in fitness under the component stressors but greater ability to grow in new stress conditions.',NULL,NULL,NULL,NULL),(108175,91,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108176,91,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108177,91,3,NULL,NULL,7,NULL,NULL,NULL),(108178,91,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108179,91,5,'snap-cap tubes',NULL,NULL,NULL,NULL,NULL),(108180,91,6,'M63',NULL,NULL,NULL,NULL,NULL),(108181,91,7,'P. aeruginosa spent medium',NULL,NULL,NULL,NULL,NULL),(108182,91,7,'pyocyanin',NULL,NULL,NULL,NULL,NULL),(108183,91,10,NULL,NULL,NULL,NULL,1,NULL),(108184,91,11,'increasing concnentrations of spent medium',NULL,NULL,NULL,NULL,NULL),(108185,91,11,'pyocyanin',NULL,NULL,NULL,NULL,NULL),(108186,91,12,NULL,NULL,NULL,NULL,0,NULL),(108187,91,17,NULL,NULL,19,NULL,NULL,NULL),(108188,91,19,NULL,NULL,NULL,NULL,1,NULL),(108189,91,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108190,91,21,'lactose/citrate selection',NULL,NULL,NULL,NULL,NULL),(108191,91,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108192,91,23,NULL,NULL,NULL,NULL,1,NULL),(108193,91,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108194,91,25,'NextSeq 500',NULL,NULL,NULL,NULL,NULL),(108195,91,26,NULL,NULL,18,NULL,NULL,NULL),(108196,91,28,NULL,NULL,45,NULL,NULL,NULL),(108197,91,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA292975',NULL,NULL,NULL,NULL,NULL),(108198,91,32,NULL,NULL,1000000000,NULL,NULL,NULL),(108199,91,33,NULL,NULL,5000000,NULL,NULL,NULL),(108200,91,34,NULL,'Ecological and interference competition between P. aeruginosa and E. coli results in parallel and convergence evolution in E. coli',NULL,NULL,NULL,NULL),(108201,91,35,NULL,'competition assays are between E. coli and P. aeruginosa, not evolved and ancestral E. coli',NULL,NULL,NULL,NULL),(108202,92,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108203,92,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108204,92,3,NULL,NULL,8,NULL,NULL,NULL),(108205,92,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108206,92,7,'glucose minimal medium in exponential phase',NULL,NULL,NULL,NULL,NULL),(108207,92,16,NULL,NULL,2000,NULL,NULL,NULL),(108208,92,17,NULL,NULL,81,NULL,NULL,NULL),(108209,92,18,'16S rRNA',NULL,NULL,NULL,NULL,NULL),(108210,92,19,NULL,NULL,NULL,NULL,1,NULL),(108211,92,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(108212,92,22,'targeted',NULL,NULL,NULL,NULL,NULL),(108213,92,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108214,92,23,NULL,NULL,NULL,NULL,1,NULL),(108215,92,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108216,92,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(108217,92,26,NULL,NULL,36,NULL,NULL,NULL),(108218,92,28,NULL,NULL,25,NULL,NULL,NULL),(108219,92,34,NULL,'3 genetic regions were found to be targeted, rpoB, pyrE and rph which resulted in 1.6 fold increase in fitness comparable, but faster than obtained in the LTEE',NULL,NULL,NULL,NULL),(108220,93,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108221,93,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108222,93,3,NULL,NULL,12,NULL,NULL,NULL),(108223,93,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108224,93,5,'microfluidic device',NULL,NULL,NULL,NULL,NULL),(108225,93,6,'LB',NULL,NULL,NULL,NULL,NULL),(108226,93,7,'carbenicillin',NULL,NULL,NULL,NULL,NULL),(108227,93,8,NULL,NULL,NULL,NULL,0,NULL),(108228,93,10,NULL,NULL,NULL,NULL,1,NULL),(108229,93,11,'cyclic exposures',NULL,NULL,NULL,NULL,NULL),(108230,93,12,NULL,NULL,NULL,NULL,1,NULL),(108231,93,13,'monolayer in microfluidic device',NULL,NULL,NULL,NULL,NULL),(108232,93,14,NULL,NULL,NULL,NULL,0,NULL),(108233,93,17,NULL,NULL,2,NULL,NULL,NULL),(108234,93,18,'kanamycin',NULL,NULL,NULL,NULL,NULL),(108235,93,19,NULL,NULL,NULL,NULL,0,NULL),(108236,93,23,NULL,NULL,NULL,NULL,0,NULL),(108237,93,32,NULL,NULL,60,NULL,NULL,NULL),(108238,93,33,NULL,NULL,30,NULL,NULL,NULL),(108239,93,34,NULL,'Development of a microfluidic lineage anlysis of evolution under intermittent antibiotic exposure.',NULL,NULL,NULL,NULL),(108240,94,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108241,94,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108242,94,3,NULL,NULL,12,NULL,NULL,NULL),(108243,94,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108244,94,5,'flask',NULL,NULL,NULL,NULL,NULL),(108245,94,6,'DM25',NULL,NULL,NULL,NULL,NULL),(108246,94,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108247,94,10,NULL,NULL,NULL,NULL,0,NULL),(108248,94,12,NULL,NULL,NULL,NULL,0,NULL),(108249,94,16,NULL,NULL,60000,NULL,NULL,NULL),(108250,94,17,NULL,NULL,9000,NULL,NULL,NULL),(108251,94,18,'T5',NULL,NULL,NULL,NULL,NULL),(108252,94,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108253,94,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108254,94,19,NULL,NULL,NULL,NULL,1,NULL),(108255,94,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108256,94,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108257,94,23,NULL,NULL,NULL,NULL,0,NULL),(108258,94,32,NULL,NULL,500000000,NULL,NULL,NULL),(108259,94,33,NULL,NULL,5000000,NULL,NULL,NULL),(108260,94,34,NULL,'The powerlaw described by Wiser et al 2013 Science predicts nicely the adaptation of an additional 10k generations and also divergence remains to exist',NULL,NULL,NULL,NULL),(108261,94,35,NULL,'follow up of Wiser et al 2013 Science',NULL,NULL,NULL,NULL),(108262,95,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108263,95,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108264,95,3,NULL,NULL,48,NULL,NULL,NULL),(108265,95,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108266,95,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(108267,95,6,'LB',NULL,NULL,NULL,NULL,NULL),(108268,95,7,'adaptation to cost of resistance mutation',NULL,NULL,NULL,NULL,NULL),(108269,95,8,NULL,NULL,NULL,NULL,1,NULL),(108270,95,9,'resistant mutants',NULL,NULL,NULL,NULL,NULL),(108271,95,10,NULL,NULL,NULL,NULL,0,NULL),(108272,95,12,NULL,NULL,NULL,NULL,0,NULL),(108273,95,14,NULL,NULL,NULL,NULL,0,NULL),(108274,95,16,NULL,NULL,280,NULL,NULL,NULL),(108275,95,17,NULL,NULL,30,NULL,NULL,NULL),(108276,95,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108277,95,18,'resistance',NULL,NULL,NULL,NULL,NULL),(108278,95,18,'sterile LB wells',NULL,NULL,NULL,NULL,NULL),(108279,95,19,NULL,NULL,NULL,NULL,1,NULL),(108280,95,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108281,95,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108282,95,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108283,95,23,NULL,NULL,NULL,NULL,1,NULL),(108284,95,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108285,95,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(108286,95,26,NULL,NULL,58,NULL,NULL,NULL),(108287,95,28,NULL,NULL,207,NULL,NULL,NULL),(108288,95,33,NULL,NULL,100000,NULL,NULL,NULL),(108289,95,34,NULL,'The costly resistance mutants still maintain in drug-free conditions by greater evolutionary adaptive potential and compensating mutations.',NULL,NULL,NULL,NULL),(108290,95,35,NULL,'reads should be in database, are not.',NULL,NULL,NULL,NULL),(108291,96,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108292,96,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108293,96,3,NULL,NULL,57,NULL,NULL,NULL),(108294,96,4,'MA',NULL,NULL,NULL,NULL,NULL),(108295,96,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108296,96,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108297,96,5,'plate',NULL,NULL,NULL,NULL,NULL),(108298,96,6,'LB',NULL,NULL,NULL,NULL,NULL),(108299,96,7,'adaptation under groEL overexpression under drift or not',NULL,NULL,NULL,NULL,NULL),(108300,96,8,NULL,NULL,NULL,NULL,1,NULL),(108301,96,9,'mutator',NULL,NULL,NULL,NULL,NULL),(108302,96,10,NULL,NULL,NULL,NULL,0,NULL),(108303,96,12,NULL,NULL,NULL,NULL,1,NULL),(108304,96,13,'plates',NULL,NULL,NULL,NULL,NULL),(108305,96,14,NULL,NULL,NULL,NULL,0,NULL),(108306,96,16,NULL,NULL,4000,NULL,NULL,NULL),(108307,96,17,NULL,NULL,180,NULL,NULL,NULL),(108308,96,19,NULL,NULL,NULL,NULL,1,NULL),(108309,96,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108310,96,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108311,96,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108312,96,23,NULL,NULL,NULL,NULL,1,NULL),(108313,96,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108314,96,24,'454',NULL,NULL,NULL,NULL,NULL),(108315,96,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(108316,96,25,'Miseq',NULL,NULL,NULL,NULL,NULL),(108317,96,25,'NA',NULL,NULL,NULL,NULL,NULL),(108318,96,26,NULL,NULL,6,NULL,NULL,NULL),(108319,96,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP058803',NULL,NULL,NULL,NULL,NULL),(108320,96,32,NULL,NULL,480000000,NULL,NULL,NULL),(108321,96,33,NULL,NULL,1,NULL,NULL,NULL),(108322,96,34,NULL,'GroEL chaperon can buffer deleterious mutations and rescue populations under drift even when expression is costly',NULL,NULL,NULL,NULL),(108323,97,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108324,97,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108325,97,3,NULL,NULL,32,NULL,NULL,NULL),(108326,97,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108327,97,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(108328,97,6,'M9',NULL,NULL,NULL,NULL,NULL),(108329,97,7,'(combinations of) antibiotics',NULL,NULL,NULL,NULL,NULL),(108330,97,8,NULL,NULL,NULL,NULL,0,NULL),(108331,97,10,NULL,NULL,NULL,NULL,1,NULL),(108332,97,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(108333,97,12,NULL,NULL,NULL,NULL,0,NULL),(108334,97,14,NULL,NULL,NULL,NULL,0,NULL),(108335,97,17,NULL,NULL,21,NULL,NULL,NULL),(108336,97,19,NULL,NULL,NULL,NULL,0,NULL),(108337,97,23,NULL,NULL,NULL,NULL,0,NULL),(108338,97,34,NULL,'Combination of antibiotics for which resistance result in collateral sensitivity towards one another, reults in slower antibiotic resistance development',NULL,NULL,NULL,NULL),(108339,98,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108340,98,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108341,98,3,NULL,NULL,14,NULL,NULL,NULL),(108342,98,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108343,98,5,'plate',NULL,NULL,NULL,NULL,NULL),(108344,98,5,'flask',NULL,NULL,NULL,NULL,NULL),(108345,98,6,'PYE',NULL,NULL,NULL,NULL,NULL),(108346,98,7,'domestication of wild strains',NULL,NULL,NULL,NULL,NULL),(108347,98,8,NULL,NULL,NULL,NULL,0,NULL),(108348,98,10,NULL,NULL,NULL,NULL,0,NULL),(108349,98,12,NULL,NULL,NULL,NULL,1,NULL),(108350,98,13,'plates',NULL,NULL,NULL,NULL,NULL),(108351,98,14,NULL,NULL,NULL,NULL,0,NULL),(108352,98,16,NULL,NULL,200,NULL,NULL,NULL),(108353,98,17,NULL,NULL,10,NULL,NULL,NULL),(108354,98,19,NULL,NULL,NULL,NULL,1,NULL),(108355,98,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108356,98,21,'resistance',NULL,NULL,NULL,NULL,NULL),(108357,98,23,NULL,NULL,NULL,NULL,1,NULL),(108358,98,34,NULL,'Phenotypic characterization of domestication of fresh E. coli isolates showed the diversity that emerged but also some level of parallelism\nDuring domestication, mainly 2 mutations were shared among replicate populations: ones in rpoS and ones in mutL',NULL,NULL,NULL,NULL),(108359,99,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108360,99,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108361,99,3,NULL,NULL,36,NULL,NULL,NULL),(108362,99,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108363,99,7,'M9',NULL,NULL,NULL,NULL,NULL),(108364,99,7,'high salt',NULL,NULL,NULL,NULL,NULL),(108365,99,7,'acid',NULL,NULL,NULL,NULL,NULL),(108366,99,7,'alkali',NULL,NULL,NULL,NULL,NULL),(108367,99,7,'alcohol stress',NULL,NULL,NULL,NULL,NULL),(108368,99,17,NULL,NULL,18,NULL,NULL,NULL),(108369,99,18,'empty control wells',NULL,NULL,NULL,NULL,NULL),(108370,99,19,NULL,NULL,NULL,NULL,1,NULL),(108371,99,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(108372,99,23,NULL,NULL,NULL,NULL,0,NULL),(108373,99,32,NULL,NULL,10000000,NULL,NULL,NULL),(108374,99,34,NULL,'Automation of frequent transferring cultures resulted in the maintainence of populations in exponential phase.',NULL,NULL,NULL,NULL),(108375,100,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108376,100,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108377,100,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108378,101,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108379,101,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108380,101,3,NULL,NULL,10,NULL,NULL,NULL),(108381,101,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108382,101,7,'42°C',NULL,NULL,NULL,NULL,NULL),(108383,101,10,NULL,NULL,NULL,NULL,0,NULL),(108384,101,12,NULL,NULL,NULL,NULL,0,NULL),(108385,101,16,NULL,NULL,1500,NULL,NULL,NULL),(108386,101,17,NULL,NULL,45,NULL,NULL,NULL),(108387,101,18,'shared mutations',NULL,NULL,NULL,NULL,NULL),(108388,101,19,NULL,NULL,NULL,NULL,1,NULL),(108389,101,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(108390,101,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108391,101,23,NULL,NULL,NULL,NULL,1,NULL),(108392,101,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108393,101,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(108394,101,26,NULL,NULL,17,NULL,NULL,NULL),(108395,101,28,NULL,NULL,25,NULL,NULL,NULL),(108396,101,34,NULL,'Key mutations resulting in improvement at 42°C were identified by parallelism and confirmed by massive parallel genome engineering and competition enrichment.',NULL,NULL,NULL,NULL),(108397,102,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108398,102,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(108399,103,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108400,103,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108401,103,3,NULL,NULL,4,NULL,NULL,NULL),(108402,103,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108403,103,5,'flask',NULL,NULL,NULL,NULL,NULL),(108404,103,6,'M9',NULL,NULL,NULL,NULL,NULL),(108405,103,7,'Osmotic (0.3 M NaCl); Acidic (pH 5.5); Oxidative (100 µM H2O2); n-butanol (0.6%)',NULL,NULL,NULL,NULL,NULL),(108406,103,10,NULL,NULL,NULL,NULL,0,NULL),(108407,103,12,NULL,NULL,NULL,NULL,0,NULL),(108408,103,16,NULL,NULL,500,NULL,NULL,NULL),(108409,103,17,NULL,NULL,72,NULL,NULL,NULL),(108410,103,18,'X-gal & IPTG staining',NULL,NULL,NULL,NULL,NULL),(108411,103,19,NULL,NULL,NULL,NULL,1,NULL),(108412,103,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108413,103,21,'X-gal & IPTG staining',NULL,NULL,NULL,NULL,NULL),(108414,103,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108415,103,23,NULL,NULL,NULL,NULL,1,NULL),(108416,103,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108417,103,25,'Illumina Genome Analyzer II',NULL,NULL,NULL,NULL,NULL),(108418,103,26,NULL,NULL,4,NULL,NULL,NULL),(108419,103,34,NULL,'Adaptation to 4 abiotic stressors shows cases of emerging cross-stress behavior (adaptation to one stressor provides fitness advantage when exposed to second stressor)',NULL,NULL,NULL,NULL),(108420,104,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108421,104,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108422,104,3,NULL,NULL,8,NULL,NULL,NULL),(108423,104,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108424,104,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108425,104,10,NULL,NULL,NULL,NULL,0,NULL),(108426,104,12,NULL,NULL,NULL,NULL,0,NULL),(108427,104,16,NULL,NULL,550,NULL,NULL,NULL),(108428,104,17,NULL,NULL,26,NULL,NULL,NULL),(108429,104,19,NULL,NULL,NULL,NULL,1,NULL),(108430,104,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108431,104,21,'methionine auxotrophy',NULL,NULL,NULL,NULL,NULL),(108432,104,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108433,104,23,NULL,NULL,NULL,NULL,1,NULL),(108434,104,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108435,104,25,'Solexa Genome Analyzer IIx',NULL,NULL,NULL,NULL,NULL),(108436,104,26,NULL,NULL,48,NULL,NULL,NULL),(108437,104,28,NULL,NULL,159,NULL,NULL,NULL),(108438,104,31,'https://trace.ddbj.nig.ac.jp/DRASearch/submission?acc=DRA000563',NULL,NULL,NULL,NULL,NULL),(108439,104,32,NULL,NULL,16000000000,NULL,NULL,NULL),(108440,104,33,NULL,NULL,16000000000,NULL,NULL,NULL),(108441,104,34,NULL,'In slow growing chemostat cultures, more diversity evolves because of mutation accumulation, hypermutation, and negative frequency dependence.',NULL,NULL,NULL,NULL),(108442,105,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108443,105,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108444,105,3,NULL,NULL,25,NULL,NULL,NULL),(108445,105,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108446,105,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(108447,105,6,'M9',NULL,NULL,NULL,NULL,NULL),(108448,105,7,'chloramphenicol',NULL,NULL,NULL,NULL,NULL),(108449,105,7,'doxycycline',NULL,NULL,NULL,NULL,NULL),(108450,105,7,'trimethoprim',NULL,NULL,NULL,NULL,NULL),(108451,105,10,NULL,NULL,NULL,NULL,1,NULL),(108452,105,11,'stepwise increases',NULL,NULL,NULL,NULL,NULL),(108453,105,12,NULL,NULL,NULL,NULL,0,NULL),(108454,105,17,NULL,NULL,25,NULL,NULL,NULL),(108455,105,19,NULL,NULL,NULL,NULL,0,NULL),(108456,105,22,'targeted',NULL,NULL,NULL,NULL,NULL),(108457,105,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108458,105,23,NULL,NULL,NULL,NULL,1,NULL),(108459,105,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108460,105,25,'Illumina Genome Analyzer II',NULL,NULL,NULL,NULL,NULL),(108461,105,26,NULL,NULL,25,NULL,NULL,NULL),(108462,105,31,'https://www.ncbi.nlm.nih.gov/sra?term=SRA046097',NULL,NULL,NULL,NULL,NULL),(108463,105,34,NULL,'Order of mutation fixation is important in evolution of antibiotic resistance under dynamically sustained drug selection.',NULL,NULL,NULL,NULL),(108464,105,35,NULL,'for 5/25 clones, data was missing in SRA',NULL,NULL,NULL,NULL),(108465,106,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108466,106,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108467,106,3,NULL,NULL,72,NULL,NULL,NULL),(108468,106,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108469,106,6,'M9 + glucose',NULL,NULL,NULL,NULL,NULL),(108470,106,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(108471,106,8,NULL,NULL,NULL,NULL,0,NULL),(108472,106,10,NULL,NULL,NULL,NULL,1,NULL),(108473,106,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(108474,106,12,NULL,NULL,NULL,NULL,0,NULL),(108475,106,14,NULL,NULL,NULL,NULL,0,NULL),(108476,106,17,NULL,NULL,15,NULL,NULL,NULL),(108477,106,19,NULL,NULL,NULL,NULL,0,NULL),(108478,106,23,NULL,NULL,NULL,NULL,0,NULL),(108479,106,34,NULL,'Engineering an E. coli with improved and reciprocal sex through conjugation results in increased adaptation rates on complex fitness landscapes',NULL,NULL,NULL,NULL),(108480,107,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108481,107,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108482,107,3,NULL,NULL,2,NULL,NULL,NULL),(108483,107,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108484,107,5,'flask',NULL,NULL,NULL,NULL,NULL),(108485,107,6,'Glucose mineral medium',NULL,NULL,NULL,NULL,NULL),(108486,107,7,'adaptation to metabolic imbalance',NULL,NULL,NULL,NULL,NULL),(108487,107,10,NULL,NULL,NULL,NULL,0,NULL),(108488,107,12,NULL,NULL,NULL,NULL,0,NULL),(108489,107,16,NULL,NULL,20,NULL,NULL,NULL),(108490,107,19,NULL,NULL,NULL,NULL,0,NULL),(108491,107,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108492,107,22,'targeted sequencing',NULL,NULL,NULL,NULL,NULL),(108493,107,23,NULL,NULL,NULL,NULL,1,NULL),(108494,107,24,'Roche Nimblegen',NULL,NULL,NULL,NULL,NULL),(108495,107,25,'CGS',NULL,NULL,NULL,NULL,NULL),(108496,107,26,NULL,NULL,10,NULL,NULL,NULL),(108497,107,34,NULL,'E. coli forces to higher production of NADPH adapts through NuoF mutations allowing for oxidation of both NADH and NADPH',NULL,NULL,NULL,NULL),(108498,108,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108499,108,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108500,108,3,NULL,NULL,10,NULL,NULL,NULL),(108501,108,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108502,108,7,'glucose',NULL,NULL,NULL,NULL,NULL),(108503,108,7,'phosphate limitation',NULL,NULL,NULL,NULL,NULL),(108504,108,16,NULL,NULL,543,NULL,NULL,NULL),(108505,108,17,NULL,NULL,37,NULL,NULL,NULL),(108506,108,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(108507,108,23,NULL,NULL,NULL,NULL,1,NULL),(108508,108,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(108509,108,25,'Southern hybridization with IS probes',NULL,NULL,NULL,NULL,NULL),(108510,108,26,NULL,NULL,120,NULL,NULL,NULL),(108511,108,32,NULL,NULL,16000000000,NULL,NULL,NULL),(108512,108,33,NULL,NULL,16000000000,NULL,NULL,NULL),(108513,108,34,NULL,'IS elements can be important drivers of evolution',NULL,NULL,NULL,NULL),(108514,109,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108515,109,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(108516,109,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108517,109,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(108518,109,7,'succinate tolerance',NULL,NULL,NULL,NULL,NULL),(108519,110,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108520,110,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108521,110,3,NULL,NULL,7,NULL,NULL,NULL),(108522,110,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108523,110,7,'glycerol',NULL,NULL,NULL,NULL,NULL),(108524,110,7,'L-lactate',NULL,NULL,NULL,NULL,NULL),(108525,110,7,'L-1,2-PDO limitation',NULL,NULL,NULL,NULL,NULL),(108526,110,16,NULL,NULL,450,NULL,NULL,NULL),(108527,110,17,NULL,NULL,50,NULL,NULL,NULL),(108528,110,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108529,110,23,NULL,NULL,NULL,NULL,1,NULL),(108530,110,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108531,110,25,'Illumina genome analyzer II',NULL,NULL,NULL,NULL,NULL),(108532,110,26,NULL,NULL,5,NULL,NULL,NULL),(108533,110,33,NULL,NULL,25000,NULL,NULL,NULL),(108534,110,34,NULL,'introduction of cumulative cell divisions as timescale and the acceleration of evolution and mutation fixation by a mutagen',NULL,NULL,NULL,NULL),(108535,111,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108536,111,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(108537,112,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108538,112,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108539,112,3,NULL,NULL,10,NULL,NULL,NULL),(108540,112,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108541,112,5,'flask',NULL,NULL,NULL,NULL,NULL),(108542,112,6,'M9',NULL,NULL,NULL,NULL,NULL),(108543,112,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(108544,112,10,NULL,NULL,NULL,NULL,0,NULL),(108545,112,12,NULL,NULL,NULL,NULL,0,NULL),(108546,112,17,NULL,NULL,50,NULL,NULL,NULL),(108547,112,19,NULL,NULL,NULL,NULL,0,NULL),(108548,112,22,'targeted',NULL,NULL,NULL,NULL,NULL),(108549,112,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108550,112,23,NULL,NULL,NULL,NULL,1,NULL),(108551,112,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(108552,112,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108553,112,26,NULL,NULL,9,NULL,NULL,NULL),(108554,112,34,NULL,'A limited number of genetic paths exist to overcome pgi deletion which can result in diverse phenotypes',NULL,NULL,NULL,NULL),(108555,113,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108556,113,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108557,113,3,NULL,NULL,3,NULL,NULL,NULL),(108558,113,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108559,113,5,'flask',NULL,NULL,NULL,NULL,NULL),(108560,113,6,'M9',NULL,NULL,NULL,NULL,NULL),(108561,113,7,'L-1,2-propanediol usage',NULL,NULL,NULL,NULL,NULL),(108562,113,8,NULL,NULL,NULL,NULL,0,NULL),(108563,113,10,NULL,NULL,NULL,NULL,1,NULL),(108564,113,11,'decreasing glycerol concentration',NULL,NULL,NULL,NULL,NULL),(108565,113,12,NULL,NULL,NULL,NULL,0,NULL),(108566,113,14,NULL,NULL,NULL,NULL,0,NULL),(108567,113,16,NULL,NULL,450,NULL,NULL,NULL),(108568,113,17,NULL,NULL,60,NULL,NULL,NULL),(108569,113,18,'PCR',NULL,NULL,NULL,NULL,NULL),(108570,113,18,'16S RNA sequencing',NULL,NULL,NULL,NULL,NULL),(108571,113,19,NULL,NULL,NULL,NULL,1,NULL),(108572,113,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(108573,113,23,NULL,NULL,NULL,NULL,0,NULL),(108574,113,32,NULL,NULL,500000000,NULL,NULL,NULL),(108575,113,33,NULL,NULL,500000,NULL,NULL,NULL),(108576,113,34,NULL,'AldA was crucial in the expansion of carbon usage in E. coli',NULL,NULL,NULL,NULL),(108577,114,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108578,114,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108579,114,3,NULL,NULL,1,NULL,NULL,NULL),(108580,114,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108581,114,7,'phosphate limitation',NULL,NULL,NULL,NULL,NULL),(108582,114,16,NULL,NULL,130,NULL,NULL,NULL),(108583,114,17,NULL,NULL,37,NULL,NULL,NULL),(108584,114,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108585,114,23,NULL,NULL,NULL,NULL,1,NULL),(108586,114,26,NULL,NULL,5,NULL,NULL,NULL),(108587,114,32,NULL,NULL,20000000000,NULL,NULL,NULL),(108588,114,33,NULL,NULL,12000000000,NULL,NULL,NULL),(108589,114,34,NULL,'diversity in regulatory mutations can arise, even when single niches are applied. All common theme: lowering stress responses',NULL,NULL,NULL,NULL),(108590,115,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108591,115,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108592,115,3,NULL,NULL,12,NULL,NULL,NULL),(108593,115,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108594,115,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108595,115,16,NULL,NULL,40000,NULL,NULL,NULL),(108596,115,17,NULL,NULL,7300,NULL,NULL,NULL),(108597,115,18,'T5',NULL,NULL,NULL,NULL,NULL),(108598,115,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108599,115,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108600,115,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108601,115,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108602,115,23,NULL,NULL,NULL,NULL,1,NULL),(108603,115,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108604,115,25,'Genome Analyzer',NULL,NULL,NULL,NULL,NULL),(108605,115,26,NULL,NULL,7,NULL,NULL,NULL),(108606,115,29,NULL,NULL,50,NULL,NULL,NULL),(108607,115,31,'https://www.ncbi.nlm.nih.gov//sra/?term=SRP001569',NULL,NULL,NULL,NULL,NULL),(108608,115,32,NULL,NULL,500000000,NULL,NULL,NULL),(108609,115,33,NULL,NULL,5000000,NULL,NULL,NULL),(108610,115,34,NULL,'population sequencing of evolving E. coli to capture diversity',NULL,NULL,NULL,NULL),(108611,116,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108612,116,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108613,116,3,NULL,NULL,11,NULL,NULL,NULL),(108614,116,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108615,116,7,'lactate limitation',NULL,NULL,NULL,NULL,NULL),(108616,116,16,NULL,NULL,1100,NULL,NULL,NULL),(108617,116,17,NULL,NULL,60,NULL,NULL,NULL),(108618,116,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108619,116,23,NULL,NULL,NULL,NULL,1,NULL),(108620,116,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(108621,116,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(108622,116,25,'Nimblegen',NULL,NULL,NULL,NULL,NULL),(108623,116,25,'Illumina 1G/2G',NULL,NULL,NULL,NULL,NULL),(108624,116,26,NULL,NULL,11,NULL,NULL,NULL),(108625,116,31,'https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-10-r118#Sec19',NULL,NULL,NULL,NULL,NULL),(108626,116,34,NULL,'differences and parallelisms in mutations across populations',NULL,NULL,NULL,NULL),(108627,117,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108628,117,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108629,117,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108630,118,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108631,118,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108632,118,3,NULL,NULL,1,NULL,NULL,NULL),(108633,118,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108634,118,7,'asparagine limitation',NULL,NULL,NULL,NULL,NULL),(108635,118,16,NULL,NULL,90,NULL,NULL,NULL),(108636,118,17,NULL,NULL,39,NULL,NULL,NULL),(108637,118,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(108638,118,23,NULL,NULL,NULL,NULL,1,NULL),(108639,118,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(108640,118,25,'Custom array',NULL,NULL,NULL,NULL,NULL),(108641,118,26,NULL,NULL,1,NULL,NULL,NULL),(108642,118,33,NULL,NULL,10000000,NULL,NULL,NULL),(108643,118,34,NULL,'Development of ADAM, an array-based discovery of adaptive mutations tool that is based on the detection of removed beneficial mutations by phage transduction of wildtype DNA',NULL,NULL,NULL,NULL),(108644,119,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108645,119,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108646,119,3,NULL,NULL,10,NULL,NULL,NULL),(108647,119,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108648,119,5,'flask',NULL,NULL,NULL,NULL,NULL),(108649,119,6,'MOPS MM+0.3M NaCl',NULL,NULL,NULL,NULL,NULL),(108650,119,7,'Osmotic stress (0.3M NaCl)',NULL,NULL,NULL,NULL,NULL),(108651,119,10,NULL,NULL,NULL,NULL,0,NULL),(108652,119,12,NULL,NULL,NULL,NULL,0,NULL),(108653,119,16,NULL,NULL,250,NULL,NULL,NULL),(108654,119,17,NULL,NULL,25,NULL,NULL,NULL),(108655,119,19,NULL,NULL,NULL,NULL,1,NULL),(108656,119,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108657,119,21,'T5 resistance (fhuA)',NULL,NULL,NULL,NULL,NULL),(108658,119,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(108659,119,23,NULL,NULL,NULL,NULL,1,NULL),(108660,119,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(108661,119,25,'NimbleGen',NULL,NULL,NULL,NULL,NULL),(108662,119,31,'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13666',NULL,NULL,NULL,NULL,NULL),(108663,119,34,NULL,'Strains under stress adapt to the lack of RpoS by a mobile genetic element that moved and allowed for transcription of a pair of key enzymes',NULL,NULL,NULL,NULL),(108664,120,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108665,120,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108666,120,3,NULL,NULL,12,NULL,NULL,NULL),(108667,120,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108668,120,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108669,120,16,NULL,NULL,40000,NULL,NULL,NULL),(108670,120,17,NULL,NULL,7300,NULL,NULL,NULL),(108671,120,18,'T5',NULL,NULL,NULL,NULL,NULL),(108672,120,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108673,120,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108674,120,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108675,120,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108676,120,23,NULL,NULL,NULL,NULL,0,NULL),(108677,120,32,NULL,NULL,500000000,NULL,NULL,NULL),(108678,120,33,NULL,NULL,5000000,NULL,NULL,NULL),(108679,120,34,NULL,'evolution of citrate usage in E. coli is contigent on potentiating mutations, arising in the first 20k generation of the LTEE',NULL,NULL,NULL,NULL),(108680,121,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108681,121,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(108682,121,3,NULL,NULL,384,NULL,NULL,NULL),(108683,121,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108684,121,7,'antibiotic (pairs)',NULL,NULL,NULL,NULL,NULL),(108685,121,16,NULL,NULL,170,NULL,NULL,NULL),(108686,121,17,NULL,NULL,15,NULL,NULL,NULL),(108687,121,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(108688,121,23,NULL,NULL,NULL,NULL,0,NULL),(108689,121,34,NULL,'Applying synergistic antibiotic pairs accelerates evolution of resistance compared to single-drug conditions or antagonistic combinations',NULL,NULL,NULL,NULL),(108690,122,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108691,122,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(108692,122,3,NULL,NULL,3,NULL,NULL,NULL),(108693,122,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108694,122,5,'flask',NULL,NULL,NULL,NULL,NULL),(108695,122,6,'M9',NULL,NULL,NULL,NULL,NULL),(108696,122,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(108697,122,10,NULL,NULL,NULL,NULL,0,NULL),(108698,122,12,NULL,NULL,NULL,NULL,0,NULL),(108699,122,16,NULL,NULL,700,NULL,NULL,NULL),(108700,122,17,NULL,NULL,60,NULL,NULL,NULL),(108701,122,18,'kanamycin resistance',NULL,NULL,NULL,NULL,NULL),(108702,122,18,'PCR',NULL,NULL,NULL,NULL,NULL),(108703,122,19,NULL,NULL,NULL,NULL,0,NULL),(108704,122,23,NULL,NULL,NULL,NULL,0,NULL),(108705,122,34,NULL,'Aerobic growth mixed-acid fermentation is made possible by deletion of all cytochrome oxidases and compensatory evolution in M9.',NULL,NULL,NULL,NULL),(108706,123,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108707,123,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108708,123,3,NULL,NULL,24,NULL,NULL,NULL),(108709,123,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108710,123,5,'flask',NULL,NULL,NULL,NULL,NULL),(108711,123,6,'DM',NULL,NULL,NULL,NULL,NULL),(108712,123,7,'acidic-alkaline range (pH 5.3; 6.3; 7.0; 7.8)',NULL,NULL,NULL,NULL,NULL),(108713,123,10,NULL,NULL,NULL,NULL,0,NULL),(108714,123,12,NULL,NULL,NULL,NULL,0,NULL),(108715,123,16,NULL,NULL,2000,NULL,NULL,NULL),(108716,123,18,'Tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108717,123,19,NULL,NULL,NULL,NULL,1,NULL),(108718,123,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108719,123,21,'Tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108720,123,23,NULL,NULL,NULL,NULL,0,NULL),(108721,123,34,NULL,'Adaptation to lower pH (5.3) was apparent, but to higher pH (7.8) far less significant gains occured. Adapted lines to pH 5.3 showed significant trade-offs, while only 33% of the pH7.8 lines showed trade-offs',NULL,NULL,NULL,NULL),(108722,124,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108723,124,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108724,124,3,NULL,NULL,8,NULL,NULL,NULL),(108725,124,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108726,124,5,'flask',NULL,NULL,NULL,NULL,NULL),(108727,124,6,'M9',NULL,NULL,NULL,NULL,NULL),(108728,124,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(108729,124,10,NULL,NULL,NULL,NULL,0,NULL),(108730,124,12,NULL,NULL,NULL,NULL,0,NULL),(108731,124,16,NULL,NULL,800,NULL,NULL,NULL),(108732,124,17,NULL,NULL,50,NULL,NULL,NULL),(108733,124,19,NULL,NULL,NULL,NULL,0,NULL),(108734,124,23,NULL,NULL,NULL,NULL,0,NULL),(108735,124,34,NULL,'Metabolic and transcriptomic investigation how evolution bypasses genetic pertubation of important metabolic genes.',NULL,NULL,NULL,NULL),(108736,125,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108737,125,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108738,125,3,NULL,NULL,72,NULL,NULL,NULL),(108739,125,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108740,125,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108741,125,16,NULL,NULL,450,NULL,NULL,NULL),(108742,125,17,NULL,NULL,40,NULL,NULL,NULL),(108743,125,18,'CFP',NULL,NULL,NULL,NULL,NULL),(108744,125,18,'YFP',NULL,NULL,NULL,NULL,NULL),(108745,125,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(108746,125,23,NULL,NULL,NULL,NULL,0,NULL),(108747,125,32,NULL,NULL,675000000,NULL,NULL,NULL),(108748,125,33,NULL,NULL,225000,NULL,NULL,NULL),(108749,125,34,NULL,'shape of DFE of benficial mutations is not important for adaptive evolution, mostly very uniform mutations that dictate adaptation which have high enough selective advantage to rise and low enough not to be very rare',NULL,NULL,NULL,NULL),(108750,126,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108751,126,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108752,126,3,NULL,NULL,1,NULL,NULL,NULL),(108753,126,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108754,126,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108755,126,16,NULL,NULL,90,NULL,NULL,NULL),(108756,126,17,NULL,NULL,26,NULL,NULL,NULL),(108757,126,18,'T5 resistance',NULL,NULL,NULL,NULL,NULL),(108758,126,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108759,126,23,NULL,NULL,NULL,NULL,0,NULL),(108760,126,32,NULL,NULL,16000000000,NULL,NULL,NULL),(108761,126,33,NULL,NULL,16000000000,NULL,NULL,NULL),(108762,126,34,NULL,'E. coli can diversify into different phenotypic clusters in an unstructured, single niche environment',NULL,NULL,NULL,NULL),(108763,127,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108764,127,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108765,127,3,NULL,NULL,50,NULL,NULL,NULL),(108766,127,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108767,127,5,'flask',NULL,NULL,NULL,NULL,NULL),(108768,127,6,'M9',NULL,NULL,NULL,NULL,NULL),(108769,127,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(108770,127,10,NULL,NULL,NULL,NULL,0,NULL),(108771,127,12,NULL,NULL,NULL,NULL,0,NULL),(108772,127,17,NULL,NULL,50,NULL,NULL,NULL),(108773,127,19,NULL,NULL,NULL,NULL,0,NULL),(108774,127,23,NULL,NULL,NULL,NULL,0,NULL),(108775,127,34,NULL,'Growth rate recovery in knockout mutants can be predicted by computational models',NULL,NULL,NULL,NULL),(108776,127,35,NULL,'dilution to maintain exponential growth',NULL,NULL,NULL,NULL),(108777,128,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108778,128,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108779,128,3,NULL,NULL,98,NULL,NULL,NULL),(108780,128,4,'MA',NULL,NULL,NULL,NULL,NULL),(108781,128,7,'stationary phase',NULL,NULL,NULL,NULL,NULL),(108782,128,17,NULL,NULL,101,NULL,NULL,NULL),(108783,128,18,'T5',NULL,NULL,NULL,NULL,NULL),(108784,128,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108785,128,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108786,128,21,'T5',NULL,NULL,NULL,NULL,NULL),(108787,128,21,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108788,128,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108789,128,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108790,128,23,NULL,NULL,NULL,NULL,0,NULL),(108791,128,34,NULL,'estimation of deleterious mutation rate in stationary phase by carrying out a MA experiment in this phase',NULL,NULL,NULL,NULL),(108792,129,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108793,129,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108794,129,3,NULL,NULL,18,NULL,NULL,NULL),(108795,129,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108796,129,7,'glucose',NULL,NULL,NULL,NULL,NULL),(108797,129,7,'nitrogen limitation at 0.1',NULL,NULL,NULL,NULL,NULL),(108798,129,7,'0.3',NULL,NULL,NULL,NULL,NULL),(108799,129,7,'0.6 per h; glucose limitation',NULL,NULL,NULL,NULL,NULL),(108800,129,7,'pH stress at 0.3 per h',NULL,NULL,NULL,NULL,NULL),(108801,129,16,NULL,NULL,100,NULL,NULL,NULL),(108802,129,17,NULL,NULL,4,NULL,NULL,NULL),(108803,129,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108804,129,23,NULL,NULL,NULL,NULL,0,NULL),(108805,129,32,NULL,NULL,16000000000,NULL,NULL,NULL),(108806,129,33,NULL,NULL,16000000000,NULL,NULL,NULL),(108807,129,34,NULL,'rpoS null mutation quickly emerge and fix under conditions which are starving but not stressfull',NULL,NULL,NULL,NULL),(108808,130,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108809,130,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108810,130,3,NULL,NULL,1,NULL,NULL,NULL),(108811,130,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108812,130,7,'thymine limitation',NULL,NULL,NULL,NULL,NULL),(108813,130,16,NULL,NULL,10000,NULL,NULL,NULL),(108814,130,17,NULL,NULL,880,NULL,NULL,NULL),(108815,130,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108816,130,23,NULL,NULL,NULL,NULL,0,NULL),(108817,130,32,NULL,NULL,10000000000,NULL,NULL,NULL),(108818,130,33,NULL,NULL,10000000000,NULL,NULL,NULL),(108819,130,34,NULL,'development of a twin chemostat and adaptation to overcome thymine starvation',NULL,NULL,NULL,NULL),(108820,131,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108821,131,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108822,131,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108823,132,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108824,132,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108825,132,3,NULL,NULL,24,NULL,NULL,NULL),(108826,132,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108827,132,5,'flasks',NULL,NULL,NULL,NULL,NULL),(108828,132,6,'DM25',NULL,NULL,NULL,NULL,NULL),(108829,132,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108830,132,8,NULL,NULL,NULL,NULL,0,NULL),(108831,132,10,NULL,NULL,NULL,NULL,0,NULL),(108832,132,12,NULL,NULL,NULL,NULL,0,NULL),(108833,132,14,NULL,NULL,NULL,NULL,0,NULL),(108834,132,16,NULL,NULL,1000,NULL,NULL,NULL),(108835,132,17,NULL,NULL,150,NULL,NULL,NULL),(108836,132,18,'T5',NULL,NULL,NULL,NULL,NULL),(108837,132,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108838,132,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108839,132,19,NULL,NULL,NULL,NULL,1,NULL),(108840,132,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108841,132,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108842,132,23,NULL,NULL,NULL,NULL,0,NULL),(108843,132,32,NULL,NULL,500000000,NULL,NULL,NULL),(108844,132,33,NULL,NULL,5000000,NULL,NULL,NULL),(108845,132,34,NULL,'While intermittent mating and recombination events clearly increased genetic diversity, it did not accelerated adaptation',NULL,NULL,NULL,NULL),(108846,133,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108847,133,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108848,133,3,NULL,NULL,50,NULL,NULL,NULL),(108849,133,4,'MA',NULL,NULL,NULL,NULL,NULL),(108850,133,7,'DM agar',NULL,NULL,NULL,NULL,NULL),(108851,133,16,NULL,NULL,7500,NULL,NULL,NULL),(108852,133,17,NULL,NULL,300,NULL,NULL,NULL),(108853,133,18,'T5',NULL,NULL,NULL,NULL,NULL),(108854,133,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108855,133,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108856,133,21,'T5',NULL,NULL,NULL,NULL,NULL),(108857,133,21,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108858,133,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108859,133,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108860,133,23,NULL,NULL,NULL,NULL,0,NULL),(108861,133,33,NULL,NULL,1,NULL,NULL,NULL),(108862,133,34,NULL,'Estimation of per microbe rate of deleterious mutations is >0.0002 in E. coli per generation',NULL,NULL,NULL,NULL),(108887,135,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108888,135,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108889,135,3,NULL,NULL,12,NULL,NULL,NULL),(108890,135,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108891,135,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108892,135,16,NULL,NULL,2000,NULL,NULL,NULL),(108893,135,17,NULL,NULL,300,NULL,NULL,NULL),(108894,135,18,'T5',NULL,NULL,NULL,NULL,NULL),(108895,135,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108896,135,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108897,135,19,NULL,NULL,NULL,NULL,1,NULL),(108898,135,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(108899,135,21,'T5',NULL,NULL,NULL,NULL,NULL),(108900,135,21,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108901,135,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108902,135,23,NULL,NULL,NULL,NULL,0,NULL),(108903,135,32,NULL,NULL,500000000,NULL,NULL,NULL),(108904,135,33,NULL,NULL,5000000,NULL,NULL,NULL),(108905,135,34,NULL,'Initial fitness gain under feast and famine regime of serial transfer in E. coli is becaus of reduced lag and increased maximum growth rate',NULL,NULL,NULL,NULL),(108906,136,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108907,136,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108908,136,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108933,138,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108934,138,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108935,138,3,NULL,NULL,1,NULL,NULL,NULL),(108936,138,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108937,138,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108938,138,16,NULL,NULL,773,NULL,NULL,NULL),(108939,138,17,NULL,NULL,113,NULL,NULL,NULL),(108940,138,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(108941,138,23,NULL,NULL,NULL,NULL,0,NULL),(108942,138,32,NULL,NULL,20000000000,NULL,NULL,NULL),(108943,138,33,NULL,NULL,20000000000,NULL,NULL,NULL),(108944,138,34,NULL,'2D protein gels and other characteristics can tell 4 clones apart from one population',NULL,NULL,NULL,NULL),(108945,139,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108946,139,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108947,139,3,NULL,NULL,12,NULL,NULL,NULL),(108948,139,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(108949,139,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108950,139,16,NULL,NULL,2000,NULL,NULL,NULL),(108951,139,17,NULL,NULL,300,NULL,NULL,NULL),(108952,139,18,'T5',NULL,NULL,NULL,NULL,NULL),(108953,139,18,'T6 resistance',NULL,NULL,NULL,NULL,NULL),(108954,139,18,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108955,139,21,'tetrazolium arabinose',NULL,NULL,NULL,NULL,NULL),(108956,139,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(108957,139,23,NULL,NULL,NULL,NULL,0,NULL),(108958,139,32,NULL,NULL,500000000,NULL,NULL,NULL),(108959,139,33,NULL,NULL,5000000,NULL,NULL,NULL),(108960,139,34,NULL,'fitness increase in all 12 populations of 37%, start of LTEE',NULL,NULL,NULL,NULL),(108984,141,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108985,141,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(108986,141,3,NULL,NULL,15,NULL,NULL,NULL),(108987,141,4,'continuous',NULL,NULL,NULL,NULL,NULL),(108988,141,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(108989,141,16,NULL,NULL,1867,NULL,NULL,NULL),(108990,141,17,NULL,NULL,272,NULL,NULL,NULL),(108991,141,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(108992,141,23,NULL,NULL,NULL,NULL,0,NULL),(108993,141,32,NULL,NULL,20000000000,NULL,NULL,NULL),(108994,141,33,NULL,NULL,20000000000,NULL,NULL,NULL),(108995,141,34,NULL,'diversification and rise of polymorphisms in mixed, simple environment',NULL,NULL,NULL,NULL),(108996,141,35,NULL,'Kurlandzka A, Rosenzweig FR & Adams J (1991) Identification of adaptive changes in an evolving population of Escherichia coli: the role of changes with regulatory and highly pleiotropic effects. Mol. Biol. Evol. 8: 261–281',NULL,NULL,NULL,NULL),(108997,142,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(108998,142,1,'Aeromonas hydrophila',NULL,NULL,NULL,NULL,NULL),(108999,142,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109000,142,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109001,142,3,NULL,NULL,6,NULL,NULL,NULL),(109002,142,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109003,142,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109004,142,6,'TSB',NULL,NULL,NULL,NULL,NULL),(109005,142,6,'MHB',NULL,NULL,NULL,NULL,NULL),(109006,142,7,'tachyplesin I',NULL,NULL,NULL,NULL,NULL),(109007,142,8,NULL,NULL,NULL,NULL,0,NULL),(109008,142,10,NULL,NULL,NULL,NULL,1,NULL),(109009,142,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(109010,142,12,NULL,NULL,NULL,NULL,0,NULL),(109011,142,14,NULL,NULL,NULL,NULL,0,NULL),(109012,142,17,NULL,NULL,96,NULL,NULL,NULL),(109013,142,19,NULL,NULL,NULL,NULL,0,NULL),(109014,142,23,NULL,NULL,NULL,NULL,0,NULL),(109015,142,34,NULL,'Resistance to AMP readily developped in 4 species',NULL,NULL,NULL,NULL),(109016,143,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109017,143,1,'Klebsiella pneumoniae',NULL,NULL,NULL,NULL,NULL),(109018,143,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109019,143,3,NULL,NULL,45,NULL,NULL,NULL),(109020,143,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109021,143,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(109022,143,6,'LB',NULL,NULL,NULL,NULL,NULL),(109023,143,7,'adaptation to conjugative plasmid carriage under drug-free or selective conditions',NULL,NULL,NULL,NULL,NULL),(109024,143,8,NULL,NULL,NULL,NULL,0,NULL),(109025,143,10,NULL,NULL,NULL,NULL,0,NULL),(109026,143,12,NULL,NULL,NULL,NULL,0,NULL),(109027,143,14,NULL,NULL,NULL,NULL,0,NULL),(109028,143,16,NULL,NULL,280,NULL,NULL,NULL),(109029,143,17,NULL,NULL,35,NULL,NULL,NULL),(109030,143,19,NULL,NULL,NULL,NULL,1,NULL),(109031,143,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109032,143,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109033,143,23,NULL,NULL,NULL,NULL,1,NULL),(109034,143,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109035,143,24,'Pacbio',NULL,NULL,NULL,NULL,NULL),(109036,143,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(109037,143,25,'RSII',NULL,NULL,NULL,NULL,NULL),(109038,143,27,NULL,NULL,18,NULL,NULL,NULL),(109039,143,29,NULL,NULL,30,NULL,NULL,NULL),(109040,143,34,NULL,'Carriage of conjugative plasmids in naive hosts elicit a cost which could be compensate for if high under selection for its maintenance by large deletions in the plasmid.',NULL,NULL,NULL,NULL),(109061,145,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109062,145,1,'Phage Lambda',NULL,NULL,NULL,NULL,NULL),(109063,145,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109064,145,3,NULL,NULL,102,NULL,NULL,NULL),(109065,145,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109066,145,5,'flask',NULL,NULL,NULL,NULL,NULL),(109067,145,6,'M9 glucose',NULL,NULL,NULL,NULL,NULL),(109068,145,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(109069,145,8,NULL,NULL,NULL,NULL,0,NULL),(109070,145,10,NULL,NULL,NULL,NULL,1,NULL),(109071,145,11,'co-evolution',NULL,NULL,NULL,NULL,NULL),(109072,145,12,NULL,NULL,NULL,NULL,0,NULL),(109073,145,14,NULL,NULL,NULL,NULL,0,NULL),(109074,145,17,NULL,NULL,28,NULL,NULL,NULL),(109075,145,19,NULL,NULL,NULL,NULL,1,NULL),(109076,145,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109077,145,23,NULL,NULL,NULL,NULL,1,NULL),(109078,145,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109079,145,24,'SOLiD',NULL,NULL,NULL,NULL,NULL),(109080,145,25,'ABI SOLiD 4',NULL,NULL,NULL,NULL,NULL),(109081,145,25,'GA II',NULL,NULL,NULL,NULL,NULL),(109082,145,26,NULL,NULL,8,NULL,NULL,NULL),(109083,145,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRA043942',NULL,NULL,NULL,NULL,NULL),(109084,145,33,NULL,NULL,100,NULL,NULL,NULL),(109085,145,34,NULL,'De novo evolution of the use of a new receptor of phage lambda to infect E. coli and this through all-or-none epistatic interactions, historical contigency and co-evolutionary dynamics\n\nThe intermediates towards new receptor use, all increase infectivity on the ancestral receptor while co-evolution of the bacteria change the landscape to favor evolution towards the use of the novel receptor.',NULL,NULL,NULL,NULL),(109086,146,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109087,146,1,'Phage M13',NULL,NULL,NULL,NULL,NULL),(109088,146,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109089,146,3,NULL,NULL,90,NULL,NULL,NULL),(109090,146,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109091,146,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(109092,146,6,'DM medium',NULL,NULL,NULL,NULL,NULL),(109093,146,7,'co-evolution between M13',NULL,NULL,NULL,NULL,NULL),(109094,146,7,'E. coli',NULL,NULL,NULL,NULL,NULL),(109095,146,8,NULL,NULL,NULL,NULL,0,NULL),(109096,146,10,NULL,NULL,NULL,NULL,1,NULL),(109097,146,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(109098,146,12,NULL,NULL,NULL,NULL,0,NULL),(109099,146,14,NULL,NULL,NULL,NULL,0,NULL),(109100,146,16,NULL,NULL,130,NULL,NULL,NULL),(109101,146,17,NULL,NULL,20,NULL,NULL,NULL),(109102,146,19,NULL,NULL,NULL,NULL,0,NULL),(109103,146,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109104,146,23,NULL,NULL,NULL,NULL,1,NULL),(109105,146,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109106,146,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(109107,146,26,NULL,NULL,11,NULL,NULL,NULL),(109108,146,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA356430',NULL,NULL,NULL,NULL,NULL),(109109,146,33,NULL,NULL,1000,NULL,NULL,NULL),(109110,146,34,NULL,'Coadapted M13 becomes parasite for ancestor E. coli but can re-evolve mutualism',NULL,NULL,NULL,NULL),(109111,146,35,NULL,'did not use/mention the genome sequencing',NULL,NULL,NULL,NULL),(109112,147,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109113,147,1,'Phage M13',NULL,NULL,NULL,NULL,NULL),(109114,147,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109115,147,3,NULL,NULL,90,NULL,NULL,NULL),(109116,147,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109117,147,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(109118,147,6,'DM medium',NULL,NULL,NULL,NULL,NULL),(109119,147,7,'co-evolution between M13',NULL,NULL,NULL,NULL,NULL),(109120,147,7,'E. coli',NULL,NULL,NULL,NULL,NULL),(109121,147,8,NULL,NULL,NULL,NULL,0,NULL),(109122,147,10,NULL,NULL,NULL,NULL,1,NULL),(109123,147,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(109124,147,12,NULL,NULL,NULL,NULL,0,NULL),(109125,147,14,NULL,NULL,NULL,NULL,0,NULL),(109126,147,17,NULL,NULL,20,NULL,NULL,NULL),(109127,147,18,'no continous shaking',NULL,NULL,NULL,NULL,NULL),(109128,147,19,NULL,NULL,NULL,NULL,1,NULL),(109129,147,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109130,147,20,'yield',NULL,NULL,NULL,NULL,NULL),(109131,147,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109132,147,23,NULL,NULL,NULL,NULL,1,NULL),(109133,147,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109134,147,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(109135,147,26,NULL,NULL,6,NULL,NULL,NULL),(109136,147,31,'https://www.ncbi.nlm.nih.gov/sra?term=SRP074196',NULL,NULL,NULL,NULL,NULL),(109137,147,34,NULL,'M13 is in fact a mutualist and can co-evolve with E. coli.',NULL,NULL,NULL,NULL),(109167,149,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109168,149,1,'Phage Mphi',NULL,NULL,NULL,NULL,NULL),(109169,149,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109170,149,3,NULL,NULL,12,NULL,NULL,NULL),(109171,149,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109172,149,5,'12-well plate',NULL,NULL,NULL,NULL,NULL),(109173,149,6,'RPMI',NULL,NULL,NULL,NULL,NULL),(109174,149,7,'adaptation to macrophage',NULL,NULL,NULL,NULL,NULL),(109175,149,8,NULL,NULL,NULL,NULL,0,NULL),(109176,149,10,NULL,NULL,NULL,NULL,0,NULL),(109177,149,12,NULL,NULL,NULL,NULL,0,NULL),(109178,149,14,NULL,NULL,NULL,NULL,1,NULL),(109179,149,15,'in situ evolution',NULL,NULL,NULL,NULL,NULL),(109180,149,16,NULL,NULL,450,NULL,NULL,NULL),(109181,149,17,NULL,NULL,30,NULL,NULL,NULL),(109182,149,18,'fluorescence resistance',NULL,NULL,NULL,NULL,NULL),(109183,149,19,NULL,NULL,NULL,NULL,1,NULL),(109184,149,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109185,149,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(109186,149,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109187,149,23,NULL,NULL,NULL,NULL,1,NULL),(109188,149,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109189,149,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(109190,149,26,NULL,NULL,7,NULL,NULL,NULL),(109191,149,32,NULL,NULL,400000000,NULL,NULL,NULL),(109192,149,33,NULL,NULL,1000000,NULL,NULL,NULL),(109193,149,34,NULL,'E. coli rapidly adapts to macrophage internalization and increases pathogenicity, by emergent small colony variants and mucoid variants with much clonal interference.\n\nMutations causal to the SCV phenotype target mainly the ETC and result in collateral sensitivity, increased aminoglycoside resistance and increased in vivo fitness',NULL,NULL,NULL,NULL),(109194,150,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109195,150,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(109196,150,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(109197,150,1,'Phage T4',NULL,NULL,NULL,NULL,NULL),(109198,150,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109199,150,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109200,150,14,NULL,NULL,NULL,NULL,1,NULL),(109201,150,15,'combination of multiple predators',NULL,NULL,NULL,NULL,NULL),(109202,150,15,'prey',NULL,NULL,NULL,NULL,NULL),(109203,151,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109204,151,1,'Aeromonas hydrophila',NULL,NULL,NULL,NULL,NULL),(109205,151,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109206,151,3,NULL,NULL,5,NULL,NULL,NULL),(109207,151,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109208,151,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109209,151,5,'plates',NULL,NULL,NULL,NULL,NULL),(109210,151,6,'LB',NULL,NULL,NULL,NULL,NULL),(109211,151,7,'growth with R1',NULL,NULL,NULL,NULL,NULL),(109212,151,8,NULL,NULL,NULL,NULL,0,NULL),(109213,151,10,NULL,NULL,NULL,NULL,0,NULL),(109214,151,12,NULL,NULL,NULL,NULL,1,NULL),(109215,151,13,'plates',NULL,NULL,NULL,NULL,NULL),(109216,151,14,NULL,NULL,NULL,NULL,0,NULL),(109217,151,16,NULL,NULL,420,NULL,NULL,NULL),(109218,151,17,NULL,NULL,42,NULL,NULL,NULL),(109219,151,18,'resistance',NULL,NULL,NULL,NULL,NULL),(109220,151,19,NULL,NULL,NULL,NULL,1,NULL),(109221,151,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109222,151,21,'resistance',NULL,NULL,NULL,NULL,NULL),(109223,151,23,NULL,NULL,NULL,NULL,0,NULL),(109224,151,33,NULL,NULL,25000000,NULL,NULL,NULL),(109225,151,34,NULL,'Despite an initial cost of bearing R1, the cost can be reverted and the evolved plasmid can even become beneficial to evolved strains, ancestor and other species.',NULL,NULL,NULL,NULL),(109226,152,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(109227,152,1,'Salmonella enterica Typhimurium',NULL,NULL,NULL,NULL,NULL),(109228,152,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109229,152,3,NULL,NULL,40,NULL,NULL,NULL),(109230,152,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109231,152,5,'plates',NULL,NULL,NULL,NULL,NULL),(109232,152,6,'M9 lactose',NULL,NULL,NULL,NULL,NULL),(109233,152,7,'cooperative lactose comsumption',NULL,NULL,NULL,NULL,NULL),(109234,152,8,NULL,NULL,NULL,NULL,1,NULL),(109235,152,9,'methionine deficient',NULL,NULL,NULL,NULL,NULL),(109236,152,10,NULL,NULL,NULL,NULL,0,NULL),(109237,152,12,NULL,NULL,NULL,NULL,1,NULL),(109238,152,13,'plates',NULL,NULL,NULL,NULL,NULL),(109239,152,14,NULL,NULL,NULL,NULL,0,NULL),(109240,152,16,NULL,NULL,20,NULL,NULL,NULL),(109241,152,17,NULL,NULL,6,NULL,NULL,NULL),(109242,152,19,NULL,NULL,NULL,NULL,1,NULL),(109243,152,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109244,152,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109245,152,23,NULL,NULL,NULL,NULL,1,NULL),(109246,152,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109247,152,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(109248,152,26,NULL,NULL,3,NULL,NULL,NULL),(109249,152,33,NULL,NULL,10000000,NULL,NULL,NULL),(109250,152,34,NULL,'While evolution resulted in paralle genetic changes, phenotypes are diverse in the enforced cooperation.',NULL,NULL,NULL,NULL),(109273,154,1,'foot-and-mouth disease virus FMDV',NULL,NULL,NULL,NULL,NULL),(109274,154,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109275,154,3,NULL,NULL,4,NULL,NULL,NULL),(109276,154,4,'MA',NULL,NULL,NULL,NULL,NULL),(109277,154,7,'BHK-21 cells',NULL,NULL,NULL,NULL,NULL),(109278,154,17,NULL,NULL,100,NULL,NULL,NULL),(109279,154,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(109280,154,23,NULL,NULL,NULL,NULL,0,NULL),(109281,154,33,NULL,NULL,1,NULL,NULL,NULL),(109282,154,34,NULL,'RNA virus MA experiment results not in a continous monotonic decrease but one showing fluctuations and resistance to virus extinction',NULL,NULL,NULL,NULL),(109283,155,1,'Geobacillus stearothermophilus',NULL,NULL,NULL,NULL,NULL),(109284,155,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109285,155,3,NULL,NULL,2,NULL,NULL,NULL),(109286,155,4,'continuous',NULL,NULL,NULL,NULL,NULL),(109287,155,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(109288,155,6,'modified LB',NULL,NULL,NULL,NULL,NULL),(109289,155,7,'temperature',NULL,NULL,NULL,NULL,NULL),(109290,155,10,NULL,NULL,NULL,NULL,1,NULL),(109291,155,11,'fixed increases',NULL,NULL,NULL,NULL,NULL),(109292,155,12,NULL,NULL,NULL,NULL,0,NULL),(109293,155,16,NULL,NULL,1500,NULL,NULL,NULL),(109294,155,17,NULL,NULL,125,NULL,NULL,NULL),(109295,155,19,NULL,NULL,NULL,NULL,1,NULL),(109296,155,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109297,155,21,'permissive temperature',NULL,NULL,NULL,NULL,NULL),(109298,155,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(109299,155,23,NULL,NULL,NULL,NULL,0,NULL),(109300,155,32,NULL,NULL,50000000000,NULL,NULL,NULL),(109301,155,33,NULL,NULL,50000000000,NULL,NULL,NULL),(109302,155,34,NULL,'Discovery of a very constrained molecular landscape in the adaptation of a single essential malaptad protein to increasing temperature',NULL,NULL,NULL,NULL),(109303,156,1,'Gephyrocapsa oceanica',NULL,NULL,NULL,NULL,NULL),(109304,156,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109305,156,3,NULL,NULL,6,NULL,NULL,NULL),(109306,156,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109307,156,6,'artificial sea water medium',NULL,NULL,NULL,NULL,NULL),(109308,156,7,'CO2 pressure',NULL,NULL,NULL,NULL,NULL),(109309,156,8,NULL,NULL,NULL,NULL,0,NULL),(109310,156,10,NULL,NULL,NULL,NULL,0,NULL),(109311,156,12,NULL,NULL,NULL,NULL,0,NULL),(109312,156,14,NULL,NULL,NULL,NULL,0,NULL),(109313,156,16,NULL,NULL,1060,NULL,NULL,NULL),(109314,156,17,NULL,NULL,712,NULL,NULL,NULL),(109315,156,19,NULL,NULL,NULL,NULL,0,NULL),(109316,156,23,NULL,NULL,NULL,NULL,0,NULL),(109317,156,32,NULL,NULL,480000000,NULL,NULL,NULL),(109318,156,33,NULL,NULL,60000,NULL,NULL,NULL),(109319,156,34,NULL,'Adaptation to high CO2 pressure leads to lower growth rate and enhanced particulate organic carbon and nitrogen production',NULL,NULL,NULL,NULL),(109320,157,1,'Haemophilus influenzae AND pB1000',NULL,NULL,NULL,NULL,NULL),(109321,157,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109322,157,3,NULL,NULL,10,NULL,NULL,NULL),(109323,157,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109324,157,6,'HTM',NULL,NULL,NULL,NULL,NULL),(109325,157,7,'cost of increased copynumber by heterozygousity in plasmid sequence',NULL,NULL,NULL,NULL,NULL),(109326,157,8,NULL,NULL,NULL,NULL,1,NULL),(109327,157,9,'high compy number heterozygosity',NULL,NULL,NULL,NULL,NULL),(109328,157,10,NULL,NULL,NULL,NULL,0,NULL),(109329,157,12,NULL,NULL,NULL,NULL,0,NULL),(109330,157,14,NULL,NULL,NULL,NULL,0,NULL),(109331,157,16,NULL,NULL,300,NULL,NULL,NULL),(109332,157,17,NULL,NULL,30,NULL,NULL,NULL),(109333,157,19,NULL,NULL,NULL,NULL,1,NULL),(109334,157,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109335,157,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(109336,157,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109337,157,23,NULL,NULL,NULL,NULL,1,NULL),(109338,157,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(109339,157,27,NULL,NULL,10,NULL,NULL,NULL),(109340,157,34,NULL,'A heterozygosity emerges in an intracellular plasmid population leading to higher copy numbers and therefore higher resistance which comes at a cost but can be overcome by experimental evolution',NULL,NULL,NULL,NULL),(109341,158,1,'Helicobacter pylori',NULL,NULL,NULL,NULL,NULL),(109342,158,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109343,158,3,NULL,NULL,18,NULL,NULL,NULL),(109344,158,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109345,158,5,'plates',NULL,NULL,NULL,NULL,NULL),(109346,158,6,'BHI-YE',NULL,NULL,NULL,NULL,NULL),(109347,158,7,'metronidazole',NULL,NULL,NULL,NULL,NULL),(109348,158,8,NULL,NULL,NULL,NULL,0,NULL),(109349,158,10,NULL,NULL,NULL,NULL,1,NULL),(109350,158,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(109351,158,12,NULL,NULL,NULL,NULL,1,NULL),(109352,158,13,'agar plate',NULL,NULL,NULL,NULL,NULL),(109353,158,14,NULL,NULL,NULL,NULL,0,NULL),(109354,158,19,NULL,NULL,NULL,NULL,0,NULL),(109355,158,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(109356,158,23,NULL,NULL,NULL,NULL,0,NULL),(109357,158,34,NULL,'While spontaneous mutation rate assessed on metronidazole was low, metronidazole resistant clones can readily be selected by serial transfer adaptive evolution.',NULL,NULL,NULL,NULL),(109358,159,1,'Helicobacter pylori',NULL,NULL,NULL,NULL,NULL),(109359,159,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109360,159,3,NULL,NULL,28,NULL,NULL,NULL),(109361,159,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109362,159,5,'plates',NULL,NULL,NULL,NULL,NULL),(109363,159,6,'MHB',NULL,NULL,NULL,NULL,NULL),(109364,159,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(109365,159,8,NULL,NULL,NULL,NULL,0,NULL),(109366,159,10,NULL,NULL,NULL,NULL,1,NULL),(109367,159,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(109368,159,12,NULL,NULL,NULL,NULL,1,NULL),(109369,159,13,'agar plate',NULL,NULL,NULL,NULL,NULL),(109370,159,14,NULL,NULL,NULL,NULL,0,NULL),(109371,159,19,NULL,NULL,NULL,NULL,0,NULL),(109372,159,23,NULL,NULL,NULL,NULL,0,NULL),(109373,159,34,NULL,'Spontaneous mutation rates on the used antibiotcs are bad predictors of long-term adaptation potential using low dose antibiotics.',NULL,NULL,NULL,NULL),(109374,160,1,'Kluyveromyces marxianus',NULL,NULL,NULL,NULL,NULL),(109375,160,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109376,161,1,'Lactobacillus rhamnosus',NULL,NULL,NULL,NULL,NULL),(109377,161,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109378,161,3,NULL,NULL,4,NULL,NULL,NULL),(109379,161,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109380,161,6,'MRS',NULL,NULL,NULL,NULL,NULL),(109381,161,7,'osmotic',NULL,NULL,NULL,NULL,NULL),(109382,161,7,'bile salt or shear stress',NULL,NULL,NULL,NULL,NULL),(109383,161,8,NULL,NULL,NULL,NULL,0,NULL),(109384,161,10,NULL,NULL,NULL,NULL,0,NULL),(109385,161,12,NULL,NULL,NULL,NULL,0,NULL),(109386,161,14,NULL,NULL,NULL,NULL,0,NULL),(109387,161,16,NULL,NULL,1000,NULL,NULL,NULL),(109388,161,17,NULL,NULL,100,NULL,NULL,NULL),(109389,161,19,NULL,NULL,NULL,NULL,0,NULL),(109390,161,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109391,161,23,NULL,NULL,NULL,NULL,1,NULL),(109392,161,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109393,161,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(109394,161,26,NULL,NULL,40,NULL,NULL,NULL),(109395,161,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP067389',NULL,NULL,NULL,NULL,NULL),(109396,161,34,NULL,'Study on the stability of the genome of LGG in respect to its industrial use and evolution.',NULL,NULL,NULL,NULL),(109397,161,35,NULL,'mutator was found',NULL,NULL,NULL,NULL),(109398,162,1,'Lactococcus lactis',NULL,NULL,NULL,NULL,NULL),(109399,162,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109400,162,3,NULL,NULL,2,NULL,NULL,NULL),(109401,162,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109402,162,5,'emulsion',NULL,NULL,NULL,NULL,NULL),(109403,162,6,'CDM + glucose',NULL,NULL,NULL,NULL,NULL),(109404,162,7,'increasing yield',NULL,NULL,NULL,NULL,NULL),(109405,162,8,NULL,NULL,NULL,NULL,1,NULL),(109406,162,9,'mutagenesis',NULL,NULL,NULL,NULL,NULL),(109407,162,10,NULL,NULL,NULL,NULL,0,NULL),(109408,162,12,NULL,NULL,NULL,NULL,1,NULL),(109409,162,13,'emulsion',NULL,NULL,NULL,NULL,NULL),(109410,162,14,NULL,NULL,NULL,NULL,0,NULL),(109411,162,17,NULL,NULL,62,NULL,NULL,NULL),(109412,162,19,NULL,NULL,NULL,NULL,0,NULL),(109413,162,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109414,162,23,NULL,NULL,NULL,NULL,1,NULL),(109415,162,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109416,162,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(109417,162,26,NULL,NULL,3,NULL,NULL,NULL),(109418,162,28,NULL,NULL,718,NULL,NULL,NULL),(109419,162,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP017852.',NULL,NULL,NULL,NULL,NULL),(109420,162,34,NULL,'Evolution in emulsion selects for higher yield, not higher growth rate',NULL,NULL,NULL,NULL),(109421,163,1,'Phage Lambda',NULL,NULL,NULL,NULL,NULL),(109422,163,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109423,163,3,NULL,NULL,18,NULL,NULL,NULL),(109424,163,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109425,163,5,'flask',NULL,NULL,NULL,NULL,NULL),(109426,163,6,'LBM9',NULL,NULL,NULL,NULL,NULL),(109427,163,7,'E. coli hosts',NULL,NULL,NULL,NULL,NULL),(109428,163,8,NULL,NULL,NULL,NULL,0,NULL),(109429,163,10,NULL,NULL,NULL,NULL,0,NULL),(109430,163,12,NULL,NULL,NULL,NULL,0,NULL),(109431,163,14,NULL,NULL,NULL,NULL,0,NULL),(109432,163,17,NULL,NULL,35,NULL,NULL,NULL),(109433,163,19,NULL,NULL,NULL,NULL,1,NULL),(109434,163,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109435,163,20,'yield',NULL,NULL,NULL,NULL,NULL),(109436,163,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(109437,163,23,NULL,NULL,NULL,NULL,1,NULL),(109438,163,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109439,163,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(109440,163,26,NULL,NULL,2,NULL,NULL,NULL),(109441,163,28,NULL,NULL,600,NULL,NULL,NULL),(109442,164,1,'Leuconostoc mesenteroides',NULL,NULL,NULL,NULL,NULL),(109443,164,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109444,164,3,NULL,NULL,1,NULL,NULL,NULL),(109445,164,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109446,164,5,'serum bottle',NULL,NULL,NULL,NULL,NULL),(109447,164,6,'MRS',NULL,NULL,NULL,NULL,NULL),(109448,164,7,'lactic acid',NULL,NULL,NULL,NULL,NULL),(109449,164,8,NULL,NULL,NULL,NULL,0,NULL),(109450,164,10,NULL,NULL,NULL,NULL,1,NULL),(109451,164,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(109452,164,12,NULL,NULL,NULL,NULL,0,NULL),(109453,164,14,NULL,NULL,NULL,NULL,0,NULL),(109454,164,17,NULL,NULL,365,NULL,NULL,NULL),(109455,164,19,NULL,NULL,NULL,NULL,0,NULL),(109456,164,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109457,164,23,NULL,NULL,NULL,NULL,1,NULL),(109458,164,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109459,164,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(109460,164,26,NULL,NULL,3,NULL,NULL,NULL),(109461,164,34,NULL,'Growth inhibition limits production of lactic acid but can be overcome by evolution towards tolerance',NULL,NULL,NULL,NULL),(109462,165,1,'Methanococcus maripaludis',NULL,NULL,NULL,NULL,NULL),(109463,165,1,'Desulfovibrio vulgaris',NULL,NULL,NULL,NULL,NULL),(109464,165,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109465,165,3,NULL,NULL,24,NULL,NULL,NULL),(109466,165,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109467,165,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109468,165,6,'coculture medium',NULL,NULL,NULL,NULL,NULL),(109469,165,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(109470,165,8,NULL,NULL,NULL,NULL,0,NULL),(109471,165,10,NULL,NULL,NULL,NULL,0,NULL),(109472,165,12,NULL,NULL,NULL,NULL,1,NULL),(109473,165,13,'static',NULL,NULL,NULL,NULL,NULL),(109474,165,14,NULL,NULL,NULL,NULL,0,NULL),(109475,165,16,NULL,NULL,300,NULL,NULL,NULL),(109476,165,17,NULL,NULL,315,NULL,NULL,NULL),(109477,165,18,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(109478,165,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(109479,165,18,'microscopy',NULL,NULL,NULL,NULL,NULL),(109480,165,19,NULL,NULL,NULL,NULL,1,NULL),(109481,165,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109482,165,20,'yield',NULL,NULL,NULL,NULL,NULL),(109483,165,23,NULL,NULL,NULL,NULL,0,NULL),(109484,165,34,NULL,'An imposed mutualism can evolve towards stability by mutations in both partners that can act antagonistically.',NULL,NULL,NULL,NULL),(109485,166,1,'Methylobacterium extorquens',NULL,NULL,NULL,NULL,NULL),(109486,166,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109487,166,3,NULL,NULL,40,NULL,NULL,NULL),(109488,166,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109489,166,5,'48-well plate',NULL,NULL,NULL,NULL,NULL),(109490,166,5,'flask',NULL,NULL,NULL,NULL,NULL),(109491,166,6,'hypho media',NULL,NULL,NULL,NULL,NULL),(109492,166,7,'methylamine limitation',NULL,NULL,NULL,NULL,NULL),(109493,166,8,NULL,NULL,NULL,NULL,1,NULL),(109494,166,9,'fae mutants',NULL,NULL,NULL,NULL,NULL),(109495,166,10,NULL,NULL,NULL,NULL,0,NULL),(109496,166,12,NULL,NULL,NULL,NULL,0,NULL),(109497,166,14,NULL,NULL,NULL,NULL,0,NULL),(109498,166,16,NULL,NULL,250,NULL,NULL,NULL),(109499,166,17,NULL,NULL,41,NULL,NULL,NULL),(109500,166,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(109501,166,19,NULL,NULL,NULL,NULL,1,NULL),(109502,166,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109503,166,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(109504,166,23,NULL,NULL,NULL,NULL,0,NULL),(109505,166,34,NULL,'Synonymous mutations were selected for during adaptive evolution to the cost of synonymous ancestral mutations.',NULL,NULL,NULL,NULL),(109506,167,1,'Methylobacterium extorquens',NULL,NULL,NULL,NULL,NULL),(109507,167,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109508,167,3,NULL,NULL,1,NULL,NULL,NULL),(109509,167,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109510,167,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109511,167,6,'minimal hypho medium + carbon source',NULL,NULL,NULL,NULL,NULL),(109512,167,7,'1-butanol',NULL,NULL,NULL,NULL,NULL),(109513,167,8,NULL,NULL,NULL,NULL,0,NULL),(109514,167,10,NULL,NULL,NULL,NULL,1,NULL),(109515,167,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(109516,167,12,NULL,NULL,NULL,NULL,0,NULL),(109517,167,14,NULL,NULL,NULL,NULL,0,NULL),(109518,167,16,NULL,NULL,200,NULL,NULL,NULL),(109519,167,18,'rifamycin resistance',NULL,NULL,NULL,NULL,NULL),(109520,167,19,NULL,NULL,NULL,NULL,0,NULL),(109521,167,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109522,167,23,NULL,NULL,NULL,NULL,1,NULL),(109523,167,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109524,167,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(109525,167,26,NULL,NULL,1,NULL,NULL,NULL),(109526,167,28,NULL,NULL,300,NULL,NULL,NULL),(109527,167,34,NULL,'1-butanol tolerance improved by increased growth rate and tolerance through a mutation in kefB which can be linked to a lot of metabolic changes',NULL,NULL,NULL,NULL),(109528,168,1,'Methylobacterium extorquens',NULL,NULL,NULL,NULL,NULL),(109529,168,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109530,168,3,NULL,NULL,2,NULL,NULL,NULL),(109531,168,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109532,168,5,'48-well plate',NULL,NULL,NULL,NULL,NULL),(109533,168,6,'modified hypho medium',NULL,NULL,NULL,NULL,NULL),(109534,168,7,'methylamine limitation',NULL,NULL,NULL,NULL,NULL),(109535,168,8,NULL,NULL,NULL,NULL,1,NULL),(109536,168,9,'loss of MaDH pathway',NULL,NULL,NULL,NULL,NULL),(109537,168,10,NULL,NULL,NULL,NULL,0,NULL),(109538,168,12,NULL,NULL,NULL,NULL,0,NULL),(109539,168,14,NULL,NULL,NULL,NULL,0,NULL),(109540,168,16,NULL,NULL,150,NULL,NULL,NULL),(109541,168,17,NULL,NULL,30,NULL,NULL,NULL),(109542,168,19,NULL,NULL,NULL,NULL,1,NULL),(109543,168,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109544,168,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(109545,168,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109546,168,23,NULL,NULL,NULL,NULL,1,NULL),(109547,168,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109548,168,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(109549,168,26,NULL,NULL,2,NULL,NULL,NULL),(109550,168,31,'https://www.ncbi.nlm.nih.gov/biosample?Db=biosample&DbFrom=bioproject&Cmd=Link&LinkName=bioproject_biosample&LinkReadableName=BioSample&ordinalpos=1&IdsFromResult=316889',NULL,NULL,NULL,NULL,NULL),(109551,168,34,NULL,'M. extorquens has 2 degenerated pathways for methylamine breakdown. Without one, it cannot depend on methylamine for Csource. Exp evolution on methylamine discovered that the reason for the degenerate pathways is actually selection, where one is important to deliver the Csource, and the other is needed for Nsource.',NULL,NULL,NULL,NULL),(109552,169,1,'Methylobacterium extorquens',NULL,NULL,NULL,NULL,NULL),(109553,169,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109554,169,3,NULL,NULL,8,NULL,NULL,NULL),(109555,169,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109556,169,5,'flask',NULL,NULL,NULL,NULL,NULL),(109557,169,6,'Hypho minimal medium',NULL,NULL,NULL,NULL,NULL),(109558,169,7,'methanol',NULL,NULL,NULL,NULL,NULL),(109559,169,10,NULL,NULL,NULL,NULL,0,NULL),(109560,169,12,NULL,NULL,NULL,NULL,0,NULL),(109561,169,16,NULL,NULL,600,NULL,NULL,NULL),(109562,169,18,'Color (pink - CM701 vs white - CM702)',NULL,NULL,NULL,NULL,NULL),(109563,169,19,NULL,NULL,NULL,NULL,1,NULL),(109564,169,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109565,169,21,'Fluorescence',NULL,NULL,NULL,NULL,NULL),(109566,169,23,NULL,NULL,NULL,NULL,1,NULL),(109567,169,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(109568,169,25,'MOgene',NULL,NULL,NULL,NULL,NULL),(109569,169,27,NULL,NULL,30,NULL,NULL,NULL),(109570,169,31,'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42116',NULL,NULL,NULL,NULL,NULL),(109571,169,34,NULL,'Changes in gene expression during adaptation consistently restored acclimatizing processes to the wild-type state after introduction of a novel metabolic pathway',NULL,NULL,NULL,NULL),(109572,170,1,'Mycobacterium tuberculosis',NULL,NULL,NULL,NULL,NULL),(109573,170,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109574,170,3,NULL,NULL,10,NULL,NULL,NULL),(109575,170,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109576,170,5,'mice',NULL,NULL,NULL,NULL,NULL),(109577,170,6,'mice',NULL,NULL,NULL,NULL,NULL),(109578,170,7,'mice',NULL,NULL,NULL,NULL,NULL),(109579,170,8,NULL,NULL,NULL,NULL,0,NULL),(109580,170,10,NULL,NULL,NULL,NULL,1,NULL),(109581,170,11,'in situ',NULL,NULL,NULL,NULL,NULL),(109582,170,12,NULL,NULL,NULL,NULL,1,NULL),(109583,170,13,'in situ',NULL,NULL,NULL,NULL,NULL),(109584,170,14,NULL,NULL,NULL,NULL,1,NULL),(109585,170,15,'in situ',NULL,NULL,NULL,NULL,NULL),(109586,170,16,NULL,NULL,108,NULL,NULL,NULL),(109587,170,17,NULL,NULL,294,NULL,NULL,NULL),(109588,170,19,NULL,NULL,NULL,NULL,0,NULL),(109589,170,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109590,170,23,NULL,NULL,NULL,NULL,1,NULL),(109591,170,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109592,170,27,NULL,NULL,20,NULL,NULL,NULL),(109593,170,30,NULL,NULL,NULL,10,NULL,NULL),(109594,170,34,NULL,'Mtb evolves under strong selection for highly beneficial mutants in a mice infection thereby purging a lot of diversity. Presence of T cells however, increase diversity.',NULL,NULL,NULL,NULL),(109595,171,1,'Mycobacterium tuberculosis',NULL,NULL,NULL,NULL,NULL),(109596,171,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109597,171,3,NULL,NULL,1,NULL,NULL,NULL),(109598,171,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109599,171,5,'25cm² flask',NULL,NULL,NULL,NULL,NULL),(109600,171,6,'RPMI',NULL,NULL,NULL,NULL,NULL),(109601,171,7,'macrophage infection',NULL,NULL,NULL,NULL,NULL),(109602,171,8,NULL,NULL,NULL,NULL,0,NULL),(109603,171,10,NULL,NULL,NULL,NULL,0,NULL),(109604,171,12,NULL,NULL,NULL,NULL,0,NULL),(109605,171,14,NULL,NULL,NULL,NULL,1,NULL),(109606,171,15,'in situ',NULL,NULL,NULL,NULL,NULL),(109607,171,16,NULL,NULL,80,NULL,NULL,NULL),(109608,171,17,NULL,NULL,70,NULL,NULL,NULL),(109609,171,19,NULL,NULL,NULL,NULL,0,NULL),(109610,171,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109611,171,23,NULL,NULL,NULL,NULL,1,NULL),(109612,171,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109613,171,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(109614,171,26,NULL,NULL,13,NULL,NULL,NULL),(109615,171,28,NULL,NULL,325,NULL,NULL,NULL),(109616,171,34,NULL,'Evolution of Mtb in macrophages for 80 generations did result in mutations however they did confer any advantage.',NULL,NULL,NULL,NULL),(109617,172,1,'Mycobacterium tuberculosis',NULL,NULL,NULL,NULL,NULL),(109618,172,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109619,172,3,NULL,NULL,40,NULL,NULL,NULL),(109620,172,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109621,172,6,'Middlebrook 7H9 broth',NULL,NULL,NULL,NULL,NULL),(109622,172,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(109623,172,8,NULL,NULL,NULL,NULL,0,NULL),(109624,172,10,NULL,NULL,NULL,NULL,0,NULL),(109625,172,12,NULL,NULL,NULL,NULL,1,NULL),(109626,172,13,'plates',NULL,NULL,NULL,NULL,NULL),(109627,172,14,NULL,NULL,NULL,NULL,0,NULL),(109628,172,17,NULL,NULL,20,NULL,NULL,NULL),(109629,172,19,NULL,NULL,NULL,NULL,1,NULL),(109630,172,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109631,172,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109632,172,23,NULL,NULL,NULL,NULL,1,NULL),(109633,172,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109634,172,25,'NextSeq',NULL,NULL,NULL,NULL,NULL),(109635,172,26,NULL,NULL,5,NULL,NULL,NULL),(109636,172,28,NULL,NULL,20,NULL,NULL,NULL),(109637,172,34,NULL,'Resistant mutants emerge predominantly from surviving persisters which are a reservoir and potentially contain a higher mutation rate',NULL,NULL,NULL,NULL),(109638,173,1,'Myxococcus xanthus',NULL,NULL,NULL,NULL,NULL),(109639,173,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109640,173,12,NULL,NULL,NULL,NULL,1,NULL),(109641,173,13,'plate',NULL,NULL,NULL,NULL,NULL),(109642,174,1,'Myxococcus xanthus',NULL,NULL,NULL,NULL,NULL),(109643,174,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109644,174,3,NULL,NULL,55,NULL,NULL,NULL),(109645,174,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109646,174,5,'plates',NULL,NULL,NULL,NULL,NULL),(109647,174,6,'CTT',NULL,NULL,NULL,NULL,NULL),(109648,174,7,'swarming',NULL,NULL,NULL,NULL,NULL),(109649,174,8,NULL,NULL,NULL,NULL,1,NULL),(109650,174,9,'partially motility deficien',NULL,NULL,NULL,NULL,NULL),(109651,174,10,NULL,NULL,NULL,NULL,0,NULL),(109652,174,12,NULL,NULL,NULL,NULL,1,NULL),(109653,174,13,'plates',NULL,NULL,NULL,NULL,NULL),(109654,174,14,NULL,NULL,NULL,NULL,0,NULL),(109655,174,16,NULL,NULL,341,NULL,NULL,NULL),(109656,174,17,NULL,NULL,680,NULL,NULL,NULL),(109657,174,18,'resistance',NULL,NULL,NULL,NULL,NULL),(109658,174,19,NULL,NULL,NULL,NULL,1,NULL),(109659,174,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109660,174,21,'kanamycin resistance',NULL,NULL,NULL,NULL,NULL),(109661,174,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109662,174,23,NULL,NULL,NULL,NULL,1,NULL),(109663,174,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109664,174,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(109665,174,26,NULL,NULL,55,NULL,NULL,NULL),(109666,174,28,NULL,NULL,100,NULL,NULL,NULL),(109667,174,34,NULL,'By selecting on increased swarming, most evolve increased swarming but by a variety of means, some only show facultative swarming and others stick to competitive interference.',NULL,NULL,NULL,NULL),(109668,175,1,'Myxococcus xanthus',NULL,NULL,NULL,NULL,NULL),(109669,175,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109670,175,12,NULL,NULL,NULL,NULL,1,NULL),(109671,175,13,'plate',NULL,NULL,NULL,NULL,NULL),(109672,176,1,'Neurospora crassa',NULL,NULL,NULL,NULL,NULL),(109673,176,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109674,176,3,NULL,NULL,12,NULL,NULL,NULL),(109675,176,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109676,176,5,'glass tubes',NULL,NULL,NULL,NULL,NULL),(109677,176,6,'VM agar',NULL,NULL,NULL,NULL,NULL),(109678,176,7,'high salinity',NULL,NULL,NULL,NULL,NULL),(109679,176,7,'low temperature',NULL,NULL,NULL,NULL,NULL),(109680,176,8,NULL,NULL,NULL,NULL,0,NULL),(109681,176,10,NULL,NULL,NULL,NULL,0,NULL),(109682,176,12,NULL,NULL,NULL,NULL,1,NULL),(109683,176,13,'solid medium strip',NULL,NULL,NULL,NULL,NULL),(109684,176,14,NULL,NULL,NULL,NULL,0,NULL),(109685,176,16,NULL,NULL,1507,NULL,NULL,NULL),(109686,176,19,NULL,NULL,NULL,NULL,1,NULL),(109687,176,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109688,176,23,NULL,NULL,NULL,NULL,0,NULL),(109689,176,34,NULL,'By divergent evolution, N. crassa specializes and isolates its reproduction.',NULL,NULL,NULL,NULL),(109690,177,1,'Nodularia sp.',NULL,NULL,NULL,NULL,NULL),(109691,177,1,'vB_NpeS-2AV2',NULL,NULL,NULL,NULL,NULL),(109692,177,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109693,177,3,NULL,NULL,6,NULL,NULL,NULL),(109694,177,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109695,177,5,'falcons',NULL,NULL,NULL,NULL,NULL),(109696,177,6,'Z8 medium',NULL,NULL,NULL,NULL,NULL),(109697,177,7,'phage exposure',NULL,NULL,NULL,NULL,NULL),(109698,177,8,NULL,NULL,NULL,NULL,0,NULL),(109699,177,10,NULL,NULL,NULL,NULL,0,NULL),(109700,177,12,NULL,NULL,NULL,NULL,1,NULL),(109701,177,13,'static',NULL,NULL,NULL,NULL,NULL),(109702,177,14,NULL,NULL,NULL,NULL,0,NULL),(109703,177,17,NULL,NULL,154,NULL,NULL,NULL),(109704,177,19,NULL,NULL,NULL,NULL,0,NULL),(109705,177,23,NULL,NULL,NULL,NULL,0,NULL),(109706,177,34,NULL,'Phage exposure to cyanobacteria leads to increased N release but also resistance which might contain a tradeoff cost as it involves lower buoyancy.',NULL,NULL,NULL,NULL),(109707,178,1,'Oenococcus oeni',NULL,NULL,NULL,NULL,NULL),(109708,178,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109709,178,4,'continuous',NULL,NULL,NULL,NULL,NULL),(109710,178,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(109711,179,1,'Phage phi6',NULL,NULL,NULL,NULL,NULL),(109712,179,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109713,179,3,NULL,NULL,18,NULL,NULL,NULL),(109714,179,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109715,179,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109716,179,6,'LC medium',NULL,NULL,NULL,NULL,NULL),(109717,179,7,'Pseudomonas syringae pathovar phaseolicola (permissive host) mixed with P. pseudoalcaligenes pathovar ERA (non-permissive host) at different population sizes',NULL,NULL,NULL,NULL,NULL),(109718,179,10,NULL,NULL,NULL,NULL,0,NULL),(109719,179,12,NULL,NULL,NULL,NULL,0,NULL),(109720,179,16,NULL,NULL,300,NULL,NULL,NULL),(109721,179,17,NULL,NULL,100,NULL,NULL,NULL),(109722,179,19,NULL,NULL,NULL,NULL,1,NULL),(109723,179,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109724,179,21,'plating on permissive',NULL,NULL,NULL,NULL,NULL),(109725,179,21,'non permissive hosts',NULL,NULL,NULL,NULL,NULL),(109726,179,23,NULL,NULL,NULL,NULL,0,NULL),(109727,179,32,NULL,NULL,10000000,NULL,NULL,NULL),(109728,179,33,NULL,NULL,100000,NULL,NULL,NULL),(109729,179,34,NULL,'While mixed host conditions selected initially for generalists take-over, lowering the percentage of new hosts allowed for the specialist to sustain itself',NULL,NULL,NULL,NULL),(109730,180,1,'Phage phi6',NULL,NULL,NULL,NULL,NULL),(109731,180,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109732,180,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109733,181,1,'Phage phi6',NULL,NULL,NULL,NULL,NULL),(109734,181,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109735,181,3,NULL,NULL,6,NULL,NULL,NULL),(109736,181,3,NULL,NULL,60,NULL,NULL,NULL),(109737,181,4,'MA',NULL,NULL,NULL,NULL,NULL),(109738,181,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109739,181,7,'MOI infection',NULL,NULL,NULL,NULL,NULL),(109740,181,7,'plaque forming on agar',NULL,NULL,NULL,NULL,NULL),(109741,181,16,NULL,NULL,50,NULL,NULL,NULL),(109742,181,16,NULL,NULL,100,NULL,NULL,NULL),(109743,181,17,NULL,NULL,20,NULL,NULL,NULL),(109744,181,19,NULL,NULL,NULL,NULL,1,NULL),(109745,181,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109746,181,21,'ability to infect P. pseudoalcaligenes',NULL,NULL,NULL,NULL,NULL),(109747,181,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(109748,181,23,NULL,NULL,NULL,NULL,0,NULL),(109749,181,32,NULL,NULL,50000000000,NULL,NULL,NULL),(109750,181,32,NULL,NULL,100000000,NULL,NULL,NULL),(109751,181,33,NULL,NULL,500,NULL,NULL,NULL),(109752,181,33,NULL,NULL,1,NULL,NULL,NULL),(109753,181,34,NULL,'Selection for robustness is relaxed under virus co-infection and resulted in higher mutation frequencies',NULL,NULL,NULL,NULL),(109754,182,1,'Phage phiX174',NULL,NULL,NULL,NULL,NULL),(109755,182,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109756,182,3,NULL,NULL,19,NULL,NULL,NULL),(109757,182,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109758,182,7,'infection of E. coli mutT with mutagenization using 250 mM hydroxylamine',NULL,NULL,NULL,NULL,NULL),(109759,182,17,NULL,NULL,145,NULL,NULL,NULL),(109760,182,18,'parallelism of mutations',NULL,NULL,NULL,NULL,NULL),(109761,182,19,NULL,NULL,NULL,NULL,1,NULL),(109762,182,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109763,182,21,'ability to infect P. pseudoalcaligenes',NULL,NULL,NULL,NULL,NULL),(109764,182,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109765,182,23,NULL,NULL,NULL,NULL,1,NULL),(109766,182,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(109767,182,25,'ABI sequencer',NULL,NULL,NULL,NULL,NULL),(109768,182,26,NULL,NULL,23,NULL,NULL,NULL),(109769,182,31,'https://www.ncbi.nlm.nih.gov/nuccore/EF380009',NULL,NULL,NULL,NULL,NULL),(109770,182,33,NULL,NULL,3,NULL,NULL,NULL),(109771,182,34,NULL,'Fitness under drift declines but reaches a plateau in time, and this because a drastic increase in the rate of beneificial mutations as fitness decreases.',NULL,NULL,NULL,NULL),(109772,183,1,'Phage phiX174',NULL,NULL,NULL,NULL,NULL),(109773,183,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109774,183,3,NULL,NULL,9,NULL,NULL,NULL),(109775,183,4,'continuous',NULL,NULL,NULL,NULL,NULL),(109776,183,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(109777,183,6,'LB',NULL,NULL,NULL,NULL,NULL),(109778,183,7,'hosts',NULL,NULL,NULL,NULL,NULL),(109779,183,7,'elevated temperature',NULL,NULL,NULL,NULL,NULL),(109780,183,8,NULL,NULL,NULL,NULL,0,NULL),(109781,183,10,NULL,NULL,NULL,NULL,1,NULL),(109782,183,11,'gradual temperature increase',NULL,NULL,NULL,NULL,NULL),(109783,183,12,NULL,NULL,NULL,NULL,0,NULL),(109784,183,14,NULL,NULL,NULL,NULL,0,NULL),(109785,183,17,NULL,NULL,22,NULL,NULL,NULL),(109786,183,18,'no parallel evolution runs',NULL,NULL,NULL,NULL,NULL),(109787,183,18,'runs at different locations',NULL,NULL,NULL,NULL,NULL),(109788,183,19,NULL,NULL,NULL,NULL,1,NULL),(109789,183,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109790,183,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109791,183,23,NULL,NULL,NULL,NULL,1,NULL),(109792,183,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(109793,183,25,'ABI377',NULL,NULL,NULL,NULL,NULL),(109794,183,27,NULL,NULL,9,NULL,NULL,NULL),(109795,183,34,NULL,'Evolution to increased temperature revealed highly parallel genetic and phenotypic evolution',NULL,NULL,NULL,NULL),(109796,184,1,'pMS0506',NULL,NULL,NULL,NULL,NULL),(109797,184,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109798,184,3,NULL,NULL,24,NULL,NULL,NULL),(109799,184,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109800,184,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109801,184,6,'LB',NULL,NULL,NULL,NULL,NULL),(109802,184,7,'host adaptation',NULL,NULL,NULL,NULL,NULL),(109803,184,8,NULL,NULL,NULL,NULL,0,NULL),(109804,184,10,NULL,NULL,NULL,NULL,0,NULL),(109805,184,12,NULL,NULL,NULL,NULL,0,NULL),(109806,184,14,NULL,NULL,NULL,NULL,0,NULL),(109807,184,16,NULL,NULL,1000,NULL,NULL,NULL),(109808,184,17,NULL,NULL,100,NULL,NULL,NULL),(109809,184,18,'antibiotc resistance',NULL,NULL,NULL,NULL,NULL),(109810,184,19,NULL,NULL,NULL,NULL,1,NULL),(109811,184,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(109812,184,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(109813,184,23,NULL,NULL,NULL,NULL,0,NULL),(109814,184,34,NULL,'Mutations in the plasmid itselve improve stability and limit costs in S. odiensis, became high copy but could no longer replicate in P. aeruginosa.',NULL,NULL,NULL,NULL),(109815,184,35,NULL,'follow up on molecular mechanisms: Yano H, Wegrzyn K, Loftie-Eaton W, Johnson J, Deckert GE, Rogers LM, Konieczny I, Top EM. 2016. Evolved plasmid-host interactions reduce plasmid interference cost. Mol Microbiol 101:743–756.',NULL,NULL,NULL,NULL),(109816,185,1,'pQBR57',NULL,NULL,NULL,NULL,NULL),(109817,185,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109818,185,3,NULL,NULL,18,NULL,NULL,NULL),(109819,185,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109820,185,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(109821,185,7,'host conjugation transfer',NULL,NULL,NULL,NULL,NULL),(109822,185,8,NULL,NULL,NULL,NULL,0,NULL),(109823,185,10,NULL,NULL,NULL,NULL,1,NULL),(109824,185,11,'cycling between hosts',NULL,NULL,NULL,NULL,NULL),(109825,185,12,NULL,NULL,NULL,NULL,1,NULL),(109826,185,13,'conjugation plates',NULL,NULL,NULL,NULL,NULL),(109827,185,14,NULL,NULL,NULL,NULL,0,NULL),(109828,185,17,NULL,NULL,72,NULL,NULL,NULL),(109829,185,18,'resistance',NULL,NULL,NULL,NULL,NULL),(109830,185,19,NULL,NULL,NULL,NULL,1,NULL),(109831,185,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109832,185,23,NULL,NULL,NULL,NULL,0,NULL),(109833,185,34,NULL,'Higher conjugation rates evolve in plasmids which also become less costly on their host but more costly on an alternative host, while this is not true for evolution to alternating hosts.',NULL,NULL,NULL,NULL),(109834,186,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109835,186,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109836,186,4,'continuous',NULL,NULL,NULL,NULL,NULL),(109837,186,5,'morbidostat',NULL,NULL,NULL,NULL,NULL),(109838,187,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109839,187,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109840,187,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109841,187,10,NULL,NULL,NULL,NULL,1,NULL),(109842,187,11,'static',NULL,NULL,NULL,NULL,NULL),(109843,188,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109844,188,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109845,188,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109846,188,7,'aztreonam resistance',NULL,NULL,NULL,NULL,NULL),(109847,188,10,NULL,NULL,NULL,NULL,1,NULL),(109848,188,11,'cyclic treatments',NULL,NULL,NULL,NULL,NULL),(109849,189,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109850,189,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109851,189,3,NULL,NULL,24,NULL,NULL,NULL),(109852,189,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109853,189,5,'microcosms',NULL,NULL,NULL,NULL,NULL),(109854,189,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(109855,189,7,'static conditions',NULL,NULL,NULL,NULL,NULL),(109856,189,8,NULL,NULL,NULL,NULL,1,NULL),(109857,189,9,'wss mutant unable to form mats through the cellulose pathway',NULL,NULL,NULL,NULL,NULL),(109858,189,10,NULL,NULL,NULL,NULL,0,NULL),(109859,189,12,NULL,NULL,NULL,NULL,1,NULL),(109860,189,13,'static',NULL,NULL,NULL,NULL,NULL),(109861,189,14,NULL,NULL,NULL,NULL,0,NULL),(109862,189,17,NULL,NULL,5,NULL,NULL,NULL),(109863,189,19,NULL,NULL,NULL,NULL,1,NULL),(109864,189,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109865,189,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(109866,189,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109867,189,23,NULL,NULL,NULL,NULL,0,NULL),(109868,189,34,NULL,'By deleting the wss operon, evolution had to find other ways to evolve mat forming under radiation in static conditions',NULL,NULL,NULL,NULL),(109869,190,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109870,190,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109871,191,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109872,191,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(109873,191,4,'continuous',NULL,NULL,NULL,NULL,NULL),(109874,191,5,'bioreactor',NULL,NULL,NULL,NULL,NULL),(109875,192,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109876,192,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109877,193,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109878,193,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109879,193,3,NULL,NULL,132,NULL,NULL,NULL),(109880,193,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109881,193,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(109882,193,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(109883,193,7,'phage predation',NULL,NULL,NULL,NULL,NULL),(109884,193,8,NULL,NULL,NULL,NULL,0,NULL),(109885,193,10,NULL,NULL,NULL,NULL,0,NULL),(109886,193,12,NULL,NULL,NULL,NULL,0,NULL),(109887,193,14,NULL,NULL,NULL,NULL,0,NULL),(109888,193,17,NULL,NULL,1,NULL,NULL,NULL),(109889,193,19,NULL,NULL,NULL,NULL,0,NULL),(109890,193,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109891,193,23,NULL,NULL,NULL,NULL,1,NULL),(109892,193,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109893,193,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(109894,193,26,NULL,NULL,18,NULL,NULL,NULL),(109895,193,34,NULL,'Multiphage predation leads to stroner population decrease but also to greater bacterial recovery because of resistance which was more general than the specific resistance in the monophage exposed populations.',NULL,NULL,NULL,NULL),(109896,194,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109897,194,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109898,194,3,NULL,NULL,36,NULL,NULL,NULL),(109899,194,4,'MA',NULL,NULL,NULL,NULL,NULL),(109900,194,5,'plates',NULL,NULL,NULL,NULL,NULL),(109901,194,6,'LB',NULL,NULL,NULL,NULL,NULL),(109902,194,7,'MA',NULL,NULL,NULL,NULL,NULL),(109903,194,10,NULL,NULL,NULL,NULL,0,NULL),(109904,194,12,NULL,NULL,NULL,NULL,0,NULL),(109905,194,16,NULL,NULL,2500,NULL,NULL,NULL),(109906,194,17,NULL,NULL,90,NULL,NULL,NULL),(109907,194,18,'colony morphotype',NULL,NULL,NULL,NULL,NULL),(109908,194,19,NULL,NULL,NULL,NULL,0,NULL),(109909,194,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109910,194,23,NULL,NULL,NULL,NULL,1,NULL),(109911,194,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109912,194,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(109913,194,26,NULL,NULL,36,NULL,NULL,NULL),(109914,194,28,NULL,NULL,58,NULL,NULL,NULL),(109915,194,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA302277',NULL,NULL,NULL,NULL,NULL),(109916,194,33,NULL,NULL,1,NULL,NULL,NULL),(109917,194,34,NULL,'Description of how spontaneous mutations are affected by location, mutation rate, ... in P. aeruginosa',NULL,NULL,NULL,NULL),(109918,195,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109919,195,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109920,195,3,NULL,NULL,6,NULL,NULL,NULL),(109921,195,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109922,195,5,'tubes',NULL,NULL,NULL,NULL,NULL),(109923,195,6,'M63',NULL,NULL,NULL,NULL,NULL),(109924,195,7,'dispersal',NULL,NULL,NULL,NULL,NULL),(109925,195,7,'reattachement',NULL,NULL,NULL,NULL,NULL),(109926,195,8,NULL,NULL,NULL,NULL,0,NULL),(109927,195,10,NULL,NULL,NULL,NULL,0,NULL),(109928,195,12,NULL,NULL,NULL,NULL,1,NULL),(109929,195,13,'polystyreen beads',NULL,NULL,NULL,NULL,NULL),(109930,195,14,NULL,NULL,NULL,NULL,0,NULL),(109931,195,16,NULL,NULL,600,NULL,NULL,NULL),(109932,195,17,NULL,NULL,90,NULL,NULL,NULL),(109933,195,19,NULL,NULL,NULL,NULL,1,NULL),(109934,195,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109935,195,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(109936,195,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109937,195,23,NULL,NULL,NULL,NULL,1,NULL),(109938,195,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109939,195,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(109940,195,27,NULL,NULL,27,NULL,NULL,NULL),(109941,195,31,'https://www.ncbi.nlm.nih.gov/bioproject/315478',NULL,NULL,NULL,NULL,NULL),(109942,195,34,NULL,'P. aeruginosa rapidly diversifies under selection of dispersal and reattachment to beads as a biofilm model. Biofilm production increases because of the emerged diversity, especially in cidigmp levels',NULL,NULL,NULL,NULL),(109943,196,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109944,196,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109945,196,3,NULL,NULL,140,NULL,NULL,NULL),(109946,196,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109947,196,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(109948,196,6,'M9 serine',NULL,NULL,NULL,NULL,NULL),(109949,196,7,'adaptation to serine limitation',NULL,NULL,NULL,NULL,NULL),(109950,196,8,NULL,NULL,NULL,NULL,1,NULL),(109951,196,9,'rpoB',NULL,NULL,NULL,NULL,NULL),(109952,196,9,'type IV pilli',NULL,NULL,NULL,NULL,NULL),(109953,196,10,NULL,NULL,NULL,NULL,0,NULL),(109954,196,12,NULL,NULL,NULL,NULL,0,NULL),(109955,196,14,NULL,NULL,NULL,NULL,0,NULL),(109956,196,16,NULL,NULL,342,NULL,NULL,NULL),(109957,196,17,NULL,NULL,45,NULL,NULL,NULL),(109958,196,19,NULL,NULL,NULL,NULL,1,NULL),(109959,196,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109960,196,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(109961,196,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(109962,196,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(109963,196,23,NULL,NULL,NULL,NULL,1,NULL),(109964,196,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(109965,196,25,'Hiseq2000',NULL,NULL,NULL,NULL,NULL),(109966,196,26,NULL,NULL,80,NULL,NULL,NULL),(109967,196,28,NULL,NULL,80,NULL,NULL,NULL),(109968,196,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP053291',NULL,NULL,NULL,NULL,NULL),(109969,196,34,NULL,'Magnitute epistasis without being diminishing returns epistasis can have profound effects on evolvability as one mutant reaches higher fitness with accumulating similar mutations.',NULL,NULL,NULL,NULL),(109970,197,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109971,197,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(109972,197,3,NULL,NULL,20,NULL,NULL,NULL),(109973,197,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109974,197,5,'24-well plate',NULL,NULL,NULL,NULL,NULL),(109975,197,6,'CAA media',NULL,NULL,NULL,NULL,NULL),(109976,197,7,'iron limitation',NULL,NULL,NULL,NULL,NULL),(109977,197,10,NULL,NULL,NULL,NULL,0,NULL),(109978,197,12,NULL,NULL,NULL,NULL,0,NULL),(109979,197,16,NULL,NULL,50,NULL,NULL,NULL),(109980,197,17,NULL,NULL,10,NULL,NULL,NULL),(109981,197,18,'empty wells',NULL,NULL,NULL,NULL,NULL),(109982,197,19,NULL,NULL,NULL,NULL,1,NULL),(109983,197,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(109984,197,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(109985,197,23,NULL,NULL,NULL,NULL,0,NULL),(109986,197,34,NULL,'Loner strains that outcompete defectors of social cooperation to uptake iron stabilize the community in a rock paper scissor game',NULL,NULL,NULL,NULL),(109987,198,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(109988,198,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(109989,198,3,NULL,NULL,24,NULL,NULL,NULL),(109990,198,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(109991,198,5,'glass universals',NULL,NULL,NULL,NULL,NULL),(109992,198,6,'casamino acids medium',NULL,NULL,NULL,NULL,NULL),(109993,198,7,'pyocin S2',NULL,NULL,NULL,NULL,NULL),(109994,198,8,NULL,NULL,NULL,NULL,0,NULL),(109995,198,10,NULL,NULL,NULL,NULL,0,NULL),(109996,198,12,NULL,NULL,NULL,NULL,0,NULL),(109997,198,14,NULL,NULL,NULL,NULL,1,NULL),(109998,198,15,'simultaneous presense of 2 strains',NULL,NULL,NULL,NULL,NULL),(109999,198,15,'iron limitation during pyocin S2 presence',NULL,NULL,NULL,NULL,NULL),(110000,198,16,NULL,NULL,70,NULL,NULL,NULL),(110001,198,17,NULL,NULL,40,NULL,NULL,NULL),(110002,198,19,NULL,NULL,NULL,NULL,0,NULL),(110003,198,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(110004,198,23,NULL,NULL,NULL,NULL,0,NULL),(110005,198,33,NULL,NULL,100,NULL,NULL,NULL),(110006,198,34,NULL,'P. aeruginosa can adapt and become resistant to natural antimicrobials but this highly depends on the conditions, specifically whether iron is present or not',NULL,NULL,NULL,NULL),(110007,199,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110008,199,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110009,199,3,NULL,NULL,21,NULL,NULL,NULL),(110010,199,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110011,199,5,'tubes',NULL,NULL,NULL,NULL,NULL),(110012,199,6,'LB',NULL,NULL,NULL,NULL,NULL),(110013,199,7,'colistin',NULL,NULL,NULL,NULL,NULL),(110014,199,8,NULL,NULL,NULL,NULL,0,NULL),(110015,199,10,NULL,NULL,NULL,NULL,1,NULL),(110016,199,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(110017,199,12,NULL,NULL,NULL,NULL,0,NULL),(110018,199,14,NULL,NULL,NULL,NULL,0,NULL),(110019,199,17,NULL,NULL,31,NULL,NULL,NULL),(110020,199,19,NULL,NULL,NULL,NULL,1,NULL),(110021,199,20,'MIC',NULL,NULL,NULL,NULL,NULL),(110022,199,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110023,199,23,NULL,NULL,NULL,NULL,1,NULL),(110024,199,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110025,199,25,'GAII',NULL,NULL,NULL,NULL,NULL),(110026,199,26,NULL,NULL,14,NULL,NULL,NULL),(110027,199,31,'https://www.ncbi.nlm.nih.gov/sra?term=ERP002381',NULL,NULL,NULL,NULL,NULL),(110028,199,34,NULL,'Evolution of colistin resistance is determined by strong synergistic interactions between multiple mutations.',NULL,NULL,NULL,NULL),(110029,200,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110030,200,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110031,200,3,NULL,NULL,36,NULL,NULL,NULL),(110032,200,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110033,200,6,'M9KB',NULL,NULL,NULL,NULL,NULL),(110034,200,7,'adaptation to cost of rifampicin resistance in rpoB',NULL,NULL,NULL,NULL,NULL),(110035,200,8,NULL,NULL,NULL,NULL,1,NULL),(110036,200,9,'8 rpoB mutants',NULL,NULL,NULL,NULL,NULL),(110037,200,10,NULL,NULL,NULL,NULL,0,NULL),(110038,200,12,NULL,NULL,NULL,NULL,0,NULL),(110039,200,14,NULL,NULL,NULL,NULL,0,NULL),(110040,200,16,NULL,NULL,300,NULL,NULL,NULL),(110041,200,17,NULL,NULL,30,NULL,NULL,NULL),(110042,200,19,NULL,NULL,NULL,NULL,1,NULL),(110043,200,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110044,200,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110045,200,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110046,200,23,NULL,NULL,NULL,NULL,1,NULL),(110047,200,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110048,200,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110049,200,26,NULL,NULL,204,NULL,NULL,NULL),(110050,200,28,NULL,NULL,44,NULL,NULL,NULL),(110051,200,31,'https://www.ebi.ac.uk/ena/data/view/PRJEB11978',NULL,NULL,NULL,NULL,NULL),(110052,200,34,NULL,'Costly resistance mutations get compensated while general adaptation occurs for less costly resistance mutations',NULL,NULL,NULL,NULL),(110053,201,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110054,201,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110055,201,3,NULL,NULL,144,NULL,NULL,NULL),(110056,201,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110057,201,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(110058,201,6,'King\'s B',NULL,NULL,NULL,NULL,NULL),(110059,201,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(110060,201,7,'/or phage LKD16',NULL,NULL,NULL,NULL,NULL),(110061,201,10,NULL,NULL,NULL,NULL,0,NULL),(110062,201,12,NULL,NULL,NULL,NULL,0,NULL),(110063,201,17,NULL,NULL,7,NULL,NULL,NULL),(110064,201,18,'control wells',NULL,NULL,NULL,NULL,NULL),(110065,201,19,NULL,NULL,NULL,NULL,0,NULL),(110066,201,23,NULL,NULL,NULL,NULL,0,NULL),(110067,201,34,NULL,'Combination of phages and antibiotics can slow down resistance evolution, especially towards antibiotics',NULL,NULL,NULL,NULL),(110068,202,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110069,202,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110070,202,3,NULL,NULL,12,NULL,NULL,NULL),(110071,202,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110072,202,5,'flasks',NULL,NULL,NULL,NULL,NULL),(110073,202,6,'LB',NULL,NULL,NULL,NULL,NULL),(110074,202,7,'ciprofloxacin antibiotic',NULL,NULL,NULL,NULL,NULL),(110075,202,8,NULL,NULL,NULL,NULL,0,NULL),(110076,202,10,NULL,NULL,NULL,NULL,0,NULL),(110077,202,12,NULL,NULL,NULL,NULL,0,NULL),(110078,202,14,NULL,NULL,NULL,NULL,0,NULL),(110079,202,16,NULL,NULL,940,NULL,NULL,NULL),(110080,202,17,NULL,NULL,94,NULL,NULL,NULL),(110081,202,19,NULL,NULL,NULL,NULL,1,NULL),(110082,202,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110083,202,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(110084,202,23,NULL,NULL,NULL,NULL,0,NULL),(110085,202,34,NULL,'Evolution in sub mix ciproflox concentrations leads to downregulation of the type III secretion system and upregulation of denitrification genes in ciprofloxacin-evolved populations.',NULL,NULL,NULL,NULL),(110086,203,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110087,203,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110088,203,3,NULL,NULL,103,NULL,NULL,NULL),(110089,203,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110090,203,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(110091,203,6,'LB',NULL,NULL,NULL,NULL,NULL),(110092,203,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(110093,203,8,NULL,NULL,NULL,NULL,0,NULL),(110094,203,10,NULL,NULL,NULL,NULL,1,NULL),(110095,203,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(110096,203,11,'changing antibiotic',NULL,NULL,NULL,NULL,NULL),(110097,203,14,NULL,NULL,NULL,NULL,0,NULL),(110098,203,17,NULL,NULL,40,NULL,NULL,NULL),(110099,203,19,NULL,NULL,NULL,NULL,0,NULL),(110100,203,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110101,203,23,NULL,NULL,NULL,NULL,1,NULL),(110102,203,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110103,203,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(110104,203,26,NULL,NULL,57,NULL,NULL,NULL),(110105,203,28,NULL,NULL,759,NULL,NULL,NULL),(110106,203,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA376615',NULL,NULL,NULL,NULL,NULL),(110107,203,34,NULL,'The evolution of antibiotic resistance is not only dependent on future treatments but also on previous treatments as some previous treatments were shown to limit future resistance to other antibiotics or future adaptation reverted initial resistances',NULL,NULL,NULL,NULL),(110108,204,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110109,204,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110110,204,3,NULL,NULL,16,NULL,NULL,NULL),(110111,204,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110112,204,5,'24-well plate',NULL,NULL,NULL,NULL,NULL),(110113,204,6,'iron-depleted CAA medium',NULL,NULL,NULL,NULL,NULL),(110114,204,7,'co-evolution between cooperators',NULL,NULL,NULL,NULL,NULL),(110115,204,7,'cheats',NULL,NULL,NULL,NULL,NULL),(110116,204,8,NULL,NULL,NULL,NULL,0,NULL),(110117,204,10,NULL,NULL,NULL,NULL,0,NULL),(110118,204,12,NULL,NULL,NULL,NULL,1,NULL),(110119,204,13,'static incubation',NULL,NULL,NULL,NULL,NULL),(110120,204,14,NULL,NULL,NULL,NULL,0,NULL),(110121,204,16,NULL,NULL,150,NULL,NULL,NULL),(110122,204,17,NULL,NULL,25,NULL,NULL,NULL),(110123,204,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(110124,204,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110125,204,19,NULL,NULL,NULL,NULL,1,NULL),(110126,204,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110127,204,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110128,204,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110129,204,23,NULL,NULL,NULL,NULL,1,NULL),(110130,204,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110131,204,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110132,204,26,NULL,NULL,195,NULL,NULL,NULL),(110133,204,28,NULL,NULL,222,NULL,NULL,NULL),(110134,204,30,NULL,NULL,NULL,5,NULL,NULL),(110135,204,33,NULL,NULL,1000000,NULL,NULL,NULL),(110136,204,34,NULL,'Co-evolution dynamics occur when cheats and cooperators evolved in a way that cooperators tried to limit exploitation and cheats tried to maximize exploitation',NULL,NULL,NULL,NULL),(110137,205,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110138,205,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110139,205,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110140,206,1,'Acinetobacter spp.',NULL,NULL,NULL,NULL,NULL),(110141,206,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110142,206,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110143,206,3,NULL,NULL,144,NULL,NULL,NULL),(110144,206,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110145,206,5,'plates',NULL,NULL,NULL,NULL,NULL),(110146,206,6,'MHB',NULL,NULL,NULL,NULL,NULL),(110147,206,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(110148,206,8,NULL,NULL,NULL,NULL,0,NULL),(110149,206,10,NULL,NULL,NULL,NULL,1,NULL),(110150,206,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(110151,206,12,NULL,NULL,NULL,NULL,1,NULL),(110152,206,13,'agar plates',NULL,NULL,NULL,NULL,NULL),(110153,206,14,NULL,NULL,NULL,NULL,0,NULL),(110154,206,17,NULL,NULL,10,NULL,NULL,NULL),(110155,206,19,NULL,NULL,NULL,NULL,0,NULL),(110156,206,23,NULL,NULL,NULL,NULL,0,NULL),(110157,206,34,NULL,'Antibiotic combinations decreased resistance development both in serial transfer long-term adaptation on plates as well as by decreasing spontaneous resistant mutants.',NULL,NULL,NULL,NULL),(110158,207,1,'DMS3vir',NULL,NULL,NULL,NULL,NULL),(110159,207,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110160,207,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110161,207,3,NULL,NULL,12,NULL,NULL,NULL),(110162,207,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110163,207,5,'tubes',NULL,NULL,NULL,NULL,NULL),(110164,207,6,'M9',NULL,NULL,NULL,NULL,NULL),(110165,207,7,'co-evoluton under migration',NULL,NULL,NULL,NULL,NULL),(110166,207,8,NULL,NULL,NULL,NULL,0,NULL),(110167,207,10,NULL,NULL,NULL,NULL,0,NULL),(110168,207,12,NULL,NULL,NULL,NULL,0,NULL),(110169,207,14,NULL,NULL,NULL,NULL,1,NULL),(110170,207,15,'inlcusion of immigration of naive hosts',NULL,NULL,NULL,NULL,NULL),(110171,207,17,NULL,NULL,35,NULL,NULL,NULL),(110172,207,19,NULL,NULL,NULL,NULL,1,NULL),(110173,207,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110174,207,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(110175,207,23,NULL,NULL,NULL,NULL,0,NULL),(110176,207,33,NULL,NULL,1000000,NULL,NULL,NULL),(110177,207,34,NULL,'Increased immigration of naive hosts rescues phage extinction and increases selection towards CRISPR-resistant hosts so that costly suface-associated resistance mechanisms become beneficial and selected for',NULL,NULL,NULL,NULL),(110178,208,1,'DMS3vir',NULL,NULL,NULL,NULL,NULL),(110179,208,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110180,208,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110181,208,3,NULL,NULL,12,NULL,NULL,NULL),(110182,208,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110183,208,5,'glass vials',NULL,NULL,NULL,NULL,NULL),(110184,208,6,'M9 glucose',NULL,NULL,NULL,NULL,NULL),(110185,208,7,'coevolution',NULL,NULL,NULL,NULL,NULL),(110186,208,8,NULL,NULL,NULL,NULL,0,NULL),(110187,208,10,NULL,NULL,NULL,NULL,1,NULL),(110188,208,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(110189,208,12,NULL,NULL,NULL,NULL,0,NULL),(110190,208,14,NULL,NULL,NULL,NULL,0,NULL),(110191,208,17,NULL,NULL,5,NULL,NULL,NULL),(110192,208,19,NULL,NULL,NULL,NULL,0,NULL),(110193,208,23,NULL,NULL,NULL,NULL,0,NULL),(110194,208,34,NULL,'If host is highly diverse, local adaptation for a predator has no use.',NULL,NULL,NULL,NULL),(110195,209,1,'Acinetobacter baumannii',NULL,NULL,NULL,NULL,NULL),(110196,209,1,'Enterobacter aerogenes',NULL,NULL,NULL,NULL,NULL),(110197,209,1,'Enterococcus faecium',NULL,NULL,NULL,NULL,NULL),(110198,209,1,'Klebsiella pneumoniae',NULL,NULL,NULL,NULL,NULL),(110199,209,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110200,209,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(110201,209,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110202,209,3,NULL,NULL,12,NULL,NULL,NULL),(110203,209,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110204,209,5,'flasks',NULL,NULL,NULL,NULL,NULL),(110205,209,6,'MHB',NULL,NULL,NULL,NULL,NULL),(110206,209,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(110207,209,8,NULL,NULL,NULL,NULL,0,NULL),(110208,209,10,NULL,NULL,NULL,NULL,1,NULL),(110209,209,11,'periodically changing between high antibiotic dose',NULL,NULL,NULL,NULL,NULL),(110210,209,11,'no dose',NULL,NULL,NULL,NULL,NULL),(110211,209,12,NULL,NULL,NULL,NULL,0,NULL),(110212,209,14,NULL,NULL,NULL,NULL,0,NULL),(110213,209,17,NULL,NULL,20,NULL,NULL,NULL),(110214,209,19,NULL,NULL,NULL,NULL,0,NULL),(110215,209,23,NULL,NULL,NULL,NULL,0,NULL),(110216,209,32,NULL,NULL,100000000000,NULL,NULL,NULL),(110217,209,34,NULL,'Persistence rapidly evolves in ESKAPE pathogens to high levels under daily treatment.',NULL,NULL,NULL,NULL),(110218,210,1,'Phage phi2-3-4',NULL,NULL,NULL,NULL,NULL),(110219,210,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(110220,210,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110221,210,3,NULL,NULL,12,NULL,NULL,NULL),(110222,210,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110223,210,5,'glass bottles',NULL,NULL,NULL,NULL,NULL),(110224,210,6,'artificial sputum medium',NULL,NULL,NULL,NULL,NULL),(110225,210,7,'adaptation to biofilm growth',NULL,NULL,NULL,NULL,NULL),(110226,210,8,NULL,NULL,NULL,NULL,0,NULL),(110227,210,10,NULL,NULL,NULL,NULL,0,NULL),(110228,210,12,NULL,NULL,NULL,NULL,1,NULL),(110229,210,13,'biofilms',NULL,NULL,NULL,NULL,NULL),(110230,210,14,NULL,NULL,NULL,NULL,1,NULL),(110231,210,15,'ASM medium',NULL,NULL,NULL,NULL,NULL),(110232,210,15,'natural temperate phages',NULL,NULL,NULL,NULL,NULL),(110233,210,16,NULL,NULL,240,NULL,NULL,NULL),(110234,210,17,NULL,NULL,120,NULL,NULL,NULL),(110235,210,19,NULL,NULL,NULL,NULL,1,NULL),(110236,210,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110237,210,21,'gentemycin',NULL,NULL,NULL,NULL,NULL),(110238,210,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110239,210,23,NULL,NULL,NULL,NULL,1,NULL),(110240,210,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110241,210,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110242,210,26,NULL,NULL,480,NULL,NULL,NULL),(110243,210,30,NULL,NULL,NULL,2.5,NULL,NULL),(110244,210,31,'https://www.ebi.ac.uk/ena/data/view/PRJEB9801',NULL,NULL,NULL,NULL,NULL),(110245,210,34,NULL,'Temperate phages accelerate evolution and make it more parallel during adaptation to lung biofilm growth',NULL,NULL,NULL,NULL),(110246,211,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110247,211,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110248,211,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110249,212,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110250,212,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110251,212,3,NULL,NULL,18,NULL,NULL,NULL),(110252,212,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110253,212,5,'96-well',NULL,NULL,NULL,NULL,NULL),(110254,212,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110255,212,7,'different bottlenecks/time in stat phase',NULL,NULL,NULL,NULL,NULL),(110256,212,8,NULL,NULL,NULL,NULL,1,NULL),(110257,212,9,'mutator',NULL,NULL,NULL,NULL,NULL),(110258,212,10,NULL,NULL,NULL,NULL,0,NULL),(110259,212,12,NULL,NULL,NULL,NULL,1,NULL),(110260,212,13,'static',NULL,NULL,NULL,NULL,NULL),(110261,212,14,NULL,NULL,NULL,NULL,0,NULL),(110262,212,16,NULL,NULL,300,NULL,NULL,NULL),(110263,212,17,NULL,NULL,60,NULL,NULL,NULL),(110264,212,18,'no shaking',NULL,NULL,NULL,NULL,NULL),(110265,212,19,NULL,NULL,NULL,NULL,0,NULL),(110266,212,23,NULL,NULL,NULL,NULL,0,NULL),(110267,212,34,NULL,'Experiments to test size hypothesis or GRH when adapting to local resource availability',NULL,NULL,NULL,NULL),(110268,213,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110269,213,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110270,213,3,NULL,NULL,190,NULL,NULL,NULL),(110271,213,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110272,213,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(110273,213,6,'PGY broth',NULL,NULL,NULL,NULL,NULL),(110274,213,7,'adaptive radiation under competition in communities',NULL,NULL,NULL,NULL,NULL),(110275,213,8,NULL,NULL,NULL,NULL,0,NULL),(110276,213,10,NULL,NULL,NULL,NULL,0,NULL),(110277,213,12,NULL,NULL,NULL,NULL,0,NULL),(110278,213,14,NULL,NULL,NULL,NULL,1,NULL),(110279,213,15,'competition in communities',NULL,NULL,NULL,NULL,NULL),(110280,213,16,NULL,NULL,10,NULL,NULL,NULL),(110281,213,17,NULL,NULL,2,NULL,NULL,NULL),(110282,213,18,'kanamycin',NULL,NULL,NULL,NULL,NULL),(110283,213,19,NULL,NULL,NULL,NULL,1,NULL),(110284,213,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110285,213,21,'kanamycin',NULL,NULL,NULL,NULL,NULL),(110286,213,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(110287,213,23,NULL,NULL,NULL,NULL,0,NULL),(110288,213,34,NULL,'The stronger the competition for nutrients and the more niches remain unused, the stronger the adaptive radiation.',NULL,NULL,NULL,NULL),(110289,214,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110290,214,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110291,214,3,NULL,NULL,42,NULL,NULL,NULL),(110292,214,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110293,214,5,'static microcosms',NULL,NULL,NULL,NULL,NULL),(110294,214,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110295,214,7,'adaptive radiation under competition with related species',NULL,NULL,NULL,NULL,NULL),(110296,214,8,NULL,NULL,NULL,NULL,0,NULL),(110297,214,10,NULL,NULL,NULL,NULL,0,NULL),(110298,214,12,NULL,NULL,NULL,NULL,1,NULL),(110299,214,13,'static environment',NULL,NULL,NULL,NULL,NULL),(110300,214,14,NULL,NULL,NULL,NULL,0,NULL),(110301,214,17,NULL,NULL,12,NULL,NULL,NULL),(110302,214,19,NULL,NULL,NULL,NULL,0,NULL),(110303,214,23,NULL,NULL,NULL,NULL,0,NULL),(110304,214,33,NULL,NULL,10000,NULL,NULL,NULL),(110305,214,34,NULL,'Competitive fitness differences decrease diversification by decreasing abundance while niche difference promote diversification.',NULL,NULL,NULL,NULL),(110306,215,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110307,215,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110308,215,3,NULL,NULL,384,NULL,NULL,NULL),(110309,215,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110310,215,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(110311,215,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110312,215,7,'rifampicin',NULL,NULL,NULL,NULL,NULL),(110313,215,8,NULL,NULL,NULL,NULL,0,NULL),(110314,215,10,NULL,NULL,NULL,NULL,0,NULL),(110315,215,12,NULL,NULL,NULL,NULL,0,NULL),(110316,215,14,NULL,NULL,NULL,NULL,0,NULL),(110317,215,16,NULL,NULL,110,NULL,NULL,NULL),(110318,215,17,NULL,NULL,14,NULL,NULL,NULL),(110319,215,19,NULL,NULL,NULL,NULL,1,NULL),(110320,215,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(110321,215,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110322,215,23,NULL,NULL,NULL,NULL,1,NULL),(110323,215,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110324,215,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110325,215,26,NULL,NULL,96,NULL,NULL,NULL),(110326,215,31,'https://www.ebi.ac.uk/ena/data/view/PRJEB14763',NULL,NULL,NULL,NULL,NULL),(110327,215,33,NULL,NULL,200000,NULL,NULL,NULL),(110328,215,34,NULL,'Intense or weak bottlenecking leads to high parallelism and low diversity, while intermediate bottlenecking leads to increased genetic diversity',NULL,NULL,NULL,NULL),(110329,216,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110330,216,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110331,216,3,NULL,NULL,200,NULL,NULL,NULL),(110332,216,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110333,216,5,'Glass microcosms',NULL,NULL,NULL,NULL,NULL),(110334,216,6,'KB (Pf)',NULL,NULL,NULL,NULL,NULL),(110335,216,6,'LB (Ec)',NULL,NULL,NULL,NULL,NULL),(110336,216,7,'Selection for access to oxygen - air-liquid surface',NULL,NULL,NULL,NULL,NULL),(110337,216,10,NULL,NULL,NULL,NULL,0,NULL),(110338,216,12,NULL,NULL,NULL,NULL,0,NULL),(110339,216,17,NULL,NULL,6,NULL,NULL,NULL),(110340,216,19,NULL,NULL,NULL,NULL,1,NULL),(110341,216,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110342,216,21,'Fluorescence',NULL,NULL,NULL,NULL,NULL),(110343,216,22,'targeted',NULL,NULL,NULL,NULL,NULL),(110344,216,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110345,216,23,NULL,NULL,NULL,NULL,1,NULL),(110346,216,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110347,216,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110348,216,27,NULL,NULL,91,NULL,NULL,NULL),(110349,216,31,'https://doi.org/10.7554/eLife.07074.006',NULL,NULL,NULL,NULL,NULL),(110350,216,34,NULL,'Parallel genetic evolution is biased by constraints - evolution first via pathways subject to negative regulation, then via promoter mutations and gene fusions and finally via activation of intragenic gain-of-function mutations',NULL,NULL,NULL,NULL),(110351,217,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110352,217,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110353,217,3,NULL,NULL,6,NULL,NULL,NULL),(110354,217,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110355,217,5,'glass bottles',NULL,NULL,NULL,NULL,NULL),(110356,217,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110357,217,7,'Loss of adaptive radiation propensity',NULL,NULL,NULL,NULL,NULL),(110358,217,8,NULL,NULL,NULL,NULL,0,NULL),(110359,217,10,NULL,NULL,NULL,NULL,0,NULL),(110360,217,12,NULL,NULL,NULL,NULL,1,NULL),(110361,217,13,'static environment',NULL,NULL,NULL,NULL,NULL),(110362,217,14,NULL,NULL,NULL,NULL,0,NULL),(110363,217,16,NULL,NULL,60,NULL,NULL,NULL),(110364,217,17,NULL,NULL,48,NULL,NULL,NULL),(110365,217,19,NULL,NULL,NULL,NULL,1,NULL),(110366,217,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110367,217,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(110368,217,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(110369,217,23,NULL,NULL,NULL,NULL,0,NULL),(110370,217,33,NULL,NULL,1,NULL,NULL,NULL),(110371,217,34,NULL,'The niche of the founder can have strong implication on the adaptive radiation it can undergo.',NULL,NULL,NULL,NULL),(110372,218,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110373,218,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110374,218,3,NULL,NULL,10,NULL,NULL,NULL),(110375,218,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110376,218,5,'glass bottles',NULL,NULL,NULL,NULL,NULL),(110377,218,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110378,218,7,'adaptive radiation in static environment',NULL,NULL,NULL,NULL,NULL),(110379,218,8,NULL,NULL,NULL,NULL,1,NULL),(110380,218,9,'adaptive radiation defective mutant',NULL,NULL,NULL,NULL,NULL),(110381,218,10,NULL,NULL,NULL,NULL,0,NULL),(110382,218,12,NULL,NULL,NULL,NULL,1,NULL),(110383,218,13,'static environment',NULL,NULL,NULL,NULL,NULL),(110384,218,14,NULL,NULL,NULL,NULL,0,NULL),(110385,218,17,NULL,NULL,12,NULL,NULL,NULL),(110386,218,19,NULL,NULL,NULL,NULL,0,NULL),(110387,218,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(110388,218,23,NULL,NULL,NULL,NULL,0,NULL),(110389,218,33,NULL,NULL,1000000,NULL,NULL,NULL),(110390,218,34,NULL,'The niche of the founder can have strong implication on the adaptive radiation it can undergo.',NULL,NULL,NULL,NULL),(110391,219,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110392,219,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110393,219,5,'plates',NULL,NULL,NULL,NULL,NULL),(110394,219,6,'PAF agar',NULL,NULL,NULL,NULL,NULL),(110395,219,7,'colony expansion',NULL,NULL,NULL,NULL,NULL),(110396,219,10,NULL,NULL,NULL,NULL,0,NULL),(110397,219,12,NULL,NULL,NULL,NULL,1,NULL),(110398,219,13,'radial expansion on plate',NULL,NULL,NULL,NULL,NULL),(110399,219,17,NULL,NULL,10,NULL,NULL,NULL),(110400,219,19,NULL,NULL,NULL,NULL,1,NULL),(110401,219,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110402,219,21,'resistance',NULL,NULL,NULL,NULL,NULL),(110403,219,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110404,219,23,NULL,NULL,NULL,NULL,1,NULL),(110405,219,24,'454',NULL,NULL,NULL,NULL,NULL),(110406,219,25,'454 FLX',NULL,NULL,NULL,NULL,NULL),(110407,219,26,NULL,NULL,1,NULL,NULL,NULL),(110408,219,32,NULL,NULL,50000000000,NULL,NULL,NULL),(110409,219,34,NULL,'P. fluorescens rapidly evolves a social interaction between a de novo point mutant that pushes the polymer lubricant producing ancestor forwards to increase territory expansion.',NULL,NULL,NULL,NULL),(110410,220,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110411,220,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110412,220,3,NULL,NULL,96,NULL,NULL,NULL),(110413,220,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110414,220,7,'serine limitation',NULL,NULL,NULL,NULL,NULL),(110415,220,18,'panB deletion',NULL,NULL,NULL,NULL,NULL),(110416,220,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(110417,220,23,NULL,NULL,NULL,NULL,0,NULL),(110418,220,32,NULL,NULL,7000,NULL,NULL,NULL),(110419,220,33,NULL,NULL,21000000,NULL,NULL,NULL),(110420,220,34,NULL,'generation of DFE of beneficial mutations which agrees best with a Weibull distribution',NULL,NULL,NULL,NULL),(110421,221,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110422,221,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110423,221,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110424,221,3,NULL,NULL,36,NULL,NULL,NULL),(110425,221,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110426,221,5,'microcosms',NULL,NULL,NULL,NULL,NULL),(110427,221,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110428,221,7,'coevolution with antibiotic selection',NULL,NULL,NULL,NULL,NULL),(110429,221,8,NULL,NULL,NULL,NULL,0,NULL),(110430,221,10,NULL,NULL,NULL,NULL,1,NULL),(110431,221,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(110432,221,12,NULL,NULL,NULL,NULL,0,NULL),(110433,221,14,NULL,NULL,NULL,NULL,1,NULL),(110434,221,15,'combination of stresses',NULL,NULL,NULL,NULL,NULL),(110435,221,16,NULL,NULL,380,NULL,NULL,NULL),(110436,221,17,NULL,NULL,66,NULL,NULL,NULL),(110437,221,19,NULL,NULL,NULL,NULL,0,NULL),(110438,221,23,NULL,NULL,NULL,NULL,0,NULL),(110439,221,34,NULL,'Extra selective pressure in the form of submic concentrations of streptomycin leads to stronger phage resistance, stronger antibiotic resistance but also extinction of phages',NULL,NULL,NULL,NULL),(110440,222,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110441,222,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110442,222,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110443,222,3,NULL,NULL,36,NULL,NULL,NULL),(110444,222,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110445,222,5,'microcosms',NULL,NULL,NULL,NULL,NULL),(110446,222,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110447,222,7,'coevoluton under different nutritional status',NULL,NULL,NULL,NULL,NULL),(110448,222,8,NULL,NULL,NULL,NULL,0,NULL),(110449,222,10,NULL,NULL,NULL,NULL,0,NULL),(110450,222,12,NULL,NULL,NULL,NULL,1,NULL),(110451,222,13,'static',NULL,NULL,NULL,NULL,NULL),(110452,222,14,NULL,NULL,NULL,NULL,0,NULL),(110453,222,17,NULL,NULL,40,NULL,NULL,NULL),(110454,222,19,NULL,NULL,NULL,NULL,0,NULL),(110455,222,23,NULL,NULL,NULL,NULL,0,NULL),(110456,222,34,NULL,'Intermediate nutritional status allows for stable coevolutionary dynamics.',NULL,NULL,NULL,NULL),(110457,223,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110458,223,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110459,223,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110460,223,3,NULL,NULL,50,NULL,NULL,NULL),(110461,223,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110462,223,5,'microcosms',NULL,NULL,NULL,NULL,NULL),(110463,223,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110464,223,7,'co-evolution under temperature shifts',NULL,NULL,NULL,NULL,NULL),(110465,223,8,NULL,NULL,NULL,NULL,0,NULL),(110466,223,10,NULL,NULL,NULL,NULL,1,NULL),(110467,223,11,'shifting temperatures with different frequencies',NULL,NULL,NULL,NULL,NULL),(110468,223,12,NULL,NULL,NULL,NULL,0,NULL),(110469,223,14,NULL,NULL,NULL,NULL,1,NULL),(110470,223,15,'combination of abiotic',NULL,NULL,NULL,NULL,NULL),(110471,223,15,'biotic stressors',NULL,NULL,NULL,NULL,NULL),(110472,223,16,NULL,NULL,24,NULL,NULL,NULL),(110473,223,17,NULL,NULL,16,NULL,NULL,NULL),(110474,223,19,NULL,NULL,NULL,NULL,1,NULL),(110475,223,20,'yield',NULL,NULL,NULL,NULL,NULL),(110476,223,23,NULL,NULL,NULL,NULL,0,NULL),(110477,223,34,NULL,'Additional abiotic selection on co-evolution dynamics can dictate when these dynamics are pronounced or not.',NULL,NULL,NULL,NULL),(110478,224,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110479,224,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110480,224,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110481,224,3,NULL,NULL,24,NULL,NULL,NULL),(110482,224,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110483,224,5,'tubes',NULL,NULL,NULL,NULL,NULL),(110484,224,6,'King\'s B',NULL,NULL,NULL,NULL,NULL),(110485,224,7,'temperature',NULL,NULL,NULL,NULL,NULL),(110486,224,7,'coevolution',NULL,NULL,NULL,NULL,NULL),(110487,224,10,NULL,NULL,NULL,NULL,1,NULL),(110488,224,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(110489,224,12,NULL,NULL,NULL,NULL,0,NULL),(110490,224,17,NULL,NULL,20,NULL,NULL,NULL),(110491,224,19,NULL,NULL,NULL,NULL,1,NULL),(110492,224,20,'yield',NULL,NULL,NULL,NULL,NULL),(110493,224,23,NULL,NULL,NULL,NULL,0,NULL),(110494,224,34,NULL,'Abiotic conditions can have crucial effect on the local adaptation of hosts to virusses and vica versa.',NULL,NULL,NULL,NULL),(110495,225,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110496,225,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110497,225,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110498,225,3,NULL,NULL,36,NULL,NULL,NULL),(110499,225,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110500,225,6,'King\'s B',NULL,NULL,NULL,NULL,NULL),(110501,225,7,'co-evolution under different bottlenecks of host',NULL,NULL,NULL,NULL,NULL),(110502,225,10,NULL,NULL,NULL,NULL,0,NULL),(110503,225,12,NULL,NULL,NULL,NULL,0,NULL),(110504,225,16,NULL,NULL,56,NULL,NULL,NULL),(110505,225,17,NULL,NULL,16,NULL,NULL,NULL),(110506,225,19,NULL,NULL,NULL,NULL,0,NULL),(110507,225,23,NULL,NULL,NULL,NULL,0,NULL),(110508,225,34,NULL,'Strong host bottleneck result in parasite extinction because of stochastic loss of sensitive hosts, responsible for phage persistence',NULL,NULL,NULL,NULL),(110509,226,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110510,226,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110511,226,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110512,226,3,NULL,NULL,6,NULL,NULL,NULL),(110513,226,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110514,226,5,'glass vials',NULL,NULL,NULL,NULL,NULL),(110515,226,6,'M9 glycerol',NULL,NULL,NULL,NULL,NULL),(110516,226,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(110517,226,8,NULL,NULL,NULL,NULL,0,NULL),(110518,226,10,NULL,NULL,NULL,NULL,0,NULL),(110519,226,12,NULL,NULL,NULL,NULL,1,NULL),(110520,226,13,'static incubation',NULL,NULL,NULL,NULL,NULL),(110521,226,14,NULL,NULL,NULL,NULL,0,NULL),(110522,226,16,NULL,NULL,400,NULL,NULL,NULL),(110523,226,17,NULL,NULL,120,NULL,NULL,NULL),(110524,226,19,NULL,NULL,NULL,NULL,1,NULL),(110525,226,20,'yield',NULL,NULL,NULL,NULL,NULL),(110526,226,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(110527,226,23,NULL,NULL,NULL,NULL,0,NULL),(110528,226,34,NULL,'Study of how the genetic distance among phages affects its local adaptation',NULL,NULL,NULL,NULL),(110529,227,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110530,227,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110531,227,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110532,227,3,NULL,NULL,24,NULL,NULL,NULL),(110533,227,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110534,227,5,'bottles',NULL,NULL,NULL,NULL,NULL),(110535,227,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110536,227,7,'co-evolution under various abiotic stress designs',NULL,NULL,NULL,NULL,NULL),(110537,227,8,NULL,NULL,NULL,NULL,0,NULL),(110538,227,10,NULL,NULL,NULL,NULL,1,NULL),(110539,227,11,'co-evolution',NULL,NULL,NULL,NULL,NULL),(110540,227,12,NULL,NULL,NULL,NULL,1,NULL),(110541,227,13,'static incubation',NULL,NULL,NULL,NULL,NULL),(110542,227,14,NULL,NULL,NULL,NULL,1,NULL),(110543,227,15,'combination of abiotic',NULL,NULL,NULL,NULL,NULL),(110544,227,15,'biotic stressors',NULL,NULL,NULL,NULL,NULL),(110545,227,17,NULL,NULL,40,NULL,NULL,NULL),(110546,227,19,NULL,NULL,NULL,NULL,0,NULL),(110547,227,23,NULL,NULL,NULL,NULL,0,NULL),(110548,227,34,NULL,'Imposing nutritional stress in structured environments can lead to the extinction of the predator in this predator-prey co-evolutionary setup',NULL,NULL,NULL,NULL),(110549,228,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110550,228,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110551,228,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110552,228,3,NULL,NULL,24,NULL,NULL,NULL),(110553,228,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110554,228,5,'flasks',NULL,NULL,NULL,NULL,NULL),(110555,228,6,'King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110556,228,7,'bacteriophage treatment with or without plasmid carriage',NULL,NULL,NULL,NULL,NULL),(110557,228,8,NULL,NULL,NULL,NULL,0,NULL),(110558,228,10,NULL,NULL,NULL,NULL,0,NULL),(110559,228,12,NULL,NULL,NULL,NULL,0,NULL),(110560,228,14,NULL,NULL,NULL,NULL,0,NULL),(110561,228,16,NULL,NULL,130,NULL,NULL,NULL),(110562,228,17,NULL,NULL,40,NULL,NULL,NULL),(110563,228,19,NULL,NULL,NULL,NULL,1,NULL),(110564,228,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110565,228,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(110566,228,23,NULL,NULL,NULL,NULL,0,NULL),(110567,228,34,NULL,'Carriage of plasmids limits the co-evolution arms race between phage and bacteria',NULL,NULL,NULL,NULL),(110568,229,1,'Phage phi2',NULL,NULL,NULL,NULL,NULL),(110569,229,1,'Pseudomonas fluorescens',NULL,NULL,NULL,NULL,NULL),(110570,229,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110571,229,3,NULL,NULL,6,NULL,NULL,NULL),(110572,229,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110573,229,5,'glass vials',NULL,NULL,NULL,NULL,NULL),(110574,229,6,'M9 glycerol',NULL,NULL,NULL,NULL,NULL),(110575,229,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(110576,229,8,NULL,NULL,NULL,NULL,0,NULL),(110577,229,10,NULL,NULL,NULL,NULL,0,NULL),(110578,229,12,NULL,NULL,NULL,NULL,1,NULL),(110579,229,13,'static incubation',NULL,NULL,NULL,NULL,NULL),(110580,229,14,NULL,NULL,NULL,NULL,0,NULL),(110581,229,16,NULL,NULL,400,NULL,NULL,NULL),(110582,229,17,NULL,NULL,120,NULL,NULL,NULL),(110583,229,19,NULL,NULL,NULL,NULL,1,NULL),(110584,229,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110585,229,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(110586,229,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(110587,229,23,NULL,NULL,NULL,NULL,0,NULL),(110588,229,34,NULL,'Arms race dynamics decelerate over time in co-evolving predator-prey systems as the cost for generalism becomes to high and is exchanged by fluctuating selection dynamics',NULL,NULL,NULL,NULL),(110589,230,1,'Pseudomonas protegens',NULL,NULL,NULL,NULL,NULL),(110590,230,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110591,230,3,NULL,NULL,3,NULL,NULL,NULL),(110592,230,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110593,230,5,'plates',NULL,NULL,NULL,NULL,NULL),(110594,230,6,'modified King\'s B medium',NULL,NULL,NULL,NULL,NULL),(110595,230,7,'swarming',NULL,NULL,NULL,NULL,NULL),(110596,230,8,NULL,NULL,NULL,NULL,0,NULL),(110597,230,10,NULL,NULL,NULL,NULL,0,NULL),(110598,230,12,NULL,NULL,NULL,NULL,1,NULL),(110599,230,13,'agar plate swarming',NULL,NULL,NULL,NULL,NULL),(110600,230,14,NULL,NULL,NULL,NULL,0,NULL),(110601,230,17,NULL,NULL,20,NULL,NULL,NULL),(110602,230,19,NULL,NULL,NULL,NULL,0,NULL),(110603,230,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(110604,230,23,NULL,NULL,NULL,NULL,0,NULL),(110605,230,34,NULL,'Spontaneous cheaters appear that cannot swarm themselves but make use of the swarming of the ancestor in the edge.',NULL,NULL,NULL,NULL),(110606,231,1,'Pseudomonas putida',NULL,NULL,NULL,NULL,NULL),(110607,231,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110608,231,3,NULL,NULL,6,NULL,NULL,NULL),(110609,231,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110610,231,5,'corn rhizosphere',NULL,NULL,NULL,NULL,NULL),(110611,231,6,'corn rhizosphere',NULL,NULL,NULL,NULL,NULL),(110612,231,7,'adaptation to rhizosphere',NULL,NULL,NULL,NULL,NULL),(110613,231,8,NULL,NULL,NULL,NULL,0,NULL),(110614,231,10,NULL,NULL,NULL,NULL,0,NULL),(110615,231,12,NULL,NULL,NULL,NULL,1,NULL),(110616,231,13,'corn rhizosphere',NULL,NULL,NULL,NULL,NULL),(110617,231,14,NULL,NULL,NULL,NULL,1,NULL),(110618,231,15,'in situ',NULL,NULL,NULL,NULL,NULL),(110619,231,17,NULL,NULL,131,NULL,NULL,NULL),(110620,231,18,'resistance',NULL,NULL,NULL,NULL,NULL),(110621,231,18,'PCR',NULL,NULL,NULL,NULL,NULL),(110622,231,19,NULL,NULL,NULL,NULL,1,NULL),(110623,231,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110624,231,21,'resistance',NULL,NULL,NULL,NULL,NULL),(110625,231,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110626,231,23,NULL,NULL,NULL,NULL,0,NULL),(110627,231,34,NULL,'P. putida rapidly adapted to the corn rhizosphere',NULL,NULL,NULL,NULL),(110628,231,35,NULL,'Expression analysis to find causal changes',NULL,NULL,NULL,NULL),(110629,232,1,'Pseudomonas syringae',NULL,NULL,NULL,NULL,NULL),(110630,232,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110631,232,3,NULL,NULL,36,NULL,NULL,NULL),(110632,232,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110633,232,5,'plants',NULL,NULL,NULL,NULL,NULL),(110634,232,6,'plants',NULL,NULL,NULL,NULL,NULL),(110635,232,7,'plants host with phages',NULL,NULL,NULL,NULL,NULL),(110636,232,8,NULL,NULL,NULL,NULL,0,NULL),(110637,232,10,NULL,NULL,NULL,NULL,1,NULL),(110638,232,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(110639,232,12,NULL,NULL,NULL,NULL,1,NULL),(110640,232,13,'in situ',NULL,NULL,NULL,NULL,NULL),(110641,232,14,NULL,NULL,NULL,NULL,1,NULL),(110642,232,15,'in situ',NULL,NULL,NULL,NULL,NULL),(110643,232,17,NULL,NULL,20,NULL,NULL,NULL),(110644,232,18,'plating',NULL,NULL,NULL,NULL,NULL),(110645,232,18,'PCR',NULL,NULL,NULL,NULL,NULL),(110646,232,18,'16S RNA sequencing',NULL,NULL,NULL,NULL,NULL),(110647,232,19,NULL,NULL,NULL,NULL,1,NULL),(110648,232,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(110649,232,20,'yield',NULL,NULL,NULL,NULL,NULL),(110650,232,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110651,232,23,NULL,NULL,NULL,NULL,1,NULL),(110652,232,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110653,232,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(110654,232,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA357646',NULL,NULL,NULL,NULL,NULL),(110655,232,33,NULL,NULL,100,NULL,NULL,NULL),(110656,232,34,NULL,'Adaptation to a novel host resulted in a stronger improvement on the normal host than evolution on this host',NULL,NULL,NULL,NULL),(110657,233,1,'Ralstonia eutropha',NULL,NULL,NULL,NULL,NULL),(110658,233,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(110659,233,3,NULL,NULL,2,NULL,NULL,NULL),(110660,233,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110661,233,5,'tubes',NULL,NULL,NULL,NULL,NULL),(110662,233,6,'TSB',NULL,NULL,NULL,NULL,NULL),(110663,233,7,'isobutanol',NULL,NULL,NULL,NULL,NULL),(110664,233,10,NULL,NULL,NULL,NULL,1,NULL),(110665,233,11,'increasing concentration of isobutanol',NULL,NULL,NULL,NULL,NULL),(110666,233,12,NULL,NULL,NULL,NULL,0,NULL),(110667,233,17,NULL,NULL,75,NULL,NULL,NULL),(110668,233,19,NULL,NULL,NULL,NULL,0,NULL),(110669,233,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(110670,233,23,NULL,NULL,NULL,NULL,0,NULL),(110671,233,34,NULL,'R. eutropha evolves to increase IBT tolerance by improved survival, allowing increased IBT production and this by mutating acrA and acrA6',NULL,NULL,NULL,NULL),(110672,234,1,'Ralstonia solanacearum',NULL,NULL,NULL,NULL,NULL),(110673,234,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110674,234,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110675,235,1,'Ralstonia solanacearum',NULL,NULL,NULL,NULL,NULL),(110676,235,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110677,235,3,NULL,NULL,18,NULL,NULL,NULL),(110678,235,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110679,235,5,'plant roots',NULL,NULL,NULL,NULL,NULL),(110680,235,6,'plant roots',NULL,NULL,NULL,NULL,NULL),(110681,235,7,'nodulization of plant roots',NULL,NULL,NULL,NULL,NULL),(110682,235,8,NULL,NULL,NULL,NULL,0,NULL),(110683,235,10,NULL,NULL,NULL,NULL,1,NULL),(110684,235,11,'in situ',NULL,NULL,NULL,NULL,NULL),(110685,235,12,NULL,NULL,NULL,NULL,1,NULL),(110686,235,13,'in situ',NULL,NULL,NULL,NULL,NULL),(110687,235,14,NULL,NULL,NULL,NULL,1,NULL),(110688,235,15,'in situ',NULL,NULL,NULL,NULL,NULL),(110689,235,16,NULL,NULL,800,NULL,NULL,NULL),(110690,235,17,NULL,NULL,672,NULL,NULL,NULL),(110691,235,18,'resistance',NULL,NULL,NULL,NULL,NULL),(110692,235,19,NULL,NULL,NULL,NULL,1,NULL),(110693,235,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110694,235,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110695,235,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110696,235,23,NULL,NULL,NULL,NULL,1,NULL),(110697,235,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110698,235,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110699,235,26,NULL,NULL,6,NULL,NULL,NULL),(110700,235,27,NULL,NULL,400,NULL,NULL,NULL),(110701,235,34,NULL,'Depending on the selection regime, the intracellular or extracellular mode of symbiosis is evolutionary enforced.',NULL,NULL,NULL,NULL),(110702,236,1,'Ralstonia solanacearum',NULL,NULL,NULL,NULL,NULL),(110703,236,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110704,236,3,NULL,NULL,6,NULL,NULL,NULL),(110705,236,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110706,236,5,'plant roots',NULL,NULL,NULL,NULL,NULL),(110707,236,6,'plant roots',NULL,NULL,NULL,NULL,NULL),(110708,236,7,'nodulization of plant roots',NULL,NULL,NULL,NULL,NULL),(110709,236,8,NULL,NULL,NULL,NULL,0,NULL),(110710,236,10,NULL,NULL,NULL,NULL,1,NULL),(110711,236,11,'in situ',NULL,NULL,NULL,NULL,NULL),(110712,236,12,NULL,NULL,NULL,NULL,1,NULL),(110713,236,13,'in situ',NULL,NULL,NULL,NULL,NULL),(110714,236,14,NULL,NULL,NULL,NULL,1,NULL),(110715,236,15,'in situ',NULL,NULL,NULL,NULL,NULL),(110716,236,16,NULL,NULL,400,NULL,NULL,NULL),(110717,236,17,NULL,NULL,336,NULL,NULL,NULL),(110718,236,18,'resistance',NULL,NULL,NULL,NULL,NULL),(110719,236,19,NULL,NULL,NULL,NULL,1,NULL),(110720,236,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110721,236,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(110722,236,23,NULL,NULL,NULL,NULL,0,NULL),(110723,236,34,NULL,'Artificial symbiosis was inforced using experimental evolution in the plant rhizosphere resulting in rapid and parallel evolution',NULL,NULL,NULL,NULL),(110724,237,1,'Ralstonia solanacearum',NULL,NULL,NULL,NULL,NULL),(110725,237,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110726,237,3,NULL,NULL,9,NULL,NULL,NULL),(110727,237,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110728,237,5,'plant roots',NULL,NULL,NULL,NULL,NULL),(110729,237,6,'plant roots',NULL,NULL,NULL,NULL,NULL),(110730,237,7,'nodulization of plant roots',NULL,NULL,NULL,NULL,NULL),(110731,237,8,NULL,NULL,NULL,NULL,0,NULL),(110732,237,10,NULL,NULL,NULL,NULL,1,NULL),(110733,237,11,'in situ',NULL,NULL,NULL,NULL,NULL),(110734,237,12,NULL,NULL,NULL,NULL,1,NULL),(110735,237,13,'in situ',NULL,NULL,NULL,NULL,NULL),(110736,237,14,NULL,NULL,NULL,NULL,1,NULL),(110737,237,15,'in situ',NULL,NULL,NULL,NULL,NULL),(110738,237,16,NULL,NULL,400,NULL,NULL,NULL),(110739,237,17,NULL,NULL,336,NULL,NULL,NULL),(110740,237,18,'resistance',NULL,NULL,NULL,NULL,NULL),(110741,237,19,NULL,NULL,NULL,NULL,1,NULL),(110742,237,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110743,237,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(110744,237,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110745,237,23,NULL,NULL,NULL,NULL,1,NULL),(110746,237,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110747,237,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(110748,237,25,'GAIIx',NULL,NULL,NULL,NULL,NULL),(110749,237,26,NULL,NULL,9,NULL,NULL,NULL),(110750,237,31,'https://www.genoscope.cns.fr/agc/microscope/expdata/evoProject.php',NULL,NULL,NULL,NULL,NULL),(110751,237,34,NULL,'The adaptation of Ralstonia solanacearum to being a symbiont in the root nodules proceeded by a temporal state of hypermutagenesis due to a mutagenesis cassette transferred on the plasmid.',NULL,NULL,NULL,NULL),(110752,238,1,'Bacillus amyloliquefaciens',NULL,NULL,NULL,NULL,NULL),(110753,238,1,'Ralstonia solanacearum',NULL,NULL,NULL,NULL,NULL),(110754,238,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110755,238,3,NULL,NULL,24,NULL,NULL,NULL),(110756,238,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110757,238,5,'48-well',NULL,NULL,NULL,NULL,NULL),(110758,238,6,'nutrient medium',NULL,NULL,NULL,NULL,NULL),(110759,238,7,'combined selection with phages or competitor producing antibiotics',NULL,NULL,NULL,NULL,NULL),(110760,238,8,NULL,NULL,NULL,NULL,0,NULL),(110761,238,10,NULL,NULL,NULL,NULL,1,NULL),(110762,238,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(110763,238,12,NULL,NULL,NULL,NULL,0,NULL),(110764,238,14,NULL,NULL,NULL,NULL,1,NULL),(110765,238,15,'combination of stresses',NULL,NULL,NULL,NULL,NULL),(110766,238,17,NULL,NULL,4,NULL,NULL,NULL),(110767,238,19,NULL,NULL,NULL,NULL,1,NULL),(110768,238,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(110769,238,20,'yield',NULL,NULL,NULL,NULL,NULL),(110770,238,23,NULL,NULL,NULL,NULL,0,NULL),(110771,238,34,NULL,'Competitors (or its antibiotics) and phages can synergistically interact to limit pathogens and their evolution because of trade-offs',NULL,NULL,NULL,NULL),(110772,239,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110773,239,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(110774,239,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110775,239,7,'increased carotenoid production',NULL,NULL,NULL,NULL,NULL),(110776,240,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110777,240,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110778,240,3,NULL,NULL,2,NULL,NULL,NULL),(110779,240,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110780,240,5,'flasks',NULL,NULL,NULL,NULL,NULL),(110781,240,6,'Delft medium',NULL,NULL,NULL,NULL,NULL),(110782,240,7,'nitrogen limitation',NULL,NULL,NULL,NULL,NULL),(110783,241,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110784,241,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110785,241,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110786,242,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110787,242,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110788,242,3,NULL,NULL,592,NULL,NULL,NULL),(110789,242,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110790,242,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(110791,242,6,'YPD',NULL,NULL,NULL,NULL,NULL),(110792,242,7,'sterility in YPD medium',NULL,NULL,NULL,NULL,NULL),(110793,242,8,NULL,NULL,NULL,NULL,0,NULL),(110794,242,10,NULL,NULL,NULL,NULL,0,NULL),(110795,242,12,NULL,NULL,NULL,NULL,1,NULL),(110796,242,13,'static',NULL,NULL,NULL,NULL,NULL),(110797,242,14,NULL,NULL,NULL,NULL,0,NULL),(110798,242,16,NULL,NULL,1000,NULL,NULL,NULL),(110799,242,17,NULL,NULL,100,NULL,NULL,NULL),(110800,242,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110801,242,18,'control wells',NULL,NULL,NULL,NULL,NULL),(110802,242,19,NULL,NULL,NULL,NULL,1,NULL),(110803,242,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110804,242,21,'bulk segregant analysis',NULL,NULL,NULL,NULL,NULL),(110805,242,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(110806,242,23,NULL,NULL,NULL,NULL,0,NULL),(110807,242,32,NULL,NULL,12800000,NULL,NULL,NULL),(110808,242,33,NULL,NULL,400000,NULL,NULL,NULL),(110809,242,34,NULL,'Cohorts often contain only one (or none) driver mutation with only a few more, only one showing expansion because of epistatic interactions',NULL,NULL,NULL,NULL),(110810,243,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110811,243,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110812,244,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110813,244,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110814,244,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110815,244,7,'heavy metals',NULL,NULL,NULL,NULL,NULL),(110816,244,10,NULL,NULL,NULL,NULL,1,NULL),(110817,244,11,'different rates of increasing metal concentration',NULL,NULL,NULL,NULL,NULL),(110824,246,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110825,246,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110826,246,3,NULL,NULL,160,NULL,NULL,NULL),(110827,246,4,'continuous',NULL,NULL,NULL,NULL,NULL),(110828,246,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(110829,246,7,'glucose',NULL,NULL,NULL,NULL,NULL),(110830,246,7,'sulfate',NULL,NULL,NULL,NULL,NULL),(110831,246,7,'phosphate limitation',NULL,NULL,NULL,NULL,NULL),(110832,246,8,NULL,NULL,NULL,NULL,0,NULL),(110833,246,10,NULL,NULL,NULL,NULL,0,NULL),(110834,246,12,NULL,NULL,NULL,NULL,0,NULL),(110835,246,14,NULL,NULL,NULL,NULL,0,NULL),(110836,246,16,NULL,NULL,300,NULL,NULL,NULL),(110837,246,18,'plating',NULL,NULL,NULL,NULL,NULL),(110838,246,18,'resistance',NULL,NULL,NULL,NULL,NULL),(110839,246,19,NULL,NULL,NULL,NULL,0,NULL),(110840,246,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110841,246,23,NULL,NULL,NULL,NULL,1,NULL),(110842,246,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110843,246,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(110844,246,26,NULL,NULL,23,NULL,NULL,NULL),(110845,246,28,NULL,NULL,97,NULL,NULL,NULL),(110846,246,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA339148',NULL,NULL,NULL,NULL,NULL),(110847,246,34,NULL,'By using experimental evolution in chemostats, isolating mutants with increased flocculation and wall growth, they could identify the most important gene for this malignant phenotype thereby reducing its occurence upon deletion.',NULL,NULL,NULL,NULL),(110851,248,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110852,248,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110853,248,3,NULL,NULL,72,NULL,NULL,NULL),(110854,248,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110855,248,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(110856,248,6,'YPD',NULL,NULL,NULL,NULL,NULL),(110857,248,7,'YPD + sexual reproduction on st',NULL,NULL,NULL,NULL,NULL),(110858,248,7,'ing variation',NULL,NULL,NULL,NULL,NULL),(110859,248,8,NULL,NULL,NULL,NULL,0,NULL),(110860,248,10,NULL,NULL,NULL,NULL,0,NULL),(110861,248,12,NULL,NULL,NULL,NULL,0,NULL),(110862,248,14,NULL,NULL,NULL,NULL,0,NULL),(110863,248,16,NULL,NULL,960,NULL,NULL,NULL),(110864,248,17,NULL,NULL,96,NULL,NULL,NULL),(110865,248,18,'nutrient markers',NULL,NULL,NULL,NULL,NULL),(110866,248,18,'antibiotics',NULL,NULL,NULL,NULL,NULL),(110867,248,18,'empty wells',NULL,NULL,NULL,NULL,NULL),(110868,248,18,'WGS',NULL,NULL,NULL,NULL,NULL),(110869,248,19,NULL,NULL,NULL,NULL,1,NULL),(110870,248,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110871,248,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110872,248,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110873,248,23,NULL,NULL,NULL,NULL,1,NULL),(110874,248,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110875,248,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(110876,248,25,'NextSeq',NULL,NULL,NULL,NULL,NULL),(110877,248,26,NULL,NULL,537,NULL,NULL,NULL),(110878,248,28,NULL,NULL,20,NULL,NULL,NULL),(110879,248,29,NULL,NULL,10,NULL,NULL,NULL),(110880,248,30,NULL,NULL,NULL,50,NULL,NULL),(110881,248,31,'https://www.ncbi.nlm.nih.gov/sra?linkname=bioproject_sra_all&from_uid=359887',NULL,NULL,NULL,NULL,NULL),(110882,248,33,NULL,NULL,100000,NULL,NULL,NULL),(110883,248,34,NULL,'Sex also speeds up adaptation on standing variation by allowing breakdown of linkage disequilibrium and transition from selection on genotypes to one on alleles.',NULL,NULL,NULL,NULL),(110884,248,35,NULL,'robotics',NULL,NULL,NULL,NULL),(110885,249,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110886,249,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110887,249,3,NULL,NULL,76,NULL,NULL,NULL),(110888,249,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110889,249,5,'96-well',NULL,NULL,NULL,NULL,NULL),(110890,249,6,'SC medium',NULL,NULL,NULL,NULL,NULL),(110891,249,7,'19 stressors including arsene',NULL,NULL,NULL,NULL,NULL),(110892,249,8,NULL,NULL,NULL,NULL,0,NULL),(110893,249,10,NULL,NULL,NULL,NULL,0,NULL),(110894,249,12,NULL,NULL,NULL,NULL,0,NULL),(110895,249,14,NULL,NULL,NULL,NULL,0,NULL),(110896,249,16,NULL,NULL,325,NULL,NULL,NULL),(110897,249,17,NULL,NULL,190,NULL,NULL,NULL),(110898,249,19,NULL,NULL,NULL,NULL,1,NULL),(110899,249,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(110900,249,20,'yield',NULL,NULL,NULL,NULL,NULL),(110901,249,20,'lag phase',NULL,NULL,NULL,NULL,NULL),(110902,249,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110903,249,23,NULL,NULL,NULL,NULL,1,NULL),(110904,249,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110905,249,24,'SOLiD',NULL,NULL,NULL,NULL,NULL),(110906,249,25,'ABI 5,500',NULL,NULL,NULL,NULL,NULL),(110907,249,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(110908,249,27,NULL,NULL,85,NULL,NULL,NULL),(110909,249,31,'https://www.ncbi.nlm.nih.gov/sra?term=SRP092403',NULL,NULL,NULL,NULL,NULL),(110910,249,32,NULL,NULL,3500000,NULL,NULL,NULL),(110911,249,33,NULL,NULL,100000,NULL,NULL,NULL),(110912,249,34,NULL,'Ultrafast evolution on arsene mediates through accumulation of mutations with strong, positive pleiotropic effects.',NULL,NULL,NULL,NULL),(110920,251,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110921,251,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110922,251,3,NULL,NULL,48,NULL,NULL,NULL),(110923,251,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110924,251,6,'YNB/CSM + carbon source',NULL,NULL,NULL,NULL,NULL),(110925,251,7,'carbon source (combination)',NULL,NULL,NULL,NULL,NULL),(110926,251,8,NULL,NULL,NULL,NULL,0,NULL),(110927,251,10,NULL,NULL,NULL,NULL,0,NULL),(110928,251,12,NULL,NULL,NULL,NULL,0,NULL),(110929,251,14,NULL,NULL,NULL,NULL,0,NULL),(110930,251,16,NULL,NULL,250,NULL,NULL,NULL),(110931,251,17,NULL,NULL,26,NULL,NULL,NULL),(110932,251,19,NULL,NULL,NULL,NULL,1,NULL),(110933,251,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110934,251,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110935,251,23,NULL,NULL,NULL,NULL,0,NULL),(110936,251,34,NULL,'De novo phenotypic heterogeneity evolves in a mixed environment likely not because of bet hedging but just as a response to NFD interactions',NULL,NULL,NULL,NULL),(110965,253,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110966,253,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(110967,253,3,NULL,NULL,24,NULL,NULL,NULL),(110968,253,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110969,253,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(110970,253,6,'YPD',NULL,NULL,NULL,NULL,NULL),(110971,253,7,'YPD adaptation of sexuals',NULL,NULL,NULL,NULL,NULL),(110972,253,7,'asexuals',NULL,NULL,NULL,NULL,NULL),(110973,253,8,NULL,NULL,NULL,NULL,0,NULL),(110974,253,10,NULL,NULL,NULL,NULL,0,NULL),(110975,253,12,NULL,NULL,NULL,NULL,0,NULL),(110976,253,14,NULL,NULL,NULL,NULL,0,NULL),(110977,253,16,NULL,NULL,1000,NULL,NULL,NULL),(110978,253,17,NULL,NULL,100,NULL,NULL,NULL),(110979,253,19,NULL,NULL,NULL,NULL,1,NULL),(110980,253,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(110981,253,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(110982,253,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(110983,253,23,NULL,NULL,NULL,NULL,1,NULL),(110984,253,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(110985,253,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(110986,253,26,NULL,NULL,8,NULL,NULL,NULL),(110987,253,27,NULL,NULL,96,NULL,NULL,NULL),(110988,253,28,NULL,NULL,80,NULL,NULL,NULL),(110989,253,29,NULL,NULL,40,NULL,NULL,NULL),(110990,253,30,NULL,NULL,NULL,10,NULL,NULL),(110991,253,31,'https://www.ncbi.nlm.nih.gov/biosample',NULL,NULL,NULL,NULL,NULL),(110992,253,32,NULL,NULL,10000000,NULL,NULL,NULL),(110993,253,33,NULL,NULL,10000,NULL,NULL,NULL),(110994,253,34,NULL,'Sexual reproduction and recombination speeds evolutionary adaptation by alleviating clonal interference and hitchhiking of deleterious mutations thereby effectively changing the molecular signature of evolution. Sexual reproduction maintains transposable elements while asexual selects against these elements.',NULL,NULL,NULL,NULL),(110995,254,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(110996,254,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(110997,254,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(110998,254,8,NULL,NULL,NULL,NULL,1,NULL),(110999,254,9,'missing polarity gene',NULL,NULL,NULL,NULL,NULL),(111000,255,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111001,255,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111002,255,3,NULL,NULL,2,NULL,NULL,NULL),(111003,255,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111004,255,5,'flasks',NULL,NULL,NULL,NULL,NULL),(111005,255,6,'Delft medium',NULL,NULL,NULL,NULL,NULL),(111006,255,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(111007,255,8,NULL,NULL,NULL,NULL,0,NULL),(111008,255,9,'N',NULL,NULL,NULL,NULL,NULL),(111009,255,11,'N',NULL,NULL,NULL,NULL,NULL),(111010,255,13,'N',NULL,NULL,NULL,NULL,NULL),(111011,255,16,NULL,NULL,168,NULL,NULL,NULL),(111012,255,17,NULL,NULL,21,NULL,NULL,NULL),(111013,255,18,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(111014,255,19,NULL,NULL,NULL,NULL,1,NULL),(111015,255,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111016,255,21,'barcodes',NULL,NULL,NULL,NULL,NULL),(111017,255,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111018,255,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111019,255,23,NULL,NULL,NULL,NULL,1,NULL),(111020,255,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111021,255,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111022,255,27,NULL,NULL,22,NULL,NULL,NULL),(111023,255,29,NULL,NULL,222000000,NULL,NULL,NULL),(111024,255,30,NULL,NULL,NULL,0.000002,NULL,NULL),(111025,255,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA310010',NULL,NULL,NULL,NULL,NULL),(111026,255,32,NULL,NULL,17000000000,NULL,NULL,NULL),(111027,255,33,NULL,NULL,70000000,NULL,NULL,NULL),(111028,255,34,NULL,'Barcode lineage tracking in yeast resulted in an unexpected DFE, overwhelming CI of small effect mutations and stochastic appearance of large effect mutations. Extensive clonal sequencing revealed a large aspect of the first mutational fitness landscape.',NULL,NULL,NULL,NULL),(111029,256,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111030,256,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111031,256,3,NULL,NULL,11,NULL,NULL,NULL),(111032,256,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111033,256,7,'nitrogen limitation',NULL,NULL,NULL,NULL,NULL),(111034,256,16,NULL,NULL,250,NULL,NULL,NULL),(111035,256,17,NULL,NULL,60,NULL,NULL,NULL),(111036,256,21,'mCherry',NULL,NULL,NULL,NULL,NULL),(111037,256,21,'mCitrine',NULL,NULL,NULL,NULL,NULL),(111038,256,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111039,256,23,NULL,NULL,NULL,NULL,1,NULL),(111040,256,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111041,256,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111042,256,25,'Illumina HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111043,256,25,'Agilent CGH array',NULL,NULL,NULL,NULL,NULL),(111044,256,26,NULL,NULL,18,NULL,NULL,NULL),(111045,256,27,NULL,NULL,11,NULL,NULL,NULL),(111046,256,28,NULL,NULL,300,NULL,NULL,NULL),(111047,256,29,NULL,NULL,300,NULL,NULL,NULL),(111048,256,30,NULL,NULL,NULL,5,NULL,NULL),(111049,256,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP032757',NULL,NULL,NULL,NULL,NULL),(111050,256,32,NULL,NULL,10000000000,NULL,NULL,NULL),(111051,256,33,NULL,NULL,10000000000,NULL,NULL,NULL),(111052,256,34,NULL,'diverse adaptive mutations arise under limiting nitrogen conditions depending on the nitrogen source, yet common themes on the level of functional modules. Many of mutations are CNV, order of mutations is determined by epistasis.',NULL,NULL,NULL,NULL),(111053,257,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111054,257,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111055,257,3,NULL,NULL,3,NULL,NULL,NULL),(111056,257,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111057,257,5,'flask',NULL,NULL,NULL,NULL,NULL),(111058,257,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111059,257,7,'3-hydroxy propionic acid',NULL,NULL,NULL,NULL,NULL),(111060,257,10,NULL,NULL,NULL,NULL,1,NULL),(111061,257,11,'Increasing concentration 3-HPA',NULL,NULL,NULL,NULL,NULL),(111062,257,12,NULL,NULL,NULL,NULL,0,NULL),(111063,257,16,NULL,NULL,200,NULL,NULL,NULL),(111064,257,17,NULL,NULL,34,NULL,NULL,NULL),(111065,257,19,NULL,NULL,NULL,NULL,0,NULL),(111066,257,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111067,257,23,NULL,NULL,NULL,NULL,1,NULL),(111068,257,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111069,257,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(111070,257,26,NULL,NULL,3,NULL,NULL,NULL),(111071,257,34,NULL,'adaptation to 3-HP is possibly mediated by reuterin (3-hydroxypropionic aldehyde) and tolerance is conferred by SFA1 gene or gluthatione',NULL,NULL,NULL,NULL),(111072,258,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111073,258,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111074,258,3,NULL,NULL,12,NULL,NULL,NULL),(111075,258,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111076,258,5,'flasks',NULL,NULL,NULL,NULL,NULL),(111077,258,6,'YPD or glucose minimum',NULL,NULL,NULL,NULL,NULL),(111078,258,7,'high salt or glucose limitation',NULL,NULL,NULL,NULL,NULL),(111079,258,8,NULL,NULL,NULL,NULL,0,NULL),(111080,258,10,NULL,NULL,NULL,NULL,0,NULL),(111081,258,12,NULL,NULL,NULL,NULL,0,NULL),(111082,258,14,NULL,NULL,NULL,NULL,0,NULL),(111083,258,16,NULL,NULL,500,NULL,NULL,NULL),(111084,258,17,NULL,NULL,75,NULL,NULL,NULL),(111085,258,18,'resistance',NULL,NULL,NULL,NULL,NULL),(111086,258,18,'barcodes',NULL,NULL,NULL,NULL,NULL),(111087,258,19,NULL,NULL,NULL,NULL,0,NULL),(111088,258,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111089,258,23,NULL,NULL,NULL,NULL,1,NULL),(111090,258,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111091,258,26,NULL,NULL,115,NULL,NULL,NULL),(111092,258,27,NULL,NULL,9,NULL,NULL,NULL),(111093,258,34,NULL,'Genomics on Dettman et al Nature 2007 to detect both broad and narrow targets of selection.',NULL,NULL,NULL,NULL),(111094,259,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111095,259,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111096,259,3,NULL,NULL,640,NULL,NULL,NULL),(111097,259,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111098,259,7,'YPD medium',NULL,NULL,NULL,NULL,NULL),(111099,259,16,NULL,NULL,740,NULL,NULL,NULL),(111100,259,17,NULL,NULL,74,NULL,NULL,NULL),(111101,259,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111102,259,18,'control wells',NULL,NULL,NULL,NULL,NULL),(111103,259,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111104,259,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111105,259,23,NULL,NULL,NULL,NULL,1,NULL),(111106,259,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111107,259,25,'Illumina HiSeq2000/2500',NULL,NULL,NULL,NULL,NULL),(111108,259,26,NULL,NULL,169,NULL,NULL,NULL),(111109,259,28,NULL,NULL,23,NULL,NULL,NULL),(111110,259,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA242140.',NULL,NULL,NULL,NULL,NULL),(111111,259,32,NULL,NULL,12800000,NULL,NULL,NULL),(111112,259,33,NULL,NULL,400000,NULL,NULL,NULL),(111113,259,34,NULL,'diminishing returns is a global theme among mutations targeting a variety of biological processes.',NULL,NULL,NULL,NULL),(111114,260,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111115,260,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111116,260,3,NULL,NULL,760,NULL,NULL,NULL),(111117,260,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111118,260,7,'rich liquid medium',NULL,NULL,NULL,NULL,NULL),(111119,260,16,NULL,NULL,400,NULL,NULL,NULL),(111120,260,17,NULL,NULL,104,NULL,NULL,NULL),(111121,260,18,'PCR',NULL,NULL,NULL,NULL,NULL),(111122,260,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111123,260,23,NULL,NULL,NULL,NULL,1,NULL),(111124,260,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111125,260,25,'Illumina HighScan SC',NULL,NULL,NULL,NULL,NULL),(111126,260,26,NULL,NULL,55,NULL,NULL,NULL),(111127,260,28,NULL,NULL,80,NULL,NULL,NULL),(111128,260,32,NULL,NULL,20000000,NULL,NULL,NULL),(111129,260,33,NULL,NULL,100000,NULL,NULL,NULL),(111130,260,34,NULL,'rapid compensation of fitness but very diverse depending on knockout and many pleiotropic effects on phenotype. Could result in diversification and change in evolvability',NULL,NULL,NULL,NULL),(111131,261,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111132,261,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111133,261,3,NULL,NULL,592,NULL,NULL,NULL),(111134,261,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111135,261,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(111136,261,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111137,261,7,'sterility in YPD medium',NULL,NULL,NULL,NULL,NULL),(111138,261,8,NULL,NULL,NULL,NULL,0,NULL),(111139,261,10,NULL,NULL,NULL,NULL,0,NULL),(111140,261,12,NULL,NULL,NULL,NULL,1,NULL),(111141,261,13,'static',NULL,NULL,NULL,NULL,NULL),(111142,261,14,NULL,NULL,NULL,NULL,0,NULL),(111143,261,16,NULL,NULL,1000,NULL,NULL,NULL),(111144,261,17,NULL,NULL,100,NULL,NULL,NULL),(111145,261,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111146,261,18,'control wells',NULL,NULL,NULL,NULL,NULL),(111147,261,19,NULL,NULL,NULL,NULL,1,NULL),(111148,261,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111149,261,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111150,261,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(111151,261,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111152,261,23,NULL,NULL,NULL,NULL,1,NULL),(111153,261,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111154,261,25,'HisSeq',NULL,NULL,NULL,NULL,NULL),(111155,261,27,NULL,NULL,480,NULL,NULL,NULL),(111156,261,29,NULL,NULL,100,NULL,NULL,NULL),(111157,261,30,NULL,NULL,NULL,1,NULL,NULL),(111158,261,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA205542',NULL,NULL,NULL,NULL,NULL),(111159,261,32,NULL,NULL,12800000,NULL,NULL,NULL),(111160,261,33,NULL,NULL,400000,NULL,NULL,NULL),(111161,261,34,NULL,'Hitchhiking and CI is pervasive in evolving yeast as detected by massive population sequencing',NULL,NULL,NULL,NULL),(111162,262,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111163,262,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111164,262,3,NULL,NULL,4,NULL,NULL,NULL),(111165,262,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111166,262,7,'sulfate limitations',NULL,NULL,NULL,NULL,NULL),(111167,262,16,NULL,NULL,250,NULL,NULL,NULL),(111168,262,18,'microscopy',NULL,NULL,NULL,NULL,NULL),(111169,262,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111170,262,23,NULL,NULL,NULL,NULL,1,NULL),(111171,262,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111172,262,26,NULL,NULL,4,NULL,NULL,NULL),(111173,262,32,NULL,NULL,1000000000,NULL,NULL,NULL),(111174,262,33,NULL,NULL,1000000000,NULL,NULL,NULL),(111175,262,34,NULL,'instructions to build your own, custom, multiplexed array of chemostats',NULL,NULL,NULL,NULL),(111176,263,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111177,263,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111178,263,3,NULL,NULL,1,NULL,NULL,NULL),(111179,263,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111180,263,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(111181,263,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111182,263,7,'ethanol',NULL,NULL,NULL,NULL,NULL),(111183,263,10,NULL,NULL,NULL,NULL,1,NULL),(111184,263,11,'fixed increases',NULL,NULL,NULL,NULL,NULL),(111185,263,12,NULL,NULL,NULL,NULL,0,NULL),(111186,263,16,NULL,NULL,141,NULL,NULL,NULL),(111187,263,17,NULL,NULL,21,NULL,NULL,NULL),(111188,263,19,NULL,NULL,NULL,NULL,0,NULL),(111189,263,22,'targeted',NULL,NULL,NULL,NULL,NULL),(111190,263,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111191,263,23,NULL,NULL,NULL,NULL,1,NULL),(111192,263,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111193,263,25,'Illumina Genome Analyzer II',NULL,NULL,NULL,NULL,NULL),(111194,263,26,NULL,NULL,6,NULL,NULL,NULL),(111195,263,32,NULL,NULL,40000000000,NULL,NULL,NULL),(111196,263,33,NULL,NULL,40000000000,NULL,NULL,NULL),(111197,263,34,NULL,'Quick adaptation to ethanol in yeast is possible and can help in understanding ethanol tolerance',NULL,NULL,NULL,NULL),(111198,264,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111199,264,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111200,264,3,NULL,NULL,35,NULL,NULL,NULL),(111201,264,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111202,264,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(111203,264,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111204,264,7,'nyastatin',NULL,NULL,NULL,NULL,NULL),(111205,264,8,NULL,NULL,NULL,NULL,0,NULL),(111206,264,10,NULL,NULL,NULL,NULL,1,NULL),(111207,264,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(111208,264,12,NULL,NULL,NULL,NULL,0,NULL),(111209,265,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111210,265,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111211,265,3,NULL,NULL,976,NULL,NULL,NULL),(111212,265,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111213,265,7,'YPD medium',NULL,NULL,NULL,NULL,NULL),(111214,265,16,NULL,NULL,550,NULL,NULL,NULL),(111215,265,17,NULL,NULL,55,NULL,NULL,NULL),(111216,265,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111217,265,18,'control wells',NULL,NULL,NULL,NULL,NULL),(111218,265,18,'ampicillin',NULL,NULL,NULL,NULL,NULL),(111219,265,18,'tetracyclin',NULL,NULL,NULL,NULL,NULL),(111220,265,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111221,265,23,NULL,NULL,NULL,NULL,0,NULL),(111222,265,32,NULL,NULL,20480000,NULL,NULL,NULL),(111223,265,33,NULL,NULL,5000,NULL,NULL,NULL),(111224,265,34,NULL,'exploitation of new mutations is in this system more important than exploration of the genotypic space',NULL,NULL,NULL,NULL),(111225,266,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111226,266,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111227,266,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111228,267,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111229,267,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111230,267,3,NULL,NULL,2160,NULL,NULL,NULL),(111231,267,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111232,267,7,'increasing salt concentrations',NULL,NULL,NULL,NULL,NULL),(111233,267,23,NULL,NULL,NULL,NULL,0,NULL),(111234,267,34,NULL,'evolutionary rescue in yeast and how this depends on the rate of environmental change and the dispersal grade of the species',NULL,NULL,NULL,NULL),(111235,268,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111236,268,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111237,268,3,NULL,NULL,592,NULL,NULL,NULL),(111238,268,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111239,268,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(111240,268,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111241,268,7,'sterility in YPD medium',NULL,NULL,NULL,NULL,NULL),(111242,268,8,NULL,NULL,NULL,NULL,0,NULL),(111243,268,10,NULL,NULL,NULL,NULL,0,NULL),(111244,268,12,NULL,NULL,NULL,NULL,1,NULL),(111245,268,13,'static',NULL,NULL,NULL,NULL,NULL),(111246,268,14,NULL,NULL,NULL,NULL,0,NULL),(111247,268,16,NULL,NULL,1000,NULL,NULL,NULL),(111248,268,17,NULL,NULL,100,NULL,NULL,NULL),(111249,268,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111250,268,18,'control wells',NULL,NULL,NULL,NULL,NULL),(111251,268,19,NULL,NULL,NULL,NULL,1,NULL),(111252,268,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111253,268,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111254,268,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(111255,268,23,NULL,NULL,NULL,NULL,0,NULL),(111256,268,32,NULL,NULL,12800000,NULL,NULL,NULL),(111257,268,33,NULL,NULL,400000,NULL,NULL,NULL),(111258,268,34,NULL,'background genotype generated during clonal preculture growth can play crucial role in following evolutionary dynamics',NULL,NULL,NULL,NULL),(111259,269,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111260,269,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(111261,269,3,NULL,NULL,2,NULL,NULL,NULL),(111262,269,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111263,269,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(111264,269,6,'YEPD',NULL,NULL,NULL,NULL,NULL),(111265,269,7,'ethanol',NULL,NULL,NULL,NULL,NULL),(111266,269,10,NULL,NULL,NULL,NULL,1,NULL),(111267,269,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(111268,269,12,NULL,NULL,NULL,NULL,0,NULL),(111269,269,16,NULL,NULL,486,NULL,NULL,NULL),(111270,269,17,NULL,NULL,192,NULL,NULL,NULL),(111271,269,19,NULL,NULL,NULL,NULL,0,NULL),(111272,269,23,NULL,NULL,NULL,NULL,0,NULL),(111273,269,32,NULL,NULL,1000000000000,NULL,NULL,NULL),(111274,269,33,NULL,NULL,1000000000000,NULL,NULL,NULL),(111275,269,34,NULL,'Stable ethanol-tolerant yeast was generation by adaptive evolution',NULL,NULL,NULL,NULL),(111276,270,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111277,270,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111278,270,3,NULL,NULL,2044,NULL,NULL,NULL),(111279,270,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111280,270,7,'salt concentrations',NULL,NULL,NULL,NULL,NULL),(111281,270,17,NULL,NULL,12,NULL,NULL,NULL),(111282,270,23,NULL,NULL,NULL,NULL,0,NULL),(111283,270,32,NULL,NULL,10000000,NULL,NULL,NULL),(111284,270,33,NULL,NULL,10,NULL,NULL,NULL),(111285,270,34,NULL,'evolutionary rescue in yeast depends heavily on the population size and a bottleneck for ER in population size was identified.',NULL,NULL,NULL,NULL),(111286,271,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111287,271,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111288,271,3,NULL,NULL,24,NULL,NULL,NULL),(111289,271,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111290,271,7,'glucose',NULL,NULL,NULL,NULL,NULL),(111291,271,7,'sulfate or phosphate limitation',NULL,NULL,NULL,NULL,NULL),(111292,271,16,NULL,NULL,200,NULL,NULL,NULL),(111293,271,21,'spontaneous canavanine resistance',NULL,NULL,NULL,NULL,NULL),(111294,271,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111295,271,23,NULL,NULL,NULL,NULL,1,NULL),(111296,271,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111297,271,25,'Affymetrix Yeast tiling array',NULL,NULL,NULL,NULL,NULL),(111298,271,26,NULL,NULL,10,NULL,NULL,NULL),(111299,271,31,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000303#s5',NULL,NULL,NULL,NULL,NULL),(111300,271,32,NULL,NULL,10000000000,NULL,NULL,NULL),(111301,271,33,NULL,NULL,10000000000,NULL,NULL,NULL),(111302,271,34,NULL,'Diverse set of mutations (SNPs, INDEls and rearrangements) in glucose and fosfate limitation but sulfate is more restricted.',NULL,NULL,NULL,NULL),(111303,272,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111304,272,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111305,272,3,NULL,NULL,8,NULL,NULL,NULL),(111306,272,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111307,272,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(111308,272,16,NULL,NULL,450,NULL,NULL,NULL),(111309,272,17,NULL,NULL,45,NULL,NULL,NULL),(111310,272,18,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111311,272,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(111312,272,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111313,272,23,NULL,NULL,NULL,NULL,1,NULL),(111314,272,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111315,272,25,'Affymetrix Yeast tiling array',NULL,NULL,NULL,NULL,NULL),(111316,272,26,NULL,NULL,5,NULL,NULL,NULL),(111317,272,31,'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11071',NULL,NULL,NULL,NULL,NULL),(111318,272,32,NULL,NULL,1000000000,NULL,NULL,NULL),(111319,272,33,NULL,NULL,100000000,NULL,NULL,NULL),(111320,272,34,NULL,'S. cerevisiae evolving populations show CI. One of the first reports to identify mutations.',NULL,NULL,NULL,NULL),(111321,273,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111322,273,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111323,273,3,NULL,NULL,12,NULL,NULL,NULL),(111324,273,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111325,273,5,'flasks',NULL,NULL,NULL,NULL,NULL),(111326,273,6,'YPD or glucose minimum',NULL,NULL,NULL,NULL,NULL),(111327,273,7,'high salt or glucose limitation',NULL,NULL,NULL,NULL,NULL),(111328,273,8,NULL,NULL,NULL,NULL,0,NULL),(111329,273,10,NULL,NULL,NULL,NULL,0,NULL),(111330,273,12,NULL,NULL,NULL,NULL,0,NULL),(111331,273,14,NULL,NULL,NULL,NULL,0,NULL),(111332,273,16,NULL,NULL,500,NULL,NULL,NULL),(111333,273,17,NULL,NULL,75,NULL,NULL,NULL),(111334,273,18,'resistance',NULL,NULL,NULL,NULL,NULL),(111335,273,18,'barcodes',NULL,NULL,NULL,NULL,NULL),(111336,273,19,NULL,NULL,NULL,NULL,1,NULL),(111337,273,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111338,273,21,'barcodes',NULL,NULL,NULL,NULL,NULL),(111339,273,23,NULL,NULL,NULL,NULL,0,NULL),(111340,273,34,NULL,'Environment-specific adaptation leads to postzygotic isolation in Saccharomyces as a first step of speciation',NULL,NULL,NULL,NULL),(111341,274,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111342,274,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111343,274,3,NULL,NULL,48,NULL,NULL,NULL),(111344,274,4,'MA',NULL,NULL,NULL,NULL,NULL),(111345,274,7,'YPD agar',NULL,NULL,NULL,NULL,NULL),(111346,274,16,NULL,NULL,4800,NULL,NULL,NULL),(111347,274,17,NULL,NULL,800,NULL,NULL,NULL),(111348,274,19,NULL,NULL,NULL,NULL,1,NULL),(111349,274,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111350,274,21,'LacZ activity',NULL,NULL,NULL,NULL,NULL),(111351,274,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(111352,274,23,NULL,NULL,NULL,NULL,0,NULL),(111353,274,32,NULL,NULL,20000000,NULL,NULL,NULL),(111354,274,33,NULL,NULL,1,NULL,NULL,NULL),(111355,274,34,NULL,'Fitness under drift declines and does so increasingly, suggesting synergistic epistasis among deleterious mutations.',NULL,NULL,NULL,NULL),(111356,275,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111357,275,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111358,275,3,NULL,NULL,40,NULL,NULL,NULL),(111359,275,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111360,275,5,'plates',NULL,NULL,NULL,NULL,NULL),(111361,275,5,'tubes',NULL,NULL,NULL,NULL,NULL),(111362,275,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111363,275,6,'CSM',NULL,NULL,NULL,NULL,NULL),(111364,275,7,'mating discrimination',NULL,NULL,NULL,NULL,NULL),(111365,275,8,NULL,NULL,NULL,NULL,0,NULL),(111366,275,10,NULL,NULL,NULL,NULL,0,NULL),(111367,275,12,NULL,NULL,NULL,NULL,1,NULL),(111368,275,13,'plates',NULL,NULL,NULL,NULL,NULL),(111369,275,14,NULL,NULL,NULL,NULL,0,NULL),(111370,275,17,NULL,NULL,180,NULL,NULL,NULL),(111371,275,18,'auxotrophic markers',NULL,NULL,NULL,NULL,NULL),(111372,275,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(111373,275,19,NULL,NULL,NULL,NULL,0,NULL),(111374,275,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(111375,275,23,NULL,NULL,NULL,NULL,0,NULL),(111376,275,33,NULL,NULL,100000,NULL,NULL,NULL),(111377,275,34,NULL,'Mating discrimination can evolve in yeast populations by strong artificial selection',NULL,NULL,NULL,NULL),(111378,276,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111379,276,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111380,276,3,NULL,NULL,32,NULL,NULL,NULL),(111381,276,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111382,276,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(111383,276,6,'minimal medium',NULL,NULL,NULL,NULL,NULL),(111384,276,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(111385,276,7,'complex stress',NULL,NULL,NULL,NULL,NULL),(111386,276,8,NULL,NULL,NULL,NULL,0,NULL),(111387,276,10,NULL,NULL,NULL,NULL,0,NULL),(111388,276,12,NULL,NULL,NULL,NULL,0,NULL),(111389,276,14,NULL,NULL,NULL,NULL,1,NULL),(111390,276,15,'complex stress',NULL,NULL,NULL,NULL,NULL),(111391,276,16,NULL,NULL,300,NULL,NULL,NULL),(111392,276,17,NULL,NULL,110,NULL,NULL,NULL),(111393,276,19,NULL,NULL,NULL,NULL,1,NULL),(111394,276,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111395,276,21,'resistance',NULL,NULL,NULL,NULL,NULL),(111396,276,23,NULL,NULL,NULL,NULL,0,NULL),(111397,276,32,NULL,NULL,1000000000,NULL,NULL,NULL),(111398,276,33,NULL,NULL,1000000000,NULL,NULL,NULL),(111399,276,34,NULL,'By comparing strains with lower sexual reproduction to ones with proficient sexual reproduction and saw a faster adaptation only in harsh conditions',NULL,NULL,NULL,NULL),(111400,277,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111401,277,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111402,277,3,NULL,NULL,100,NULL,NULL,NULL),(111403,277,4,'MA',NULL,NULL,NULL,NULL,NULL),(111404,277,7,'YPD agar',NULL,NULL,NULL,NULL,NULL),(111405,277,16,NULL,NULL,600,NULL,NULL,NULL),(111406,277,17,NULL,NULL,72,NULL,NULL,NULL),(111407,277,21,'geneticin resistance',NULL,NULL,NULL,NULL,NULL),(111408,277,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(111409,277,23,NULL,NULL,NULL,NULL,0,NULL),(111410,277,33,NULL,NULL,1,NULL,NULL,NULL),(111411,277,34,NULL,'Per generation rate of spontaneous mutations is low with high mean fitness costs per mtuation while mutators increases the number of slightly deleterious mutations',NULL,NULL,NULL,NULL),(111412,278,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111413,278,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111414,278,3,NULL,NULL,3,NULL,NULL,NULL),(111415,278,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111416,278,7,'glucose limitation',NULL,NULL,NULL,NULL,NULL),(111417,278,10,NULL,NULL,NULL,NULL,0,NULL),(111418,278,12,NULL,NULL,NULL,NULL,0,NULL),(111419,278,16,NULL,NULL,500,NULL,NULL,NULL),(111420,278,19,NULL,NULL,NULL,NULL,1,NULL),(111421,278,20,'yield',NULL,NULL,NULL,NULL,NULL),(111422,278,23,NULL,NULL,NULL,NULL,0,NULL),(111423,278,34,NULL,'Evolution of transcriptome is highly parallel between 3 different populations of yeast evolving on glucose-minimal medium',NULL,NULL,NULL,NULL),(111424,279,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111425,279,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(111426,279,3,NULL,NULL,7,NULL,NULL,NULL),(111427,279,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111428,279,5,'turbidostat',NULL,NULL,NULL,NULL,NULL),(111429,279,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111430,279,7,'acetic acid',NULL,NULL,NULL,NULL,NULL),(111431,279,10,NULL,NULL,NULL,NULL,1,NULL),(111432,279,11,'linear concentration rise of acetic acid',NULL,NULL,NULL,NULL,NULL),(111433,279,12,NULL,NULL,NULL,NULL,0,NULL),(111434,279,17,NULL,NULL,3,NULL,NULL,NULL),(111435,279,19,NULL,NULL,NULL,NULL,0,NULL),(111436,279,23,NULL,NULL,NULL,NULL,0,NULL),(111437,279,34,NULL,'Bakers yeast can be quickly improved for application in sour dough production through experimental evolution',NULL,NULL,NULL,NULL),(111438,280,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111439,280,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111440,280,3,NULL,NULL,1,NULL,NULL,NULL),(111441,280,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111442,280,5,'chemostat',NULL,NULL,NULL,NULL,NULL),(111443,280,7,'organic phosphate limitation',NULL,NULL,NULL,NULL,NULL),(111444,280,10,NULL,NULL,NULL,NULL,0,NULL),(111445,280,12,NULL,NULL,NULL,NULL,0,NULL),(111446,280,16,NULL,NULL,1000,NULL,NULL,NULL),(111447,280,17,NULL,NULL,240,NULL,NULL,NULL),(111448,280,19,NULL,NULL,NULL,NULL,0,NULL),(111449,280,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(111450,280,23,NULL,NULL,NULL,NULL,0,NULL),(111451,280,32,NULL,NULL,1000000000,NULL,NULL,NULL),(111452,280,33,NULL,NULL,1000000000,NULL,NULL,NULL),(111453,280,34,NULL,'3 steps of evolution were detected in a chemostat, the last one improving clumping as a side-effect.',NULL,NULL,NULL,NULL),(111454,281,1,'Acinetobacter baylyi',NULL,NULL,NULL,NULL,NULL),(111455,281,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111456,281,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111457,281,3,NULL,NULL,30,NULL,NULL,NULL),(111458,281,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111459,281,5,'plates',NULL,NULL,NULL,NULL,NULL),(111460,281,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111461,281,7,'co-evolution between killer',NULL,NULL,NULL,NULL,NULL),(111462,281,7,'sensitive',NULL,NULL,NULL,NULL,NULL),(111463,281,7,'dsRNA virus',NULL,NULL,NULL,NULL,NULL),(111464,281,8,NULL,NULL,NULL,NULL,0,NULL),(111465,281,10,NULL,NULL,NULL,NULL,1,NULL),(111466,281,11,'coevolution',NULL,NULL,NULL,NULL,NULL),(111467,281,12,NULL,NULL,NULL,NULL,1,NULL),(111468,281,13,'plates',NULL,NULL,NULL,NULL,NULL),(111469,281,14,NULL,NULL,NULL,NULL,0,NULL),(111470,281,16,NULL,NULL,500,NULL,NULL,NULL),(111471,281,17,NULL,NULL,140,NULL,NULL,NULL),(111472,281,18,'antibiotc resistance',NULL,NULL,NULL,NULL,NULL),(111473,281,19,NULL,NULL,NULL,NULL,1,NULL),(111474,281,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111475,281,21,'antibiotic resistance',NULL,NULL,NULL,NULL,NULL),(111476,281,23,NULL,NULL,NULL,NULL,0,NULL),(111477,281,34,NULL,'Different levels of co-evolution occured in this tripartite system, between killer and non killer cell and between killer and dsRNA virus',NULL,NULL,NULL,NULL),(111478,282,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(111479,282,1,'Saccharomyces uvarum',NULL,NULL,NULL,NULL,NULL),(111480,282,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111481,282,3,NULL,NULL,3,NULL,NULL,NULL),(111482,282,4,'continuous',NULL,NULL,NULL,NULL,NULL),(111483,282,7,'ammonium limitation',NULL,NULL,NULL,NULL,NULL),(111484,282,16,NULL,NULL,200,NULL,NULL,NULL),(111485,282,17,NULL,NULL,40,NULL,NULL,NULL),(111486,282,21,'GFP',NULL,NULL,NULL,NULL,NULL),(111487,282,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111488,282,23,NULL,NULL,NULL,NULL,1,NULL),(111489,282,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111490,282,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111491,282,25,'Illumina HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111492,282,25,'Agilent CGH array',NULL,NULL,NULL,NULL,NULL),(111493,282,26,NULL,NULL,4,NULL,NULL,NULL),(111494,282,28,NULL,NULL,20,NULL,NULL,NULL),(111495,282,31,'https://www.ncbi.nlm.nih.gov/bioproject/PRJNA172024',NULL,NULL,NULL,NULL,NULL),(111496,282,34,NULL,'interspecific yeast hybrids independently can acquire genome rearrangements for swift and potentially otherwise unavailable evolutionary innovations',NULL,NULL,NULL,NULL),(111497,283,1,'Saccharomyces paradoxus',NULL,NULL,NULL,NULL,NULL),(111498,283,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111499,283,3,NULL,NULL,144,NULL,NULL,NULL),(111500,283,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111501,283,5,'12-well plate',NULL,NULL,NULL,NULL,NULL),(111502,283,7,'carbon limitation',NULL,NULL,NULL,NULL,NULL),(111503,283,10,NULL,NULL,NULL,NULL,0,NULL),(111504,283,12,NULL,NULL,NULL,NULL,0,NULL),(111505,283,16,NULL,NULL,180,NULL,NULL,NULL),(111506,283,17,NULL,NULL,294,NULL,NULL,NULL),(111507,283,19,NULL,NULL,NULL,NULL,0,NULL),(111508,283,23,NULL,NULL,NULL,NULL,0,NULL),(111509,283,34,NULL,'Plastic or direct response of strains adapted to long term starvation is less to novel stressors but if they survived initial novel stress, they adapted most to this new stres',NULL,NULL,NULL,NULL),(111510,283,35,NULL,'robotics, evolutionary rescue',NULL,NULL,NULL,NULL),(111511,284,1,'Saccharomyces paradoxus',NULL,NULL,NULL,NULL,NULL),(111512,284,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111513,284,3,NULL,NULL,780,NULL,NULL,NULL),(111514,284,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111515,284,5,'12-well plate',NULL,NULL,NULL,NULL,NULL),(111516,284,7,'alcohol',NULL,NULL,NULL,NULL,NULL),(111517,284,7,'high temperature',NULL,NULL,NULL,NULL,NULL),(111518,284,7,'pH10',NULL,NULL,NULL,NULL,NULL),(111519,284,7,'salt stress',NULL,NULL,NULL,NULL,NULL),(111520,284,7,'carbon limitation',NULL,NULL,NULL,NULL,NULL),(111521,284,10,NULL,NULL,NULL,NULL,0,NULL),(111522,284,12,NULL,NULL,NULL,NULL,0,NULL),(111523,284,16,NULL,NULL,17,NULL,NULL,NULL),(111524,284,17,NULL,NULL,28,NULL,NULL,NULL),(111525,284,19,NULL,NULL,NULL,NULL,0,NULL),(111526,284,23,NULL,NULL,NULL,NULL,0,NULL),(111527,284,34,NULL,'Plastic or direct response of strains adapted to long term starvation is less to novel stressors but if they survived initial novel stress, they adapted most to this new stres',NULL,NULL,NULL,NULL),(111528,284,35,NULL,'robotics, evolutionary rescue',NULL,NULL,NULL,NULL),(111529,285,1,'Saccharomyces paradoxus',NULL,NULL,NULL,NULL,NULL),(111530,285,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111531,285,3,NULL,NULL,144,NULL,NULL,NULL),(111532,285,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111533,285,5,'12-well plates',NULL,NULL,NULL,NULL,NULL),(111534,285,6,'minimal medium',NULL,NULL,NULL,NULL,NULL),(111535,285,7,'12 carbon sources',NULL,NULL,NULL,NULL,NULL),(111536,285,8,NULL,NULL,NULL,NULL,0,NULL),(111537,285,10,NULL,NULL,NULL,NULL,0,NULL),(111538,285,12,NULL,NULL,NULL,NULL,0,NULL),(111539,285,14,NULL,NULL,NULL,NULL,0,NULL),(111540,285,16,NULL,NULL,180,NULL,NULL,NULL),(111541,285,17,NULL,NULL,294,NULL,NULL,NULL),(111542,285,19,NULL,NULL,NULL,NULL,1,NULL),(111543,285,20,'yield',NULL,NULL,NULL,NULL,NULL),(111544,285,23,NULL,NULL,NULL,NULL,0,NULL),(111545,285,34,NULL,'Mutational degradation explains fitness trade-offs and specialisation.',NULL,NULL,NULL,NULL),(111546,285,35,NULL,'robotics were used',NULL,NULL,NULL,NULL),(111547,286,1,'Saccharomyces paradoxus',NULL,NULL,NULL,NULL,NULL),(111548,286,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111549,286,3,NULL,NULL,96,NULL,NULL,NULL),(111550,286,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111551,286,5,'multiwell plates',NULL,NULL,NULL,NULL,NULL),(111552,286,6,'YPD',NULL,NULL,NULL,NULL,NULL),(111553,286,7,'salt concentration',NULL,NULL,NULL,NULL,NULL),(111554,286,8,NULL,NULL,NULL,NULL,0,NULL),(111555,286,10,NULL,NULL,NULL,NULL,1,NULL),(111556,286,11,'fixed increase in salt',NULL,NULL,NULL,NULL,NULL),(111557,286,12,NULL,NULL,NULL,NULL,0,NULL),(111558,286,14,NULL,NULL,NULL,NULL,0,NULL),(111559,286,17,NULL,NULL,40,NULL,NULL,NULL),(111560,286,19,NULL,NULL,NULL,NULL,1,NULL),(111561,286,20,'colony diameter',NULL,NULL,NULL,NULL,NULL),(111562,286,23,NULL,NULL,NULL,NULL,0,NULL),(111563,286,34,NULL,'Mutations that protect against lethal stress emerge already at lower stress levels and do so more when population is bigger.',NULL,NULL,NULL,NULL),(111564,287,1,'Salmonella enterica',NULL,NULL,NULL,NULL,NULL),(111565,287,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111566,287,3,NULL,NULL,72,NULL,NULL,NULL),(111567,287,3,NULL,NULL,15,NULL,NULL,NULL),(111568,287,4,'MA',NULL,NULL,NULL,NULL,NULL),(111569,287,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111570,287,7,'GC agar',NULL,NULL,NULL,NULL,NULL),(111571,287,7,'compensation for deleterious accumulation in LB',NULL,NULL,NULL,NULL,NULL),(111572,287,16,NULL,NULL,6750,NULL,NULL,NULL),(111573,287,16,NULL,NULL,100,NULL,NULL,NULL),(111574,287,17,NULL,NULL,270,NULL,NULL,NULL),(111575,287,19,NULL,NULL,NULL,NULL,1,NULL),(111576,287,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(111577,287,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111578,287,23,NULL,NULL,NULL,NULL,1,NULL),(111579,287,24,'Hybridization array',NULL,NULL,NULL,NULL,NULL),(111580,287,32,NULL,NULL,100000000,NULL,NULL,NULL),(111581,287,32,NULL,NULL,1000000000,NULL,NULL,NULL),(111582,287,33,NULL,NULL,1,NULL,NULL,NULL),(111583,287,33,NULL,NULL,1000000,NULL,NULL,NULL),(111584,287,34,NULL,'RecA-independent genomic deletions occur 50x more in MMR deficient mutants during MA on solid agar',NULL,NULL,NULL,NULL),(111585,288,1,'Salmonella enterica Typhimurium',NULL,NULL,NULL,NULL,NULL),(111586,288,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111587,288,3,NULL,NULL,5,NULL,NULL,NULL),(111588,288,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111589,288,5,'mice',NULL,NULL,NULL,NULL,NULL),(111590,288,7,'adaption to mice infection',NULL,NULL,NULL,NULL,NULL),(111591,288,8,NULL,NULL,NULL,NULL,0,NULL),(111592,288,10,NULL,NULL,NULL,NULL,1,NULL),(111593,288,11,'induction of immune system',NULL,NULL,NULL,NULL,NULL),(111594,288,12,NULL,NULL,NULL,NULL,1,NULL),(111595,288,13,'mice body',NULL,NULL,NULL,NULL,NULL),(111596,288,14,NULL,NULL,NULL,NULL,1,NULL),(111597,288,15,'in situ evolution',NULL,NULL,NULL,NULL,NULL),(111598,288,17,NULL,NULL,42,NULL,NULL,NULL),(111599,288,18,'barcoding',NULL,NULL,NULL,NULL,NULL),(111600,288,19,NULL,NULL,NULL,NULL,0,NULL),(111601,288,22,'marker divergence',NULL,NULL,NULL,NULL,NULL),(111602,288,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111603,288,23,NULL,NULL,NULL,NULL,1,NULL),(111604,288,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111605,288,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111606,288,26,NULL,NULL,8,NULL,NULL,NULL),(111607,288,28,NULL,NULL,200,NULL,NULL,NULL),(111608,288,31,'https://www.ncbi.nlm.nih.gov/sra?linkname=bioproject_sra_all&from_uid=313139',NULL,NULL,NULL,NULL,NULL),(111609,288,33,NULL,NULL,10000,NULL,NULL,NULL),(111610,288,34,NULL,'Salmonella Typhimurium can quickly undergo patho-adaptive evolution in mice',NULL,NULL,NULL,NULL),(111611,289,1,'Salmonella enterica Typhimurium',NULL,NULL,NULL,NULL,NULL),(111612,289,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111613,289,3,NULL,NULL,12,NULL,NULL,NULL),(111614,289,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111615,289,5,'tubes',NULL,NULL,NULL,NULL,NULL),(111616,289,6,'LB',NULL,NULL,NULL,NULL,NULL),(111617,289,7,'adaptation to knockout gene',NULL,NULL,NULL,NULL,NULL),(111618,289,10,NULL,NULL,NULL,NULL,0,NULL),(111619,289,12,NULL,NULL,NULL,NULL,0,NULL),(111620,289,16,NULL,NULL,300,NULL,NULL,NULL),(111621,289,17,NULL,NULL,30,NULL,NULL,NULL),(111622,289,19,NULL,NULL,NULL,NULL,1,NULL),(111623,289,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(111624,289,22,'targeted',NULL,NULL,NULL,NULL,NULL),(111625,289,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111626,289,23,NULL,NULL,NULL,NULL,1,NULL),(111627,289,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111628,289,25,'Illumina HiSeq2000/2500',NULL,NULL,NULL,NULL,NULL),(111629,289,27,NULL,NULL,24,NULL,NULL,NULL),(111630,289,30,NULL,NULL,NULL,80,NULL,NULL),(111631,289,34,NULL,'H-NS buffers the secretion of SPI-1 excretion system that defines the species',NULL,NULL,NULL,NULL),(111632,290,1,'Salmonella enterica Typhimurium',NULL,NULL,NULL,NULL,NULL),(111633,290,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111634,290,3,NULL,NULL,12,NULL,NULL,NULL),(111635,290,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111636,290,5,'tubes',NULL,NULL,NULL,NULL,NULL),(111637,290,6,'LB',NULL,NULL,NULL,NULL,NULL),(111638,290,7,'batch growth or exponential growth',NULL,NULL,NULL,NULL,NULL),(111639,290,8,NULL,NULL,NULL,NULL,0,NULL),(111640,290,10,NULL,NULL,NULL,NULL,0,NULL),(111641,290,12,NULL,NULL,NULL,NULL,0,NULL),(111642,290,14,NULL,NULL,NULL,NULL,0,NULL),(111643,290,16,NULL,NULL,80,NULL,NULL,NULL),(111644,290,17,NULL,NULL,14,NULL,NULL,NULL),(111645,290,19,NULL,NULL,NULL,NULL,1,NULL),(111646,290,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111647,290,21,'colony morphology',NULL,NULL,NULL,NULL,NULL),(111648,290,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(111649,290,23,NULL,NULL,NULL,NULL,0,NULL),(111650,290,34,NULL,'Salmonella Typhimurium quickly looses a stable colony morphology under laboratory grwoth conditions',NULL,NULL,NULL,NULL),(111651,291,1,'Schizophyllum commune',NULL,NULL,NULL,NULL,NULL),(111652,291,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111653,291,3,NULL,NULL,24,NULL,NULL,NULL),(111654,291,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111655,291,5,'plate',NULL,NULL,NULL,NULL,NULL),(111656,291,6,'minimal medium',NULL,NULL,NULL,NULL,NULL),(111657,291,7,'minimal medium',NULL,NULL,NULL,NULL,NULL),(111658,291,8,NULL,NULL,NULL,NULL,0,NULL),(111659,291,10,NULL,NULL,NULL,NULL,0,NULL),(111660,291,12,NULL,NULL,NULL,NULL,1,NULL),(111661,291,13,'plates',NULL,NULL,NULL,NULL,NULL),(111662,291,14,NULL,NULL,NULL,NULL,0,NULL),(111663,291,16,NULL,NULL,13000,NULL,NULL,NULL),(111664,291,17,NULL,NULL,540,NULL,NULL,NULL),(111665,291,19,NULL,NULL,NULL,NULL,1,NULL),(111666,291,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(111667,291,23,NULL,NULL,NULL,NULL,0,NULL),(111668,291,34,NULL,'Dikaryotic mycelia showed improved fitness over time while haploid monokaryota did not and coevolution between nuclei occured.',NULL,NULL,NULL,NULL),(111669,292,1,'Sf6',NULL,NULL,NULL,NULL,NULL),(111670,292,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111671,292,3,NULL,NULL,10,NULL,NULL,NULL),(111672,292,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111673,292,5,'flasks',NULL,NULL,NULL,NULL,NULL),(111674,292,6,'LB',NULL,NULL,NULL,NULL,NULL),(111675,292,7,'faster development on Shigella flexneri without preferred receptor',NULL,NULL,NULL,NULL,NULL),(111676,292,8,NULL,NULL,NULL,NULL,0,NULL),(111677,292,10,NULL,NULL,NULL,NULL,0,NULL),(111678,292,12,NULL,NULL,NULL,NULL,0,NULL),(111679,292,14,NULL,NULL,NULL,NULL,0,NULL),(111680,292,16,NULL,NULL,200,NULL,NULL,NULL),(111681,292,17,NULL,NULL,20,NULL,NULL,NULL),(111682,292,19,NULL,NULL,NULL,NULL,0,NULL),(111683,292,22,'targeted',NULL,NULL,NULL,NULL,NULL),(111684,292,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111685,292,23,NULL,NULL,NULL,NULL,1,NULL),(111686,292,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111687,292,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(111688,292,26,NULL,NULL,26,NULL,NULL,NULL),(111689,292,34,NULL,'Evolved phages reduce their genome size to allow faster devlopment and packing.',NULL,NULL,NULL,NULL),(111690,293,1,'Sindbis virus',NULL,NULL,NULL,NULL,NULL),(111691,293,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111692,293,3,NULL,NULL,144,NULL,NULL,NULL),(111693,293,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111694,293,5,'25cm² flasks',NULL,NULL,NULL,NULL,NULL),(111695,293,7,'host adaptation',NULL,NULL,NULL,NULL,NULL),(111696,293,8,NULL,NULL,NULL,NULL,0,NULL),(111697,293,10,NULL,NULL,NULL,NULL,1,NULL),(111698,293,11,'changing ratio of permissive host to novel host',NULL,NULL,NULL,NULL,NULL),(111699,293,12,NULL,NULL,NULL,NULL,0,NULL),(111700,293,14,NULL,NULL,NULL,NULL,0,NULL),(111701,293,17,NULL,NULL,50,NULL,NULL,NULL),(111702,293,19,NULL,NULL,NULL,NULL,1,NULL),(111703,293,20,'yield',NULL,NULL,NULL,NULL,NULL),(111704,293,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111705,293,23,NULL,NULL,NULL,NULL,1,NULL),(111706,293,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(111707,293,27,NULL,NULL,145,NULL,NULL,NULL),(111708,293,29,NULL,NULL,4,NULL,NULL,NULL),(111709,293,31,'https://www.ncbi.nlm.nih.gov/nuccore/KT121582',NULL,NULL,NULL,NULL,NULL),(111710,293,33,NULL,NULL,10000,NULL,NULL,NULL),(111711,293,34,NULL,'Gradual transfer to novel hosts improveds fitness improvement and leads to less variable peaks in the fitness landscape.',NULL,NULL,NULL,NULL),(111712,294,1,'Sphingobium sp',NULL,NULL,NULL,NULL,NULL),(111713,294,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111714,294,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111715,294,7,'predation',NULL,NULL,NULL,NULL,NULL),(111716,295,1,'Sporomusa ovata',NULL,NULL,NULL,NULL,NULL),(111717,295,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(111718,295,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111719,296,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111720,296,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111721,297,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111722,297,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111723,297,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111724,297,7,'antibiotic',NULL,NULL,NULL,NULL,NULL),(111725,297,10,NULL,NULL,NULL,NULL,1,NULL),(111726,297,11,'cycling of type of antibiotic',NULL,NULL,NULL,NULL,NULL),(111727,297,12,NULL,NULL,NULL,NULL,1,NULL),(111728,297,13,'plates',NULL,NULL,NULL,NULL,NULL),(111729,298,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111730,298,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111731,298,3,NULL,NULL,35,NULL,NULL,NULL),(111732,298,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111733,298,5,'microtiter plate',NULL,NULL,NULL,NULL,NULL),(111734,298,6,'MHB',NULL,NULL,NULL,NULL,NULL),(111735,298,7,'AMPs or antibiotics (combinations)',NULL,NULL,NULL,NULL,NULL),(111736,298,8,NULL,NULL,NULL,NULL,0,NULL),(111737,298,10,NULL,NULL,NULL,NULL,1,NULL),(111738,298,11,'fixed increasing concentrations',NULL,NULL,NULL,NULL,NULL),(111739,298,12,NULL,NULL,NULL,NULL,0,NULL),(111740,298,14,NULL,NULL,NULL,NULL,0,NULL),(111741,298,17,NULL,NULL,28,NULL,NULL,NULL),(111742,298,18,'colony morphology',NULL,NULL,NULL,NULL,NULL),(111743,298,19,NULL,NULL,NULL,NULL,0,NULL),(111744,298,23,NULL,NULL,NULL,NULL,1,NULL),(111745,298,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111746,298,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111747,298,26,NULL,NULL,17,NULL,NULL,NULL),(111748,298,28,NULL,NULL,134,NULL,NULL,NULL),(111749,298,32,NULL,NULL,2800000000,NULL,NULL,NULL),(111750,298,33,NULL,NULL,28000000,NULL,NULL,NULL),(111751,298,34,NULL,'S aureus readily evolves increased resistance to AMPs targetting cell wall/membrane. Distinct mutations in genes responsible for cell enveloppe synthesis result in AMP resistance that causes increased survival in an insect model.',NULL,NULL,NULL,NULL),(111752,299,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111753,299,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111754,299,3,NULL,NULL,32,NULL,NULL,NULL),(111755,299,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111756,299,5,'96-well plate',NULL,NULL,NULL,NULL,NULL),(111757,299,6,'mammalian ionic environment medium',NULL,NULL,NULL,NULL,NULL),(111758,299,7,'AMP',NULL,NULL,NULL,NULL,NULL),(111759,299,8,NULL,NULL,NULL,NULL,0,NULL),(111760,299,10,NULL,NULL,NULL,NULL,1,NULL),(111761,299,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(111762,299,12,NULL,NULL,NULL,NULL,0,NULL),(111763,299,14,NULL,NULL,NULL,NULL,1,NULL),(111764,299,15,'mammalian medium',NULL,NULL,NULL,NULL,NULL),(111765,299,16,NULL,NULL,322,NULL,NULL,NULL),(111766,299,19,NULL,NULL,NULL,NULL,1,NULL),(111767,299,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(111768,299,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111769,299,23,NULL,NULL,NULL,NULL,1,NULL),(111770,299,34,NULL,'AMP pressure selects for increased resistance without apparent costs, cross resistance to antibiotics and maintaining virulence',NULL,NULL,NULL,NULL),(111771,300,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111772,300,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111773,300,3,NULL,NULL,36,NULL,NULL,NULL),(111774,300,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111775,300,5,'multiwell plates',NULL,NULL,NULL,NULL,NULL),(111776,300,6,'modified LB',NULL,NULL,NULL,NULL,NULL),(111777,300,7,'antibiotics combinations',NULL,NULL,NULL,NULL,NULL),(111778,300,10,NULL,NULL,NULL,NULL,1,NULL),(111779,300,11,'increasing concentrations',NULL,NULL,NULL,NULL,NULL),(111780,300,12,NULL,NULL,NULL,NULL,0,NULL),(111781,300,17,NULL,NULL,10,NULL,NULL,NULL),(111782,300,19,NULL,NULL,NULL,NULL,0,NULL),(111783,300,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111784,300,23,NULL,NULL,NULL,NULL,1,NULL),(111785,300,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111786,300,24,'Ion Torrent',NULL,NULL,NULL,NULL,NULL),(111787,300,26,NULL,NULL,37,NULL,NULL,NULL),(111788,300,28,NULL,NULL,125,NULL,NULL,NULL),(111789,300,34,NULL,'Evolution of resistance is slowed down by combination therapy regardless of the interaction type between the antibiotics at low subMIC concentration yet the extent to which resistance evolves to a combination depends on the interaction between the antbiotics which moreover can change.',NULL,NULL,NULL,NULL),(111810,302,1,'Enterococcus faecalis',NULL,NULL,NULL,NULL,NULL),(111811,302,1,'Staphylococcus aureus',NULL,NULL,NULL,NULL,NULL),(111812,302,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111813,302,3,NULL,NULL,15,NULL,NULL,NULL),(111814,302,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111815,302,5,'worm',NULL,NULL,NULL,NULL,NULL),(111816,302,6,'TSB/THB',NULL,NULL,NULL,NULL,NULL),(111817,302,7,'co-evolution',NULL,NULL,NULL,NULL,NULL),(111818,302,8,NULL,NULL,NULL,NULL,0,NULL),(111819,302,10,NULL,NULL,NULL,NULL,1,NULL),(111820,302,11,'in situ',NULL,NULL,NULL,NULL,NULL),(111821,302,12,NULL,NULL,NULL,NULL,1,NULL),(111822,302,13,'in situ',NULL,NULL,NULL,NULL,NULL),(111823,302,14,NULL,NULL,NULL,NULL,1,NULL),(111824,302,15,'in situ',NULL,NULL,NULL,NULL,NULL),(111825,302,17,NULL,NULL,20,NULL,NULL,NULL),(111826,302,18,'rifampicin selection',NULL,NULL,NULL,NULL,NULL),(111827,302,19,NULL,NULL,NULL,NULL,1,NULL),(111828,302,20,'yield',NULL,NULL,NULL,NULL,NULL),(111829,302,21,'rifampicin resistance',NULL,NULL,NULL,NULL,NULL),(111830,302,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111831,302,23,NULL,NULL,NULL,NULL,1,NULL),(111832,302,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111833,302,25,'HiSeq4000',NULL,NULL,NULL,NULL,NULL),(111834,302,27,NULL,NULL,42,NULL,NULL,NULL),(111835,302,31,'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJEB13385',NULL,NULL,NULL,NULL,NULL),(111836,302,34,NULL,'Fluctuating selection dynamics of local adaption rules the co-evolutionary dynamics between defence microbes and pathogens',NULL,NULL,NULL,NULL),(111893,305,1,'S-TIM4 myovirus',NULL,NULL,NULL,NULL,NULL),(111894,305,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111895,305,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111896,306,1,'Streptococcus mitis',NULL,NULL,NULL,NULL,NULL),(111897,306,1,'Streptococcus oralis',NULL,NULL,NULL,NULL,NULL),(111898,306,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111899,306,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111900,307,1,'Streptococcus pyogenes',NULL,NULL,NULL,NULL,NULL),(111901,307,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111902,307,3,NULL,NULL,15,NULL,NULL,NULL),(111903,307,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111904,307,5,'plates',NULL,NULL,NULL,NULL,NULL),(111905,307,6,'MHB',NULL,NULL,NULL,NULL,NULL),(111906,307,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(111907,307,8,NULL,NULL,NULL,NULL,0,NULL),(111908,307,10,NULL,NULL,NULL,NULL,1,NULL),(111909,307,11,'increasing concentration',NULL,NULL,NULL,NULL,NULL),(111910,307,12,NULL,NULL,NULL,NULL,1,NULL),(111911,307,13,'agar plates',NULL,NULL,NULL,NULL,NULL),(111912,307,14,NULL,NULL,NULL,NULL,0,NULL),(111913,307,19,NULL,NULL,NULL,NULL,0,NULL),(111914,307,23,NULL,NULL,NULL,NULL,0,NULL),(111915,307,34,NULL,'Development of a gradient plate method to evolve antibiotic resistance.',NULL,NULL,NULL,NULL),(111916,307,35,NULL,'antibiotic gradient plate very similar to Kishony',NULL,NULL,NULL,NULL),(111917,308,1,'Streptomyces coelicolor',NULL,NULL,NULL,NULL,NULL),(111918,308,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(111919,308,3,NULL,NULL,12,NULL,NULL,NULL),(111920,308,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111921,308,5,'plates',NULL,NULL,NULL,NULL,NULL),(111922,308,6,'SFM',NULL,NULL,NULL,NULL,NULL),(111923,308,7,'submic streptomycin',NULL,NULL,NULL,NULL,NULL),(111924,308,8,NULL,NULL,NULL,NULL,0,NULL),(111925,308,10,NULL,NULL,NULL,NULL,0,NULL),(111926,308,12,NULL,NULL,NULL,NULL,1,NULL),(111927,308,13,'plates',NULL,NULL,NULL,NULL,NULL),(111928,308,14,NULL,NULL,NULL,NULL,0,NULL),(111929,308,16,NULL,NULL,500,NULL,NULL,NULL),(111930,308,17,NULL,NULL,125,NULL,NULL,NULL),(111931,308,19,NULL,NULL,NULL,NULL,1,NULL),(111932,308,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111933,308,21,'resistance',NULL,NULL,NULL,NULL,NULL),(111934,308,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111935,308,23,NULL,NULL,NULL,NULL,1,NULL),(111936,308,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111937,308,25,'HiSeq4000',NULL,NULL,NULL,NULL,NULL),(111938,308,26,NULL,NULL,22,NULL,NULL,NULL),(111939,308,30,NULL,NULL,NULL,50,NULL,NULL),(111940,308,33,NULL,NULL,100000,NULL,NULL,NULL),(111941,308,34,NULL,'Selection at submic can lead to resistances above mic without tradeoff costs although spontaneous resistances carry costs.',NULL,NULL,NULL,NULL),(111942,309,1,'Sulfolobus solfataricus',NULL,NULL,NULL,NULL,NULL),(111943,309,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111944,309,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111945,309,5,'flasks',NULL,NULL,NULL,NULL,NULL),(111946,309,6,'basal salts medium',NULL,NULL,NULL,NULL,NULL),(111947,309,7,'acidification',NULL,NULL,NULL,NULL,NULL),(111948,309,7,'high temp',NULL,NULL,NULL,NULL,NULL),(111949,309,8,NULL,NULL,NULL,NULL,0,NULL),(111950,309,10,NULL,NULL,NULL,NULL,1,NULL),(111951,309,11,'increasing acidity',NULL,NULL,NULL,NULL,NULL),(111952,309,12,NULL,NULL,NULL,NULL,0,NULL),(111953,309,14,NULL,NULL,NULL,NULL,0,NULL),(111954,309,16,NULL,NULL,200,NULL,NULL,NULL),(111955,309,17,NULL,NULL,1068,NULL,NULL,NULL),(111956,309,18,'plating',NULL,NULL,NULL,NULL,NULL),(111957,309,19,NULL,NULL,NULL,NULL,1,NULL),(111958,309,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(111959,309,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(111960,309,23,NULL,NULL,NULL,NULL,1,NULL),(111961,309,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111962,309,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111963,309,26,NULL,NULL,3,NULL,NULL,NULL),(111964,309,28,NULL,NULL,500,NULL,NULL,NULL),(111965,309,34,NULL,'After 3 years, Sulfolobus solfataricus can grow at pH 0.8 and 80°C',NULL,NULL,NULL,NULL),(111966,310,1,'Thermococcus onnurineus',NULL,NULL,NULL,NULL,NULL),(111967,310,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(111968,311,1,'Thermotoga maritima',NULL,NULL,NULL,NULL,NULL),(111969,311,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111970,311,7,'maltose catabolism',NULL,NULL,NULL,NULL,NULL),(111971,311,8,NULL,NULL,NULL,NULL,0,NULL),(111972,311,10,NULL,NULL,NULL,NULL,0,NULL),(111973,311,12,NULL,NULL,NULL,NULL,0,NULL),(111974,311,14,NULL,NULL,NULL,NULL,0,NULL),(111975,311,19,NULL,NULL,NULL,NULL,0,NULL),(111976,311,23,NULL,NULL,NULL,NULL,1,NULL),(111977,311,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(111978,311,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(111979,311,26,NULL,NULL,1,NULL,NULL,NULL),(111980,311,34,NULL,'Genome instability underlies adaptation to maltose',NULL,NULL,NULL,NULL),(111981,312,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(111982,312,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(111983,312,3,NULL,NULL,22,NULL,NULL,NULL),(111984,312,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(111985,312,5,'Nicotiana tabacum',NULL,NULL,NULL,NULL,NULL),(111986,312,6,'Nicotiana tabacum',NULL,NULL,NULL,NULL,NULL),(111987,312,7,'novel genes',NULL,NULL,NULL,NULL,NULL),(111988,312,8,NULL,NULL,NULL,NULL,1,NULL),(111989,312,9,'novel genes',NULL,NULL,NULL,NULL,NULL),(111990,312,10,NULL,NULL,NULL,NULL,0,NULL),(111991,312,12,NULL,NULL,NULL,NULL,1,NULL),(111992,312,13,'in situe',NULL,NULL,NULL,NULL,NULL),(111993,312,14,NULL,NULL,NULL,NULL,1,NULL),(111994,312,15,'in situ',NULL,NULL,NULL,NULL,NULL),(111995,312,17,NULL,NULL,189,NULL,NULL,NULL),(111996,312,19,NULL,NULL,NULL,NULL,1,NULL),(111997,312,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(111998,312,21,'fluorescence',NULL,NULL,NULL,NULL,NULL),(111999,312,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112000,312,23,NULL,NULL,NULL,NULL,1,NULL),(112001,312,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112002,312,25,'HiSeq2500',NULL,NULL,NULL,NULL,NULL),(112003,312,27,NULL,NULL,36,NULL,NULL,NULL),(112004,312,29,NULL,NULL,40000,NULL,NULL,NULL),(112005,312,30,NULL,NULL,NULL,1,NULL,NULL),(112006,312,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP090412',NULL,NULL,NULL,NULL,NULL),(112007,312,34,NULL,'HGT can be highly constrained in RNA plant virusses but also beneficial',NULL,NULL,NULL,NULL),(112008,313,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(112009,313,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112010,313,3,NULL,NULL,7,NULL,NULL,NULL),(112011,313,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112012,313,5,'Arabidopsis thaliana',NULL,NULL,NULL,NULL,NULL),(112013,313,6,'Arabidopsis thaliana',NULL,NULL,NULL,NULL,NULL),(112014,313,7,'new host',NULL,NULL,NULL,NULL,NULL),(112015,313,8,NULL,NULL,NULL,NULL,0,NULL),(112016,313,10,NULL,NULL,NULL,NULL,0,NULL),(112017,313,12,NULL,NULL,NULL,NULL,1,NULL),(112018,313,13,'in situ',NULL,NULL,NULL,NULL,NULL),(112019,313,14,NULL,NULL,NULL,NULL,1,NULL),(112020,313,15,'in situ',NULL,NULL,NULL,NULL,NULL),(112021,313,17,NULL,NULL,119,NULL,NULL,NULL),(112022,313,19,NULL,NULL,NULL,NULL,1,NULL),(112023,313,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112024,313,21,'yield',NULL,NULL,NULL,NULL,NULL),(112025,313,23,NULL,NULL,NULL,NULL,1,NULL),(112026,313,34,NULL,'Fitness landscapes are not static but shift from one environment to another leading to small but significant changes that explain viral specification.\n\nHigh-order epistasis is prevalent and can result in synthetic lethality and thus holes in the landscape.\n\nWhile most lineages went extinct, one prevailed and acquired 5 mutations.',NULL,NULL,NULL,NULL),(112027,314,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(112028,314,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112029,314,3,NULL,NULL,15,NULL,NULL,NULL),(112030,314,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112031,314,5,'N. tabacum plants',NULL,NULL,NULL,NULL,NULL),(112032,314,6,'N. tabacum plants',NULL,NULL,NULL,NULL,NULL),(112033,314,7,'plant infectivity with gene duplication',NULL,NULL,NULL,NULL,NULL),(112034,314,10,NULL,NULL,NULL,NULL,0,NULL),(112035,314,12,NULL,NULL,NULL,NULL,1,NULL),(112036,314,13,'plant',NULL,NULL,NULL,NULL,NULL),(112037,314,17,NULL,NULL,189,NULL,NULL,NULL),(112038,314,19,NULL,NULL,NULL,NULL,1,NULL),(112039,314,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112040,314,21,'RT-qPCR',NULL,NULL,NULL,NULL,NULL),(112041,314,22,'targeted',NULL,NULL,NULL,NULL,NULL),(112042,314,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112043,314,23,NULL,NULL,NULL,NULL,1,NULL),(112044,314,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112045,314,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(112046,314,27,NULL,NULL,18,NULL,NULL,NULL),(112047,314,30,NULL,NULL,NULL,1,NULL,NULL),(112048,314,34,NULL,'Gene order is strictly regulated in TEV and any change is severely constrained by costs that cannot even be recovered in a long-term evolution experiment despite the signs of adaptive evolution.',NULL,NULL,NULL,NULL),(112049,315,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(112050,315,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112051,315,3,NULL,NULL,21,NULL,NULL,NULL),(112052,315,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112053,315,5,'N. tabacum plants',NULL,NULL,NULL,NULL,NULL),(112054,315,6,'N. tabacum plants',NULL,NULL,NULL,NULL,NULL),(112055,315,7,'plant infectivity with gene misorder',NULL,NULL,NULL,NULL,NULL),(112056,315,10,NULL,NULL,NULL,NULL,0,NULL),(112057,315,12,NULL,NULL,NULL,NULL,1,NULL),(112058,315,13,'plant',NULL,NULL,NULL,NULL,NULL),(112059,315,17,NULL,NULL,252,NULL,NULL,NULL),(112060,315,19,NULL,NULL,NULL,NULL,1,NULL),(112061,315,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112062,315,21,'RT-qPCR',NULL,NULL,NULL,NULL,NULL),(112063,315,22,'targeted',NULL,NULL,NULL,NULL,NULL),(112064,315,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112065,315,23,NULL,NULL,NULL,NULL,1,NULL),(112066,315,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112067,315,25,'HiSeq',NULL,NULL,NULL,NULL,NULL),(112068,315,27,NULL,NULL,21,NULL,NULL,NULL),(112069,315,30,NULL,NULL,NULL,1,NULL,NULL),(112070,315,34,NULL,'Gene order is strictly regulated in TEV and any change is severely constrained by costs that cannot even be recovered in a long-term evolution experiment despite the signs of adaptive evolution.',NULL,NULL,NULL,NULL),(112090,317,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(112091,317,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112092,317,3,NULL,NULL,15,NULL,NULL,NULL),(112093,317,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112094,317,5,'Arabidopsis thaliana',NULL,NULL,NULL,NULL,NULL),(112095,317,6,'Arabidopsis thaliana',NULL,NULL,NULL,NULL,NULL),(112096,317,7,'host adaptation',NULL,NULL,NULL,NULL,NULL),(112097,317,8,NULL,NULL,NULL,NULL,0,NULL),(112098,317,10,NULL,NULL,NULL,NULL,0,NULL),(112099,317,12,NULL,NULL,NULL,NULL,1,NULL),(112100,317,13,'plant',NULL,NULL,NULL,NULL,NULL),(112101,317,14,NULL,NULL,NULL,NULL,0,NULL),(112102,317,17,NULL,NULL,315,NULL,NULL,NULL),(112103,317,19,NULL,NULL,NULL,NULL,1,NULL),(112104,317,20,'Malthusian growth rate',NULL,NULL,NULL,NULL,NULL),(112105,317,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112106,317,23,NULL,NULL,NULL,NULL,1,NULL),(112107,317,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(112108,317,27,NULL,NULL,15,NULL,NULL,NULL),(112109,317,34,NULL,'Experimental evolution on local hosts lead to generalist for hard-to-infect hosts and trade-offs in other conditions',NULL,NULL,NULL,NULL),(112110,317,35,NULL,'follow-up on transcriptome effects: Hillung J, García-García F, Dopazo J, Cuevas JM, Elena SF. 2016. The transcriptomics of an experimentally evolved plant-virus interaction. Sci Rep 6:24901.',NULL,NULL,NULL,NULL),(112132,319,1,'Tobacco etch virus',NULL,NULL,NULL,NULL,NULL),(112133,319,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112134,319,3,NULL,NULL,20,NULL,NULL,NULL),(112135,319,4,'MA',NULL,NULL,NULL,NULL,NULL),(112136,319,7,'infection of Chenopodium quinoa',NULL,NULL,NULL,NULL,NULL),(112137,319,17,NULL,NULL,77,NULL,NULL,NULL),(112138,319,19,NULL,NULL,NULL,NULL,1,NULL),(112139,319,20,'Malthusian growth rate',NULL,NULL,NULL,NULL,NULL),(112140,319,22,'deducted from phenotype/fitness',NULL,NULL,NULL,NULL,NULL),(112141,319,23,NULL,NULL,NULL,NULL,0,NULL),(112142,319,33,NULL,NULL,1,NULL,NULL,NULL),(112143,319,34,NULL,'The dynamics of fitness decline in MA experiments indicate that antagonistic epistasis is at play between deleterious mutations',NULL,NULL,NULL,NULL),(112144,320,1,'Trichoderma citrinoviride',NULL,NULL,NULL,NULL,NULL),(112145,320,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112146,321,1,'Trichoderma citrinoviride',NULL,NULL,NULL,NULL,NULL),(112147,321,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(112148,321,3,NULL,NULL,10,NULL,NULL,NULL),(112149,321,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112150,321,5,'tubes',NULL,NULL,NULL,NULL,NULL),(112151,321,6,'minimal filter paper medium',NULL,NULL,NULL,NULL,NULL),(112152,321,7,'paper cellulose as carbon source',NULL,NULL,NULL,NULL,NULL),(112153,321,10,NULL,NULL,NULL,NULL,0,NULL),(112154,321,12,NULL,NULL,NULL,NULL,0,NULL),(112155,321,16,NULL,NULL,540,NULL,NULL,NULL),(112156,321,17,NULL,NULL,125,NULL,NULL,NULL),(112157,321,19,NULL,NULL,NULL,NULL,0,NULL),(112158,321,23,NULL,NULL,NULL,NULL,0,NULL),(112159,321,32,NULL,NULL,121000000,NULL,NULL,NULL),(112160,321,33,NULL,NULL,5000000,NULL,NULL,NULL),(112161,321,34,NULL,'Using experimental evolution to increase cellulose deconstruction, a complex multistep process by T. citrinoviride.',NULL,NULL,NULL,NULL),(112162,322,1,'Trichodesmium',NULL,NULL,NULL,NULL,NULL),(112163,322,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112164,322,3,NULL,NULL,12,NULL,NULL,NULL),(112165,322,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112166,322,6,'aquil medium',NULL,NULL,NULL,NULL,NULL),(112167,322,7,'CO2 pressure',NULL,NULL,NULL,NULL,NULL),(112168,322,8,NULL,NULL,NULL,NULL,0,NULL),(112169,322,10,NULL,NULL,NULL,NULL,0,NULL),(112170,322,12,NULL,NULL,NULL,NULL,0,NULL),(112171,322,14,NULL,NULL,NULL,NULL,0,NULL),(112172,322,16,NULL,NULL,850,NULL,NULL,NULL),(112173,322,17,NULL,NULL,1642,NULL,NULL,NULL),(112174,322,19,NULL,NULL,NULL,NULL,1,NULL),(112175,322,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(112176,322,23,NULL,NULL,NULL,NULL,0,NULL),(112177,322,32,NULL,NULL,1100000,NULL,NULL,NULL),(112178,322,33,NULL,NULL,700000,NULL,NULL,NULL),(112179,322,34,NULL,'When adapted at elevated CO2 pressures, Trichodesmium also shows increased fitness at normal CO2',NULL,NULL,NULL,NULL),(112180,323,1,'Vesicular stomatitis virus',NULL,NULL,NULL,NULL,NULL),(112181,323,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112182,323,3,NULL,NULL,3,NULL,NULL,NULL),(112183,323,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112184,323,5,'12-well plate',NULL,NULL,NULL,NULL,NULL),(112185,323,6,'Dulbecco’s modified Eagle’s minimum essential medium',NULL,NULL,NULL,NULL,NULL),(112186,323,7,'adaptation to IFN',NULL,NULL,NULL,NULL,NULL),(112187,323,8,NULL,NULL,NULL,NULL,1,NULL),(112188,323,9,'defective matrix protein',NULL,NULL,NULL,NULL,NULL),(112189,323,10,NULL,NULL,NULL,NULL,0,NULL),(112190,323,12,NULL,NULL,NULL,NULL,1,NULL),(112191,323,13,'cell monolayer',NULL,NULL,NULL,NULL,NULL),(112192,323,14,NULL,NULL,NULL,NULL,0,NULL),(112193,323,16,NULL,NULL,40,NULL,NULL,NULL),(112194,323,17,NULL,NULL,20,NULL,NULL,NULL),(112195,323,18,'genotype',NULL,NULL,NULL,NULL,NULL),(112196,323,19,NULL,NULL,NULL,NULL,0,NULL),(112197,323,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112198,323,23,NULL,NULL,NULL,NULL,1,NULL),(112199,323,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(112200,323,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112201,323,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112202,323,26,NULL,NULL,4,NULL,NULL,NULL),(112203,323,27,NULL,NULL,4,NULL,NULL,NULL),(112204,323,28,NULL,NULL,2,NULL,NULL,NULL),(112205,323,29,NULL,NULL,5000,NULL,NULL,NULL),(112206,323,30,NULL,NULL,NULL,0.1,NULL,NULL),(112207,323,33,NULL,NULL,10000,NULL,NULL,NULL),(112208,323,34,NULL,'IFN suppression of VSV with a defective IFN deflector leads to compensatory mutations or just fast growing VSV',NULL,NULL,NULL,NULL),(112209,324,1,'Vesicular stomatitis virus',NULL,NULL,NULL,NULL,NULL),(112210,324,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112211,324,3,NULL,NULL,21,NULL,NULL,NULL),(112212,324,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112213,324,5,'25cm² flasks',NULL,NULL,NULL,NULL,NULL),(112214,324,6,'Dulbecco’s modified Eagle’s minimum essential me- dium with 10 per cent fetal bovine serum',NULL,NULL,NULL,NULL,NULL),(112215,324,6,'1 per cent penicil- lin',NULL,NULL,NULL,NULL,NULL),(112216,324,6,'streptomycin.',NULL,NULL,NULL,NULL,NULL),(112217,324,7,'host',NULL,NULL,NULL,NULL,NULL),(112218,324,10,NULL,NULL,NULL,NULL,0,NULL),(112219,324,11,'Y',NULL,NULL,NULL,NULL,NULL),(112220,324,12,NULL,NULL,NULL,NULL,1,NULL),(112221,324,13,'diverse hosts mixed',NULL,NULL,NULL,NULL,NULL),(112222,324,16,NULL,NULL,100,NULL,NULL,NULL),(112223,324,17,NULL,NULL,25,NULL,NULL,NULL),(112224,324,19,NULL,NULL,NULL,NULL,1,NULL),(112225,324,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112226,324,21,'monoclonal antibody resistance',NULL,NULL,NULL,NULL,NULL),(112227,324,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112228,324,23,NULL,NULL,NULL,NULL,1,NULL),(112229,324,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(112230,324,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112231,324,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(112232,324,27,NULL,NULL,22,NULL,NULL,NULL),(112233,324,29,NULL,NULL,6384,NULL,NULL,NULL),(112234,324,30,NULL,NULL,NULL,1,NULL,NULL),(112235,324,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP066436',NULL,NULL,NULL,NULL,NULL),(112236,324,34,NULL,'Larger number of low variants underly the variance among populations in host-use when evolved under heterogeneous conditions that generally lead to generalists',NULL,NULL,NULL,NULL),(112237,325,1,'Vesicular stomatitis virus',NULL,NULL,NULL,NULL,NULL),(112238,325,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112239,325,3,NULL,NULL,16,NULL,NULL,NULL),(112240,325,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112241,325,5,'25cm² flasks',NULL,NULL,NULL,NULL,NULL),(112242,325,6,'Dulbecco’s modified Eagle’s minimum essential medium',NULL,NULL,NULL,NULL,NULL),(112243,325,7,'host (immune response)',NULL,NULL,NULL,NULL,NULL),(112244,325,8,NULL,NULL,NULL,NULL,0,NULL),(112245,325,10,NULL,NULL,NULL,NULL,1,NULL),(112246,325,11,'cycling between immune deficient',NULL,NULL,NULL,NULL,NULL),(112247,325,11,'efficient hosts',NULL,NULL,NULL,NULL,NULL),(112248,325,12,NULL,NULL,NULL,NULL,1,NULL),(112249,325,13,'monolayer in microfluidic device',NULL,NULL,NULL,NULL,NULL),(112250,325,14,NULL,NULL,NULL,NULL,0,NULL),(112251,325,16,NULL,NULL,100,NULL,NULL,NULL),(112252,325,17,NULL,NULL,50,NULL,NULL,NULL),(112253,325,19,NULL,NULL,NULL,NULL,1,NULL),(112254,325,20,'yield',NULL,NULL,NULL,NULL,NULL),(112255,325,23,NULL,NULL,NULL,NULL,0,NULL),(112256,325,34,NULL,'Host specialisation is more determined by the presence/absence of innate immunity than by the cell type and generalists did not emerge, rather fluctuating specialists',NULL,NULL,NULL,NULL),(112257,326,1,'Vesicular stomatitis virus',NULL,NULL,NULL,NULL,NULL),(112258,326,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112259,326,3,NULL,NULL,5,NULL,NULL,NULL),(112260,326,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112261,326,5,'25cm² flasks',NULL,NULL,NULL,NULL,NULL),(112262,326,6,'Dulbecco’s modified Eagle’s minimum essential medium',NULL,NULL,NULL,NULL,NULL),(112263,326,7,'BHK-21 cells at high MOI',NULL,NULL,NULL,NULL,NULL),(112264,326,8,NULL,NULL,NULL,NULL,0,NULL),(112265,326,10,NULL,NULL,NULL,NULL,0,NULL),(112266,326,12,NULL,NULL,NULL,NULL,0,NULL),(112267,326,14,NULL,NULL,NULL,NULL,0,NULL),(112268,326,17,NULL,NULL,20,NULL,NULL,NULL),(112269,326,19,NULL,NULL,NULL,NULL,1,NULL),(112270,326,20,'yield',NULL,NULL,NULL,NULL,NULL),(112271,326,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112272,326,23,NULL,NULL,NULL,NULL,1,NULL),(112273,326,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(112274,326,27,NULL,NULL,11,NULL,NULL,NULL),(112275,326,29,NULL,NULL,8,NULL,NULL,NULL),(112276,326,30,NULL,NULL,NULL,100,NULL,NULL),(112277,326,34,NULL,'NFDS results in oscillatory cycles of VSV infection where DIPs outcompete full lengths and vica versa.',NULL,NULL,NULL,NULL),(112278,327,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(112279,336,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112280,358,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112297,255,26,NULL,NULL,418,NULL,NULL,NULL),(112298,255,28,NULL,NULL,20,NULL,NULL,NULL),(112299,152,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112300,152,21,'blue white lacZ screening',NULL,NULL,NULL,NULL,NULL),(112301,302,7,'adaptation to worm host',NULL,NULL,NULL,NULL,NULL),(112302,302,17,NULL,NULL,45,NULL,NULL,NULL),(112303,302,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112304,302,26,NULL,NULL,13,NULL,NULL,NULL),(112305,302,33,NULL,NULL,15,NULL,NULL,NULL),(112306,98,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112307,98,24,'Pacbio',NULL,NULL,NULL,NULL,NULL),(112308,98,25,'GAIIx',NULL,NULL,NULL,NULL,NULL),(112309,98,25,'RSII',NULL,NULL,NULL,NULL,NULL),(112310,98,26,NULL,NULL,25,NULL,NULL,NULL),(112311,98,28,NULL,NULL,175,NULL,NULL,NULL),(112312,149,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112313,149,26,NULL,NULL,12,NULL,NULL,NULL),(112314,149,27,NULL,NULL,133,NULL,NULL,NULL),(112315,149,31,'https://www.ncbi.nlm.nih.gov/sra/?term=SRP074322',NULL,NULL,NULL,NULL,NULL),(112316,145,20,'competitive fitness',NULL,NULL,NULL,NULL,NULL),(112317,145,22,'targeted analysis',NULL,NULL,NULL,NULL,NULL),(112318,11,22,'targeted',NULL,NULL,NULL,NULL,NULL),(112319,11,22,'WG analysis',NULL,NULL,NULL,NULL,NULL),(112320,11,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112321,11,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(112322,11,26,NULL,NULL,2,NULL,NULL,NULL),(112323,313,24,'Sanger',NULL,NULL,NULL,NULL,NULL),(112324,313,25,'ABI 3100',NULL,NULL,NULL,NULL,NULL),(112325,313,26,NULL,NULL,1,NULL,NULL,NULL),(112326,635,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112327,32,3,NULL,NULL,87,NULL,NULL,NULL),(112328,32,34,NULL,'Selection is less efficient during range expansions due to genetic drift. However, when imposing artificial selection, beneficial mutations are still produced that can counteract the negative effect of expansion load',NULL,NULL,NULL,NULL),(112329,710,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112330,710,1,'Klebsiella pneumoniae',NULL,NULL,NULL,NULL,NULL),(112331,710,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112332,710,3,NULL,NULL,18,NULL,NULL,NULL),(112333,710,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112334,710,5,'96-well plates',NULL,NULL,NULL,NULL,NULL),(112335,710,6,'MHBII',NULL,NULL,NULL,NULL,NULL),(112336,710,7,'ciprofloxacin',NULL,NULL,NULL,NULL,NULL),(112337,710,8,NULL,NULL,NULL,NULL,1,NULL),(112338,710,9,'Mutator',NULL,NULL,NULL,NULL,NULL),(112339,710,16,NULL,NULL,116,NULL,NULL,NULL),(112340,710,18,'Checkerboard pattern',NULL,NULL,NULL,NULL,NULL),(112341,710,23,NULL,NULL,NULL,NULL,1,NULL),(112342,710,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112343,710,25,'HiSeq4000',NULL,NULL,NULL,NULL,NULL),(112344,710,26,NULL,NULL,5,NULL,NULL,NULL),(112345,710,34,NULL,'Evolution towards resistance to existing antibiotics can lead to resistance to new ones under development, even when these new ones target multiple targets.',NULL,NULL,NULL,NULL),(112346,711,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112347,711,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112348,711,3,NULL,NULL,10,NULL,NULL,NULL),(112349,711,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112350,711,6,'Davis Minimal Broth with dextrose 1g/l',NULL,NULL,NULL,NULL,NULL),(112351,711,7,'Gallium',NULL,NULL,NULL,NULL,NULL),(112352,711,17,NULL,NULL,10,NULL,NULL,NULL),(112353,711,23,NULL,NULL,NULL,NULL,1,NULL),(112354,711,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112355,711,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112356,711,27,NULL,NULL,10,NULL,NULL,NULL),(112357,711,29,NULL,NULL,40,NULL,NULL,NULL),(112358,711,32,NULL,NULL,1000000000,NULL,NULL,NULL),(112359,711,33,NULL,NULL,10000000,NULL,NULL,NULL),(112360,712,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112361,712,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112362,712,3,NULL,NULL,4,NULL,NULL,NULL),(112363,712,4,'continuous',NULL,NULL,NULL,NULL,NULL),(112364,712,6,'M9 glucose',NULL,NULL,NULL,NULL,NULL),(112365,712,7,'gentamicin',NULL,NULL,NULL,NULL,NULL),(112366,712,10,NULL,NULL,NULL,NULL,1,NULL),(112367,712,11,'Increasing concentrations',NULL,NULL,NULL,NULL,NULL),(112368,712,16,NULL,NULL,NULL,NULL,NULL,NULL),(112369,712,17,NULL,NULL,3,NULL,NULL,NULL),(112370,712,23,NULL,NULL,NULL,NULL,1,NULL),(112371,712,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112372,712,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(112373,712,26,NULL,NULL,8,NULL,NULL,NULL),(112374,712,31,'https://www.ncbi.nlm.nih.gov/bioproject/PRJNA429575',NULL,NULL,NULL,NULL,NULL),(112428,15,36,NULL,NULL,NULL,NULL,NULL,'15BurkMA.xlsx'),(112429,15,40,NULL,NULL,NULL,NULL,1,NULL),(112430,721,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(112431,721,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112432,721,3,NULL,NULL,12,NULL,NULL,NULL),(112433,721,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112434,721,40,NULL,NULL,NULL,NULL,1,NULL),(112435,43,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(112436,722,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(112437,722,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112438,722,3,NULL,NULL,24,NULL,NULL,NULL),(112439,722,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112440,722,5,'24-well plate',NULL,NULL,NULL,NULL,NULL),(112441,722,5,'microtiter plates',NULL,NULL,NULL,NULL,NULL),(112442,722,6,'SCFM',NULL,NULL,NULL,NULL,NULL),(112443,722,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(112444,722,31,'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA414086',NULL,NULL,NULL,NULL,NULL),(112445,722,40,NULL,NULL,NULL,NULL,1,NULL),(112446,723,1,'Pseudomonas aeruginosa',NULL,NULL,NULL,NULL,NULL),(112447,723,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112448,723,3,NULL,NULL,39,NULL,NULL,NULL),(112449,723,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112450,723,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(112451,723,7,'ciprofloxacin',NULL,NULL,NULL,NULL,NULL),(112452,723,7,'meropenem',NULL,NULL,NULL,NULL,NULL),(112453,723,7,'tobramycin',NULL,NULL,NULL,NULL,NULL),(112454,723,40,NULL,NULL,NULL,NULL,1,NULL),(112455,724,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112456,724,1,'Salmonella enterica',NULL,NULL,NULL,NULL,NULL),(112457,724,1,'Klebsiella pneumonia',NULL,NULL,NULL,NULL,NULL),(112458,724,1,'Acinetobacter baumannii',NULL,NULL,NULL,NULL,NULL),(112459,724,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112460,724,3,NULL,NULL,310,NULL,NULL,NULL),(112461,724,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112462,724,23,NULL,NULL,NULL,NULL,1,NULL),(112463,724,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112464,724,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112465,724,40,NULL,NULL,NULL,NULL,1,NULL),(112466,725,1,'Xenorhabdus nematophila',NULL,NULL,NULL,NULL,NULL),(112467,725,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112468,725,3,NULL,NULL,16,NULL,NULL,NULL),(112469,725,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112470,725,7,'Galleria mellonella infection with/without mutualistic nematode',NULL,NULL,NULL,NULL,NULL),(112471,725,40,NULL,NULL,NULL,NULL,0,NULL),(112472,726,1,'Saccharomyces cerevisiae',NULL,NULL,NULL,NULL,NULL),(112473,726,2,'Study (the evolution of complex) traits',NULL,NULL,NULL,NULL,NULL),(112474,726,3,NULL,NULL,8,NULL,NULL,NULL),(112475,726,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112476,726,6,'YPG',NULL,NULL,NULL,NULL,NULL),(112477,726,7,'high/low expression of 8 reporter genes',NULL,NULL,NULL,NULL,NULL),(112478,726,17,NULL,NULL,72,NULL,NULL,NULL),(112479,726,22,'segregant analysis',NULL,NULL,NULL,NULL,NULL),(112480,726,23,NULL,NULL,NULL,NULL,1,NULL),(112481,726,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112482,726,25,'NextSeq500',NULL,NULL,NULL,NULL,NULL),(112483,726,40,NULL,NULL,NULL,NULL,0,NULL),(112484,727,1,'Gluconobacter frateurii',NULL,NULL,NULL,NULL,NULL),(112485,727,2,'Optimize strains for application',NULL,NULL,NULL,NULL,NULL),(112486,727,3,NULL,NULL,1,NULL,NULL,NULL),(112487,727,4,'serial transfer',NULL,NULL,NULL,NULL,NULL),(112488,727,7,'temperature',NULL,NULL,NULL,NULL,NULL),(112489,727,7,'high temperature',NULL,NULL,NULL,NULL,NULL),(112490,727,23,NULL,NULL,NULL,NULL,1,NULL),(112491,727,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112492,727,25,'HiSeq2000',NULL,NULL,NULL,NULL,NULL),(112493,727,26,NULL,NULL,1,NULL,NULL,NULL),(112494,727,40,NULL,NULL,NULL,NULL,0,NULL),(112522,737,1,'My new species',NULL,NULL,NULL,NULL,NULL),(112523,737,1,'Another new species',NULL,NULL,NULL,NULL,NULL),(112524,737,2,'The goal of this experiment',NULL,NULL,NULL,NULL,NULL),(112525,737,34,NULL,'The major outcome of this experiment can be described here',NULL,NULL,NULL,NULL),(112526,737,10,NULL,NULL,NULL,NULL,0,NULL),(112859,762,1,'Escherichia coli',NULL,NULL,NULL,NULL,NULL),(112860,762,2,'Study evolution',NULL,NULL,NULL,NULL,NULL),(112861,762,3,NULL,NULL,3,NULL,NULL,NULL),(112862,762,4,'continuous',NULL,NULL,NULL,NULL,NULL),(112863,762,5,'tube',NULL,NULL,NULL,NULL,NULL),(112864,762,6,'LB',NULL,NULL,NULL,NULL,NULL),(112865,762,7,'antibiotics',NULL,NULL,NULL,NULL,NULL),(112866,762,8,NULL,NULL,NULL,NULL,1,NULL),(112867,762,10,NULL,NULL,NULL,NULL,1,NULL),(112868,762,12,NULL,NULL,NULL,NULL,1,NULL),(112869,762,14,NULL,NULL,NULL,NULL,1,NULL),(112870,762,15,'antibiotic combinations',NULL,NULL,NULL,NULL,NULL),(112871,762,17,NULL,NULL,12,NULL,NULL,NULL),(112872,762,19,NULL,NULL,NULL,NULL,1,NULL),(112873,762,20,'growth rate',NULL,NULL,NULL,NULL,NULL),(112874,762,22,'whole-genome analysis',NULL,NULL,NULL,NULL,NULL),(112875,762,23,NULL,NULL,NULL,NULL,1,NULL),(112876,762,24,'Illumina',NULL,NULL,NULL,NULL,NULL),(112877,762,25,'MiSeq',NULL,NULL,NULL,NULL,NULL),(112878,762,31,'https://www.ncbi.nlm.nih.gov/bioproject/PRJNA328094/',NULL,NULL,NULL,NULL,NULL),(112879,762,36,NULL,NULL,NULL,NULL,1,NULL);
/*!40000 ALTER TABLE `experiments_fields` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `experiments_references`
--

DROP TABLE IF EXISTS `experiments_references`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `experiments_references` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `experiment_id` int(11) DEFAULT NULL,
  `reference_id` int(11) DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `uniq_exp_ref` (`experiment_id`,`reference_id`),
  KEY `fk_ref_experiment_id` (`experiment_id`),
  KEY `fk_reference_id` (`reference_id`),
  CONSTRAINT `fk_ref_experiment_id` FOREIGN KEY (`experiment_id`) REFERENCES `experiments` (`id`) ON DELETE CASCADE,
  CONSTRAINT `fk_reference_id` FOREIGN KEY (`reference_id`) REFERENCES `references` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB AUTO_INCREMENT=6031 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `experiments_references`
--

LOCK TABLES `experiments_references` WRITE;
/*!40000 ALTER TABLE `experiments_references` DISABLE KEYS */;
INSERT INTO `experiments_references` VALUES (5251,1,11249),(5252,2,11250),(5253,3,11251),(5254,4,11252),(5255,5,11253),(5970,7,11254),(5257,7,11255),(5258,8,11256),(5259,9,11257),(5982,11,11258),(5261,11,11259),(5262,12,11260),(5984,12,11261),(5983,12,11262),(5265,15,11263),(5967,15,11264),(5267,17,11265),(5268,18,11266),(5269,19,11267),(5270,20,11268),(5969,22,11269),(5272,22,11270),(5273,23,11271),(5274,24,11272),(5276,26,11274),(5974,27,11273),(5277,27,11275),(5278,28,11276),(5279,29,11277),(5280,30,11278),(5281,31,11279),(5282,32,11280),(5996,32,11955),(5283,33,11281),(5284,34,11282),(5285,35,11283),(5286,36,11284),(5287,37,11285),(5987,37,11382),(5988,37,11385),(5989,37,11388),(5288,38,11286),(5289,39,11287),(5292,42,11290),(5293,43,11291),(5295,45,11293),(5296,46,11294),(5297,47,11295),(5298,48,11296),(5299,49,11297),(5300,50,11298),(5301,51,11299),(5302,52,11300),(5303,53,11301),(5304,54,11302),(5305,55,11303),(5306,56,11304),(5307,57,11305),(5308,58,11306),(5309,59,11307),(5310,60,11308),(5311,61,11309),(5312,62,11310),(5313,63,11311),(5314,64,11312),(5315,65,11313),(5316,66,11314),(5963,67,11591),(5318,68,11316),(5319,69,11317),(5320,70,11318),(5321,71,11319),(5322,72,11320),(5323,73,11321),(5324,74,11322),(5960,75,11322),(5326,76,11324),(5327,77,11325),(5328,78,11326),(5329,79,11327),(5976,79,11596),(5330,80,11328),(5331,81,11329),(5332,82,11330),(5333,83,11331),(5334,84,11332),(5961,85,11332),(5336,86,11334),(5962,87,11334),(5338,88,11336),(5339,89,11337),(5340,90,11338),(5341,91,11339),(5342,92,11340),(5343,93,11341),(5344,94,11342),(5345,95,11343),(5346,96,11344),(5347,97,11345),(5975,98,11292),(5348,98,11346),(5349,99,11347),(5350,100,11348),(5351,101,11349),(5980,102,11288),(5352,102,11350),(5353,103,11351),(5354,104,11352),(5355,105,11353),(5356,106,11354),(5357,107,11355),(5358,108,11356),(5359,109,11357),(5360,110,11358),(5361,111,11359),(5362,112,11360),(5363,113,11361),(5364,114,11362),(5365,115,11363),(5366,116,11364),(5367,117,11365),(5368,118,11366),(5369,119,11367),(5370,120,11368),(5371,121,11369),(5372,122,11370),(5373,123,11371),(5374,124,11372),(5375,125,11373),(5376,126,11374),(5377,127,11375),(5378,128,11376),(5379,129,11377),(5380,130,11378),(5381,131,11379),(5382,132,11380),(5383,133,11381),(5385,135,11383),(5386,136,11384),(5388,138,11386),(5389,139,11387),(5391,141,11389),(5392,142,11390),(5393,143,11391),(5981,145,11392),(5395,145,11393),(5396,146,11394),(5397,147,11395),(5979,149,11396),(5399,149,11397),(5400,150,11398),(5401,151,11399),(5402,152,11400),(5971,152,11401),(5404,154,11402),(5405,155,11403),(5406,156,11404),(5407,157,11405),(5408,158,11406),(5409,159,11407),(5410,160,11408),(5411,161,11409),(5412,162,11410),(5413,163,11411),(5414,164,11412),(5415,165,11413),(5416,166,11414),(5417,167,11415),(5418,168,11416),(5419,169,11417),(5420,170,11418),(5421,171,11419),(5422,172,11420),(5423,173,11421),(5424,174,11422),(5425,175,11423),(5426,176,11424),(5427,177,11425),(5428,178,11426),(5429,179,11427),(5430,180,11428),(5431,181,11429),(5432,182,11430),(5433,183,11431),(5434,184,11432),(5435,185,11433),(5436,186,11434),(5437,187,11435),(5438,188,11436),(5439,189,11437),(5440,190,11438),(5441,191,11439),(5442,192,11440),(5443,193,11441),(5444,194,11442),(5445,195,11443),(5446,196,11444),(5447,197,11445),(5448,198,11446),(5449,199,11447),(5450,200,11448),(5451,201,11449),(5452,202,11450),(5453,203,11451),(5454,204,11452),(5455,205,11453),(5456,206,11454),(5457,207,11455),(5458,208,11456),(5459,209,11457),(5460,210,11458),(5461,211,11459),(5462,212,11460),(5463,213,11461),(5464,214,11462),(5465,215,11463),(5466,216,11464),(5467,217,11465),(5957,218,11465),(5469,219,11467),(5470,220,11468),(5471,221,11469),(5472,222,11470),(5473,223,11471),(5474,224,11472),(5475,225,11473),(5476,226,11474),(5477,227,11475),(5478,228,11476),(5479,229,11477),(5480,230,11478),(5481,231,11479),(5482,232,11480),(5483,233,11481),(5484,234,11482),(5485,235,11483),(5486,236,11484),(5487,237,11485),(5488,238,11486),(5489,239,11487),(5490,240,11488),(5491,241,11489),(5492,242,11490),(5493,243,11491),(5494,244,11492),(5986,244,11493),(5985,244,11498),(5496,246,11494),(5498,248,11496),(5499,249,11497),(5501,251,11499),(5503,253,11501),(5998,253,11957),(5504,254,11502),(5968,255,11500),(5505,255,11503),(5506,256,11504),(5507,257,11505),(5508,258,11506),(5509,259,11507),(5510,260,11508),(5511,261,11509),(5512,262,11510),(5513,263,11511),(5514,264,11512),(5515,265,11513),(5995,266,11495),(5516,266,11514),(5517,267,11515),(5518,268,11516),(5519,269,11517),(5520,270,11518),(5521,271,11519),(5522,272,11520),(5523,273,11521),(5524,274,11522),(5525,275,11523),(5526,276,11524),(5527,277,11525),(5528,278,11526),(5529,279,11527),(5530,280,11528),(5531,281,11529),(5532,282,11530),(5533,283,11531),(5958,284,11531),(5535,285,11533),(5536,286,11534),(5537,287,11535),(5538,288,11536),(5539,289,11537),(5540,290,11538),(5541,291,11539),(5542,292,11540),(5543,293,11541),(5544,294,11542),(5545,295,11543),(5546,296,11544),(5547,297,11545),(5548,298,11546),(5965,298,11549),(5966,298,11676),(5549,299,11547),(5550,300,11548),(5552,302,11550),(5972,302,11551),(5973,302,11552),(5555,305,11553),(5556,306,11554),(5557,307,11555),(5558,308,11556),(5559,309,11557),(5560,310,11558),(5561,311,11559),(5562,312,11560),(5563,313,11561),(5990,313,11564),(5991,313,11566),(5564,314,11562),(5959,315,11562),(5567,317,11565),(5569,319,11567),(5570,320,11568),(5571,321,11569),(5572,322,11570),(5573,323,11571),(5574,324,11572),(5575,325,11573),(5576,326,11574),(5577,327,11575),(5578,328,11576),(5579,329,11577),(5580,330,11578),(5581,331,11579),(5582,332,11580),(5583,333,11581),(5584,334,11582),(5585,335,11583),(5586,336,11584),(5588,338,11586),(5589,339,11587),(5590,340,11588),(5591,341,11589),(5592,342,11590),(5594,344,11592),(5595,345,11593),(5596,346,11594),(5597,347,11595),(5599,349,11597),(5600,350,11598),(5601,351,11599),(5602,352,11600),(5603,353,11601),(5604,354,11602),(5605,355,11603),(5606,356,11604),(5607,357,11605),(5608,358,11606),(5609,359,11607),(5610,360,11608),(5611,361,11609),(5613,363,11611),(5614,364,11612),(5615,365,11613),(5616,366,11614),(5617,367,11615),(5618,368,11616),(5619,369,11617),(5620,370,11618),(5621,371,11619),(5622,372,11620),(5623,373,11621),(5624,374,11622),(5625,375,11623),(5626,376,11624),(5627,377,11625),(5628,378,11626),(5629,379,11627),(5630,380,11628),(5631,381,11629),(5632,382,11630),(5634,384,11632),(5635,385,11633),(5636,386,11634),(5637,387,11635),(5638,388,11636),(5639,389,11637),(5641,391,11639),(5642,392,11640),(5643,393,11641),(5644,394,11642),(5645,395,11643),(5646,396,11644),(5647,397,11645),(5649,399,11647),(5651,401,11649),(5652,402,11650),(5653,403,11651),(5654,404,11652),(5657,407,11655),(5658,408,11656),(5659,409,11657),(5660,410,11658),(5662,412,11660),(5663,413,11661),(5664,414,11662),(5666,416,11664),(5667,417,11665),(5668,418,11666),(5669,419,11667),(5670,420,11668),(5671,421,11669),(5977,422,11610),(5672,422,11670),(5673,423,11671),(5674,424,11672),(5675,425,11673),(5676,426,11674),(5677,427,11675),(5679,429,11677),(5680,430,11678),(5681,431,11679),(5682,432,11680),(5683,433,11681),(5684,434,11682),(5685,435,11683),(5686,436,11684),(5687,437,11685),(5689,439,11687),(5690,440,11688),(5691,441,11689),(5692,442,11690),(5693,443,11691),(5694,444,11692),(5695,445,11693),(5696,446,11694),(5697,447,11695),(5698,448,11696),(5699,449,11697),(5700,450,11698),(5701,451,11699),(5702,452,11700),(5703,453,11701),(5704,454,11702),(5705,455,11703),(5706,456,11704),(5707,457,11705),(5708,458,11706),(5710,460,11708),(5711,461,11709),(5712,462,11710),(5713,463,11711),(5714,464,11712),(5715,465,11713),(5716,466,11714),(5717,467,11715),(5718,468,11716),(5719,469,11717),(5720,470,11718),(5721,471,11719),(5722,472,11720),(5723,473,11721),(5724,474,11722),(5725,475,11723),(5726,476,11724),(5728,478,11726),(5729,479,11727),(5731,481,11729),(5732,482,11730),(5733,483,11731),(5734,484,11732),(5735,485,11733),(5736,486,11734),(5737,487,11735),(5739,489,11737),(5740,490,11738),(5741,491,11739),(5742,492,11740),(5743,493,11741),(5744,494,11742),(5745,495,11743),(5746,496,11744),(5747,497,11745),(5748,498,11746),(5749,499,11747),(5750,500,11748),(5752,502,11750),(5753,503,11751),(5755,505,11753),(5756,506,11754),(5757,507,11755),(5758,508,11756),(5759,509,11757),(5760,510,11758),(5762,512,11760),(5763,513,11761),(5764,514,11762),(5765,515,11763),(5766,516,11764),(5768,518,11766),(5769,519,11767),(5770,520,11768),(5771,521,11769),(5772,522,11770),(5773,523,11771),(5775,525,11773),(5776,526,11774),(5777,527,11775),(5778,528,11776),(5779,529,11777),(5780,530,11778),(5781,531,11779),(5782,532,11780),(5784,534,11782),(5785,535,11783),(5786,536,11784),(5787,537,11785),(5788,538,11786),(5789,539,11787),(5790,540,11788),(5791,541,11789),(5793,543,11791),(5794,544,11792),(5795,545,11793),(5796,546,11794),(5797,547,11795),(5798,548,11796),(5799,549,11797),(5800,550,11798),(5801,551,11799),(5802,552,11800),(5804,554,11802),(5805,555,11803),(5806,556,11804),(5807,557,11805),(5808,558,11806),(5809,559,11807),(5810,560,11808),(5811,561,11809),(5812,562,11810),(5813,563,11811),(5814,564,11812),(5815,565,11813),(5816,566,11814),(5817,567,11815),(5818,568,11816),(5819,569,11817),(5820,570,11818),(5821,571,11819),(5822,572,11820),(5823,573,11821),(5824,574,11822),(5826,576,11824),(5827,577,11825),(5828,578,11826),(5829,579,11827),(5830,580,11828),(5831,581,11829),(5832,582,11830),(5833,583,11831),(5834,584,11832),(5835,585,11833),(5836,586,11834),(5837,587,11835),(5838,588,11836),(5840,590,11838),(5841,591,11839),(5842,592,11840),(5843,593,11841),(5844,594,11842),(5845,595,11843),(5846,596,11844),(5847,597,11845),(5848,598,11846),(5849,599,11847),(5853,603,11851),(5854,604,11852),(5855,605,11853),(5856,606,11854),(5857,607,11855),(5858,608,11856),(5860,610,11858),(5861,611,11859),(5862,612,11860),(5866,616,11864),(5867,617,11865),(5868,618,11866),(5869,619,11867),(5870,620,11868),(5871,621,11869),(5872,622,11870),(5873,623,11871),(5874,624,11872),(5875,625,11873),(5876,626,11874),(5877,627,11875),(5879,629,11877),(5880,630,11878),(5882,632,11880),(5883,633,11881),(5884,634,11882),(5994,635,11861),(5993,635,11862),(5992,635,11879),(5885,635,11883),(5886,636,11884),(5888,638,11886),(5891,641,11889),(5892,642,11890),(5893,643,11891),(5894,644,11892),(5895,645,11893),(5896,646,11894),(5897,647,11895),(5898,648,11896),(5899,649,11897),(5900,650,11898),(5901,651,11899),(5902,652,11900),(5903,653,11901),(5904,654,11902),(5905,655,11903),(5906,656,11904),(5907,657,11905),(5908,658,11906),(5909,659,11907),(5910,660,11908),(5911,661,11909),(5912,662,11910),(5913,663,11911),(5914,664,11912),(5915,665,11913),(5916,666,11914),(5917,667,11915),(5918,668,11916),(5919,669,11917),(5920,670,11918),(5921,671,11919),(5922,672,11920),(5924,674,11922),(5925,675,11923),(5926,676,11924),(5927,677,11925),(5928,678,11926),(5929,679,11927),(5930,680,11928),(5931,681,11929),(5932,682,11930),(5933,683,11931),(5934,684,11932),(5935,685,11933),(5936,686,11934),(5937,687,11935),(5938,688,11936),(5939,689,11937),(5940,690,11938),(5941,691,11939),(5942,692,11940),(5943,693,11941),(5944,694,11942),(5946,696,11944),(5947,697,11945),(5949,699,11947),(5950,700,11948),(5952,702,11950),(5954,704,11952),(5955,705,11953),(5956,706,11954),(5997,710,11956),(5999,711,11958),(6000,712,11959),(6012,721,11964),(6013,722,11965),(6014,723,11966),(6015,724,11967),(6016,725,11968),(6017,725,11969),(6018,726,11970),(6019,727,11971),(6020,727,11972),(6025,737,11954),(6026,737,11975),(6030,762,11982);
/*!40000 ALTER TABLE `experiments_references` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `fields`
--

DROP TABLE IF EXISTS `fields`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `fields` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `unit` varchar(100) COLLATE utf8_unicode_ci DEFAULT NULL,
  `description` text COLLATE utf8_unicode_ci DEFAULT NULL,
  `type_column` enum('value_VARCHAR','value_TEXT','value_INT','value_DOUBLE','value_BOOL','value_ATTACH') COLLATE utf8_unicode_ci DEFAULT NULL,
  `options` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `link` tinyint(1) NOT NULL DEFAULT 0,
  `required` tinyint(1) NOT NULL DEFAULT 0,
  `group` tinyint(1) DEFAULT 0,
  `weight` int(4) DEFAULT NULL,
  `group_id` int(11) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `fk_group_id` (`group_id`),
  CONSTRAINT `fk_group_id` FOREIGN KEY (`group_id`) REFERENCES `fields` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB AUTO_INCREMENT=73 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `fields`
--

LOCK TABLES `fields` WRITE;
/*!40000 ALTER TABLE `fields` DISABLE KEYS */;
INSERT INTO `fields` VALUES (1,'Species','','','value_VARCHAR','',0,1,0,1,NULL),(2,'Goal','','Goal of experiment','value_VARCHAR','Study (the evolution of complex) traits, Study evolution, Optimize strains for application',0,1,0,2,NULL),(3,'Number of Lines','','','value_INT','',0,1,0,3,NULL),(4,'Setup of the experiment','','','value_VARCHAR','serial transfer, continuous, MA',0,1,0,4,NULL),(5,'Recipient','','','value_VARCHAR','',0,0,0,5,NULL),(6,'Medium','','','value_VARCHAR','',0,0,0,6,NULL),(7,'Selective Condition','','','value_VARCHAR','',0,0,0,7,NULL),(8,'Cripple Mutant','','','value_BOOL','',0,0,1,8,NULL),(9,'How Cripple Mutant','','','value_VARCHAR','',0,0,0,9,8),(10,'Changing Environments','','','value_BOOL','',0,0,1,10,NULL),(11,'How Changing Environment','','','value_VARCHAR','',0,0,0,11,10),(12,'Structured or Heterogeneous Environment','','','value_BOOL','',0,0,1,12,NULL),(13,'How Structured or Heterogeneous Environment','','','value_VARCHAR','',0,0,0,13,12),(14,'Natural conditions/complex environments','','','value_BOOL','',0,0,1,14,NULL),(15,'How Natural conditions/complex Environment','','','value_VARCHAR','',0,0,0,15,14),(16,'Number of Generations','','','value_INT','',0,0,0,16,NULL),(17,'Absolute Time','days','','value_INT','',0,0,0,17,NULL),(18,'Determination of Contamination','','','value_VARCHAR','',0,0,0,18,NULL),(19,'Fitness Determination','','','value_BOOL','',0,0,1,19,NULL),(20,'Fitness Parameter','','','value_VARCHAR','',0,0,0,20,19),(21,'Means to perform direct competition experiments','','','value_VARCHAR','',0,0,0,21,NULL),(22,'Detection/tracking of beneficial mutations by','','','value_VARCHAR','',0,0,0,22,NULL),(23,'Whole Genome Sequencing','','Was WGS used?','value_BOOL','',0,0,1,23,NULL),(24,'WGS Technology','','','value_VARCHAR','',0,0,0,24,23),(25,'WGS Platform','','','value_VARCHAR','',0,0,0,25,23),(26,'Number of Clones Sequenced','','','value_INT','',0,0,0,26,23),(27,'Number of Populations Sequenced','','','value_INT','',0,0,0,27,23),(28,'Coverage Clones','','','value_INT','',0,0,0,28,23),(29,'Coverage Populations','','','value_INT','',0,0,0,29,23),(30,'Detection Limit','%','','value_DOUBLE','',0,0,0,30,23),(31,'Link to NGS Dataset','','','value_VARCHAR','',1,0,0,31,23),(32,'Maximum Population Size','','','value_INT','',0,0,0,32,NULL),(33,'Minimum Population Size','','','value_INT','',0,0,0,33,NULL),(34,'Major Outcomes','','Describe main results','value_TEXT','',0,0,0,34,NULL),(35,'Remarks','','','value_TEXT','',0,0,0,35,NULL),(36,'Mutation data','','Is there mutation data associated with the experiment?','value_BOOL','',0,0,1,36,NULL),(40,'Mutation data complete?','','Is the provided file filtered or does it contain the full list of mutations','value_BOOL','',0,1,0,37,36),(41,'Chromosome ID','','Chromosome ID used to compare sequence output with','value_VARCHAR','',0,0,0,38,36),(42,'Start Position','','First genomic position of variant','value_INT','',0,0,0,39,36),(43,'End Position','','Last genomic position of variant','value_INT','',0,0,0,40,36),(44,'Type','','Type of genomic mutation','value_VARCHAR','SNP, DEL, INS, INV, MNV, REPL, AMP',0,0,0,41,36),(45,'Reference Amino Acid','','Reference base at this position','value_TEXT','',0,0,0,42,36),(46,'Alternate Amino Acid','','Mutant base at this position','value_TEXT','',0,0,0,43,36),(47,'Gene','','Gene effected by mutation','value_TEXT','',0,0,0,44,36),(48,'Change in Amino Acid','','AA Change at the defined location','value_VARCHAR','',0,0,0,45,36),(49,'Population','','Population of Origin','value_VARCHAR','',0,0,0,46,36),(50,'Clone','','Clone ID that was sequenced','value_VARCHAR','',0,0,0,47,36),(51,'Time','days','Timepoint of sampling during evolution','value_VARCHAR','',0,0,0,48,36),(52,'Frequency','','Frequency of allele in the sample','value_DOUBLE','',0,0,0,49,36),(53,'Comments','','Free field to make comments (abbreviation of anibiotic)','value_VARCHAR','',0,0,0,50,36),(54,'Impact','','[MUTFUNC] Is the mutation predicted to impact function? \'1\' if yes, \'0\' if no','value_BOOL','',0,0,0,51,36),(55,'Score','','[MUTFUNC] Sift score, any mutation with a score below 0.05 is considered deleterious','value_DOUBLE','',0,0,0,52,36),(56,'Information Content','','[MUTFUNC] Information content at this position of the alignment (a high value indicates strong conservation, where the maximum value is 4.32)','value_DOUBLE','',0,0,0,53,36),(57,'Free Energy Change','','[MUTFUNC] Predicted change in free energy of unfolding, where a value above 0 indicates a destabilising mutation','value_DOUBLE','',0,0,0,54,36),(58,'PDB ID','','[MUTFUNC] PDB identifier or homology model identifier of the structure containing the mutation','value_VARCHAR','',0,0,0,55,36),(59,'Sequence','','[MUTFUNC] Sequence of the linear motif','value_VARCHAR','',0,0,0,56,36),(60,'Accession','','[MUTFUNC] ELM accession for the linear motif','value_VARCHAR','',0,0,0,57,36),(61,'Modification Type','','[MUTFUNC] Type of post-translational modifications','value_VARCHAR','',0,0,0,57,36),(62,'Site Function','','[MUTFUNC] Function of this phosphorylation site, if any','value_VARCHAR','',0,0,0,58,36),(63,'Function Evidence','','[MUTFUNC] Evidence of site function, if any','value_VARCHAR','',0,0,0,59,36),(64,'Predicted Kinase','','[MUTFUNC] Kinase predicted to lose phosphorylation at this site','value_VARCHAR','',0,0,0,60,36),(65,'Probability of Kinase Loss','','[MUTFUNC] Probability of kinase losing phosphorylation at this site','value_DOUBLE','',0,0,0,61,36),(66,'Knockout p-value','','[MUTFUNC] P-value of over or under-expression for the downstream gene when the transcription factor is knocked out','value_DOUBLE','',0,0,0,62,36),(67,'Transcription Factor','','[MUTFUNC] Transcription factor predicted to bind this binding site','value_VARCHAR','',0,0,0,63,36),(68,'Category of Mutation','','[MUTFUNC] Category of Mutfunc Mutation','value_TEXT','',0,0,0,64,36),(69,'Interaction Score','','[MECHISMO]','value_DOUBLE','',0,0,0,65,36),(70,'Total Interaction Score','','[MECHISMO]','value_DOUBLE','',0,0,0,66,36),(71,'CELLO2GO Probabilities','','[CELLO2GO]','value_VARCHAR','',0,0,0,67,36),(72,'Intracellular Location','','[CELLO2GO]','value_VARCHAR','',0,0,0,68,36);
/*!40000 ALTER TABLE `fields` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `mutations_fields`
--

DROP TABLE IF EXISTS `mutations_fields`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `mutations_fields` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `experiment_id` int(11) NOT NULL,
  `mutation_id` int(11) NOT NULL,
  `field_id` int(11) DEFAULT NULL,
  `value_VARCHAR` varchar(255) DEFAULT NULL,
  `value_TEXT` mediumtext DEFAULT NULL,
  `value_INT` bigint(20) DEFAULT NULL,
  `value_DOUBLE` double DEFAULT NULL,
  `value_BOOL` tinyint(1) DEFAULT NULL,
  `value_ATTACH` varchar(255) DEFAULT NULL,
  KEY `ix_mutations_fields_index` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=168224 DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `mutations_fields`
--

LOCK TABLES `mutations_fields` WRITE;
/*!40000 ALTER TABLE `mutations_fields` DISABLE KEYS */;
/*!40000 ALTER TABLE `mutations_fields` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `references`
--

DROP TABLE IF EXISTS `references`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `references` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `authors` text COLLATE utf8_unicode_ci DEFAULT NULL,
  `title` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
  `journal` varchar(100) COLLATE utf8_unicode_ci DEFAULT NULL,
  `year` int(5) DEFAULT NULL,
  `pages` varchar(50) COLLATE utf8_unicode_ci DEFAULT NULL,
  `pubmed_id` int(11) DEFAULT NULL,
  `url` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `pubmed_id` (`pubmed_id`)
) ENGINE=InnoDB AUTO_INCREMENT=11983 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `references`
--

LOCK TABLES `references` WRITE;
/*!40000 ALTER TABLE `references` DISABLE KEYS */;
INSERT INTO `references` VALUES (11249,'Moffatt JH, Harper M, Harrison P, Hale JDF, Vinogradov E, Seemann T, Henry R, Crane B, St Michael F, Cox AD, Adler B, Nation RL, Li J, Boyce JD','Colistin resistance in Acinetobacter baumannii is mediated by complete loss of lipopolysaccharide production','Antimicrob Agents Chemother',2010,'54:4971-4977',20855724,'http://aac.asm.org/content/54/12/4971.short'),(11250,'McLeman A, Sierocinski P, Hesse E, Buckling A, Perron G, Hülter N, Johnsen PJ, Vos M','No effect of natural transformation on the evolution of resistance to bacteriophages in the Acinetobacter baylyi model system','Sci Reports',2016,'6:37144',27869203,'https://www.nature.com/articles/srep37144'),(11251,'Jezequel N, Lagomarsino MC, Heslot F, Thomen P','Long-term diversity and genome adaptation of Acinetobacter baylyi in a minimal-medium chemostat','Genome Biol',2013,'5:87-97',23254395,'https://academic.oup.com/gbe/article/5/1/87/728571#89426439'),(11252,'de Crécy E, Metzgar D, Allen C, Pénicaud M, Lyons BJ, Hansen CJ, de Crécy-Lagard V a','Development of a novel continuous culture device for experimental evolution of bacterial populations','Appl Microbiol Biotechnol',2007,'77:489-496',17896105,'https://link.springer.com/article/10.1007/s00253-007-1168-5'),(11253,'Vila-Farrés X, Ferrer-Navarro M, Callarisa AE, Martí S, Espinal P, Gupta S, Rolain J-MM, Giralt E, Vila J','Loss of LPS is involved in the virulence and resistance to colistin of colistin-resistant Acinetobacter nosocomialis mutants selected in vitro','J Antimicrob Chemother',2015,'70:2981-2986',26311838,'https://academic.oup.com/jac/article/70/11/2981/2364076'),(11254,'Zhang J, van den Heuvel J, Debets AJM, Verweij PE, Melchers WJG, Zwaan BJ, Schoustra SE','Evolution of cross-resistance to medical triazoles in Aspergillus fumigatus through selection pressure of environmental fungicides','Proc. R. Soc. B',2017,'284:20170635',28931745,''),(11255,'Zhang J, Debets AJM, Verweij PE, Melchers WJG, Zwaan BJ, Schoustra SE','Asexual sporulation facilitates adaptation : The emergence of azole resistance in Aspergillus fumigatus','Evolution',2015,'69:2573-2586',26315993,''),(11256,'Bruggeman J, Debets AJM, Wijngaarden PJ, DeVisser JAGM, Hoekstra RF','Sex slows down the accumulation of deleterious mutations in the homothallic fungus Aspergillus nidulans','Genetics',2003,'164:479-485',12807769,'http://www.genetics.org/content/164/2/479.long'),(11257,'Rosenberg J, Yeak KC, Commichau FM','A two-step evolutionary process establishes a non-native vitamin B6 pathway in Bacillus subtilis','Env Microbiol',2017,'20:156-168',29027347,''),(11258,'Waters SM, Zeigler DR, Nicholson WL','Experimental evolution of enhanced growth by Bacillus subtilis at low atmospheric pressure: Genomic changes revealed by whole-genome sequencing','Appl Environ Microbiol',2015,'81:7525-7532',26296725,'http://aem.asm.org/content/81/21/7525'),(11259,'Nicholson WL, Fajardo-Cavazos P, Fedenko J, Ortíz-Lugo JL, Rivas-Castillo A, Waters SM, Schuerger AC','Exploring the low-pressure growth limit: evolution of Bacillus subtilis in the laboratory to enhanced growth at 5 kilopascals.','Appl Environ Microbiol',2010,'76(22): 7559-65',20889789,'https://doi.org/10.1128/AEM.01126-10'),(11260,'Kloesener MH, Bose J, Schulte RD','Experimental evolution with a multicellular host causes diversification within and between microbial parasite populations-Differences in emerging phenotypes of two different parasite strains.','Evolution',2017,'71(9): 2194-2205',28714591,'https://doi.org/10.1111/evo.13306'),(11261,'Schulte RD, Makus C, Schulenburg H','Host-parasite coevolution favours parasite genetic diversity and horizontal gene transfer','J Evol Biol',2013,'1–5.',23865952,''),(11262,'Schulte RD, Makus C, Hasert B, Michiels NK, Schulenburg H','Multiple reciprocal adaptations and rapid genetic change upon experimental coevolution of an animal host and its microbial parasite','PNAS',2010,'107:7359-7364',20368449,''),(11263,'Dillon MM, Cooper VS','The fitness effects of spontaneous mutations nearly unseen by selection in a bacterium with multiple chromosomes','Genetics',2016,'',27672096,'http://www.genetics.org/cgi/doi/10.1534/genetics.116.193060'),(11264,'Dillon MM, Sung W, Lynch M, Cooper VS, Cooper VS','The rate and molecular spectrum of spontaneous mutations in the GC-rich multi- chromosome genome of Burkholderia cenocepacia','Genetics',2015,'12115:935-946',25971664,'http://www.genetics.org/content/200/3/935'),(11265,'Weil T, Santamaría R, Lee W, Rung J, Tocci N, Abbey D, Bezerra AR, Carreto L, Moura GR, Bayés M, Gut IG, Csikasz-Nagy A, Cavalieri D, Berman J, Santos MAS','Adaptive Mistranslation Accelerates the Evolution of Fluconazole Resistance and Induces Major Genomic and Gene Expression Alterations in Candida albicans','mSphere',2017,'2:e00167-17',28808688,''),(11266,'Healey KR, Zhao Y, Perez WB, Lockhart SR, Sobel JD, Farmakiotis D, Kontoyiannis DP, Sanglard D, Taj-Aldeen SJ, Alexander BD, Jimenez-Ortigosa C, Shor E, Perlin DS','Prevalent mutator genotype identified in fungal pathogen Candida glabrata promotes multi-drug resistance','Nat Commun',2016,'7:11128',27020939,'http://www.nature.com/doifinder/10.1038/ncomms11128'),(11267,'Boyd M, Rosenzweig F, Herron MD','Analysis of motility in multicellular Chlamydomonas reinhardtii evolved under predation','Plos One',2018,'13:e0192184',29381766,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0192184'),(11268,'Herron MD, Borin JM, Boswell JC, Walker J, Knox CA, Boyd M, Rosenzweig F, Ratcliff WC','De novo origin of multicellularity in response to predation','biorxiv',2018,'',30787483,'https://www.biorxiv.org/content/early/2018/01/12/247361'),(11269,'Kraemer S, Böndel KB, Ness RW, Keightley PD, Colegrave N','Fitness change in relation to mutation number in spontaneous mutation accumulation lines of Chlamydomonas reinhardtii','Evolution',2017,'1–23.',28884790,''),(11270,'Morgan AD, Ness RW, Keightley PD, Colegrave N','Spontaneous mutation accumulation in multiple strains of the green alga, Chlamydomonas reinhardtii.','Evolution',2014,'68(9): 2589-602',24826801,'https://doi.org/10.1111/evo.12448'),(11271,'Korona R, Nakatsu CH, Forney LJ, Lenski RE','Evidence for multiple adaptive peaks from populations of bacteria evolving in a structured habitat','Proc Natl Acad Sci',1994,'91:9037-9041',8090765,'http://www.pnas.org/content/91/19/9037'),(11272,'Pfeifer E, Gätgens C, Polen T, Frunzke J','Adaptive laboratory evolution of Corynebacterium glutamicum towards higher growth rates on glucose minimal medium','Sci Reports',2017,'7:16780',29196644,''),(11273,'Zhou A, Lau R, Baran R, Ma J, von Netzer F, Shi W, Gorman-Lewis D, Kempher ML, He Z, Qin Y, Shi Z, Zane GM, Wu L, Bowen BP, Northen TR, Hillesland KL, Stahl DA, Wall JD, Arkin AP, Zhou J','Key Metabolites and Mechanistic Changes for Salt Tolerance in an Experimentally Evolved Sulfate-Reducing Bacterium, Desulfovibrio vulgaris','mbio',2017,'8:e01780-17',29138306,''),(11274,'Pellerin A, Anderson-Trocmé L, Whyte LG, Zane GM, Wall JD, Wing B a','Sulfur isotope fractionation during the evolutionary adaptation of a sulfate-reducing bacterium','Appl. Environ. Microbiol.',2015,'81:2676-2689',25662968,'http://aem.asm.org/content/81/8/2676'),(11275,'Zhou A, Baidoo E, He Z, Mukhopadhyay A, Baumohl JK, Benke P, Joachimiak MP, Xie M, Song R, Arkin AP, Hazen TC, Keasling JD, Wall JD, Stahl DA, Zhou J','Characterization of NaCl tolerance in Desulfovibrio vulgaris Hildenborough through experimental evolution.','ISME J',2013,'7(9): 1790-802',23575373,'https://doi.org/10.1038/ismej.2013.60'),(11276,'Schlu ter L, Lohbeck KT, Gro ger JP, Riebesell U, Reusch TBH','Long-term dynamics of adaptive evolution in a globally important phytoplankton species to ocean acidification','Sci Adv',2016,'2:e1501660-e1501660',27419227,'http://advances.sciencemag.org/content/2/7/e1501660'),(11277,'Beabout K, Hammerstrom TG, Wang TT, Bhatty M, Christie PJ, Saxer G, Shamoo Y','Rampant parasexuality evolves in a hospital pathogen during antibiotic selection','Mol Biol Evol',2015,'32:2585-2597',26060280,'https://academic.oup.com/mbe/article/32/10/2585/1208182'),(11278,'Maddamsetti R, Lenski RE','Analysis of bacterial genomes from an evolution experiment with horizontal gene transfer shows that recombination can sometimes overwhelm selection','Plos Genet',2018,'14:e1007199',29385126,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007199'),(11279,'Aziz RK, Monk JM, Andrews KA, Nhan J, Khaw VL, Wong H, Palsson BØ, Charusanti P','The aldehyde dehydrogenase, AldA, is essential for L-1,2-propanediol utilization in laboratory-evolved Escherichia coli','Microbiol Res',2017,'194:47-52',27938862,'http://aem.asm.org/content/76/13/4158'),(11280,'Bosshard L, Dupanloup I, Tenaillon O, Bruggmann R, Ackermann M, Peischl S, Excoffier L','Accumulation of Deleterious Mutations During Bacterial Range Expansions','Genetics',2017,'1–40',28821588,''),(11281,'Bottery MJ, Wood AJ, Brockhurst MA','Adaptive modulation of antibiotic resistance through intragenomic coevolution','Nat Ecol Evol',2017,'1:1364-1369',28890939,''),(11282,'Chib S, Ali F, Seshasayee ASN','Genomewide Mutational Diversity in Escherichia coli Population Evolving in Prolonged Stationary Phase','mSphere',2017,'2:e00059-17',28567442,''),(11283,'Consuegra J, Plucain J, Gaffé J, Hindré T, Schneider D','Genetic Basis of Exploiting Ecological Opportunity During the Long-Term Diversification of a Bacterial Population','J Mol Evol',2017,'1–11',28744786,''),(11284,'Creamer KE, Ditmars FS, Basting PJ, Kunka KS, Hamdallah IN, Bush SP, Scott Z, He A, Penix SR, Gonzales AS, Eder EK, Camperchioli D, Berndt A, Clark MW, Rouhier KA, Slonczewski JL','Benzoate and salicylate tolerant strains lose antibiotic resistance dduring laboratory evolution of Escherichia coli K-12','Appl Environ Microbiol',2016,'',27793830,'http://aem.asm.org/lookup/doi/10.1128/AEM.02736-16'),(11285,'Deatherage DE, Kepner JL, Bennett AF, Lenski RE, Barrick JE','Specificity of genome evolution in experimental populations of <i>Escherichia coli</i> evolved at different temperatures.','Proc Natl Acad Sci U S A',2017,'114(10): E1904-E1912',28202733,'https://doi.org/10.1073/pnas.1616132114'),(11286,'Finn TJ, Shewaramani S, Leahy SC, Janssen PH, Moon CD','Dynamics and genetic diversification of Escherichia coli during experimental adaptation to an anaerobic environment','PeerJ',2017,'5:e3244',28480139,''),(11287,'Fraebel DT, Mickalide H, Schnitkey D, Merritt J, Kuhlman TE, Kuehn S','Environment determines evolutionary trajectory in a constrained phenotypic space','Elife',2017,'6:1-32',28346136,''),(11288,'Guo Y, Winkler J, Kao K','Insights on Osmotic Tolerance Mechanisms in Escherichia coli Gained from an rpoC Mutation','Bioengineering',2017,'4:61',28952540,''),(11290,'Horinouchi T, Suzuki S, Kotani H, Tanabe K, Sakata N, Shimizu H, Furusawa C','Prediction of Cross-resistance and Collateral Sensitivity by Gene Expression profiles and Genomic Mutations','biorxiv',2017,'',NULL,''),(11291,'Levin-Reisman I, Ronin I, Gefen O, Braniss I, Shoresh N, Balaban NQ','Antibiotic tolerance facilitates the evolution of resistance','Science',2017,'355:826-830',28183996,'http://science.sciencemag.org/content/355/6327/826'),(11292,'Liu B, Eydallin G, Maharjan RP, Feng L, Wang L, Ferenci T','Natural Escherichia coli isolates rapidly acquire genetic changes upon laboratory domestication','Microbiology',2016,'',28198347,'http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000405#tab2'),(11293,'Long CP, Gonzalez JE, Feist AM, Palsson BØ, Antoniewicz MR','Dissecting the genetic and metabolic mechanisms of adaptation to the knockout of a major metabolic enzyme in Escherichia coli','PNAS',2017,'115 (1) 222-227',29255023,''),(11294,'Lopatkin AJ, Meredith HR, Srimani JK, Pfeiffer C, Durrett R, You L','Persistence and reversal of plasmid-mediated antibiotic resistance','Nat Commun',2017,'1–10',29162798,''),(11295,'Marchal M, Goldschmidt F, Derksen-Müller SN, Panke S, Ackermann M, Johnson DR','A passive mutualistic interaction promotes the evolution of spatial structure within microbial populations','BMC Evol Biol',2017,'17:106',28438135,'https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-017-0950-y'),(11296,'Mohamed ET, Wang S, Lennen RM, Herrgård MJ, Simmons BA, Singer SW, Feist AM','Generation of a platform strain for ionic liquid tolerance using adaptive laboratory evolution','Microb Cell Fact',2017,'16:204',29145855,''),(11297,'Mundhada H, Seoane JM, Schneider K, Koza A, Christensen HB, Klein T, Phaneuf P V, Herrgard M, Feist AM, Nielsen AT, Miguel JS, Schneider K, Koza A, Christensen HB, Klein T, Phaneuf P V, Herrgard M, Feist AM, Nielsen AT','Increased production of L-serine in Escherichia coli through Adaptive Laboratory Evolution','Metab Eng',2017,'39:141-150',27908688,'https://www.sciencedirect.com/science/article/pii/S1096717616302348?via%3Dihub'),(11298,'Peabody V GL, Li H, Kao KC','Sexual recombination and increased mutation rate expedite evolution of Escherichia coli in varied fitness landscapes','Nat Commun',2017,'8:2112',29235478,'https://www.nature.com/articles/s41467-017-02323-4'),(11299,'Peng F, Widmann S, Wunsche A, Duan K, Donovan KA, Dobson RCJ, Lenski RE, Cooper TF','Effects of beneficial mutations in pykF gene vary over time and across replicate populations in a long-term experiment with bacteria','Mol Biol Evol',2017,'',29069429,'https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msx279'),(11300,'Reynolds TS, Courtney CM, Erickson KE, Wolfe LM, Chatterjee A, Nagpal P, Gill RT','ROS Mediated Selection for Increased NADPH Availability in Escherichia coli','Biotech Bioeng',2017,'',28710857,''),(11301,'Sandberg TE, Lloyd CJ, Palsson BO, Feist AM','Laboratory evolution to alternating substrate environments yields distinct phenotypic and genetic adaptive strategies','Appl Environ Microbiol',2017,'83:1-47',28455337,'http://aem.asm.org/content/83/13/e00410-17'),(11302,'Sievert C, Nieves LM, Panyon LA, Loeffler T, Morris C, Cartwright RA, Wang X','Experimental evolution reveals a novel avenue to release catabolite repression via mutations in XylR','bioRxiv',2017,'114:1-22',28655843,'http://www.pnas.org/content/114/28/7349'),(11303,'Singh T, Hyun M, Sniegowski P','Evolution of mutation rates in hypermutable populations of Escherichia coli propagated at very small effective population size','Biol Lett',2017,'13:1-10',28250208,'http://rsbl.royalsocietypublishing.org/content/13/3/20160849'),(11304,'Steinrueck M, Guet CC','Complex chromosomal neighborhood effects determine the adaptive potential of a gene under selection','Elife',2017,'6:1-26',28738969,''),(11305,'Swings T, Van den Bergh B, Wuyts S, Oeyen E, Voordeckers K, Verstrepen KJ, Fauvart M, Verstraeten AN, Michiels J, van Den Bergh B, Kevin J','Adaptive tuning of mutation rates allows fast response to lethal stress in Escherichia coli','eLife',2017,'6:1-24',28460660,'https://elifesciences.org/articles/22939'),(11306,'Wünsche A, Dinh DM, Satterwhite RS, Arenas CD, Stoebel DM, Cooper TF','Diminishing-returns epistasis decreases adaptability along an evolutionary trajectory','Nat Ecol Evol',2017,'1:61',28812657,'https://www.nature.com/articles/s41559-016-0061'),(11307,'Zampieri M, Enke T, Chubukov V, Ricci V, Piddock L, Sauer U','Metabolic constraints on the evolution of antibiotic resistance','Mol Syst Biol',2017,'1–14.',28265005,'http://msb.embopress.org/content/13/3/917'),(11308,'Aguilar-Rodríguez J, Sabater-Munoz B, Berlanga V, Alvarez-Ponce D, Wagner A, Fares MA, Sabater-Muñoz B','The molecular chaperone DnaK accelerates protein evolution','bioRxiv',2016,'',27497316,'https://www.biorxiv.org/content/early/2016/02/22/040600'),(11309,'Alvarez-Ponce D, Sabater-Muñoz B, Toft C, Ruiz-González MX, Fares MA','Essentiality is a strong determinant of protein rates of evolution during mutation accumulation experiments in Escherichia coli','Genome Biol Evol',2016,'8:2914-2927',27566759,'http://gbe.oxfordjournals.org/lookup/doi/10.1093/gbe/evw205'),(11310,'Azevedo M, Sousa A, Moura De Sousa J, Thompson JA, Proença JT, Gordo I','Trade-offs of Escherichia coli adaptation to an intracellular lifestyle in macrophages','PLoS One',2016,'11:e0146123',26752723,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0146123'),(11311,'Baym M, Lieberman TD, Kelsic ED, Chait R, Gross R, Yelin I, Kishony R','Spatiotemporal microbial evolution on antibiotic landscapes','Science',2016,'353:1147-1151',27609891,'http://science.sciencemag.org/content/353/6304/1147'),(11312,'Cottinet D, Condamine F, Bremond N, Griffiths AD, Rainey PB, de Visser JAGM, Baudry J, Bibette J','Lineage tracking for probing heritable phenotypes at single-cell resolution','PLoS One',2016,'11:e0152395',27077662,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0152395'),(11313,'Dillon MM, Rouillard NP, Van Dam B, Gallet R, Cooper VS','Diverse phenotypic and genetic responses to short-term selection in evolving Escherichia coli populations','Evolution',2016,'70:586-599',26995338,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12868/abstract'),(11314,'Hammerling MJ, Gollihar J, Mortensen C, Alnahhas RN, Ellington AD, Barrick JE','Expanded genetic codes create new mutational routes to rifampicin resistance in Escherichia coli','Mol Biol Evol',2016,'33:2054-2063',27189550,'http://mbe.oxfordjournals.org/lookup/doi/10.1093/molbev/msw094'),(11316,'Karve SM, Bhave D, Nevgi D, Dey S','Escherichia coli populations adapt to complex, unpredictable fluctuations by minimizing trade-offs across environments','J Evol Biol',2016,'',27575521,'http://onlinelibrary.wiley.com/doi/10.1111/jeb.12972/abstract'),(11317,'Lescat M, Launay A, Ghalayini M, Magnan M, Glodt J, Pintard C, Dion S, Denamur E, Tenaillon O','Using long-term experimental evolution to uncover the patterns and determinants of molecular evolution of an Escherichia coli natural isolate in the streptomycin-treated mouse gut','Mol Ecol',2016,'',27661780,'http://onlinelibrary.wiley.com/doi/10.1111/mec.13851/abstract'),(11318,'Long H, Miller SF, Strauss C, Zhao C, Cheng L, Ye Z, Griffin K, Te R, Lee H, Chen C-C, Lynch M','Antibiotic treatment enhances the genome-wide mutation rate of target cells','Proc Natl Acad Sci',2016,'113:E2498-E2505',27091991,'http://www.pnas.org/content/113/18/E2498'),(11319,'Moreau PL, Loiseau L','Characterization of acetic acid-detoxifying Escherichia coli evolved under phosphate starvation conditions','Microb Cell Fact',2016,'15:42',26895825,'http://www.microbialcellfactories.com/content/15/1/42'),(11320,'Peabody G, Winkler JD, Fountain W, Castro DA, Leiva-Aravena E, Kao KC','Benefits of a recombination-proficient Escherichia coli system for adaptive laboratory evolution','Appl Environ Microbiol',2016,'82:6736-6747',27613685,''),(11321,'Phillips KN, Castillo G, Wünsche A, Cooper TF','Adaptation of Escherichia coli to glucose promotes evolvability in lactose','Evolution',2016,'70:465-470',26748670,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12849/abstract'),(11322,'Plucain J, Suau A, Cruveiller S, Médigue C, Schneider D, Le Gac M','Contrasting effects of historical contingency on phenotypic and genomic trajectories during a two-step evolution experiment with bacteria','BMC Evol Biol',2016,'16:86',27108090,'https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-016-0662-8'),(11324,'Rajaraman E, Agarwal A, Crigler J, Seipelt-Thiemann R, Altman E, Eiteman MA','Transcriptional analysis and adaptive evolution of Escherichia coli strains growing on acetate','Appl Microbiol Biotechnol',2016,'100:7777-7785',27448288,'https://link.springer.com/article/10.1007%2Fs00253-016-7724-0'),(11325,'Rigato E, Fusco G','Enhancing effect of phenotype mutational robustness on adaptation in Escherichia coli','J Exp Zool Part B Mol Dev Evol',2016,'326:31-37',26612084,'http://onlinelibrary.wiley.com/doi/10.1002/jez.b.22662/full'),(11326,'San Millan A, Escudero JA, Gifford DR, Mazel D, MacLean RC','Multicopy plasmids potentiate the evolution of antibiotic resistance in bacteria','Nat Ecol Evol',2016,'1:10',28812563,'https://www.nature.com/articles/s41559-016-0010'),(11327,'Sandberg TE, Long CP, Gonzalez JE, Feist AM, Antoniewicz MR, Palsson BO','Evolution of E. coli on [U-13C]Glucose Reveals a Negligible Isotopic Influence on Metabolism and Physiology.','PLoS One',2016,'11(3): e0151130',26964043,'https://doi.org/10.1371/journal.pone.0151130'),(11328,'Spagnolo F, Rinaldi C, Sajorda DR, Dykhuizen DE','Evolution of resistance to continuously increasing streptomycin concentrations in populations of Escherichia coli','Antimicrob Agents Chemother',2016,'60:1336-1342',26666944,'http://aac.asm.org/content/60/3/1336'),(11329,'Stone LK, Baym M, Lieberman TD, Chait R, Clardy J, Kishony R','Compounds that select against the tetracycline-resistance efflux pump','Nat Chem Biol',2016,'12:902-904',27642863,'http://www.nature.com/doifinder/10.1038/nchembio.2176'),(11330,'Suzuki S, Horinouchi T, Furusawa C','Phenotypic changes associated with the fitness cost in antibiotic resistant Escherichia coli strains','Mol BioSyst',2016,'12:414-420',26625335,'http://pubs.rsc.org/en/Content/ArticleLanding/2016/MB/C5MB00590F#!divAbstract'),(11331,'Szappanos B, Fritzemeier J, Csörgó B, Lázár V, Lu X, Fekete G, Bálint B, Herczeg R, Nagy I, Notebaart RA, Lercher MJ, Pál C, Papp B, Pa C','Adaptive evolution of complex innovations through stepwise metabolic niche expansion','Nat Commun',2016,'7:11607',27197754,'http://www.nature.com/doifinder/10.1038/ncomms11607'),(11332,'Tenaillon O, Barrick JE, Ribeck N, Deatherage DE, Blanchard JL, Dasgupta A, Wu GC, Wielgoss S, Cruveiller S, Médigue C, Schneider D, Lenski RE','Tempo and mode of genome evolution in a 50,000-generation experiment','Nature',2016,'536:165-170',27479321,'http://www.nature.com/doifinder/10.1038/nature18959'),(11334,'Van den Bergh B, Michiels JE, Wenseleers T, Windels EM, Boer PV, Kestemont D, De Meester L, Verstrepen KJ, Verstraeten N, Fauvart M, Michiels J','Frequency of antibiotic application drives rapid evolutionary adaptation of Escherichia coli persistence.','Nat Microbiol',2016,'1: 16020',27572640,'https://doi.org/10.1038/nmicrobiol.2016.20'),(11336,'Van Hofwegen DJ, Hovde CJ, Minnich SA','Rapid evolution of citrate utilization by Escherichia coli by direct selection requires citT and dctA','J Bacteriol',2016,'198:1022-1034',26833416,'http://jb.asm.org/content/198/7/1022'),(11337,'Harden MM, He A, Creamer KE, Clark MW, Hamdallah I, Martinez K a, Kresslein RL, Bush SP, Slonczewski JL','Acid-adapted strains of Escherichia coli K-12 obtained by experimental evolution','Appl Environ Microbiol',2015,'81:1932-1941',25556191,'http://aem.asm.org/content/81/6/1932'),(11338,'Karve SM, Daniel S, Chavhan YD, Anand A, Kharola SS, Dey S','Escherichia coli populations in unpredictably fluctuating environments evolve to face novel stresses through enhanced efflux activity','J Evol Biol',2015,'28:1131-1143',25865653,'http://onlinelibrary.wiley.com/doi/10.1111/jeb.12640/abstract'),(11339,'Khare A, Tavazoie S','Multifactorial competition and resistance in a two-species bacterial system','PLOS Genet',2015,'11:e1005715',26647077,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005715'),(11340,'LaCroix RA, Sandberg TE, O’Brien EJ, Utrilla J, Ebrahim A, Guzman GI, Szubin R, Palsson BO, Feist AM','Use of Adaptive Laboratory Evolution To Discover Key Mutations Enabling Rapid Growth of Escherichia coli K-12 MG1655 on Glucose Minimal Medium','Appl Environ Microbiol',2015,'81:17-30',25304508,'http://aem.asm.org/content/81/1/17.long'),(11341,'Lambert G, Kussell E','Quantifying selective pressures driving bacterial evolution using lineage analysis','Phys Rev',2015,'5:11016',26213639,'https://journals.aps.org/prx/abstract/10.1103/PhysRevX.5.011016'),(11342,'Lenski RE, Wiser MJ, Ribeck N, Blount ZD, Nahum JR, Morris JJ, Zaman L, Turner CB, Wade BD, Maddamsetti R, Burmeister AR, Baird EJ, Bundy J, Grant NA, Card KJ, Rowles M, Weatherspoon K, Papoulis SE, Sullivan R, Clark C, Mulka JS, Hajela N','Sustained fitness gains and variability in fitness trajectories in the long-term evolution experiment with Escherichia coli','Proc R Soc B Biol Sci',2015,'282:20152292',26674951,'http://rspb.royalsocietypublishing.org/content/282/1821/20152292'),(11343,'Moura De Sousa J, Sousa A, Bourgard C, Gordo I','Potential for adaptation overrides cost of resistance','Future Microbiology',2015,'10:1415-1431',26343510,'http://www.futuremedicine.com/doi/10.2217/fmb.15.61'),(11344,'Sabater-Munoz B, Prats-escriche M, Montagud-Martínez R, López-Cerdán A, Toft C, Aguilar-Rodríguez J, Wagner A, Fares MA','Fitness trade-offs determine the role of the molecular chaperonin GroEL in buffering mutations','Mol Biol Evol',2015,'32:2681-2693',26116858,'https://academic.oup.com/mbe/article/32/10/2681/1210991'),(11345,'Suzuki S, Horinouchi T, Furusawa C','Suppression of antibiotic resistance acquisition by combined use of antibiotics','J Biosci Bioeng',2015,'120:467-469',25746894,'https://www.sciencedirect.com/science/article/pii/S138917231500064X?via%3Dihub'),(11346,'Eydallin G, Ryall B, Maharjan R, Ferenci T','The nature of laboratory domestication changes in freshly isolated Escherichia coli strains.','Environ Microbiol',2014,'16(3): 813-28',23889812,'https://doi.org/10.1111/1462-2920.12208'),(11347,'Horinouchi T, Minamoto T, Suzuki S, Shimizu H, Furusawa C','Development of an automated culture system for laboratory evolution','Journal of Laboratory Automation',2014,'19:478-482',24526062,'http://journals.sagepub.com/doi/abs/10.1177/2211068214521417?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub%3Dpubmed&'),(11348,'Munck C, Gumpert HK, Wallin AI, Wang HH, Sommer MO','Prediction of resistance development against drug combinations by collateral responses to component drugs.','Sci Transl Med',2014,'6(262): 262ra156',25391482,'https://doi.org/10.1126/scitranslmed.3009940'),(11349,'Sandberg TE, Pedersen M, LaCroix RA, Ebrahim A, Bonde MT, Herrgård MJ, Palsson B, Sommer MO a, Feist AM','Evolution of Escherichia coli to 42°C and subsequent genetic engineering reveals adaptive mechanisms and novel mutations','Mol Biol Evol',2014,'31:2647-2662',25015645,'http://mbe.oxfordjournals.org/content/early/2014/07/09/molbev.msu209.short'),(11350,'Winkler JD, Garcia C, Olson M, Callaway E, Kao KC','Evolved osmotolerant Escherichia coli mutants frequently exhibit defective N-acetylglucosamine catabolism and point mutations in cell shape-regulating protein MreB.','Appl Environ Microbiol',2014,'80(12): 3729-40',24727267,'https://doi.org/10.1128/AEM.00499-14'),(11351,'Dragosits M, Mozhayskiy V, Quinones-Soto S, Park J, Tagkopoulos I','Evolutionary potential, cross-stress behavior and the genetic basis of acquired stress resistance in Escherichia coli','Mol Syst Biol',2013,'9:643',23385483,'http://msb.embopress.org/content/9/1/643.long#DC5'),(11352,'Maharjan RP, Ferenci T, Reeves PR, Li Y, Liu B, Wang L','The multiplicity of divergence mechanisms in a single evolving population','Genome Biol',2012,'13:R41',22682524,'https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-6-r41'),(11353,'Toprak E, Veres A, Michel J-B, Chait R, Hartl DL, Kishony R','Evolutionary paths to antibiotic resistance under dynamically sustained drug selection','Nat Genet',2012,'44:101-105',22179135,'https://www.nature.com/articles/ng.1034'),(11354,'Winkler JD, Kao KC','Harnessing recombination to speed adaptive evolution in Escherichia coli','Metab Eng',2012,'14:487-495',22842472,'https://www.sciencedirect.com/science/article/pii/S1096717612000742?via%3Dihub'),(11355,'Auriol C, Bestel-Corre G, Claude J-B, Soucaille P, Meynial-Salles I','Stress-induced evolution of Escherichia coli points to original concepts in respiratory cofactor selectivity','Proc Natl Acad Sci',2011,'108:1278-1283',21205901,'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3029715/'),(11356,'Gaffé J, McKenzie C, Maharjan RP, Coursange E, Ferenci T, Schneider D','Insertion sequence-driven evolution of Escherichia coli in chemostats','J Mol Evol',2011,'72:398-412',21399911,'https://link.springer.com/article/10.1007%2Fs00239-011-9439-2'),(11357,'Kwon Y, Kim S, Lee SY, Kim P','Long-term continuous adaptation of Escherichia coli to high succinate stress and transcriptome analysis of the tolerant strain','JBIOSC',2011,'111:26-30',20829106,''),(11358,'Lee D-H, Feist AM, Barrett CL, Palsson BØ','Cumulative number of cell divisions as a meaningful timescale for adaptive laboratory evolution of Escherichia coli','PLoS One',2011,'6:e26172',22028828,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0026172'),(11359,'Wang X, Miller EN, Yomano LP, Zhang X, Shanmugam KT, Ingram LO','Increased Furfural Tolerance Due to Overexpression of NADH-Dependent Oxidoreductase FucO in Escherichia coli Strains Engineered for the Production of Ethanol and Lactate ᰔ','Appl Env Microiol',2011,'77:5132-5140',21685167,''),(11360,'Charusanti P, Conrad TM, Knight EM, Venkataraman K, Fong NL, Xie B, Gao Y, Palsson BØ','Genetic basis of growth adaptation of Escherichia coli after deletion of pgi, a major metabolic gene','PLoS Genet',2010,'6:e1001186',21079674,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1001186'),(11361,'Lee D-H, Palsson BØ','Adaptive evolution of Escherichia coli K-12 MG1655 during growth on a Nonnative carbon source, L-1,2-propanediol','Appl Environ Microbiol',2010,'76:4158-68',20435762,'https://www.sciencedirect.com/science/article/pii/S094450131630800X?via%3Dihub'),(11362,'Wang L, Spira B, Rice DP, Feng L, Maharjan RP, Li X, Li F, McKenzie C, Reeves PR, Ferenci T, Zhou Z, Feng L, Maharjan RP, Li X, Li F, McKenzie C, Reeves PR, Ferenci T','Divergence involving global regulatory gene mutations in an Escherichia coli population evolving under phosphate limitation','Genome Biol',2010,'2:478-487',20639316,'https://academic.oup.com/gbe/article/doi/10.1093/gbe/evq035/574394'),(11363,'Barrick JE, Lenski RE, tool=pmcentrez, re','Genome-wide mutational diversity in an evolving population of Escherichia coli','Cold Spring Harb. Symp. Quant. Biol.',2009,'74:119-29',19776167,'http://symposium.cshlp.org/content/74/119'),(11364,'Conrad TM, Joyce AR, Applebee MK, Barrett CL, Xie B, Gao Y, Palsson BØ','Whole-genome resequencing of Escherichia coli K-12 MG1655 undergoing short-term laboratory evolution in lactate minimal media reveals flexible selection of adaptive mutations','Genome Biol',2009,'10:R118',19849850,'https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-10-r118#Sec19'),(11365,'Gac M Le, Doebeli M','ENVIRONMENTAL VISCOSITY DOES NOT AFFECT THE EVOLUTION OF COOPERATION DURING EXPERIMENTAL EVOLUTION OF COLICIGENIC BACTERIA','Evolution',2009,'',19674096,''),(11366,'Goodarzi H, Hottes AKAAK, Tavazoie S','Global discovery of adaptive mutations','Nat Methods',2009,'6:581-583',19597501,'http://www.nature.com/doifinder/10.1038/nmeth.1352'),(11367,'Stoebel DM, Hokamp K, Last MS, Dorman CJ','Compensatory evolution of gene regulation in response to stress by Escherichia coli lacking RpoS','PLoS Genet',2009,'5:e1000671',19798444,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000671#s4'),(11368,'Blount ZD, Borland CZ, Lenski RE','Historical contingency and the evolution of a key innovation in an experimental population of Escherichia coli','Proc Natl Acad Sci',2008,'105:7899-906',18524956,'http://www.pnas.org/content/105/23/7899'),(11369,'Hegreness MJ, Shoresh N, Damian D, Hartl DL, Kishony R','Accelerated evolution of resistance in multidrug environments','Proc Natl Acad Sci',2008,'105:13977-13981',18779569,'http://www.pnas.org/content/105/37/13977'),(11370,'Portnoy V a, Herrgård MJ, Palsson BØ','Aerobic fermentation of D-glucose by an evolved cytochrome oxidase-deficient Escherichia coli strain','Appl Environ Microbiol',2008,'74:7561-7569',18952873,'http://aem.asm.org/content/74/24/7561'),(11371,'Hughes BS, Cullum AJ, Bennett AF','Evolutionary adaptation to environmental pH in experimental lineages of Escherichia coli','Evolution',2007,'61:1725-1734',17598751,'http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2007.00139.x/abstract'),(11372,'Fong SS, Nanchen A, Palsson BØ, Sauer U','Latent pathway activation and increased pathway capacity enable Escherichia coli adaptation to loss of key metabolic enzymes','J Biol Chem',2006,'281:8024-8033',16319065,'http://www.jbc.org/content/281/12/8024.long'),(11373,'Hegreness MJ, Shoresh N, Hartl DL, Kishony R','An equivalence principle for the incorporation of favorable mutations in asexual populations','Science',2006,'311:1615-1617',16543462,'http://science.sciencemag.org.kuleuven.ezproxy.kuleuven.be/content/311/5767/1615'),(11374,'Maharjan RP, Seeto S, Notley-McRobb L, Ferenci T','Clonal adaptive radiation in a constant environment','Science',2006,'313:514-517',16825532,'http://www.sciencemag.org/cgi/pmidlookup?view=long&pmid=16825532'),(11375,'Fong SS, Palsson BØ','Metabolic gene-deletion strains of Escherichia coli evolve to computationally predicted growth phenotypes','Nat Genet',2004,'36:1056-8',15448692,'https://www.nature.com/articles/ng1432'),(11376,'Loewe L, Textor V, Scherer S','High deleterious genomic mutation rate in stationary phase of Escherichia coli','Science',2003,'302:1558-1560',14645846,'http://science.sciencemag.org/content/302/5650/1558'),(11377,'Notley-McRobb L, King T, Ferenci T','rpoS mutations and loss of general stress resistance in Escherichia coli populations as a consequence of conflict between competing stress responses','J Bacteriol',2002,'184:806-811',11790751,'http://jb.asm.org/content/184/3/806.short'),(11378,'de Crécy-Lagard V a, Bellalou J, Mutzel R, Marlière P','Long term adaptation of a microbial population to a permanent metabolic constraint: overcoming thymineless death by experimental evolution of Escherichia coli','BMC Biotechnol',2001,'1:10',11737878,'https://bmcbiotechnol.biomedcentral.com/articles/10.1186/1472-6750-1-10'),(11379,'Vasi FK, Lenski RE','Ecological strategies and fitness tradeoffs inEscherichia coli mutants adapted to prolonged starvation','J Genet',1999,'78:43-49',NULL,''),(11380,'Souza V, Turner PE, Lenski RE','Long-term experimental evolution in Escherichia coli','J Evol Biol',1997,'10:743-769',NULL,'http://onlinelibrary.wiley.com/doi/10.1046/j.1420-9101.1997.10050743.x/full'),(11381,'Kibota T, Lynch M, 2012]','Estimate of the genomic mutation rate deleterious to overall fitness in E coli','Nature',1996,'381:694-696',8649513,'https://www.nature.com/articles/381694a0'),(11382,'Mongold JA, Bennett AF, Lenski RE','Evolutionary adaptation to temperature','Evolution',1996,'50:35-43',28568880,'http://www.jstor.org/stable/2410778?origin=crossref'),(11383,'Vasi FK, Travisano M, Lenski RE','Long-term experimental evolution in Escherichia coli','The American Naturalist',1994,'144:432-456',NULL,'http://www.journals.uchicago.edu/doi/10.1086/285685'),(11384,'Zambrano, María Mercedes, Siegele DA, Tormo A, A, Kolter R, Zambrano M, Siegele D, Almiron M, Tormo A, Kolter R','Microbial competition: Escherichia coli mutants that take over stationary phase cultures','Science',1993,'259:1757-1760',7681219,''),(11385,'Bennett AF, Lenski RE, Mittler JE','Evolutionary adaptation to temperature','Evolution',1992,'46:16-30',28564952,'https://www.jstor.org/stable/2409801?seq=1#page_scan_tab_contents'),(11386,'Kurlandzka A, Rosenzweig FR, Adams J','Identification of adaptive changes in an evolving population of Escherichia coli: the role of changes with regulatory and highly pleiotropic effects','Mol Biol Evol',1991,'8:261-281',2072859,'https://academic.oup.com/mbe/article/8/3/261/1044771'),(11387,'Lenski RE, Rose M, Simpson S, Tadler S','Long-term experimental evolution in Escherichia coli','Am Nat',1991,'138:1315-1341',NULL,'https://www.jstor.org/stable/2462549'),(11388,'Bennett AF, Dao KM, Lenski RE','Rapid evolution in response to high-temperature selection','Nature',1990,'346:79-81',2195353,'https://www.nature.com/articles/346079a0'),(11389,'Helling R, Vargas CN, Adams J, 2012]','Evolution of Escherichia coli during growth in a constant environment','Genetics',1987,'358:349-358',3301527,'http://www.genetics.org/content/116/3/349.short'),(11390,'Hong J, Hu J, Ke F','Experimental induction of bacterial resistance to the antimicrobial peptide tachyplesin I and investigation of the resistance mechanisms','Antimicrob Agents Chemother',2016,'60:6067-6075',27480861,'http://aac.asm.org/content/60/10/6067'),(11391,'Porse A, Schønning K, Munck C, Sommer MOAA','Survival and Evolution of a Large Multidrug Resistance Plasmid in New Clinical Bacterial Hosts','Mol Biol Evol',2016,'33:2860-2873',27501945,'https://academic.oup.com/mbe/article/33/11/2860/2271987'),(11392,'Burmeister AR, Lenski RE, Meyer JR','Host coevolution alters the adaptive landscape of a virus','Proc R Soc B Biol Sci',2016,'283:20161528',27683370,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2016.1528'),(11393,'Meyer JR, Dobias DT, Weitz JS, Barrick JE, Quick RT, Lenski RE','Repeatability and contingency in the evolution of a key innovation in phage lambda.','Science',2012,'335(6067): 428-32',22282803,'https://doi.org/10.1126/science.1214449'),(11394,'Shapiro JW, Turner PE','Evolution of mutualism from parasitism in experimental virus populations','Evolution',2018,'72:707-712',29380361,'http://onlinelibrary.wiley.com/doi/10.1111/evo.13440/abstract'),(11395,'Shapiro JW, Williams ESCP, Turner PE','Evolution of parasitism and mutualism between filamentous phage M13 and Escherichia coli','PeerJ',2016,'4:e2060',27257543,'https://peerj.com/articles/2060/'),(11396,'Ramiro RS, Costa H, Gordo I','Macrophage adaptation leads to parallel evolution of genetically diverse Escherichia coli small-colony variants with increased fitness in vivo and antibiotic collateral sensitivity','Evol Appl',2016,'9:994-1004',27606007,'http://doi.wiley.com/10.1111/eva.12397'),(11397,'Miskinyte M, Sousa A, Ramiro RS, de Sousa JAM, Kotlinowski J, Caramalho I, Magalhães S, Soares MP, Gordo I','The genetic basis of Escherichia coli pathoadaptation to macrophages','Plos Pathogens',2013,'9:e1003802',24348252,'http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1003802'),(11398,'Hiltunen T, Kaitala V, Laakso J, Becks L','Evolutionary contribution to coexistence of competitors in microbial food webs','Proc R Soc B Biol Sci',2017,'284:20170415',29021178,''),(11399,'Dionisio F, Conceição IC, Marques ACR, Fernandes L, Gordo I','The evolution of a conjugative plasmid and its ability to increase bacterial fitness','Biol Lett',2005,'1:250-252',17148179,'http://rsbl.royalsocietypublishing.org/content/1/2/250.short'),(11400,'Douglas SM, Chubiz LM, Harcombe WR, Ytreberg FM, Marx CJ','Parallel Mutations Result in a Wide Range of Cooperation and Community Consequences in a Two-Species Bacterial Consortium.','PLoS One',2016,'11(9): e0161837',27617746,'https://doi.org/10.1371/journal.pone.0161837'),(11401,'Harcombe W','Novel cooperation experimentally evolved between species','Evolution',2010,'64:2166-2172',20100214,'http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2010.00959.x/abstract'),(11402,'Lazaro E, Escarmís C, Perez-Mercader J, Manrubia SC, Domingo E, Lázaro E, Pérez-Mercader J','Resistance of virus to extinction on bottleneck passages: study of a decaying and fluctuating pattern of fitness loss','PNAS',2003,'100:10830-10835',12960384,'http://www.pnas.org/content/100/19/10830'),(11403,'Couñago R, Chen S, Shamoo Y','In vivo molecular evolution reveals biophysical origins of organismal fitness','Mol Cell',2006,'22:441-449',16713575,'http://www.cell.com/molecular-cell/fulltext/S1097-2765(06)00260-7?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276506002607%3Fshowall%3Dtrue'),(11404,'Tong S, Gao K, Hutchins DA','Adaptive evolution in the coccolithophore Gephyrocapsa oceanica following 1000 generations of selection under elevated CO 2','Glob Chan Biol',2018,'12:3218-3221',29356310,'http://onlinelibrary.wiley.com/doi/10.1111/gcb.14065/abstract'),(11405,'Santos-Lopez A, Bernabe-Balas C, Ares-Arroyo M, Ortega-Huedo R, Hoefer A, San Millan A, Gonzalez-Zorn B','A naturally occurring SNP in plasmid pB1000 produces a reversible increase in antibiotic resistance','Antimicrob Agents Chemother',2016,'',27895020,'http://aac.asm.org/content/early/2016/11/22/AAC.01735-16'),(11406,'Wang G, Wilson TJM, Jiang Q, Taylor DE','Spontaneous mutations that confer antibiotic Resistance in Helicobacter pylori','Antimicrob Agents Chemother',2001,'45:727-733',11181351,'http://aac.asm.org/content/45/3/727.abstract'),(11407,'Haas CE, Nix DE, Schentag JJ','In vitro selection of resistant Helicobacter pylori','Antimicrob Agents Chemother',1990,'34:1637-1641',2285275,'http://aac.asm.org/content/34/9/1637'),(11408,'Saini P, Beniwal A, Kokkiligadda A, Vij S','Evolutionary adaptation of Kluyveromyces marxianus strain for efficient conversion of whey lactose to bioethanol','Process Biochem',2017,'',NULL,''),(11409,'Douillard FP, Ribbera A, Xiao K, Ritari J, Rasinkangas P, Paulin L, Palva A, Hao Y, de Vos WM','Polymorphisms, chromosomal rearrangements, and mutator phenotype development during experimental evolution of Lactobacillus rhamnosus GG','Appl Environ Microbiol',2016,'82:3783-3792',27084020,'http://aem.asm.org/content/82/13/3783'),(11410,'Bachmann H, Fischlechner M, Rabbers I, Barfa N, Branco dos Santos F, Molenaar D, Teusink B','Availability of public goods shapes the evolution of competing metabolic strategies','Proc Natl Acad Sci',2013,'110:14302-7',23940318,'http://www.pnas.org/content/110/35/14302'),(11411,'Meyer JR, Dobias DT, Medina SJ, Servilio L, Gupta A, Lenski RE','Ecological speciation of bacteriophage lambda in allopatry and sympatry','Science',2016,'354:1301-1304',27884940,'http://science.sciencemag.org/content/354/6317/1301'),(11412,'Ju SY, Kim JH, Lee PC','Long-term adaptive evolution of Leuconostoc mesenteroides for enhancement of lactic acid tolerance and production','Biotechnol Biofuels',2016,'9:240',27843489,'https://biotechnologyforbiofuels.biomedcentral.com/articles/10.1186/s13068-016-0662-3'),(11413,'Hillesland KL, Stahl DA','Rapid evolution of stability and productivity at the origin of a microbial mutualism','Proc Natl Acad Sci',2010,'107:2124-2129',20133857,'http://www.pnas.org/content/107/5/2124'),(11414,'Agashe D, Sane M, Phalnikar K, Diwan GD, Habibullah A, Martinez-Gomez NC, Sahasrabuddhe V, Polachek W, Wang J, Chubiz LM, Marx CJ','Large-effect beneficial synonymous mutations mediate rapid and parallel adaptation in a bacterium','Mol Biol Evol',2016,'33:1542-1553',26908584,'https://academic.oup.com/mbe/article/33/6/1542/2579642'),(11415,'Hu B, Yang Y-M, Beck DAC, Wang Q-W, Chen W-J, Yang J, Lidstrom ME, Yang S','Comprehensive molecular characterization of Methylobacterium extorquens AM1 adapted for 1-butanol tolerance','Biotechnol Biofuels',2016,'9:84',NULL,'https://biotechnologyforbiofuels.biomedcentral.com/articles/10.1186/s13068-016-0497-y'),(11416,'Nayak DDD, Agashe D, Lee M-CC, Marx CJJ','Selection Maintains Apparently Degenerate Metabolic Pathways due to Tradeoffs in Using Methylamine for Carbon versus Nitrogen','Curr Biol',2016,'26:1416-1426',27212407,'http://www.cell.com/current-biology/fulltext/S0960-9822(16)30351-7?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0960982216303517%3Fshowall%3Dtrue'),(11417,'Carroll SM, Marx CJ','Evolution after introduction of a novel metabolic pathway consistently leads to restoration of wild-type physiology','PLoS Genet',2013,'9:e1003427',23593025,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003427#s4'),(11418,'Copin R, Wang X, Louie E, Escuyer V, Coscolla M, Gagneux S, Palmer GH, Ernst JD','Within Host Evolution Selects for a Dominant Genotype of Mycobacterium tuberculosis while T Cells Increase Pathogen Genetic Diversity','PLOS Pathog',2016,'12:e1006111',27973588,'http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006111'),(11419,'Guerrini V, Subbian S, Santucci P, Canaan S, Gennaro ML, Pozzi G','Experimental Evolution of Mycobacterium tuberculosis in Human Macrophages Results in Low-Frequency Mutations Not Associated with Selective Advantage','PLoS One',2016,'11:e0167989',27959952,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0167989'),(11420,'Sebastian J, Swaminath S, Nair RR, Jakkala K, Pradhan A, Ajitkumar P','De Novo Emergence of Genetically Resistant Mutants of Mycobacterium tuberculosis from the Persistence Phase Cells formed against Antituberculosis Drugs In Vitro','Antimicrob Agents Chemother',2016,'',27895008,'http://aac.asm.org/content/61/2/e01343-16'),(11421,'Zee PC, Liu J, Velicer GJ','Pervasive, yet idiosyncratic, epistatic pleiotropy during adaptation in a behaviourally complex microbe','J Evol Biol',2017,'30:257-269',27862537,''),(11422,'Rendueles O, Velicer GJ','Evolution by flight and fight: diverse mechanisms of adaptation by actively motile microbes','ISME J',2016,'1-14',27662568,'https://www.nature.com/articles/ismej2016115'),(11423,'Velicer GJ, Yu Y-TN','Evolution of novel cooperative swarming in the bacterium Myxococcus xanthus','Nature',2003,'425:75-78',12955143,''),(11424,'Dettman JR, Anderson JB, Kohn LM','Divergent adaptation promotes reproductive isolation among experimental populations of the filamentous fungus Neurospora','BMC Evol Biol',2008,'8:1-14',18237415,'https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-8-35'),(11425,'Cairns J, Coloma S, Sivonen K, Hiltunen T','Evolving interactions between diazotrophic cyanobacterium and phage mediate nitrogen release and host competitive ability','R Soc Open Sci',2016,'',28083116,'http://rsos.royalsocietypublishing.org/content/3/12/160839'),(11426,'Betteridge AL, Sumby KM, Sundstrom JF, Grbin PR, Jiranek V','Application of directed evolution to develop ethanol tolerant Oenococcus oeni for more efficient malolactic fermentation','Appl Microbiol Biotechnol',2017,'',29150706,''),(11427,'Bono LM, Gensel CL, Pfennig DW, Burch CL','Evolutionary rescue and the coexistence of generalist and specialist competitors: an experimental test','Proc R Soc B Biol Sci',2015,'282:20151932',26702041,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2015.1932'),(11428,'Bono LM, Gensel CL, Pfennig DW, Burch CL','Competition and the origins of novelty: experimental evolution of niche-width expansion in a virus','Biol Letters',2012,'',23075527,''),(11429,'Montville R, Froissart R, Remold SK, Tenaillon O, Turner PE','Evolution of mutational robustness in an RNA virus','PLoS Biol',2005,'3:e381',16248678,'http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0030381'),(11430,'Silander OK, Tenaillon O, Chao L, render','Understanding the evolutionary fate of finite populations: the dynamics of mutational effects','PLoS Biol',2007,'5:e94',17407380,'http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0050094'),(11431,'Bull JJ, Badgett M, Wichman HA, Huelsenbeck J, Hillis D, Gulati A, Ho C, Molineux IJ','Exceptional Convergent Evolution in a Virus','Genetics',1997,'147:1497-1507',9409816,'http://www.genetics.org/content/147/4/1497.short'),(11432,'Sota M, Yano H, M Hughes J, Daughdrill GW, Abdo Z, Forney LJ, Top EM','Shifts in the host range of a promiscuous plasmid through parallel evolution of its replication initiation protein','ISME J',2010,'4:1568-1580',20520653,'http://www.nature.com/articles/ismej201072'),(11433,'Kottara A, Hall JPJ, Harrison E, Brockhurst MA','Multi-host environments select for host-generalist conjugative plasmids','BMC Evol Biol',2016,'16:70',27039285,'https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-016-0642-z'),(11434,'Dößelmann B, Willmann M, Steglich M, Bunk B, Nübel U, Peter S, Neher RA','Rapid and consistent evolution of colistin resistance in extensively drug-resistant pseudomonas aeruginosa during morbidostat culture','Antimicrob Agents Chemotherp',2017,'61:1-16',28630206,''),(11435,'Granato ET, Kümmerli R','The path to re-evolve cooperation is constrained in Pseudomonas aeruginosa','BMC Evol Biol',2017,'17:1-11',28893176,''),(11436,'Jorth P, McLean K, Ratjen A, Secor PR, Bautista GE, Ravishankar S, Rezayat A, Garudathri J, Harrison JJ, Harwood RA, Penewit K, Waalkes A, Singh PK, Salipante SJ','Evolved Aztreonam Resistance Is Multifactorial and Can Produce Hypervirulence in Pseudomonas aeruginosa','mbio',2017,'8:e00517-17',29089424,''),(11437,'Lind PA, Farr AD, Rainey PB','Evolutionary convergence in experimental Pseudomonas populations','ISME J',2016,'11:1-12',27911438,'http://www.nature.com/articles/ismej2016157'),(11438,'Lowery NV, McNally L, Ratcliff WC, Brown SP','Division of Labor, Bet Hedging, and the Evolution of Mixed Biofilm Investment Strategies','Mbio',2017,'8:e00672-17',28790201,''),(11439,'Mehta HH, Prater AG, Shamoo Y','Using experimental evolution to identify druggable targets that could inhibit the evolution of antimicrobial resistance','J Antibiot (Tokyo',2017,'1-8',28928474,''),(11440,'Melnyk AH, McCloskey N, Hinz AJ, Dettman J, Kassen R','Evolution of Cost-Free Resistance under Fluctuating Drug Selection in Pseudomonas aeruginosa','mSphere',2017,'2:e00158-17',28744479,''),(11441,'Betts A, Gifford DR, MacLean RC, King KC','Parasite diversity drives rapid host dynamics and evolution of resistance in a bacteria-phage system','Evolution',2016,'70:969-978',27005577,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12909/abstract'),(11442,'Dettman JR, Sztepanacz JL, Kassen R','The properties of spontaneous mutations in the opportunistic pathogen Pseudomonas aeruginosa','BMC Genomics',2016,'17:27',26732503,'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702332/pdf/12864_2015_Article_2244.pdf'),(11443,'Flynn KM, Dowell G, Johnson TM, Koestler BJ, Waters CM, Cooper VS','Evolution of ecological diversity in biofilms of Pseudomonas aeruginosa by altered cyclic diguanylate signaling','J Bacteriol',2016,'198:2608-2618',27021563,'http://jb.asm.org/content/198/19/2608'),(11444,'Gifford DR, Toll-Riera M, MacLean RC','Epistatic interactions between ancestral genotype and beneficial mutations shape evolvability in Pseudomonas aeruginosa','Evolution',2016,'70:1659-1666',27230588,'http://doi.wiley.com/10.1111/evo.12958'),(11445,'Inglis RF, Biernaskie JM, Gardner A, Kümmerli R','Presence of a loner strain maintains cooperation and diversity in well-mixed bacterial communities','Proc R Soc B Biol Sci',2016,'283:20152682',26763707,'http://rspb.royalsocietypublishing.org/content/283/1822/20152682'),(11446,'Inglis RF, Scanlan PD, Buckling A','Iron availability shapes the evolution of bacteriocin resistance in Pseudomonas aeruginosa','ISME J',2016,'10:2060-2066',26905630,'https://www.nature.com/articles/ismej201615'),(11447,'Jochumsen N, Marvig RL, Damkiær S, Jensen RL, Paulander W, Molin S, Jelsbak L, Folkesson A','The evolution of antimicrobial peptide resistance in Pseudomonas aeruginosa is shaped by strong epistatic interactions','Nat Commun',2016,'7:13002',27694971,'https://www.nature.com/articles/ncomms13002'),(11448,'Qi Q, Toll-Riera M, Heilbron K, Preston GM, MacLean RC','The genomic basis of adaptation to the fitness cost of rifampicin resistance in Pseudomonas aeruginosa','Proc R Soc B Biol Sci',2016,'283:20152452',26763710,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2015.2452'),(11449,'Torres-Barceló C, Franzon B, Vasse M, Hochberg ME','Long-term effects of single and combined introductions of antibiotics and bacteriophages on populations of Pseudomonas aeruginosa','Evol Appl',2016,'9:583-595',27099623,'http://onlinelibrary.wiley.com/doi/10.1111/eva.12364/abstract;jsessionid=530EEEC6BE58BA0F2442C4ED470322F5.f03t04'),(11450,'Wassermann T, Meinike Jørgensen K, Ivanyshyn K, Bjarnsholt T, Khademi SMH, Jelsbak L, Høiby N, Ciofu O','The phenotypic evolution of Pseudomonas aeruginosa populations changes in the presence of subinhibitory concentrations of ciprofloxacin','Microbiology',2016,'162:865-875',26953154,'http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000273'),(11451,'Yen P, Papin JA','History of Antibiotic Adaptation Influences Microbial Evolutionary Dynamics Affiliations','PLOS Biol',2016,'',28792497,'http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001586'),(11452,'Kümmerli R, Santorelli LA, Granato ET, Dumas Z, Dobay A, Griffin AS, West SA','Co-evolutionary dynamics between public good producers and cheats in the bacterium Pseudomonas aeruginosa','J Evol Biol',2015,'28:2264-2274',26348785,'http://onlinelibrary.wiley.com/doi/10.1111/jeb.12751/abstract'),(11453,'Perron GG, Gonzalez A, Buckling A','Source-sink dynamics shape the evolution of antibiotic resistance and its pleiotropic fitness cost','Proc R Soc B Biol Sci',2007,'274:2351-2356',17650474,''),(11454,'Drago L, De Vecchi E, Nicola L, Tocalli L, Gismondo MR','In vitro selection of resistance in Pseudomonas aeruginosa and Acinetobacter spp','J Antimicrob Chemother',2005,'56:353-359',15967767,'https://academic.oup.com/jac/article/56/2/353/837303'),(11455,'Houte S Van, Buckling A, Westra ER, Chabas H, van Houte S, Høyland-Kroghsbo NM','Immigration of susceptible hosts triggers the evolution of alternative parasite defence strategies','Proc R Soc B Biol Sci',2016,'283:20160721',27581884,'http://rspb.royalsocietypublishing.org/content/283/1837/20160721'),(11456,'Morley D, Broniewski JM, Westra ER, Buckling A, van Houte S','Host diversity limits the evolution of parasite local adaptation','Mol Ecol',2016,'',27862566,'http://onlinelibrary.wiley.com/doi/10.1111/mec.13917/abstract'),(11457,'Michiels JE, Van den Bergh B, Verstraeten AN, Fauvart M, Michiels J','In vitro emergence of high persistence upon periodic aminoglycoside challenge in the ESKAPE pathogens','AAC',2016,'60:4630-4637',27185802,'http://aac.asm.org/content/60/8/4630'),(11458,'Davies E V., James CE, Williams D, O’Brien S, Fothergill JL, Haldenby S, Paterson S, Winstanley C, Brockhurst MA','Temperate phages both mediate and drive adaptive evolution in pathogen biofilms','Proc Natl Acad Sci',2016,'113:8266-8271',27382184,'http://www.pnas.org/lookup/doi/10.1073/pnas.1520056113'),(11459,'Tan J, Rattray JB, Yang X, Jiang L','Spatial storage effect promotes biodiversity during adaptive radiation','Proc R Soc London B Biol Sci',2017,'284:20170841',28701564,''),(11460,'Gounand I, Daufresne T, Gravel D, Bouvier C, Bouvier T, Combe M, Gougat-Barbera C, Torres-Barceló C, Mouquet N','Size evolution in microorganisms masks trade-offs predicted by the growth rate hypothesis','Proc R Soc B Biol Sci',2016,'283:20162272',28003453,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2016.2272'),(11461,'Jousset A, Eisenhauer N, Merker M, Mouquet N, Scheu S','High functional diversity stimulates diversification in experimental microbial communities','Sci Adv',2016,'2:e1600124-e1600124',27386573,'http://advances.sciencemag.org/content/2/6/e1600124'),(11462,'Tan J, Slattery MR, Yang X, Jiang L','Phylogenetic context determines the role of competition in adaptive radiation','Proc R Soc B Biol Sci',2016,'283:20160241',27335414,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2016.0241'),(11463,'Vogwill T, Phillips RL, Gifford DR, MacLean RC','Divergent evolution peaks under intermediate population bottlenecks during bacterial experimental evolution','Proc R Soc B Biol Sci',2016,'283:20160749',27466449,'http://rspb.royalsocietypublishing.org/lookup/doi/10.1098/rspb.2016.0749'),(11464,'Lind PA, Farr AD, Rainey PB','Experimental evolution reveals hidden diversity in evolutionary pathways','Elife',2015,'4:e07074',25806684,'https://elifesciences.org/articles/07074#s2'),(11465,'Flohr RCE, Blom CJ, Rainey PB, Beaumont HJE','Founder niche constrains evolutionary adaptive radiation','Proc Natl Acad Sci',2013,'110:20663-20668',24306929,'http://www.pnas.org/content/110/51/20663'),(11467,'Kim W, Levy SB, Foster KR','Rapid radiation in bacteria leads to a division of labour','Nat Commun',2008,'354:1395-1405',26852925,'http://www.nature.com/doifinder/10.1038/ncomms10508'),(11468,'Barrett RDH, MacLean RC, Bell G','Mutations of intermediate effect are responsible for adaptation in evolving Pseudomonas fluorescens populations','Biol Lett',2006,'2:236-238',17148371,'http://rsbl.royalsocietypublishing.org/content/2/2/236'),(11469,'Cairns J, Becks L, Jalasvuori M, Hiltunen T','Sublethal steptomycin concentrations and lytic bacteriophage together promote resistance evolution','Philos Trans R Soc',2016,'',27920385,'http://rstb.royalsocietypublishing.org/content/372/1712/20160040'),(11470,'Zhao X-F, Hao Y-Q, Zhang Q-G','Stability of A Coevolving Host-parasite System Peaks at Intermediate Productivity','PLoS One',2017,'12:e0168560',28076419,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0168560'),(11471,'Duncan AB, Dusi E, Jacob F, Ramsayer J, Hochberg ME, Kaltz O','Hot spots become cold spots: Coevolution in variable temperature environments','J Evol Biol',2016,'',27711983,'http://onlinelibrary.wiley.com/doi/10.1111/jeb.12985/abstract'),(11472,'Gorter FA, Scanlan PD, Buckling A','Adaptation to abiotic conditions drives local adaptation in bacteria and viruses coevolving in heterogeneous environments','Biol Lett',2016,'12:20150879',26888914,'http://rsbl.royalsocietypublishing.org/content/12/2/20150879'),(11473,'Hesse E, Buckling A','Host population bottlenecks drive parasite extinction during antagonistic coevolution','Evolution',2016,'70:235-240',26661325,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12837/abstract'),(11474,'Scanlan PD, Hall AR, Buckling A','Parasite genetic distance and local adaptation in coevolving bacteria-bacteriophage populations','Mol Ecol',2016,'',27775190,'http://onlinelibrary.wiley.com/doi/10.1111/mec.13897/abstract'),(11475,'Wright RCT, Brockhurst MA, Harrison E','Ecological conditions determine extinction risk in co-evolving bacteria-phage populations','BMC Evol Biol',2016,'16:227',27776482,'https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-016-0808-8'),(11476,'Harrison E, Truman J, Wright RCT, Spiers AJ, Paterson S, Brockhurst MA','Plasmid carriage can limit bacteria-phage coevolution','Biol Lett',2015,'11:3-6',26268992,'http://rsbl.royalsocietypublishing.org/content/11/8/20150361'),(11477,'Hall AR, Scanlan PD, Morgan AD, Buckling A','Host – parasite coevolutionary arms races give way to fluctuating selection','Ecol Lett',2011,'14:635-642',21521436,'http://onlinelibrary.wiley.com/doi/10.1111/j.1461-0248.2011.01624.x/abstract'),(11478,'Song C, Kidarsa TA, van de Mortel JE, Loper JE, Raaijmakers JM','Living on the edge: emergence of spontaneous gac mutations in Pseudomonas protegens during swarming motility','Environ Microbiol',2016,'18:3453-3465',26945503,'http://onlinelibrary.wiley.com/doi/10.1111/1462-2920.13288/abstract'),(11479,'Quesada JM, Fernández M, Soriano MI, Barrientos-Moreno L, Llamas MA, Espinosa-Urgel M','Rhizosphere selection of Pseudomonas putida KT2440 variants with increased fitness associated to changes in gene expression','Environ Microbiol Rep',2016,'8:842-850',27487165,'http://onlinelibrary.wiley.com/doi/10.1111/1758-2229.12447/abstract'),(11480,'Meaden S, Koskella B','Adaptation of the pathogen, Pseudomonas syringae, during experimental evolution on a native versus alternative host plant','Mol Ecol',2017,'26:1790-1801',28207977,'http://onlinelibrary.wiley.com/doi/10.1111/mec.14060/abstract'),(11481,'Bernardi AC, Gai CS, Lu J, Sinskey AJ, Brigham CJ','Experimental evolution and gene knockout studies reveal AcrA-mediated isobutanol tolerance in Ralstonia eutropha','J Biosci Bioeng',2015,'122:64-69',26811221,'http://www.sciencedirect.com/science/article/pii/S1389172315004958?via%3Dihub'),(11482,'Barberis P, Guidot A, Jiang W, Ferdy J, Th C','Multihost Experimental Evolution of the Pathogen Ralstonia solanacearum Unveils Genes Involved in Adaptation to Plants','MBE',2018,'31:2913-2928',25086002,''),(11483,'Marchetti M, Clerissi C, Yousfi Y, Gris C, Bouchez O, Rocha E, Cruveiller S, Jauneau A, Capela D, Masson-Boivin C','Experimental evolution of rhizobia may lead to either extra- or intracellular symbiotic adaptation depending on the selection regime','Mol Ecol',2017,'26:1818-1831',27770459,'http://onlinelibrary.wiley.com/doi/10.1111/mec.13895/abstract'),(11484,'Marchetti M, Jauneau A, Capela D, Remigi P, Gris C, Batut J, Masson-Boivin C','Shaping Bacterial Symbiosis With Legumes by Experimental Evolution','MPMI',2014,'27:956-964',25105803,'https://apsjournals.apsnet.org/doi/10.1094/MPMI-03-14-0083-R'),(11485,'Remigi P, Capela D, Clerissi C, Tasse L, Torchet R, Bouchez O, Batut J, Cruveiller S, Rocha EPC, Masson-Boivin C','Transient Hypermutagenesis Accelerates the Evolution of Legume Endosymbionts following Horizontal Gene Transfer','Plos Biol',2014,'12:e1001942',25181317,'http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001942'),(11486,'Wang XX, Wei Z, Li M, Shan A, Mei X, Jousset A, Shen Q, Xu Y, Friman V-P','Parasites and competitors suppress bacterial pathogen synergistically due to evolutionary trade-offs','Evolution',2017,'71:733-746',27925169,'http://onlinelibrary.wiley.com/doi/10.1111/evo.13143/abstract'),(11487,'Reyes LH, Kao KC','Growth-Coupled Carotenoids Production Using Adaptive Laboratory Evolution','Book',2018,'319–330',29170968,''),(11488,'Blundell JR, Schwartz K, Francois D, Fisher DS, Sherlock G, Levy SF','The dynamics of adaptive genetic diversity during the early stages of clonal evolution.','Nat Ecol Evol',2019,'3(2): 293-301',30598529,'https://doi.org/10.1038/s41559-018-0758-1'),(11489,'Bódi Z, Farkas Z, Nevozhay D, Kalapis D, Lázár V, Csörgő B, Nyerges Á, Szamecz B, Fekete G, Papp B, Araújo H, Oliveira JL, Moura G, Santos MAS, Székely T, Balázsi G, Pál C','Phenotypic heterogeneity promotes adaptive evolution','PLoS Biol',2017,'15:1-26',28486496,''),(11490,'Buskirk SW, Peace RE, Lang GI','Hitchhiking and epistasis give rise to cohort dynamics in adapting populations','Proc Natl Acad Sci',2017,'',28720700,'http://www.pnas.org/content/114/31/8330'),(11491,'Delneri D, Lovell SC, Naseeb S, Ames RM','Rapid functional and evolutionary changes follow gene duplication in yeast','Proc. R. Soc. B',2017,'',28835561,''),(11492,'Gorter FA, Aarts MGM, Zwaan BJ, de Visser JAGM','Local Fitness Landscapes Predict Yeast Evolutionary Dynamics in Directionally Changing Environments.','Genetics',2018,'208(1): 307-322',29141909,'https://doi.org/10.1534/genetics.117.300519'),(11493,'Gorter FA, Derks MFL, Heuvel J Van Den, Aarts MGM, Zwaan BJ, Ridder D De, Visser JAGM De','Genomics of Adaptation Depends on the Rate of Environmental Change in Experimental Yeast Populations','Mol Biol Evol',2018,'34:2613-2626',28957501,''),(11494,'Hope EA, Amorosi CJ, Miller AW, Dang K, Heil CS, Dunham MJ, Smukowski Heil C','Experimental evolution reveals favored adaptive routes to cell aggregation in yeast','Genetics',2017,'206:1153-1167',28450459,'http://www.genetics.org/content/206/2/1153'),(11495,'Jacobeen S, Pentz JT, Graba EC, Brandys CG, Ratcliff WC, Yunker PJ','Cellular packing, mechanical stress and the evolution of multicellularity','Nature Physics',2017,'1-6',NULL,''),(11496,'Kosheleva K, Desai MM','Recombination alters the dynamics of adaptation on standing variation in laboratory yeast populations','Mol Biol Evol',2017,'1-43',29069452,'https://academic.oup.com/mbe/article-abstract/35/1/180/4563780'),(11497,'Gjuvsland AB, Zörgö E, Samy JK, Stenberg S, Demirsoy IH, Roque F, Maciaszczyk‐Dziubinska E, Migocka M, Alonso‐Perez E, Zackrisson M, Wysocki R, Tamás MJ, Jonassen I, Omholt SW, Warringer J','Disentangling genetic and epigenetic determinants of ultrafast adaptation','Mol Syst Bio',2016,'12:892',27979908,'http://msb.embopress.org/content/12/12/892'),(11498,'Gorter FA, Aarts MMG, Zwaan BJ, Visser JAGM De','Dynamics of Adaptation in Experimental Yeast Populations Exposed to Gradual and Abrupt Change in Heavy Metal Concentration','Am Nat',2016,'187',27277407,''),(11499,'Healey D, Axelrod K, Gore J','Negative frequency-dependent interactions can underlie phenotypic heterogeneity in a clonal microbial population','Mol Syst Biol',2016,'877',27487817,'http://msb.embopress.org/content/12/8/877.long'),(11500,'Venkataram S, Dunn B, Li Y, Agarwala A, Chang J, Ebel ER, Geiler-Samerotte K, Hérissant L, Blundell JR, Levy SF, Fisher DS, Sherlock G, Petrov DA','Development of a Comprehensive Genotype-to-Fitness Map of Adaptation-Driving Mutations in Yeast','Cell',2016,'166:1585-1596',27594428,'http://linkinghub.elsevier.com/retrieve/pii/S0092867416310108'),(11501,'McDonald MJ, Rice DP, Desai MM','Sex speeds adaptation by altering the dynamics of molecular evolution.','Nature',2016,'531(7593): 233-6',26909573,'https://doi.org/10.1038/nature17143'),(11502,'Laan L, Koschwanez JH, Murray AW','Evolutionary adaptation after crippling cell polarization follows reproducible trajectories','Elife',2015,'4:1-18',26426479,''),(11503,'Levy SF, Blundell JR, Venkataram S, Petrov D a, Fisher DS, Sherlock G','Quantitative evolutionary dynamics using high-resolution lineage tracking','Nature',2015,'519:181-186',25731169,'https://www.nature.com/articles/nature14279'),(11504,'Hong J, Gresham D','Molecular specificity, convergence and constraint shape adaptive evolution in nutrient-poor nvironments','PLoS Genetics',2014,'10:e1004041',24415948,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004041'),(11505,'Kildegaard KR, Hallström BM, Blicher TH, Sonnenschein N, Jensen NB, Sherstyk S, Harrison SJ, Maury J, Herrgård MJ, Juncker AS, Forster J, Nielsen J, Borodina I','Evolution reveals a glutathione-dependent mechanism of 3-hydroxypropionic acid tolerance','Metab Eng',2014,'26:57-66',25263954,'http://www.sciencedirect.com/science/article/pii/S1096717614001189?via%3Dihub#s0010'),(11506,'Kohn LM, Anderson JB','The underlying structure of adaptation under strong selection in twelve experimental yeast populations','Eukaryot Cell',2014,'13:1200-1206',25016004,'http://ec.asm.org/content/13/9/1200.long'),(11507,'Kryazhimskiy S, Rice DP, Jerison ER, Desai MM','Global epistasis makes adaptation predictable despite sequence-level stochasticity','Science',2014,'344:1519-1522',24970088,'http://science.sciencemag.org/content/344/6191/1519.long'),(11508,'Szamecz B, Boross G, Kalapis D, Kovács K, Fekete G, Farkas Z, Lázár V, Hrtyan M, Kemmeren P, Groot Koerkamp MJ a, Rutkai E, Holstege FCP, Papp B, Pál C','The genomic landscape of compensatory evolution','PLoS Biol',2014,'12:e1001935',25157590,'http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001935'),(11509,'Lang GI, Rice DP, Hickman MJ, Sodergren E, Weinstock GM, Botstein D, Desai MM','Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations','Nature',2013,'500:571-574',23873039,'http://www.nature.com/doifinder/10.1038/nature12344'),(11510,'Miller AW, Befort C, Kerr EO, Dunham MJ','Design and use of multiplexed chemostat arrays','J Vis Exp',2013,'',23462663,'https://www.jove.com/video/50262/design-and-use-of-multiplexed-chemostat-arrays'),(11511,'Avrahami-Moyal L, Engelberg D, Wenger JW, Sherlock G, Braun S','Turbidostat culture of Saccharomyces cerevisiae W303-1A under selective pressure elicited by ethanol selects for mutations in SSD1 and UTH1','FEMS Yeast Res',2012,'29:997-1003',22443114,'https://academic.oup.com/femsyr/article/12/5/521/495657'),(11512,'Gerstein AC, Lo DS, Otto SP','Parallel Genetic Changes and Nonparallel Gene – Environment Interactions Characterize the Evolution of Drug Resistance in Yeast','Genetics',2012,'192:241-252',22714405,''),(11513,'Kryazhimskiy S, Rice DP, Desai MM','Population subdivision and adaptation in asexual populations of saccharomyces cerevisiae','Evolution',2012,'66:1931-1941',22671557,'http://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2011.01569.x/abstract'),(11514,'Ratcliff WC, Denison RF, Borrello M, Travisano M','Experimental evolution of multicellularity.','Proc Natl Acad Sci U S A',2012,'109(5): 1595-600',22307617,'https://doi.org/10.1073/pnas.1115323109'),(11515,'Bell G, Gonzalez A','Adaptation and evolutionary rescue in metapopulations experiencing environmental deterioration','Science',2011,'332:1327-1330',21659606,'http://science.sciencemag.org.kuleuven.ezproxy.kuleuven.be/content/332/6035/1327'),(11516,'Lang GI, Botstein D, Desai MM','Genetic variation and the fate of beneficial mutations in asexual populations','Genetics',2011,'188:647-661',21546542,'http://www.genetics.org/content/188/3/647'),(11517,'Stanley D, Fraser S, Chambers PJ, Rogers P, Stanley G a','Generation and characterisation of stable ethanol-tolerant mutants of Saccharomyces cerevisiae','J Ind Microbiol Biotechnol',2010,'37:139-149',19902282,'https://link.springer.com/article/10.1007%2Fs10295-009-0655-3'),(11518,'Bell G, Gonzalez A','Evolutionary rescue can prevent extinction following environmental change','Ecol Lett',2009,'12:942-948',19659574,'http://onlinelibrary.wiley.com/doi/10.1111/j.1461-0248.2009.01350.x/abstract'),(11519,'Gresham D, Desai MM, Tucker CM, Jenq HT, Pai D a, Ward A, DeSevo CG, Botstein D, Dunham MJ','The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast','PLoS Genet',2008,'4:e1000303',19079573,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000303'),(11520,'Kao KC, Sherlock G','Molecular characterization of clonal interference during adaptive evolution in asexual populations of Saccharomyces cerevisiae','Nat Genet',2008,'40:1499-1504',19029899,'http://science.sciencemag.org/content/311/5767/1615'),(11521,'Dettman JR, Sirjusingh C, Kohn LM, Anderson JB','Incipient speciation by divergent adaptation and antagonistic epistasis in yeast','Nature',2007,'447',17538619,'https://www.nature.com/articles/nature05856'),(11522,'Dickinson WJ','Synergistic fitness interactions and a high frequency of beneficial changes among mutations accumulated under relaxed selection in Saccharomyces cerevisiae','Genetics',2008,'178:1571-1578',18245361,'http://www.genetics.org/content/early/2008/02/01/genetics.107.080853'),(11523,'Leu JY, Murray AW','Experimental evolution of mating discrimination in budding yeast','Curr Biol',2006,'16:280-286',16461281,'https://www.sciencedirect.com/science/article/pii/S0960982205015897'),(11524,'Goddard MR, Charles H, Godfray J, Burt A','Sex increases the efficacy of natural selection in experimental yeast populations','Nature',2005,'434:636-640',15800622,'https://www.nature.com/articles/nature03405'),(11525,'Zeyl CW, de Visser JAGM, 2012]','Estimates of the rate and distribution of fitness effects of spontaneous mutation in Saccharomyces cerevisiae','Genetics',2001,'157:53-61',11139491,'http://www.genetics.org/content/157/1/53.short'),(11526,'Ferea TL, Botstein D, Brown PO, Rosenzweig FR','Systematic changes in gene expression patterns following adaptive evolution in yeast','Proc Natl Acad Sci',1999,'96:9721-9726',10449761,'http://www.pnas.org/content/96/17/9721.long'),(11527,'Aarnio TH, Suihko ML, Kauppinen VS','Isolation of acetic acid-tolerant Baker’s yeast variants in a turbidostat','Appl Biochem Biotechnol',1991,'27:55-63',NULL,'https://link.springer.com/article/10.1007/BF02921515'),(11528,'Francis JC, Hansche PE','Directed evolution of metabolic pathways in microbial populations','Genetics',1972,'70:59-73',4552227,'http://www.genetics.org/content/70/1/59.long'),(11529,'Pieczynska MD, Wloch-Salamon D, Korona R, de Visser JAGM','Rapid multiple-level coevolution in experimental populations of yeast killer and nonkiller strains','Evolution',2016,'70:1342-1353',27168531,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12945/abstract'),(11530,'Dunn B, Paulish T, Stanbery A, Piotrowski JS, Koniges G, Kroll E, Louis EJ, Liti G, Sherlock G, Rosenzweig FR','Recurrent rearrangement during adaptive evolution in an interspecific yeast hybrid suggests a model for rapid introgression','PLoS Genet',2013,'9:e1003366',23555283,'http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003366'),(11531,'Samani P, Bell G','The ghosts of selection past reduces the probability of plastic rescue but increases the likelihood of evolutionary rescue to novel stressors in experimental populations of wild yeast','Ecol Lett',2016,'19:289-298',26777785,'http://onlinelibrary.wiley.com/doi/10.1111/ele.12566/abstract'),(11533,'Samani P, Bell G','Experimental evolution of the grain of metabolic specialization in yeast','Ecol Evol',2016,'6:3912-3922',27516854,'http://onlinelibrary.wiley.com/doi/10.1002/ece3.2151/abstract'),(11534,'Samani P, Bell G','Adaptation of experimental yeast populations to stressful conditions in relation to population size','J Evol Biol',2010,'23:791-796',20149025,'http://dx.doi.org/10.1111/j.1420-9101.2010.01945.x'),(11535,'Nilsson AI, Koskiniemi S, Eriksson S, Kugelberg E, Hinton JCD, Andersson DI','Bacterial genome size reduction by experimental evolution','PNAS',2005,'102:12112-12116',16099836,'http://www.pnas.org/content/102/34/12112'),(11536,'Søndberg E, Jelsbak L','Salmonella Typhimurium undergoes distinct genetic adaption during chronic infections of mice','BMC Microbiol',2016,'16:30',26955808,'https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-016-0646-2'),(11537,'Ali SS, Soo J, Rao C, Leung AS, Ngai DH-M, Ensminger AW, Navarre WW','Silencing by H-NS potentiated the evolution of Salmonella','PLoS Pathog',2014,'10:e1004500',25375226,'http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1004500'),(11538,'Davidson CJ, White AP, Surette MG','Evolutionary loss of the rdar morphotype in Salmonella as a result of high mutation rates during laboratory passage','ISME J',2008,'2:293-307',18256702,'https://www.nature.com/articles/ismej20084'),(11539,'Clark TA, Anderson JB','Dikaryons of the basidiomycete fungus Schizophyllum commune: Evolution in long-term culture','Genetics',2004,'167:1663-1675',15342506,'http://www.genetics.org/content/167/4/1663'),(11540,'Dover JA, Burmeister AR, Molineux IJ, Parent KN','Evolved populations of Shigella flexneri phage Sf6 acquire large deletions, altered genomic architecture, and faster life cycles','Genome Biol Evol',2016,'8:2827-2840',27497318,'https://academic.oup.com/gbe/article/8/9/2827/2236259'),(11541,'Morley VJ, Mendiola SY, Turner PE','Rate of novel host invasion affects adaptability of evolving RNA virus lineages','Proc R Soc B Biol Sci',2015,'282:20150801',26246544,'http://rspb.royalsocietypublishing.org/content/282/1813/20150801.long'),(11542,'Baumgartner M, Roffler S, Wicker T, Pernthaler J','Letting go: bacterial genome reduction solves the dilemma of adapting to predation mortality in a substrate-restricted environment','ISME journal',2017,'11:2258-2266',28585936,''),(11543,'Zhang T, Tremblay P-L','An Adaptive Laboratory Evolution Method to Accelerate Autotrophic Metabolism','Book',2018,'',29170958,''),(11544,'Cao S, Huseby DL, Brandis G, Diarmaid H','Alternative Evolutionary Pathways for Drug-Resistant Small Colony Variant Mutants in Staphylococcus aureus','mBio',2017,'8:1-15',28634236,''),(11545,'Jiao YJ, Baym M, Veres A, Kishony R','Population diversity jeopardizes the efficacy of antibiotic cycling','biorxiv',2016,'1-38',NULL,''),(11546,'Johnston PR, Dobson AJ, Rolff J','Genomic signatures of experimental adaptation to antimicrobial peptides in Staphylococcus aureus','G3 Genes|Genomes|Genetics',2016,'6:1535-1539',27172179,'http://www.g3journal.org/content/6/6/1535'),(11547,'Kubicek-Sutherland JZ, Lofton H, Vestergaard M, Hjort K, Ingmer H, Andersson DI','Antimicrobial peptide exposure selects for Staphylococcus aureus resistance to human defence peptides','J Antimicrob Chemother dkw',2016,'',27650186,'http://www.jac.oxfordjournals.org/lookup/doi/10.1093/jac/dkw381'),(11548,'Evgrafov MR De, Gumpert H, Munck C, Thomsen TT, Sommer MOA','Collateral resistance and sensitivity modulate evolution of high-level resistance to drug combination treatment in Staphylococcus aureus','Mol Biol Evol',2015,'32:1175-1185',26538142,'https://academic.oup.com/mbe/article/32/5/1175/1123081'),(11549,'Dobson AJ, Purves J, Kamysz W, Rolff J','Comparing Selection on S. aureus between Antimicrobial Peptides and Common Antibiotics','Plos One',2013,'8:3-7',24204634,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0076521#s4'),(11550,'Ford SA, Williams D, Paterson S, King KC','Coevolutionary dynamics between a defensive microbe and a pathogen driven by fluctuating selection','Mol Ecol',2016,'',27862515,'http://onlinelibrary.wiley.com/doi/10.1111/mec.13906/abstract'),(11551,'Ford SA, Kao D, Williams D, King KC','Microbe-mediated host defence drives the evolution of reduced pathogen virulence','Nat Commun',2016,'7:1-9',27845328,'http://dx.doi.org/10.1038/ncomms13430'),(11552,'King KC, Brockhurst MA, Vasieva O, Paterson S, Betts A, Ford SA, Frost CL, Horsburgh MJ, Haldenby S, Hurst GD','Rapid evolution of microbe-mediated protection against pathogens in a worm host','ISME J',2016,'10:1915-1924',26978164,'http://www.nature.com/doifinder/10.1038/ismej.2015.259'),(11553,'Enav H, Kirzner S, Lindell D, Mandel-gutfreund Y','Adaptation to sub-optimal hosts is a driver of viral diversification in','bioRxiv',2018,'1-27',30409965,''),(11554,'Adams HM, Joyce LR, Guan Z, Akins RL, Palmer KL','Streptococcus mitis and S. oralis Lack a Requirement for CdsA, the Enzyme Required for Synthesis of Major Membrane Phospholipids in Bacteria Hannah','Antimicrob Agents Chemotherp',2017,'61:1-11',28223392,''),(11555,'Carsenti-Etesse H, Roger PM, Dunais B, Durgeat S, Mancini G, Bensoussan M, Dellamonica P','Gradient plate method to induce Streptococcus pyogenes resistance','J Antimicrob Chemother',1999,'44:439-443',10588303,'https://academic.oup.com/jac/article/44/4/439/671652'),(11556,'Westhoff S, Leeuwe T Van, Zhang Z, Wezel G Van, Rozen DE, van Leeuwe TM, Qachach O, Zhang Z, van Wezel GP, Rozen DE','The evolution of no-cost resistance at sub-MIC concentrations of streptomycin in Streptomyces coelicolor','ISME J',2016,'1-11',28094796,'http://www.nature.com/doifinder/10.1038/ismej.2016.194'),(11557,'McCarthy S, Johnson T, Pavlik BJ, Payne S, Schackwitz W, Martin J, Lipzen A, Keffeler E, Blum P','Expanding the Limits of Thermoacidophily by Adaptive Evolution','AEM',2016,'82:857-867',26590281,'http://aem.asm.org/content/82/3/857.full'),(11558,'Jung H-C, Lee SHS-M, Lee SHS-M, An YJ, Lee J-H, Lee HS, Kang SG','Adaptive evolution of a hyperthermophilic archaeon pinpoints a formate transporter as a critical factor for the growth enhancement on formate','Sci Rep',2017,'7:6124',28733620,''),(11559,'Singh R, Gradnigo J, White D, Lipzen A, Martin J, Schackwitz W, Moriyama E, Blum P','Complete Genome Sequence of an Evolved Thermotoga maritima Isolate','Gen Ann',2015,'3:9-10',26021931,'http://genomea.asm.org/content/3/3/e00557-15'),(11560,'Willemsen A, Zwart MP, Ambrós S, Carrasco JL, Elena SF','2b or not 2b : Experimental evolution of functional exogenous sequences in a plant RNA virusGenome Biology and Evolution','Genome Biol Evol',2017,'',28137747,'https://academic.oup.com/gbe/article/9/2/279/2896743'),(11561,'Cervera H, Lalić J, Elena SF','Effect of host species on topography of the fitness landscape for a plant RNA irus','J Virol',2016,'90:10160-10169',27581976,'http://jvi.asm.org/content/90/22/10160'),(11562,'Willemsen A, Zwart MP, Tromas N, Majer E, Daros J-A, Elena SF','Multiple barriers to the evolution of alternative gene orders in a positive-strand RNA virus','Genetics',2016,'202:1503-1521',26868766,'http://www.genetics.org/content/202/4/1503'),(11564,'Lalić J, Elena SF','The impact of high-order epistasis in the within-host fitness of a positive-sense plant RNA virus','J Evol Biol',2015,'28:2236-2247',26344415,'http://onlinelibrary.wiley.com/doi/10.1111/jeb.12748/abstract'),(11565,'Hillung J, Cuevas JM, Valverde S, Elena SF','Experimental evolution of an emerging plant virus in host genotypes that differ in their susceptibility to infection','Evolution',2014,'2467-2480',24889935,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12458/abstract'),(11566,'Agudelo-Romero P, Carbonell P, Perez-Amador M a, Elena SF','Virus adaptation by manipulation of host’s gene expression','Plos one',2008,'',18545680,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0002397'),(11567,'de la Iglesia F, Elena SF','Fitness declines in Tobacco etch virus upon serial bottleneck transfers','Journal of Virology',2007,'81:4941-4947',17344305,'http://jvi.asm.org/content/81/10/4941.abstract'),(11568,'Lin H, Kazlauskas RJ, Travisano M, Travisano M','Developmental evolution facilitates rapid evolution of dispersal','Sci Rep',2017,'2-10',29162942,''),(11569,'Lin H, Travisano M, Kazlauskas RJ','Experimental evolution of Trichoderma citrinoviride for faster deconstruction of cellulose','PLoS One',2016,'11:e0147024',26820897,'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147024'),(11570,'Walworth NG, Lee MD, Fu F-X, Hutchins DA, Webb EA','Molecular and physiological evidence of genetic assimilation to high CO 2 in the marine nitrogen fixer Trichodesmium','Proc Natl Acad Sci',2016,'',27830646,'http://www.pnas.org/content/113/47/E7367'),(11571,'Garijo R, Cuevas JM, Briz Á, Sanjuán R','Constrained evolvability of interferon suppression in an RNA virus','Sci Rep',2016,'6:24722',27098004,'https://www.nature.com/articles/srep24722'),(11572,'Morley VJ, Sistrom M, Usme-Ciro JA, Remold SK, Turner PE','Evolution in spatially mixed host environments increases divergence for evolved fitness and intrapopulation genetic diversity in RNA viruses','Virus Evol',2016,'2:vev022',27774292,'https://academic.oup.com/ve/article/2/1/vev022/1753359'),(11573,'Wasik BR, Muñoz-Rojas AR, Okamoto KW, Miller-Jensen K, Turner PE','Generalized selection to overcome innate immunity selects for host breadth in an RNA virus','Evolution',2016,'70:270-281',26882316,'http://onlinelibrary.wiley.com/doi/10.1111/evo.12845/abstract'),(11574,'Williams ESCPCP, Morales NM, Wasik BR, Brusic V, Whelan SPJ, Turner PE','Repeatable population dynamics among vesicular stomatitis virus lineages evolved under high co-infection','Front Microbiol',2016,'',27065953,'https://www.frontiersin.org/articles/10.3389/fmicb.2016.00370/full'),(11575,'Beabout K, McCurry MD, Mehta H, Shah AA, Pulukuri KK, Rigol S, Wang Y, Nicolaou KC, Shamoo Y','Experimental Evolution of Diverse Strains as a Method for the Determination of Biochemical Mechanisms of Action for Novel Pyrrolizidinone Antibiotics','ACS Inf Dis',2017,'',28942642,''),(11576,'Bedhomme S, Perez Pantoja D, Bravo IG','Plasmid and clonal interference during post-horizontal gene transfer evolution','Mol Ecol',2017,'38:42-49',28206693,''),(11577,'Bonhoeffer S, Allen RC, Engelsta J, Mcdonald BA, Hall AR, Allen RC, Hall AR','Reversing resistance : different routes and common themes across pathogens','Proc Biol Sci',2017,'',28954914,''),(11578,'Couce A, Caudwell LV, Feinauer C, Hindré T, Feugeas J, Weigt M, Lenski RE, Schneider D, Tenaillon O','Mutator genomes decay, despite sustained fitness gains, in a long-term experiment with bacteria','Proc Natl Acad Sci',2017,'114:E9026-E9035',29073099,''),(11579,'LeClair JS','The Impact of Population Bottlenecks on the Social Lives of Microbes','J Stat Phys',2017,'',NULL,''),(11580,'Erickson KE, Otoupal PB, Chatterjee A','Transcriptome-Level Signatures in Gene Expression and Gene Expression Variability during Bacterial Adaptive Evolution','mSphere',2017,'2:e00009-17',28217741,''),(11581,'Fronhofer EA, Gut S, Altermatt F','Evolution of density-dependent movement during replicated experimental range expansions','bioRxiv',2017,'',28977712,''),(11582,'Gallet R, Froissart R, Ravigné V','Maintenance of Polymorphism in Spatially Heterogeneous Environments: An Experimental Test of Hard and Soft Selection','bioRxiv',2017,'',29882597,''),(11583,'Good BH, McDonald MJ, Barrick JE, Lenski RE, Desai MM','The dynamics of molecular evolution over 60,000 generations','Nature',2017,'551:45-50',29045390,''),(11584,'Hall JPJ, Williams D, Paterson S, Harrison E, Brockhurst MA','Positive selection inhibits gene mobilization and transfer in soil bacterial communities','Nat Ecol Evol',2017,'',28890938,''),(11586,'Hall RN, Capucci L, Matthaei M, Esposito S, Kerr PJ, Frese M, Strive T','An in vivo system for directed experimental evolution of rabbit haemorrhagic disease virus','PLoS One',2017,'12:e0173727',28288206,''),(11587,'Harrison E, Hall JJP, Paterson S, Spiers AJ, Brockhurst MA','Conflicting selection alters the trajectory of molecular evolution in a tripartite bacteria-plasmid-phage interaction','Mol Ecol',2017,'38:42-49',28247474,''),(11588,'Holmskov M, Moreira-maia CR, Ramstad E, Krogh HB, Groth C, Gillies D, Zwi M, Skoog M, Gluud C, Simonsen E','','Copyright',2017,'1-19',NULL,''),(11589,'Jia N, Ding M-Z, Zou Y, Gao F, Yuan Y-J','Comparative genomics and metabolomics analyses of the adaptation mechanism in Ketogulonicigenium vulgare-Bacillus thuringiensis consortium','Sci Rep',2017,'7:46759',28440340,''),(11590,'Kacar B, Garmendia E, Tuncbag N, Andersson DI, Hughes D','Functional constraints on replacing an essential gene with its ancient and modern homologs','MBio',2017,'8:1-30',28851849,''),(11591,'Kacar B, Ge X, Sanyal S, Gaucher EA','Experimental Evolution of Escherichia coli Harboring an Ancient Translation Protein','J Mol Evol',2017,'84:69-84',28233029,''),(11592,'Kalily E, Hollander A, Korin B, Cymerman I, Yaron S','Salmonella Senftenberg adaptation to linalool and its association with antibiotic resistance and environmental persistence','Appl Environ Microbiol',2017,'',28258149,''),(11593,'Laanto E, Hoikkala V, Ravantti J, Sundberg L-R','Long-term genomic coevolution of host-parasite interaction in natural environment','bioRxiv',2017,'',28740072,''),(11594,'LaCroix RA, Palsson BØ, Feist AM','A Model for Designing Adaptive Laboratory Evolution Experiments','Appl Environ Microbiol',2017,'',28159796,''),(11595,'Leister D','Experimental evolution in photoautotrophic microorganisms as a means of enhancing chloroplast functions','Essays Biochem',2017,'',28887328,''),(11596,'Long CP, Gonzalez JE, Feist AM, Palsson BØ, Antoniewicz MR','Fast growth phenotype of E. coli K-12 from adaptive laboratory evolution does not require intracellular flux rewiring','Metab Eng',2017,'44:100-107',28951266,''),(11597,'López García de Lomana A, Kaur A, Turkarslan S, Beer KD, Mast FD, Smith JJ, Aitchison JD, Baliga NS','Adaptive Prediction Emerges Over Short Evolutionary Time Scales','Genome Biol Evol',2017,'9:1616-1623',28854640,''),(11598,'Mcnamara KB, Simmons LW','Experimental evolution reveals differences between phenotypic and evolutionary responses to population density','J Evol Biol',2017,'38:42-49',28675768,''),(11599,'Morley VJ, Turner PE','Dynamics of molecular evolution in RNA virus populations depend on sudden versus gradual environmental change','Evolution',2017,'1-31',28121018,''),(11600,'Moura De Sousa J, Balbontı R, Durão P, Gordo I, Dur P, Gordo I, Sousa JM De, Balbontı R','Multidrug-resistant bacteria compensate for the epistasis between resistances','Plos Biology',2017,'',28419091,''),(11601,'Moxon R, Kussell E','The impact of bottlenecks on microbial survival, adaptation and phenotypic switching in host-pathogen interactions','Evolution',2017,'',28983912,''),(11602,'Niehus R, Picot A, Oliveira NM, Mitri S, Foster KR','The evolution of siderophore production as a competitive trait','Evolution',2017,'19-52',28323325,''),(11603,'Nishimura I, Kurokawa M, Liu L, Ying B-W','Coordinated Changes in Mutation and Growth Rates Induced by Genome Reduction','MBio 8:e',2017,'8:e00676-17',28679744,''),(11604,'Phan K, Ferenci T','The fitness costs and trade-off shapes associated with the exclusion of nine antibiotics by OmpF porin channels','ISME J',2017,'11:1472-1482',28072422,''),(11605,'Sabater-Muñoz B, Toft C, Alvarez-Ponce D, Fares MA','Chance and necessity in the genome evolution of endosymbiotic bacteria of insects','ISME J',2017,'11:1291-1304',28323281,''),(11606,'Sackman AM, McGee LW, Morrison AJ, Pierce J, Anisman J, Hamilton H, Sanderbeck S, Newman C, Rokyta DR','Mutation-Driven Parallel Evolution during Viral Adaptation','Mol Biol Evol',2017,'34:3243-3253',29029274,''),(11607,'Salverda MLM, Koomen J, Koopmanschap B, Zwart MP, de Visser JAGM','Adaptive benefits from small mutation supplies in an antibiotic resistance enzyme','Proc Natl Acad Sci',2017,'',29133391,''),(11608,'Sanchez MR, Miller AW, Liachko I, Sunshine AB, Lynch B, Huang M, Alcantara E, DeSevo CG, Pai DA, Tucker CM, Hoang ML, Dunham MJ','Differential paralog divergence modulates genome evolution across yeast species','PLOS Genet 13:e',2017,'13:e1006585',28196070,''),(11609,'Song J, Rensing C, Holm PE, Virta M, Brandt KK','Comparison of Metals and Tetracycline as Selective Agents for Development of Tetracycline Resistant Bacterial Communities in Agricultural Soil','Environ Sci Technol',2017,'',28198616,''),(11610,'Sousa A, Ramiro RS, Barroso-Batista J, Güleresi D, Lourenço M, Gordo I','Recurrent Reverse Evolution Maintains Polymorphism After Strong Bottlenecks in Commensal Gut Bacteria','Mol Biol Evol',2017,'',28961745,''),(11611,'Stalder T, Rogers LM, Renfrow C, Yano H, Smith Z, Top EM','Emerging patterns of plasmid-host coevolution that stabilize antibiotic resistance','Sci Rep',2017,'7:1-10',28687759,''),(11612,'Stern A, Yeh M Te, Zinger T, Smith M, Wright C, Ling G, Nielsen R, Macadam A, Andino R','The Evolutionary Pathway to Virulence of an RNA Virus','Cell',2017,'169:35-46',28340348,''),(11613,'Tan J, Yang X, Jiang L','Species ecological similarity modulates the importance of colonization history for adaptive radiation','Evolution',2017,'1-9',28444894,''),(11614,'Tashiro Y, Eida H, Ishii S, Futamata H, Okabe S','Generation of Small Colony Variants in Biofilms by &lt;i&gt;Escherichia coli&lt;/i&gt; Harboring a Conjugative F Plasmid','Microbes Environ',2017,'0:1-7',28302951,''),(11615,'Tognon M, Köhler T, Gdaniec BG, Hao Y, Lam JS, Beaume M, Luscher A, Buckling A, van Delden C','Co-evolution with Staphylococcus aureus leads to lipopolysaccharide alterations in Pseudomonas aeruginosa','ISME J',2017,'1-11',28548661,''),(11616,'Turkarslan S, Raman A V, Thompson AW, Arens CE, Gillespie MA, Netzer F Von, Hillesland KL, Stolyar S, López A, Lomana G De, Reiss DJ, Gorman-lewis D, Zane GM, Ranish JA, Wall JD, Stahl D a, Baliga NS','Mechanism for microbial population collapse in a fluctuating resource environment','Mol Syst Biol',2017,'1-16',28320772,''),(11617,'Vázquez-García I, Salinas F, Li J, Fischer A, Barré B, Hallin J, Bergström A, Alonso-Perez E, Warringer J, Mustonen V, Liti G','Clonal Heterogeneity Influences the Fate of New Adaptive Mutations','Cell Rep',2017,'21:732-744',29045840,''),(11618,'Wang J, Dong X, Shao Y, Guo H, Pan L, Hui W, Kwok L-Y, Zhang H, Zhang W','Genome adaptive evolution of Lactobacillus casei under long-term antibiotic selection pressures','BMC Genomics',2017,'18:320',28438179,''),(11619,'Willemsen A, Zwart MP, Elena SF','High virulence does not necessarily impede viral adaptation to a new host: A case study using a plant RNA virus','BMC Evolutionary Biology',2016,'',28103791,''),(11620,'Winans NJ, Walter A, Chouaia B, Chaston JM, Douglas AE, Newell PD','A genomic investigation of ecological differentiation between free-living and Drosophila -associated bacteria','Mol Ecol',2017,'38:42-49',28667798,''),(11621,'Wu B, Zwart MP, Sánchez-Navarro JA, Elena SF','Within-host Evolution of Segments Ratio for the Tripartite Genome of Alfalfa Mosaic Virus','Sci Rep',2017,'7:1-35',28694514,''),(11622,'Zee PC, Velicer GJ','Parallel emergence of negative epistasis across experimental lineages','Evolution',2017,'71:1088-1095',28128449,''),(11623,'Churton NW V, Misra R V, Howlin RP, Allan RN, Jefferies J, Faust SN, Gharbia SE, Edwards RJ, Clarke SC, Webb JS','Parallel Evolution in Streptococcus pneumoniae Biofilms','Genome Biol Evol',2016,'8:1316-1326',27190203,''),(11624,'Kram KE, Geiger C, Ismail WM, Lee H, Tang H, Foster PL, Finkel SE','Adaptation of Escherichia coli to Serial Passage in Complex Medium: Evidence of Parallel Evolution','mSystems',2016,'1–12',28289732,''),(11625,'Manhart M, Adkar B V, Shakhnovich EI','Tradeoffs between microbial growth phases lead to frequency-dependent and non-transitive selection','Proc Biol Sci',2016,'',29445020,''),(11626,'Rogozin IB, Belinky F, Pavlenko V, Shabalina SA, Kristensen DM, Koonin E V','Evolutionary switches between two serine codon sets are driven by selection','Proc Natl Acad Sci',2016,'',27799560,''),(11627,'Souza GD, Kost C','Experimental evolution of metabolic dependency in bacteria','PLOS Biology',2016,'348:1-27',27814362,''),(11628,'Bailey SF, Rodrigue N, Kassen R','The effect of selection environment on the probability of parallel evolution','Mol Biol Evol',2015,'32:1436-1448',25761765,''),(11629,'Baym M, Kryazhimskiy S, Lieberman TD, Chung H, Desai MM, Kishony R','Inexpensive multiplexed library preparation for megabase-sized genomes','PLoS One 10:e',2015,'10:e0128036',26000737,''),(11630,'Carroll SM, Chubiz LM, Agashe D, Marx CJ','Parallel and divergent evolutionary solutions for the optimization of an engineered central metabolism in Methylobacterium extorquens AM1','Microorganisms',2015,'3:152-174',27682084,''),(11632,'Durão P, Trindade S, Sousa A, Gordo I, Dur??o P, Trindade S, Sousa A, Gordo I','Multiple resistance at no cost: Rifampicin and streptomycin a dangerous liaison in the spread of antibiotic resistance','Mol Biol Evol',2015,'32:2675-2680',26130082,''),(11633,'Ellis CN, Traverse CC, Mayo-Smith L, Buskirk SW, Cooper VS','Character displacement and the evolution of niche complementarity in a model biofilm community','Evolution',2015,'69:283-293',25494960,''),(11634,'Friman V-P, Guzman LM, Reuman DC, Bell T','Bacterial adaptation to sublethal antibiotic gradients can change the ecological properties of multitrophic microbial communities','Proc R Soc',2015,'282:20142920',25833854,''),(11635,'Fumagalli M, Osella M, Thomen P, Heslot F, Lagomarsino M','Speed of evolution in large asexual populations with diminishing returns','J Theor Biol',2015,'365:9-11',25308509,''),(11636,'Gallie J, Libby E, Bertels F, Remigi P, Jendresen CB, Ferguson GC, Desprat N, Buffing MF, Sauer U, Beaumont HJE, Martinussen J, Kilstrup M, Rainey PB','Bistability in a metabolic network underpins the de novo evolution of colony switching in Pseudomonas fluorescens','PLoS Bio',2015,'13:e1002109',25763575,''),(11637,'Graves JL, Tajkarimi M, Cunningham Q, Campbell A, Nonga H, Harrison SH, Barrick JE','Rapid evolution of silver nanoparticle resistance in Escherichia coli','Front Genet',2015,'6:1-13',25741363,''),(11639,'Guo J, Han N, Zhang Y, Wang H, Zhang X, Su L, Liu C, Li J, Chen C, Liu C','Use of genome sequencing to assess nucleotide structure variation of Staphylococcus aureus strains cultured in spaceflight on Shenzhou-X, under simulated microgravity and on the ground','Microbiol Res',2015,'170:61-68',25304992,''),(11640,'Hao Y-Q, Brockhurst MA, Petchey OL, Zhang Q-G, Letters E','Evolutionary rescue can be impeded by temporary environmental amelioration','Ecol Lett',2015,'18:892-898',26119065,''),(11641,'Harrison E, Wood a J, Dytham C, Pitchford JW, Truman J, Spiers A, Paterson S, Brockhurst MA','Bacteriophages limit the existence conditions for conjugative plasmids','Mbio',2015,'6:e00586-15',26037122,''),(11642,'Hiltunen T, Ayan GB, Becks L','Environmental fluctuations restrict eco-evolutionary dynamics in predator – prey system','Proc R Soc',2015,'282:20150013',25994670,''),(11643,'Horinouchi T, Suzuki S, Hirasawa T, Ono N, Yomo T, Shimizu H, Furusawa C','Phenotypic convergence in bacterial adaptive evolution to ethanol stress','BMC Evol Biol',2015,'15:180',26334309,''),(11644,'Ketola T, Saarinen K','Experimental evolution in fluctuating environments: Tolerance measurements at constant temperatures incorrectly predict the ability to tolerate fluctuating temperatures','J Evol Biol',2015,'28:800-806',25704064,''),(11645,'Koskella B','Understanding adaptation and diversification: Insights from the study of microbial experimental evolution','Evolution',2015,'69:279-280',25809535,''),(11647,'Lin W, Rocco MJ, Bertozzi-villa A, Kussell E, Al WLINET','Populations adapt to fluctuating selection using derived and ancestral allelic diversity','Evolution',2015,'69:1448-1460',25908222,''),(11649,'Maddamsetti R, Lenski RE, Barrick JE','Adaptation, clonal interference, and frequency dependent interactions in a long term evolution experiment with Escherichia coli','Genetics',2015,'200:619-631',25911659,''),(11650,'Marietou A, Nguyen ATT, Allen EE, Bartlett D','Adaptive laboratory evolution of Escherichia coli K-12 MG1655 for growth at high hydrostatic pressure','Front Microbiol',2015,'5:749',25610434,''),(11651,'Mechler L, Herbig A, Paprotka K, Fraunholz M, Nieselt K, Bertram R','A novel point mutation promotes growth phase-dependent daptomycin tolerance in Staphylococcus aureus','Antimicrob Agents Chemother',2015,'59:5366-5376',26100694,''),(11652,'Minter EJA, Watts PC, Lowe CD, Brockhurst MA','Negative frequency-dependent selection is intensified at higher population densities in protist populations','Biol Lett',2015,'11:20150192',26063750,''),(11655,'Palmer AC, Toprak E, Baym M, Kim S, Veres A, Bershtein S, Kishony R','Delayed commitment to evolutionary fate in antibiotic resistance fitness landscapes','Nat Commun',2015,'6:7385',26060115,''),(11656,'Perry EB, Barrick JE, Bohannan BJM','The molecular and genetic basis of repeatable coevolution between Escherichia coli and bacteriophage T3 in a laboratory microcosm','PLoS One',2015,'10:e0130639',26114300,''),(11657,'Rendueles O, Amherd M, Velicer GJ','Positively frequency-dependent interference competition maintains diversity and pervades a natural population of cooperative microbes','Curr Biol',2015,'25:1673-1681',26051889,''),(11658,'Rendueles O, Zee PC, Dinkelacker I, Amherd M, Wielgoss S, Velicer GJ','Rapid and widespread de novo evolution of kin discrimination','Proc Natl Acad Sci',2015,'112:9076-9081',26150498,''),(11660,'Rice DP, Good BH, Desai MM','The evolutionarily stable distribution of fitness effects','Genetics',2015,'200:321-329',25762525,''),(11661,'Satterwhite RS, Cooper TF','Constraints on adaptation of Escherichia coli to mixed-resource environments increase over time','Evolution',2015,'59:2067-2078',26103008,''),(11662,'Scanlan PD, Hall AR, Blackshields G, Friman V-P, Jr MRD, Goldberg JB, Buckling A, Davis MR, Goldberg JB, Buckling A','Coevolution with bacteriophages drives genome-wide host evolution and constrains the acquisition of abiotic-beneficial mutations','Mol Biol Evol',2015,'32:1425-1435',25681383,''),(11664,'Taylor TB, Mulley G, Dills AH, Alsohim AS, Mcguffin LJ, Studholme DJ, Silby MW, Brockhurst MA, Johnson LJ, Jackson RW','Evolutionary resurrection of flagellar motility via rewiring of the nitrogen regulation system','Science',2015,'347:1014-1017',25722415,''),(11665,'Turner CB, Blount ZD, Mitchell DH, Lenski RE','Evolution and coexistence in response to a key innovation in a long-term evolution experiment with Escherichia coli','bioRxiv doi.org',2015,'',NULL,''),(11666,'Turner CB, Wade BD, Meyer JR, Lenski RE','Evolution of organismal stoichiometry in a 50,000-generation experiment with Escherichia coli','bioRxiv doi.org',2015,'',NULL,''),(11667,'Voordeckers K, Verstrepen KJ','Experimental evolution of the model eukaryote Saccharomyces cerevisiae yields insight into the molecular mechanisms underlying adaptation','Curr Opin Microbiol',2015,'28:1-9',26202939,''),(11668,'Wasik BR, Bhushan A, Ogbunugafor CB, Turner PE','Delayed transmission selects for increased survival of vesicular stomatitis virus','Evolution',2015,'69:117-125',25311513,''),(11669,'Acevedo A, Brodsky L, Andino R','Mutational and fitness landscapes of an RNA virus revealed through population sequencing','Nature',2014,'505:686-690',24284629,''),(11670,'Barroso-Batista J, Sousa A, Lourenço M, Bergman ML, Sobral D, Demengeot J, Xavier KB, Gordo I','The first steps of adaptation of Escherichia coli to the gut are dominated by soft sweeps.','PLoS Genet',2014,'10(3): e1004182',24603313,'https://doi.org/10.1371/journal.pgen.1004182'),(11671,'Bos J, Zhang Q, Vyawahare S, Rogers E, Rosenberg SM, Austin RH','Emergence of antibiotic resistance from multinucleated bacterial filaments','Proc Natl Acad Sci',2014,'112:178-183',25492931,''),(11672,'Burke MK, Liti G, Long AD','Standing genetic variation drives repeatable experimental evolution in outcrossing populations of Saccharomyces cerevisiae','Mol Biol Evol',2014,'31:3228-3239',25172959,''),(11673,'Carja O, Furrow RE, Feldman MW','The Role of Migration in the Evolution of Phenotypic Switching','Proc Biol Sci',2014,'281:20141677',25232136,''),(11674,'Carja O, Liberman U, Feldman MW','The evolution of phenotypic switching in subdivided populations','Genetics',2014,'196:1185-1197',24496012,''),(11675,'Cooper VS, Staples RK, Traverse CC, Ellis CN','Parallel evolution of small colony variants in Burkholderia cenocepacia biofilms','Genomics',2014,'104:447-452',25263109,''),(11676,'Dobson AJ, Purves J, Rolff J','Increased survival of experimentally evolved antimicrobial peptide-resistant Staphylococcus aureus in an animal host','Evol Appl',2014,'7:905-912',25469169,''),(11677,'Fothergill JL, Neill DR, Loman N, Winstanley C, Kadioglu A','Pseudomonas aeruginosa adaptation in the nasopharyngeal reservoir leads to migration and persistence in the lungs','Nat Commun',2014,'5:4780',25179232,''),(11678,'Frenkel EM, Good BH, Desai MM','The fates of mutant lineages and the distribution of fitness effects of beneficial mutations in laboratory budding yeast opulations','Genetics',2014,'196:1217-1226',24514901,''),(11679,'Fridman O, Goldberg A, Ronin I, Shoresh N, Balaban NQ','Optimization of lag time underlies antibiotic tolerance in evolved bacterial populations','Nature',2014,'513:1-9',25043002,''),(11680,'Friman V-P, Jousset A, Buckling A','Rapid prey evolution can alter the structure of predator-prey communities','J Evol Biol',2014,'27:374-380',24372926,''),(11681,'Graham JK, Smith ML, Simons AM','Experimental evolution of bet hedging under manipulated environmental uncertainty in Neurospora crassa','Proc R Soc B Biol Sci',2014,'281:20140706',24870047,''),(11682,'Guyot S, Pottier L, Hartmann A, Ragon M, Hauck Tiburski J, Molin P, Ferret E, Gervais P','Extremely rapid acclimation of Escherichia coli to high temperature over a few generations of a fed-batch culture during slow warming','Microbiologyopen',2014,'3:52-63',24357618,''),(11683,'Heilbron K, Toll-Riera M, Kojadinovic M, MacLean RC','Fitness is strongly influenced by rare mutations of large effect in a microbial mutation accumulation experiment','Genetics',2014,'197:981-990',24814466,''),(11684,'Helliwell KE, Collins S, Kazamia E, Purton S, Wheeler GL, Smith AG','Fundamental shift in vitamin B12 eco-physiology of a model alga demonstrated by experimental evolution','ISME J',2014,'9:1446-1455',25526368,''),(11685,'Hiltunen T, Becks L','Consumer co-evolution as an important component of the eco-evolutionary feedback','Nat Commun',2014,'5:5226',25335515,''),(11687,'Kim S, Lieberman TD, Kishony R','Alternating antibiotic treatments constrain evolutionary paths to multidrug resistance','Proc Natl Acad Sci',2014,'111:14494-14499',25246554,''),(11688,'Kim W, Racimo F, Schluter J, Levy SB, Foster KR','Importance of positioning for microbial evolution','Proc Natl Acad Sci',2014,'111:E1639-47',24715732,''),(11689,'Kinnersley M a, Wenger J, Kroll E, Adams J, Sherlock G, Rosenzweig FR','Ex Uno Plures: Clonal reinforcement drives evolution of a simple microbial community','PLoS Genet 10:e',2014,'10:e1004430',24968217,''),(11690,'Laehnemann D, Peña-Miller R, Rosenstiel P, Beardmore RE, Jansen G, Schulenburg H','Genomics of rapid adaptation to antibiotics: Convergent evolution and scalable sequence amplification','Genome Biol Evol',2014,'6:1287-1301',24850796,''),(11691,'Lang GI, Desai MM','The spectrum of adaptive mutations in experimental evolution','Genomics',2014,'104:412-416',25269377,''),(11692,'Lázár V, Nagy I, Spohn R, Csörgó B, Györkei Á, Nyerges Á, Horváth B, Vörös A, Busa-Fekete R, Hrtyan M, Bogos B, Méhi O, Fekete G, Szappanos B, Kégl B, Papp B, Pál C','Genome-wide analysis captures the determinants of the antibiotic cross-resistance interaction network','Nat Commun',2014,'5:4352',25000950,''),(11693,'Leiby N, Marx CJ, and not tradeoffs','Metabolic erosion primarily through mutation accumulation, and not tradeoffs, drives limited evolution of substrate specificity in Escherichia coli','PLoS Biol',2014,'12:e1001789',24558347,''),(11694,'Morris JJ, Papoulis S, Lenski RE','Coexistence of evolving bacteria stabilized by a shared Black Queen function','Evolution',2014,'68:2960-2971',24989794,''),(11695,'New AM, Cerulus B, Govers SK, Perez-Samper G, Zhu B, Boogmans S, Xavier JBJB, Verstrepen KJ','Different Levels of Catabolite Repression Optimize Growth in Stable and Variable Environments','PLoS Biol',2014,'12:17-20',24453942,''),(11696,'Oz T, Guvenek A, Yildiz S, Karaboga E, Tamer YT, Mumcuyan N, Ozan VB, Senturk GH, Cokol M, Yeh PJ, Toprak E','Strength of selection pressure is an important parameter contributing to the complexity of antibiotic resistance evolution','Mol Biol Evol',2014,'31:2387-2401',24962091,''),(11697,'Perfeito L, Sousa A, Bataillon T, Gordo I','Rates of fitness decline and rebound suggest pervasive epistasis','Evolution',2014,'68:150-162',24372601,''),(11698,'Plucain J, Hindré T, Gac M Le, Tenaillon O, Cruveiller S, Medigue C, Leiby N, Harcombe WR, Marx CJ, Lenski RE, Schneider D','Epistasis and allele specificity in the emergence of a stable polymorphism in Escherichia coli','Science',2014,'343:1366-1369',24603152,''),(11699,'Puentes-Téllez PE, Kovács ÁT, Kuipers OP, van Elsas JD','Comparative genomics and transcriptomics analysis of experimentally evolved Escherichia coli MC1000 in complex environments','Environ Microbiol',2014,'16:856-870',24033913,''),(11700,'Quandt EM, Deatherage DE, Ellington AD, Georgiou G, Barrick JE','Recursive genomewide recombination and sequencing reveals a key refinement step in the evolution of a metabolic innovation in Escherichia coli','Proc Natl Acad Sci',2014,'111:2217-2222',24379390,''),(11701,'Rajon E, Desouhant E, Chevalier M, Débias F, Menu F','The evolution of bet-hedging in response to local ecological conditions','Am Nat',2014,'184:E1-E15',24921607,''),(11702,'Raynes Y, Sniegowski PD','Experimental evolution and the dynamics of genomic mutation rate modifiers','Heredity',2014,'113:375-380',24849169,''),(11703,'Schenk MF, Witte S, Salverda MLM, Koopmanschap B, Krug J, de Visser JAGM','Role of pleiotropy during adaptation of TEM-1 β -lactamase to two novel antibiotics','Evol Appl',2014,'8:248-260',25861383,''),(11704,'Solopova A, van Gestel J, Weissing FJ, Bachmann H, Teusink B, Kok J, Kuipers OP','Bet-hedging during bacterial diauxic shift','Proc Natl Acad Sci',2014,'111:7427-7432',24799698,''),(11705,'Stelkens RB, Brockhurst MA, Hurst GD, Greig D','Hybridization facilitates evolutionary rescue','Evol Appl',2014,'7:1209-1217',25558281,''),(11706,'Sutera V, Rozen DE, Burmeister WP, Schneider D, Maurin M','Evolution toward high-level fluoroquinolone resistance in Francisella species','J Antimicrob Chemother',2014,'69:101-110',23963236,''),(11708,'Wildenberg G a, Murray AW','Evolving a 24-hr oscillator in budding yeast','Elife',2014,'3:e04875',25383925,''),(11709,'Alto BW, Wasik BR, Morales NM, Turner PE','Stochastic temperatures impede rna virus adaptation','Evolution',2013,'67:969-979',23550749,''),(11710,'Angst DC, Hall AR','The cost of antibiotic resistance depends on evolutionary history in Escherichia coli','BMC Evol Biol',2013,'13:163',23914906,''),(11711,'Bell G','Experimental evolution of heterotrophy in a green alga','Evolution',2013,'67:468-476',23356618,''),(11712,'Flynn KM, Cooper TF, Moore FB-GG, Cooper VS','The Environment Affects Epistatic Interactions to Alter the Topology of an Empirical Fitness Landscape','PLoS Genet',2013,'9:6',23593024,''),(11713,'Friman V-P, Buckling A','Effects of predation on real-time host-parasite coevolutionary dynamics','Ecol Lett',2013,'16:39-46',23013242,''),(11714,'Gifford DR, Maclean RC','Evolutionary reversals of antibiotic resistance in experimental populations of Pseudomonas aeruginosa','Evolution',2013,'67:2973-2981',24094347,''),(11715,'Gong LI, Suchard M a, Bloom JD','Stability-mediated epistasis constrains the evolution of an influenza protein','Elife',2013,'2:e00631',23682315,''),(11716,'Herron MD, Doebeli M','Parallel evolutionary cynamics of adaptive diversification in Escherichia coli','PLoS Biol',2013,'11:e1001490',23431270,''),(11717,'Jeong J, Cho N, Jung D, Bang D','Genome-scale genetic engineering in Escherichia coli','Biotechnol Adv',2013,'31:804-810',23624241,''),(11718,'Jiang W, Bikard D, Cox D, Zhang F, Marraffini L a','RNA-guided editing of bacterial genomes using CRISPR-Cas systems','Nat Biotechnol',2013,'31:233-239',23360965,''),(11719,'Jørgensen KM, Wassermann T, Jensen PØ, Wang H, Molin S, Høiby N, Ciofu O','Sub-lethal ciprofloxacin treatment leads to rapid development of high-level ciprofloxacin resistance during long-term experimental evolution of Pseudomonas aeruginosa','Antimicrob Agents Chemother',2013,'57:4215-4221',23774442,''),(11720,'Koschwanez JH, Foster KR, Murray AW','Improved use of a public good selects for the evolution of undifferentiated multicellularity','Elife',2013,'2:e00367',23577233,''),(11721,'Kvitek DJ, Sherlock G','Whole genome, whole population sequencing reveals that loss of signaling networks is the major adaptive strategy in a constant environment','PLoS Genet',2013,'9:e1003972',24278038,''),(11722,'Laszlo AH, Derrington IM, Brinkerhoff H, Langford KW, Nova IC, Samson JM, Bartlett JJ, Pavlenok M, Gundlach JH','Detection and mapping of 5-methylcytosine and 5-hydroxymethylcytosine with nanopore MspA','Proc Natl Acad Sci',2013,'110:18904-18909',24167255,''),(11723,'Lázár V, Pal Singh G, Spohn R, Nagy I, Horváth B, Hrtyan M, Busa-Fekete R, Bogos B, Méhi O, Csörgó B, Pósfai G, Fekete G, Szappanos B, Kégl B, Papp B, Pál C','Bacterial evolution of antibiotic hypersensitivity','Mol Syst Biol',2013,'9:700',24169403,''),(11724,'Lindsey H a, Gallie J, Taylor S, Kerr B','Evolutionary rescue from extinction is contingent on a lower rate of environmental change','Nature',2013,'494:463-467',23395960,''),(11726,'Maharjan RP, Ferenci T','Epistatic interactions determine the mutational pathways and coexistence of lineages in clonal Escherichia coli populations','Evolution',2013,'67:2762-2768',24033182,''),(11727,'Maharjan RP, Gaffé J, Plucain J, Schliep M, Wang L, Feng L, Tenaillon O, Ferenci T, Schneider D','A case of adaptation through a mutation in a tandem duplication during experimental evolution in Escherichia coli','BMC Genomics',2013,'14:441',23822838,''),(11729,'Pena-Miller R, Laehnemann D, Jansen G, Fuentes-hernandez A, Rosenstiel P, Schulenburg H, Beardmore RE','When the most potent combination of antibiotics selects for the greatest bacterial load: the smile-frown transition','PLoS Biol',2013,'11:e1001540',23630452,''),(11730,'Puentes-Téllez PE, Hansen MA, Sørensen SJ, van Elsas JD','Adaptation and heterogeneity of Escherichia coli MC1000 growing in complex environments','Appl Environ Microbiol',2013,'79:1008-1017',23204426,''),(11731,'Ratcliff WC, Herron MD, Howell K, Pentz JT, Rosenzweig FR, Travisano M','Experimental evolution of an alternating uni- and multicellular life cycle in Chlamydomonas reinhardtii','Nat Commun',2013,'4:2742',24193369,''),(11732,'Ratcliff WC, Pentz JT, Travisano M','Tempo and mode of multicellular adaptation in experimentally evolved Saccharomyces cerevisiae','Evolution',2013,'67:1573-1581',23730752,''),(11733,'Reyes LH, Abdelaal AS, Kao KC','Genetic determinants for N-butanol tolerance in evolved E','Appl Environ Microbiol',2013,'79:5313-5320',23811509,''),(11734,'Rodríguez-Verdugo A, Gaut BS, Tenaillon O','Evolution of Escherichia coli rifampicin resistance in an antibiotic-free environment during thermal stress','BMC Evol Biol',2013,'13:50',23433244,''),(11735,'Schenk MF, Szendro IG, Salverda MLM, Krug J, de Visser JAGM','Patterns of epistasis between beneficial mutations in an antibiotic resistance gene','Mol Biol Evol',2013,'30:1779-1787',23676768,''),(11737,'Torres-Barceló C, Cabot G, Oliver A, Buckling A, MacLean RC','A trade-off between oxidative stress resistance and DNA repair plays a role in the evolution of elevated mutation rates in bacteria','Proc R Soc B Biol Sci',2013,'280:20130007-20130007',23446530,''),(11738,'Traverse CC, Mayo-Smith LM, Poltak SR, Cooper VS','Tangled bank of experimentally evolved Burkholderia biofilms reflects selection during chronic infections','Proc Natl Acad Sci',2013,'110:E250-E259',23271804,''),(11739,'Wiser MJ, Ribeck N, Lenski RE','Long-term dynamics of adaptation in asexual populations','Science',2013,'1364:1364-1367',24231808,''),(11740,'Wright E a, Fothergill JL, Paterson S, Brockhurst MA, Winstanley C','Sub-inhibitory concentrations of some antibiotics can drive diversification of Pseudomonas aeruginosa populations in artificial sputum medium','BMC Microbiol',2013,'13:170',23879797,''),(11741,'Yi X, Dean AM','Bounded population sizes, fluctuating selection and the tempo and mode of coexistence','Proc Natl Acad Sci',2013,'110:16945-16950',24077262,''),(11742,'Arnoldini M, Mostowy R, Bonhoeffer S, Ackermann M','Evolution of stress response in the face of unreliable environmental signals','PLoS Comput Biol',2012,'8:e1002627',22916000,''),(11743,'Bachmann H, Starrenburg MJC, Molenaar D, Kleerebezem M, van Hylckama Vlieg JET','Microbial domestication signatures of Lactococcus lactis can be reproduced by experimental evolution','Genome Res',2012,'22:115-124',22080491,''),(11744,'Bedhomme S, Lafforgue G, Elena SF','Multihost experimental evolution of a plant RNA virus reveals local adaptation and host-specific mutations','Mol Biol Evol',2012,'29:1481-1492',22319146,''),(11745,'Blaby IK, Lyons BJ, Wroclawska-Hughes E, Phillips GCF, Pyle TP, Chamberlin SG, Benner S a, Lyons TJ, De Crécy-Lagard V, Crécy E De','Experimental evolution of a facultative thermophile from a mesophilic ancestor','Appl Environ Microbiol',2012,'78:144-155',22020511,''),(11746,'Blount ZD, Barrick JE, Davidson CJ, Lenski RE','Genomic analysis of a key innovation in an experimental Escherichia coli population','Nature',2012,'489:513-518',22992527,''),(11747,'Chou H-HH, Marx CJ','Optimization of Gene Expression through Divergent Mutational Paths','Cell Rep',2012,'1:133-140',22832162,''),(11748,'Chubiz LM, Lee M-C, Delaney NF, Marx CJ','FREQ-Seq: a rapid, cost-effective, sequencing-based method to determine allele frequencies directly from mixed populations','PLoS One',2012,'7:e47959',23118913,''),(11750,'Gordo I, Sousa A, Magalha S, Magalhães S, Gordo I','Cost of antibiotic resistance and the geometry of adaptation','Mol Biol Evol',2012,'29:1417-1428',22144641,''),(11751,'Hughes JM, Lohman BK, Deckert GE, Nichols EP, Settles M, Abdo Z, Top EM','The role of clonal interference in the evolutionary dynamics of plasmid-host adaptation','Mbio',2012,'3:e00077-12',22761390,''),(11753,'Lalic J, Elena SF, Bataillon T, Joyce P, Lalić J','Epistasis between mutations is host-dependent for an RNA virus','Biol Lett',2012,'9:20120396',22809724,''),(11754,'Lalić J, Elena SF','Magnitude and sign epistasis among deleterious mutations in a positive-sense plant RNA virus','Heredity (Edinb',2012,'109:71-77',22491062,''),(11755,'Lee M-C, Marx CJ','Repeated, selection-driven genome reduction of accessory genes in experimental populations','PLoS Genet',2012,'8:e1002651',22589730,''),(11756,'Leiby N, Harcombe WR, Marx CJ','Multiple long-term, experimentally-evolved populations of Escherichia coli acquire dependence upon citrate as an iron chelator for optimal growth on glucose','BMC Evol Biol',2012,'12:151',22909317,''),(11757,'Levy SF, Ziv N, Siegal ML','Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant','PLoS Biol',2012,'10:e1001325',22589700,''),(11758,'Morris JJ, Lenski RE, Zinser ER','The Black Queen Hypothesis: evolution of dependencies through adaptive gene loss','Mbio',2012,'3:e00036-12',22448042,''),(11760,'Nasvall J, Sun L, Roth JR, Andersson DI','Real-Time Evolution of New Genes by Innovation, Amplification, and Divergence','Science',2012,'338:384-387',23087246,''),(11761,'Piotrowski JS, Nagarajan S, Kroll E, Stanbery A, Chiotti KE, Kruckeberg AL, Dunn B, Sherlock G, Rosenzweig FR','Different selective pressures lead to different genomic outcomes as newly-formed hybrid yeasts evolve','BMC Evol Biol',2012,'12:46',22471618,''),(11762,'Ram Y, Hadany L','The evolution of stress-induced hypermutation in asexual populations','Evolution',2012,'66:2315-2328',22759304,''),(11763,'Reyes LH, Almario MP, Winkler JD, Orozco MM, Kao KC','Visualizing evolution in real time to determine the molecular mechanisms of n-butanol tolerance in Escherichia coli','Metab Eng',2012,'14:579-590',22652227,''),(11764,'Reyes LH, Winkler JD, Kao KC','Visualizing evolution in real-time method for strain engineering','Front Microbiol',2012,'3:198',22661973,''),(11766,'Starrfelt J, Kokko H','Bet-hedging-a triple trade-off between means, variances and correlations','Biol Rev',2012,'87:742-755',22404978,''),(11767,'Tenaillon O, Rodríguez-Verdugo A, Gaut RL, Mcdonald P, Bennett AF, Long AD, Gaut BS','The molecular diversity of adaptive convergence','Science',2012,'335:457-461',22282810,''),(11768,'Walkiewicz K, Cardenas A, Sun CL, Bacorn C, Saxer G, Shamoo Y, Benitez AS','Small changes in enzyme function can lead to surprisingly large fitness effects during adaptive evolution of antibiotic resistance','Proc Natl Acad Sci',2012,'109:21408-21413',23236139,''),(11769,'Wong A, Rodrigue N, Kassen R','Genomics of adaptation during experimental evolution of the opportunistic pathogen Pseudomonas aeruginosa','PLoS Genet',2012,'8:e1002928',23028345,''),(11770,'Beardmore RE, Gudelj I, Lipson D a, Hurst LD','Metabolic trade-offs and the maintenance of the fittest and the flattest','Nature',2011,'472:342-346',21441905,''),(11771,'Chou H-H, Chiu H-C, Delaney NF, Segrè D, Marx CJ','Diminishing returns epistasis among beneficial mutations decelerates adaptation','Science',2011,'332:1190-1192',21636771,''),(11773,'Coffey LL, Vignuzzi M','Host alternation of chikungunya virus increases fitness while restricting population diversity and adaptability to novel selective pressures','J Virol',2011,'85:1025-1035',21047966,''),(11774,'Deng Y, Fong SS','Laboratory evolution and multi-platform genome re-sequencing of the cellulolytic actinobacterium Thermobifida fusca','J Biol Chem',2011,'286:39958-39966',21914801,''),(11775,'Goldman RP, Travisano M','Experimental evolution of ultraviolet radiation resistance in Escherichia coli','Evolution',2011,'65:3486-3498',22133220,''),(11776,'Gullberg E, Cao S, Berg OG, Ilbäck C, Sandegren L, Hughes D, Andersson DI','Selection of resistant bacteria at very low antibiotic concentrations','PLoS Pathog',2011,'7:e1002158',21811410,''),(11777,'Hall AR, MacLean RC','Epistasis buffers the fitness effects of rifampicin- resistance mutations in Pseudomonas aeruginosa','Evolution',2011,'65:2370-2379',21790582,''),(11778,'Jayaraman R','Hypermutation and stress adaptation in bacteria','J Genet',2011,'90:383-391',21869495,''),(11779,'Khan AI, Dinh DM, Schneider D, Lenski RE, Cooper TF','Negative epistasis between beneficial mutations in an evolving bacterial population','Science',2011,'332:1193-1196',21636772,''),(11780,'Kvitek DJ, Sherlock G','Reciprocal sign epistasis between frequently experimentally evolved adaptive mutations causes a rugged fitness landscape','PLoS Genet',2011,'7:e1002056',21552329,''),(11782,'Minty JJ, Lesnefsky A a, Lin F, Chen Y, Zaroff T a, Veloso AB, Xie B, McConnell C a, Ward RJ, Schwartz DR, Rouillard J-M, Gao Y, Gulari E, Lin XN','Evolution combined with genomic study elucidates genetic bases of isobutanol tolerance in Escherichia coli','Microb Cell Fact',2011,'10:18',21435272,''),(11783,'Rainey PB, Beaumont HJE, Ferguson GC, Gallie J, Kost C, Libby E, Zhang X-X','The evolutionary emergence of stochastic phenotype switching in bacteria','Microb Cell Fact',2011,'1:S14',21995592,''),(11784,'Rokyta DR, Joyce P, Caudle S, Miller C','Epistasis between beneficial mutations and the phenotype-to-fitness map for a ssDNA virus','PLoS Genet',2011,'7:e1002075',21655079,''),(11785,'Salverda MLM, Dellus E, Gorter FA, Debets AJM, van der Oost J, Hoekstra RF, Tawfik DS, de Visser JAGM','Initial mutations direct alternative pathways of protein evolution','PLoS Genet',2011,'7:e1001321',21408208,''),(11786,'Schiffels S, Szöllosi GJ, Mustonen V, Lässig M','Emergent neutrality in adaptive asexual evolution','Genetics',2011,'189:1361-1375',21926305,''),(11787,'Simons AM','Modes of response to environmental change and the elusive empirical evidence for bet hedging','Proc R Soc B Biol Sci',2011,'278:1601-1609',21411456,''),(11788,'Vanlint D, Mitchell R, Bailey E','Rapid acquisition of gigapascal-high-pressure resistance by Escherichia coli','Mbio',2011,'2:e00130',21264062,''),(11789,'Venail PA, Kaltz O, Olivieri I, Pommier T, Mouquet N','Diversification in temporally heterogeneous environments: Effect of the grain in experimental bacterial populations','J Evol Biol',2011,'24:2485-2495',21899638,''),(11791,'Wang Y, Manow R, Finan C, Wang J, Garza E, Zhou S','Adaptive evolution of nontransgenic Escherichia coli KC01 for improved ethanol tolerance and homoethanol fermentation from xylose','J Ind Microbiol Biotechnol',2011,'38:1371-1377',21188614,''),(11792,'Wenger JW, Piotrowski JS, Nagarajan S, Chiotti K, Sherlock G, Rosenzweig FR','Hunger artists: Yeast adapted to carbon limitation show trade-offs under carbon sufficiency','PLoS Genet',2011,'7:e1002202',21829391,''),(11793,'Woods RJ, Barrick JE, Cooper TF, Shrestha U, Kauth MR, Lenski RE','Second-order selection for evolvability in a large Escherichia coli population','Science',2011,'331:1433-1436',21415350,''),(11794,'Zhang Q, Lambert G, Liao D, Kim H, Robin K, Tung C, Pourmand N, Austin RH','Acceleration of emergence of bacterial antibiotic resistance in connected microenvironments','Science',2011,'333:1764-1767',21940899,''),(11795,'Atsumi S, Wu T-Y, Machado IMP, Huang W-C, Chen P-Y, Pellegrini M, Liao JC','Evolution, genomic analysis, and reconstruction of isobutanol tolerance in Escherichia coli','Mol Syst Biol',2010,'6:449',21179021,''),(11796,'Barrick JE, Kauth MR, Strelioff CC, Lenski RE','Escherichia coli rpoB mutants have increased evolvability in proportion to their fitness defects','Mol Biol Evol',2010,'27:1338-1347',20106907,''),(11797,'Bloom JD, Gong L, Baltimore D','Permissive secondary mutations enable the evolution of influenza oseltamivir resistance','Science',2010,'328:1272-1275',20522774,''),(11798,'Cooper TF, Lenski RE','Experimental evolution with E. coli in diverse resource environments. I. Fluctuating environments promote divergence of replicate populations','BMC Evol Biol',2010,'10:11',20070898,''),(11799,'Cox J, Schubert AM, Travisano M, Putonti C','Adaptive evolution and inherent tolerance to extreme thermal environments','BMC Evol Biol',2010,'10:75',20226044,''),(11800,'Ellis CN, Cooper VS','Experimental adaptation of Burkholderia cenocepacia to onion medium reduces host range','Appl Environ Microbiol',2010,'76:2387-2396',20154121,''),(11802,'Goodarzi H, Bennett BD, Amini S, Reaves ML, Hottes AK, Rabinowitz JD, Tavazoie S','Regulatory and metabolic rewiring during laboratory evolution of ethanol tolerance in E','Mol Syst Biol',2010,'6:378',20531407,''),(11803,'Harriott MM, Noverr MC','Ability of Candida albicans mutants to induce Staphylococcus aureus vancomycin resistance during polymicrobial biofilm formation','Antimicrob Agents Chemother',2010,'54:3746-3755',20566760,''),(11804,'Horinouchi T, Tamaoka K, Furusawa C, Ono N, Suzuki S, Hirasawa T, Yomo T, Shimizu H','Transcriptome analysis of parallel-evolved Escherichia coli strains under ethanol stress','BMC Genomics',2010,'11:579',20955615,''),(11805,'Kishimoto T, Iijima L, Tatsumi M, Ono N, Oyake A, Hashimoto T, Matsuo M, Okubo M, Suzuki S, Mori K, Kashiwagi A, Furusawa C, Ying B-W, Yomo T','Transition from positive to neutral in mutation fixation along with continuing rising fitness in thermal adaptive evolution','PLoS Genet',2010,'6:e1001164',20975944,''),(11806,'Kohanski M a, DePristo M a, Collins JJ','Sublethal antibiotic treatment leads to multidrug resistance via radical-induced mutagenesis','Mol Cell',2010,'37:311-320',20159551,''),(11807,'MacLean RC, Perron GG, Gardner A','Diminishing returns from beneficial mutations and pervasive epistasis shape the fitness landscape for rifampicin resistance in Pseudomonas aeruginosa','Genetics',2010,'186:1345-1354',20876562,''),(11808,'Meyer JR, Agrawal A a, Quick RT, Dobias DT, Schneider D, Lenski RE','Parallel changes in host resistance to viral infection during 45,000 generations of relaxed selection','Evolution',2010,'64:3024-3034',20550574,''),(11809,'Paterson S, Vogwill T, Buckling A, Benmayor R, Spiers AJ, Thomson NR, Quail M, Smith F, Walker D, Libberton B, Fenton A, Hall N, Brockhurst MA','Antagonistic coevolution accelerates molecular evolution','Nature',2010,'464:275-278',20182425,''),(11810,'Racey D, Inglis RF, Harrison F, Oliver A, Buckling A','The effect of elevated mutation rates on the evolution of cooperation and virulence of Pseudomonas aeruginosa','Evolution',2010,'64:515-521',19686265,''),(11811,'Saxer G, Doebeli M, Travisano M','The repeatability of adaptive radiation during long-term experimental evolution of Escherichia coli in a multiple nutrient environment','PLoS One',2010,'5:e14184',21152028,''),(11812,'Trindade S, Perfeito L, Gordo I','Rate and effects of spontaneous mutations that affect fitness in mutator Escherichia coli','Philos Trans R Soc Lond B Biol Sci',2010,'365:1177-1186',20308092,''),(11813,'Barrick JE, Yu DS, Yoon SH, Jeong H, Oh TK, Schneider D, Lenski RE, Kim JF','Genome evolution and adaptation in a long-term experiment with Escherichia coli','Nature',2009,'461:1243-1247',19838166,''),(11814,'Beaumont HJE, Gallie J, Kost C, Ferguson GC, Rainey PB','Experimental evolution of bet hedging','Nature',2009,'462:90-93',19890329,''),(11815,'Collins S, De Meaux J','Adaptation to different rates of environmental change in chlamydomonas','Evolution',2009,'63:2953-2965',19619223,''),(11816,'Crecy E de, Jaronski S, Lyons BJ, Lyons TJ, Keyhani NO, de Crécy E, Jaronski S, Lyons BJ, Lyons TJ, Keyhani NO','Directed evolution of a filamentous fungus for thermotolerance','BMC Biotechnol',2009,'9:74',19709419,''),(11817,'Handel A, Rozen DE','The impact of population size on the evolution of asexual microbes on smooth versus rugged fitness landscapes','BMC Evol Biol',2009,'9:236',19765292,''),(11818,'Harris DR, Pollock S V, Wood E a, Goiffon RJ, Klingele AJ, Cabot EL, Schackwitz WS, Martin J, Eggington J, Durfee TJ, Middle CM, Norton JE, Popelars MC, Li H, Klugman S a., Hamilton LL, Bane LB, Pennacchio L a, Albert TJ, Perna NT, Cox MM, Battista JR','Directed Evolution of Ionizing Radiation Resistance in Escherichia coli','J Bacteriol',2009,'191:5240-5252',19502398,''),(11819,'Johnson PJT, Dubnau D, Levin BR','Episodic selection and the maintenance of competence and natural transformation in Bacillus subtilis','Genetics',2009,'181:1521-1533',19189946,''),(11820,'Kinnersley M a, Holben WE, Rosenzweig FR','E Unibus Plurum: genomic analysis of an experimentally evolved polymorphism in Escherichia coli','PLoS Genet',2009,'5:e1000713',19893610,''),(11821,'Kraigsley AM, Finkel SE','Adaptive evolution in single species bacterial biofilms','FEMS Microbiol Lett',2009,'293:135-140',19239496,''),(11822,'Lee M-C, Chou H-H, Marx CJ','Asymmetric, bimodal trade-offs during adaptation of Methylobacterium to distinct growth substrates','Evolution',2009,'63:2816-2830',19545267,''),(11824,'Philippe N, Pelosi L, Lenski RE, Schneider D','Evolution of penicillin-binding protein 2 concentration and cell shape during a long-term experiment with Escherichia coli','J Bacteriol',2009,'191:909-921',19047356,''),(11825,'Rokyta DR, Abdo Z, Wichman H a','The genetics of adaptation for eight microvirid bacteriophages','J Mol Evol',2009,'69:229-39',19693424,''),(11826,'Rozen DE, Philippe N, Arjan de Visser J, Lenski RE, Schneider D','Death and cannibalism in a seasonal environment facilitate bacterial coexistence','Ecol Lett',2009,'12:34-44',19019196,''),(11827,'Saxer G, Doebeli M, Travisano M','Spatial structure leads to ecological breakdown and loss of diversity','Proc Biol Sci',2009,'276:2065-2070',19324788,''),(11828,'Schoustra SE, Bataillon T, Gifford DR, Kassen R','The properties of adaptive walks in evolving populations of fungus','PLoS Biol',2009,'7:e1000250',19956798,''),(11829,'Simons AM','Fluctuating natural selection accounts for the evolution of diversification bet hedging','Proc R Soc B Biol Sci',2009,'276:1987-1992',19324774,''),(11830,'Stanek MT, Cooper TF, Lenski RE','Identification and dynamics of a beneficial mutation in a long-term evolution experiment with Escherichia coli','BMC Evol Biol',2009,'9:302',20040094,''),(11831,'Trindade S, Sousa A, Xavier KB, Dionisio F, Ferreira MG, Gordo I','Positive epistasis drives the acquisition of multidrug resistance','PLoS Genet',2009,'5:e1000578',19629166,''),(11832,'Ward H, Perron GG, MacLean RC','The cost of multiple drug resistance in Pseudomonas aeruginosa','J Evol Biol',2009,'22:997-1003',19298493,''),(11833,'Acar M, Mettetal JT, van Oudenaarden A','Stochastic switching as a survival strategy in fluctuating environments','Nat Genet',2008,'40:471-475',18362885,''),(11834,'Agudelo-Romero P, de la Iglesia F, Elena SF','The pleiotropic cost of host-specialization in Tobacco etch potyvirus','Infect Genet Evol',2008,'8:806-814',18765302,''),(11835,'Benmayor R, Buckling A, Bonsall MB, Brockhurst MA, Hodgson DJ','The interactive effects of parasites, disturbance, and productivity on experimental adaptive radiations','Evolution',2008,'62:467-477',18039322,''),(11836,'Coffey LL, Vasilakis N, Brault AC, Powers AM, Tripet F, Weaver SC','Arbovirus evolution in vivo is constrained by host alternation','Proc Natl Acad Sci',2008,'105:6970-6975',18458341,''),(11838,'Michel J-B, Yeh PJ, Chait R, Moellering RC, Kishony R','Drug interactions modulate the potential for evolution of resistance','Proc Natl Acad Sci',2008,'105:14918-14923',18815368,''),(11839,'Ostrowski E a, Woods RJ, Lenski RE','The genetic basis of parallel and divergent phenotypic responses in evolving populations of Escherichia coli','Proc Biol Sci',2008,'275:277-84',18029306,''),(11840,'Perron GG, Gonzalez a, Buckling A','The rate of environmental change drives adaptation to an antibiotic','J Evol Biol',2008,'21:1724-1731',18681913,''),(11841,'Rokyta DR, Beisel ÆCJ, Joyce ÆP, Ferris ÆMT, Burch CL, Wichman ÆHA, Beisel CJ, Joyce P, Ferris MT, Burch CL, Wichman HA','Beneficial fitness effects are not exponential for two viruses','J Mol Evol',2008,'67:368-376',18779988,''),(11842,'Rozen DE, Habets MGJL, Handel A, de Visser JAGM','Heterogeneous adaptive trajectories of small populations on complex fitness landscapes','PLoS One',2008,'3:e1715',18320036,''),(11843,'Sleight SC, Orlic C, Schneider D, Lenski RE','Genetic basis of evolutionary adaptation by Escherichia coli to stressful cycles of freezing, thawing and growth','Genetics',2008,'180:431-443',18757947,''),(11844,'Spencer CC, Tyerman J, Bertrand M, Doebeli M','Adaptation increases the likelihood of diversification in an experimental bacterial lineage','Proc Natl Acad Sci',2008,'105:1585-1589',18216261,''),(11845,'Veening J-W, Stewart EJ, Berngruber TW, Taddei F, Kuipers OP, Hamoen LW','Bet-hedging and epigenetic inheritance in bacterial cell development','Proc Natl Acad Sci',2008,'105:4393-8',18326026,''),(11846,'Bennett AF, Lenski RE','An experimental test of evolutionary trade-offs during temperature adaptation','Proc Natl Acad Sci',2007,'104:8649-8654',17494741,''),(11847,'Desai MM, Fisher DS, Murray AW','The speed of evolution and maintenance of variation in asexual populations','Curr Biol',2007,'17:385-94',17331728,''),(11851,'Gallet R, Alizon S, Comte P-A, Gutierrez A, Depaulis F, van Baalen M, Michel E, Müller-Graf CDM','Predation and disturbance interact to shape prey species diversity','Am Nat',2007,'170:143-154',17853998,''),(11852,'Hansen SK, Rainey PB, Haagensen J a J, Molin S','Evolution of species interactions in a biofilm community','Nature',2007,'445:533-536',17268468,''),(11853,'Hughes BS, Cullum AJ, Bennett AF','An experimental evolutionary study on adaptation to temporally fluctuating pH in Escherichia coli','Physiol Biochem Zool',2007,'80:406-421',17508336,''),(11854,'Jasmin JN, Kassen R','On the experimental evolution of specialization and diversity in heterogeneous environments','Ecol Lett',2007,'10:272-281',17355566,''),(11855,'Maharjan RP, Seeto S, Ferenci T','Divergence and redundancy of transport and metabolic rate-yield strategies in a single Escherichia coli population','J Bacteriol',2007,'189:2350-2358',17158684,''),(11856,'Maughan H, Masel J, Birky CW, Nicholson WL','The roles of mutation accumulation and selection in loss of sporulation in experimental populations of Bacillus subtilis','Genetics',2007,'177:937-948',17720926,''),(11858,'Perfeito L, Fernandes L, Mota C, Gordo I','Adaptive mutations in bacteria: high rate and small effects','Science',2007,'317:813-815',17690297,''),(11859,'Rozen DE, McGee L, Levin BR, Klugman KP','Fitness costs of fluoroquinolone resistance in Streptococcus pneumoniae','Antimicrob Agents Chemother',2007,'51:412-416',17116668,''),(11860,'Sleight SC, Lenski RE','Evolutionary adaptation to freeze-thaw-growth cycles in Escherichia coli','Physiol Biochem Zool',2007,'80:370-385',17508333,''),(11861,'Spencer CC, Bertrand M, Travisano M, Doebeli M','Adaptive diversification in genes that regulate resource use in Escherichia coli','PLoS Genet',2007,'3:e15',17238290,''),(11862,'Spencer CC, Saxer G, Travisano M, Doebeli M','Seasonal resource oscillations maintain diversity in bacterial microcosms','Evol Ecol Res',2007,'9:775-787',19079742,''),(11864,'Codoñer FM, Daròs JA, Solé R V, Elena SF','The Fittest versus the Flattest: Experimental Confirmation of the Quasispecies Effect with Subviral Pathogens','PLoS Pathog',2006,'2:e136',17196038,''),(11865,'Duffy S, Turner PE, Burch CL','Pleiotropic costs of niche expansion in the RNA bacteriophage Φ6','Genetics',2006,'172:751-757',16299384,''),(11866,'Fiegna F, Yu Y-TN, Kadam S V, Velicer GJ','Evolution of an obligate social cheater to a superior cooperator','Nature',2006,'441:310-314',16710413,''),(11867,'Habets MGJL, Rozen DE, Hoekstra RF, de Visser JAGM','The effect of population structure on the adaptive radiation of microbial populations evolving in spatially structured environments','Ecol Lett',2006,'9:1041-1048',16925653,''),(11868,'Kassen R, Bataillon T','Distribution of fitness effects among beneficial mutations before selection in experimental populations of bacteria','Nat Genet',2006,'38:484-488',16550173,''),(11869,'Sleight SC, Wigginton NS, Lenski RE','Increased susceptibility to repeated freeze-thaw cycles in Escherichia coli following long-term evolution in a benign environment','BMC Evol Biol',2006,'6:104',17147797,''),(11870,'Velicer GJ, Raddatz G, Keller H, Deiss S, Lanz C, Dinkelacker I, Schuster SC','Comprehensive mutation identification in an evolved bacterial cooperator and its cheating ancestor','Proc Natl Acad Sci',2006,'103:8107-8112',16707573,''),(11871,'Weinreich DM, Delaney NF, Depristo M a, Hartl DL','Darwinian evolution can follow only very few mutational paths to fitter proteins','Science',2006,'312:111-114',16601193,''),(11872,'Woods RJ, Schneider D, Winkworth CL, Riley M a, Lenski RE','Tests of parallel molecular evolution in a long-term experiment with Escherichia coli','Proc Natl Acad Sci',2006,'103:9107-12',16705032,''),(11873,'Barrett RDH, MacLean RC, Bell G','Experimental evolution of Pseudomonas fluorescens in simple and complex environments','Am Nat',2005,'166:470-480',16224703,''),(11874,'Greene IP, Wang E, Deardorff ER, Milleron R, Domingo E, Weaver SC','Effect of alternating passage on adaptation of sindbis virus to vertebrate and invertebrate cells','J Virol',2005,'79:14253-14260',16254360,''),(11875,'Kussell E, Leibler S','Phenotypic diversity, population growth, and information in fluctuating environments','Science',2005,'309:2075-2078',16123265,''),(11877,'Ostrowski EA, Rozen DE, Lenski RE','Pleiotropic effects of beneficial mutations in Escherichia coli','Evolution',2005,'59:2343-2352',16396175,''),(11878,'Rozen DE, Schneider D, Lenski RE','Long-term experimental evolution in Escherichia coli','J Mol Evol',2005,'61:171-180',15999245,''),(11879,'Tyerman J, Havard N, Saxer G, Travisano M, Doebeli M','Unparallel diversification in bacterial microcosms','Proc Biol Sci',2005,'272:1393-1398',16006323,''),(11880,'Alcántara-Díaz D, Breña-Valle M, Serment-Guerrero J','Divergent adaptation of Escherichia coli to cyclic ultraviolet light exposures','Mutagenesis',2004,'19:349-354',15388806,''),(11881,'Brockhurst M a, Rainey PB, Buckling A','The effect of spatial heterogeneity and parasites on the evolution of host diversity','Proc Biol Sci',2004,'271:107-111',15002778,''),(11882,'Burch CL, Chao L','Epistasis and its relationship to canalization in the RNA virus φ6','Genetics',2004,'167:559-567',15238511,''),(11883,'Friesen ML, Saxer G, Travisano M, Doebeli M','Experimental evidence for sympatric ecological diversification due to frequency-dependent competition in Escherichia coli.','Evolution',2004,'58(2): 245-60',15068343,''),(11884,'Joseph SB, Hall DW','Spontaneous mutations in diploid Saccharomyces cerevisiae: More beneficial than expected','Genetics',2004,'168:1817-1825',15611159,''),(11886,'Remold SK, Lenski RE','Pervasive joint influence of epistasis and plasticity on mutational effects in Escherichia coli','Nat Genet',2004,'36:423-426',15072075,''),(11889,'Cooper TF, Rozen DE, Lenski RE','Parallel changes in gene expression after 20,000 generations of evolution in Escherichia coli','Proc Natl Acad Sci',2003,'100:1072-1077',12538876,''),(11890,'Szafraniec K, Wloch DM, Sliwa P, BORTS RH, Korona R','Small fitness effects and weak genetic interactions between deleterious mutations in heterozygous loci of the yeast Saccharomyces cerevisiae','Genet Res',2003,'82:19-31',14621268,''),(11891,'Cooper VS','Long-term experimental evolution in Escherichia coli','BMC Evol Biol',2002,'2:12',12199912,''),(11892,'de Visser JAGM, Lenski RE','Long-term experimental evolution in Escherichia coli','BMC Evol Biol',2002,'8:1-8',12443537,''),(11893,'Elena SF','Restrictions to RNA virus adaptation: An experimental approach','Int J Gen Mol Microbiol',2002,'81:135-142',12448712,''),(11894,'Kerr B, Riley M a, Feldman MW, Bohannan BJM','Local dispersal promotes biodiversity in a real-life game of rock-paper-scissors','Nature',2002,'418:171-174',12110887,''),(11895,'Notley-McRobb L, Seeto S, Ferenci T','Enrichment and elimination of mutY mutators in Escherichia coli populations','Genetics',2002,'162:1055-1062',12454055,''),(11896,'Rozen DE, de Visser JAGM, Gerrish PJ','Fitness effects of fixed beneficial mutations in microbial populations','Curr Biol',2002,'12:1040-1045',12123580,''),(11897,'Cullum AJ, Bennett AF, Lenski RE','Evolutionary adaptation to temperature','Evolution',2001,'55:2194-2202',11794780,''),(11898,'Giraud A, Matic I, Tenaillon O, Clara A, Radman M, Fons M, Taddei F','Costs and benefits of high mutation rates: adaptive evolution of bacteria in the mouse gut','Science',2001,'291:2606-2608',11283373,''),(11899,'Imhof M, Schlötterer C','Fitness effects of advantageous mutations in evolving Escherichia coli populations','Proc Natl Acad Sci',2001,'98:1113-1117',11158603,''),(11900,'Ooper VASC, Ennett ALFB, Enski RIEL, Cooper VS, Bennett AF, Lenski RE','Evolution of thermal dependence of growth rate of Escherichia coli populations during 20,000 generations in a constant environment','Evolution',2001,'55:889-896',11430649,''),(11901,'Remold SK, Lenski RE','Contribution of individual random mutations to genotype-by-environment interactions in Escherichia coli','Proc Natl Acad Sci',2001,'98:11388-11393',11572987,''),(11902,'Riehle MM, Bennett AF, Long AD','Genetic architecture of thermal adaptation in Escherichia coli','Proc Natl Acad Sci',2001,'98:525-530',11149947,''),(11903,'Tenaillon O, Taddei F, Radmian M, Matic I','Second-order selection in bacterial evolution: selection acting on mutation and recombination rates in the course of adaptation','Res Microbiol',2001,'152:11-16',11281320,''),(11904,'Wilke CO, Wang JLJ, Ofria C, Lenski RE, Adami C','Evolution of digital organisms at high mutation rates leads to survival of the flattest','Nature',2001,'412:331-333',11460163,''),(11905,'Wloch DM, Borts RH, Korona R','Epistatic interactions of spontaneous mutations in haploid strains of the yeast Saccharomyces cerevisiae','J Evol Biol',2001,'14:310-316',NULL,''),(11906,'Wloch DM, Szafraniec K, Borts RH, Korona R','Direct estimate of the mutation rate and the distribution of fitness effects in the yeast Saccharomyces cerevisiae','Genetics',2001,'159:441-452',11606524,''),(11907,'Burch CL, Chao L','Evolvability of an RNA virus is determined by its mutational neighbourhood','Nature',2000,'406:625-628',10949302,''),(11908,'Cooper VS, Lenski RE','The population genetics of ecological specialization in evolving Escherichia coli populations','Nature',2000,'407:736-739',11048718,''),(11909,'Rozen DE, Lenski RE','Long-Term Experimental Evolution in Escherichia coli','Am Nat',2000,'155:24-35',10657174,''),(11910,'Schneider D, Duperchy E, Coursange E, Lenski RE, Blot M','Long-term experimental evolution in Escherichia coli','Genetics',2000,'156:477-488',11014799,''),(11911,'Turner PE, Elena SF','Cost of host radiation in an RNA virus','Genetics',2000,'156:1465-1470',11102349,''),(11912,'de Visser JAGM, Zeyl CW, Gerrish PJ, Blanchard JL, Lenski RE','Diminishing returns from mutation supply rate in asexual populations','Science',1999,'283:404-406',9888858,''),(11913,'Elena SF','Little evidence for synergism among deleterious mutations in a nonsegmented RNA virus','J Mol Evol',1999,'49:703-707',10552052,''),(11914,'Hopper KR','Risk-spreading and bet-hedging in insect population biology','Annu Rev Entomol',1999,'44:535-560',15012381,''),(11915,'Miralles R, Gerrish PJ, Moya A, Elena SF','Clonal interference and the evolution of RNA viruses','Science',1999,'285:1745-1747',10481012,''),(11916,'Papadopoulos D, Schneider D, Meier-Eiss J, Arber W, Lenski RE, Blot M','Genomic evolution during a 10,000-generation experiment with bacteria','Proc Natl Acad Sci',1999,'96:3807-3812',10097119,''),(11917,'Velicer GJ, Lenski RE','Evolutionary trade-offs under conditions of resource abundance and scarcity: Experiments with bacteria','Ecology',1999,'80:1168-1179',NULL,''),(11918,'Wichman HA','Different Trajectories of Parallel Evolution During Viral Adaptation','Science',1999,'285:422-424',10411508,''),(11919,'Akanuma S, Yamagishi A, Tanaka N, Oshima T','Serial increase in the thermal stability of 3-isopropylmalate dehydrogenase from Bacillus subtilis by experimental evolution','Protein Sci',1998,'7:698-705',9541402,''),(11920,'Baranowski E, Sevilla NN, Ruiz-Jarabo CM, Go G, Domingo E, Gómez-Mariano G, Baranowski E, Domingo E','An RNA virus can adapt to the multiplicity of infection','J Gen Virol',1998,'79:2971-2980',9880011,''),(11922,'Gerrish PJ, Lenski RE','The fate of competing beneficial mutations in an asexual population','Genetica',1998,'103:127-144',9720276,''),(11923,'Kassen R, Bell G','Experimental evolution in Chlamydomonas','Heredity',1998,'80:732-741',NULL,''),(11924,'Rainey PB, Travisano M','Adaptive radiation in a heterogeneous environment','Nature',1998,'394:69-72',9665128,''),(11925,'Treves DS, Manning S, Adams J','Repeated evolution of an acetate-crossfeeding polymorphism in long-term populations of Escherichia coli','Mol Biol Evol',1998,'15:789-797',9656481,''),(11926,'Bell G, Reboud X','Experimental evolution in Chlamydomonas II. Genetic variation in strongly contrasted environments','Heredity',1997,'78:498-506',NULL,''),(11927,'de Visser JAGM, Hoekstra RF, Van Ende H Den','An experimental test for synergistic epistasis and its application in Chlamydomonas','Genetics',1997,'145:815-819',9055090,''),(11928,'Elena SF, Lenski RE','Long-term experimental evolution in Escherichia coli: VII. Mechanisms maintaining genetic variability within populations','Evolution',1997,'51:1059-1067',28565475,''),(11929,'Reboud X, Bell G','Experimental evolution in Chlamydomonas','Heredity',1997,'78:507-514',NULL,''),(11930,'Sniegowski PD, Gerrish PJ, Lenski RE','Evolution of high mutation rates in experimental populations of E. coli','Nature',1997,'387:703-705',9192894,''),(11931,'Bennett AF, Lenski RE','Evolutionary adaptation to temperature V Adaptive mechanisms and correlated responses in experimental lines of Escherichia coli','Evolution',1996,'50:493-503',28568929,''),(11932,'de Visser JAGM, Hoekstra RF, Ende H Van Den','The effect of sex and deleterious mutations on fitness in Chlamydomonas','Proc R Soc B Biol Sci',1996,'263:193-200',NULL,''),(11933,'Elena SF, Cooper VS, Lenski RE','Punctuated evolution caused by selection of rare beneficial mutations','Science',1996,'272:1802-4',8650581,''),(11934,'Mongold J a, Lenski RE','Experimental rejection of a nonadaptive explanation for increased cell size in Escherichia coli','Changes',1996,'178:5333-5334',8752359,''),(11935,'Novella IS, Cilnis M, Elena SF, Kohn J, Moya A, Domingo E, Holland JJ','Large-population passages of vesicular stomatitis virus in interferon-treated cells select variants of only limited resistance','J Virol',1996,'70:6414-6417',8709273,''),(11936,'Turner PE, Souza V, Lenski RE','Tests of ecological mechanisms promoting the stable coexistence of two bacterial genotypes','Ecology',1996,'77:2119-2129',11539425,''),(11937,'Travisano M, Mongold JA, Bennett AF, Lenski RE, chance','Experimental tests of the roles of adaptation, chance, and history in evolution','Science',1995,'267:87-90',7809610,''),(11938,'Travisano M, Vasi F, Lenski RE','LONG-TERM EXPERIMENTAL EVOLUTION IN ESCHERICHIA COLI. III. VARIATION AMONG REPLICATE POPULATIONS IN CORRELATED RESPONSES TO NOVEL ENVIRONMENTS.','Evolution',1995,'49(1): 189-200',28593661,'https://doi.org/10.1111/j.1558-5646.1995.tb05970.x'),(11939,'Lenski RE, Travisano M','Dynamics of adaptation and diversification: a 10,000-generation experiment with bacterial populations','Proc Natl Acad Sci',1994,'91:6808-6814',8041701,''),(11940,'Leroi AM, Lenski RE, Bennett AF','Evolutionary adaptation to temperature. III. Adaptation of Escherichia coli to a temporally varying environment','Evolution',1994,'48:1222-1229',28564463,''),(11941,'Rosenzweig FR, Sharp RR, Treves DS, Adams J, Rosenzweig RF','Microbial evolution in a simple unstructured environment: genetic differentiation in Escherichia coli','Genetics',1994,'917:903-917',7982572,''),(11942,'Bennett AF, Lenski RE','Evolutionary adaptation to temperature II. Thermal niches of expeirmental lines of Escherichia coli','Evolution',1993,'47:1-12',28568084,''),(11944,'Clarke DK, Duarte E a, Moya A, Elena SF, Domingo E, Holland J','Genetic bottlenecks and population passages cause profound fitness differences in RNA viruses','J Virol',1993,'67:222-228',8380072,''),(11945,'Lenski RE, Bennett AF','Evolutionary response of Escherichia coli to thermal stress','Am Nat',1993,'142:S47-S64',19425951,''),(11947,'Duarte E a, Clarke D, Moya A, Domingo E, Holland J','Rapid fitness losses in mammalian RNA virus clones due to Muller’s ratchet','Proc Natl Acad Sci',1992,'89:6015-6019',1321432,''),(11948,'Chao L','Fitness of RNA virus decreased by Muller’s ratchet','Nature',1990,'348:454-455',2247152,''),(11950,'Lenski RE','Experimental Studies of Pleiotropy and Epistasis in Escherichia coli','Evolution',1988,'42:425-432',28564005,''),(11952,'Lenski RE, Hattingh SE','Coexistence of two competitors on one resource and one inhibitor: a chemostat model based on bacteria and antibiotics','J Theor Biol',1986,'122:83-93',3796009,''),(11953,'Paquin C, Adams J','Frequency of fixation of adaptive mutations is higher in evolving diploid than haploid yeast populations','Nature',1983,'302:495-500',6339947,''),(11954,'Atwood KC, Schneider LK, Ryan FJ','Periodic selection in Escherichia coli','Proc Natl Acad Sci',1951,'37:146-155',14808170,''),(11955,'Bosshard L, Peischl S, Ackermann M, Excoffier L','Mutational and selective processes involved in evolution during bacterial range expansions.','Mol Biol Evol',2019,'',31241150,'https://doi.org/10.1093/molbev/msz148'),(11956,'Szili P, Draskovits G, Révész T, Bogar F, Balogh D, Martinek T, Daruka L, Spohn R, Vásárhelyi BM, Czikkely M, Kintses B, Grézal G, Ferenc G, Pál C, Nyerges Á','Rapid evolution of reduced susceptibility against a balanced dual-targeting antibiotic through stepping-stone mutations.','Antimicrob Agents Chemother',2019,'',31235632,'https://doi.org/10.1128/AAC.00207-19'),(11957,'Bast J, Jaron KS, Schuseil D, Roze D, Schwander T','Asexual reproduction reduces transposable element load in experimental yeast populations.','Elife',2019,'8',31486772,'https://doi.org/10.7554/eLife.48548'),(11958,'Graves JL, Ewunkem AJ, Ward Jason, Staley Constance, Thomas Misty D, Rhinehardt Kristen L, Han Jian, Harrison Scott H','Experimental Evolution of Gallium Resistance in Escherichia coli.','Evolution, Medicine, and Public Health',2019,'',NULL,'https://doi.org/10.1093/emph/eoz025'),(11959,'Ibacache-Quiroga C, Oliveros JC, Couce A, Blázquez J','Parallel Evolution of High-Level Aminoglycoside Resistance in <i>Escherichia coli</i> Under Low and High Mutation Supply Rates.','Front Microbiol',2018,'9: 427',29615988,'https://doi.org/10.3389/fmicb.2018.00427'),(11964,'Sanz-García F, Hernando-Amado S, Martínez JL','Mutation-Driven Evolution of Pseudomonas aeruginosa in the Presence of either Ceftazidime or Ceftazidime-Avibactam.','Antimicrob Agents Chemother',2018,'62(10)',30082283,'https://doi.org/10.1128/AAC.01379-18'),(11965,'Imamovic L, Ellabaan MMH, Dantas Machado AM, Citterio L, Wulff T, Molin S, Krogh Johansen H, Sommer MOA','Drug-Driven Phenotypic Convergence Supports Rational Treatment Strategies of Chronic Infections.','Cell',2018,'172(1-2): 121-134.e14',29307490,'https://doi.org/10.1016/j.cell.2017.12.012'),(11966,'Wardell SJT, Rehman A, Martin LW, Winstanley C, Patrick WM, Lamont IL','A large-scale whole-genome comparison shows that experimental evolution in response to antibiotics predicts changes in naturally evolved clinical <i>Pseudomonas aeruginosa</i>.','Antimicrob Agents Chemother',2019,'',31570397,'https://doi.org/10.1128/AAC.01619-19'),(11967,'Spohn R, Daruka L, Lázár V, Martins A, Vidovics F, Grézal G, Méhi O, Kintses B, Számel M, Jangir PK, Csörgő B, Györkei Á, Bódi Z, Faragó A, Bodai L, Földesi I, Kata D, Maróti G, Pap B, Wirth R, Papp B, Pál C','Integrated evolutionary analysis reveals antimicrobial peptides with limited resistance.','Nat Commun',2019,'10(1): 4538',31586049,'https://doi.org/10.1038/s41467-019-12364-6'),(11968,'Bhattacharya A, Toro Díaz VC, Morran LT, Bashey F','Evolution of increased virulence is associated with decreased spite in the insect-pathogenic bacterium Xenorhabdus nematophila.','Biol Lett',2019,'15(8): 20190432',31455168,'https://doi.org/10.1098/rsbl.2019.0432'),(11969,'Morran LT, Penley MJ, Byrd VS, Meyer AJ, O\'Sullivan TS, Bashey F, Goodrich-Blair H, Lively CM','Nematode-bacteria mutualism: Selection within the mutualism supersedes selection outside of the mutualism.','Evolution',2016,'70(3): 687-95',26867502,'https://doi.org/10.1111/evo.12878'),(11970,'You ST, Jhou YT, Kao CF, Leu JY','Experimental evolution reveals a general role for the methyltransferase Hmt1 in noise buffering.','PLoS Biol',2019,'17(10): e3000433',31613873,'https://doi.org/10.1371/journal.pbio.3000433'),(11971,'Hattori H, Yakushi T, Matsutani M, Moonmangmee D, Toyama H, Adachi O, Matsushita K','High-temperature sorbose fermentation with thermotolerant Gluconobacter frateurii CHM43 and its mutant strain adapted to higher temperature.','Appl Microbiol Biotechnol',2012,'95(6): 1531-40',22434571,'https://doi.org/10.1007/s00253-012-4005-4'),(11972,'Matsumoto N, Hattori H, Matsutani M, Matayoshi C, Toyama H, Kataoka N, Yakushi T, Matsushita K','A Single-Nucleotide Insertion in a Drug Transporter Gene Induces a Thermotolerance Phenotype in Gluconobacter frateurii by Increasing the NADPH/NADP<sup>+</sup> Ratio via Metabolic Change.','Appl Environ Microbiol',2018,'84(10)',29549098,'https://doi.org/10.1128/AEM.00354-18'),(11975,'a list of authors goes here','this is the title of the new paper','Journal Abbr.',2019,'',NULL,''),(11982,'a list of authors goes here','this is the title of the new paper','Journal Abbr.',2019,'',28596757,'https://doi.org/10.3389/fmicb.2017.00893');
/*!40000 ALTER TABLE `references` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `sessions`
--

DROP TABLE IF EXISTS `sessions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `sessions` (
  `token` varchar(255) DEFAULT NULL,
  `created` datetime DEFAULT current_timestamp(),
  UNIQUE KEY `token` (`token`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `sessions`
--

LOCK TABLES `sessions` WRITE;
/*!40000 ALTER TABLE `sessions` DISABLE KEYS */;
INSERT INTO `sessions` VALUES ('c3dd7546-278d-474c-bbe4-93b96916e877','2021-04-19 07:34:33');
/*!40000 ALTER TABLE `sessions` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2021-04-20  8:24:57
